{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import os\n",
    "os.chdir(\"..\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "\"\"\"\n",
    "Iterate over the PubMED articles that mention infecious diseases from the\n",
    "disease ontology.\n",
    "\"\"\"\n",
    "import rdflib\n",
    "from pylru import lrudecorator\n",
    "import pubcrawler.article as pubcrawler\n",
    "from annotator.keyword_annotator import KeywordAnnotator\n",
    "from annotator.annotator import AnnoDoc\n",
    "import re\n",
    "import json\n",
    "import pymongo"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Loading disease ontology...\n",
      "disease ontology loaded\n"
     ]
    }
   ],
   "source": [
    "print(\"Loading disease ontology...\")\n",
    "disease_ontology = rdflib.Graph()\n",
    "disease_ontology.parse(\n",
    "    \"http://purl.obolibrary.org/obo/doid.owl\",\n",
    "    format=\"xml\"\n",
    ")\n",
    "print(\"disease ontology loaded\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<Graph identifier=N282c4c34511b47809860361834f167da (<class 'rdflib.graph.Graph'>)>"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "disease_ontology"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def get_annotation_keywords():\n",
    "    qres = disease_ontology.query(\"\"\"\n",
    "    prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "    prefix obo: <http://purl.obolibrary.org/obo/>\n",
    "    prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "    SELECT ?entity ?label\n",
    "    WHERE {\n",
    "        # only resolve diseases by infectious agent\n",
    "        ?entity rdfs:subClassOf* obo:DOID_0050117\n",
    "        ; oboInOwl:hasNarrowSynonym|oboInOwl:hasRelatedSynonym|oboInOwl:hasExactSynonym|rdfs:label ?label\n",
    "    }\n",
    "    \"\"\")\n",
    "    def remove_parenthetical_notes(label):\n",
    "        label = re.sub(r\"\\s\\(.*\\)\",\"\", label)\n",
    "        label = re.sub(r\"\\s\\[.*\\]\",\"\", label)\n",
    "        assert(len(label) > 0)\n",
    "        return label\n",
    "    return list(set([remove_parenthetical_notes(str(r[1])) for r in qres]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "with open(\"annotation_keywords\", \"w+\") as f:\n",
    "    for item in get_annotation_keywords():\n",
    "        f.write(\"{}\\n\".format(item))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def str_escape(s):\n",
    "    return json.dumps(s)[1:-1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "@lrudecorator(500)\n",
    "def resolve_keyword(keyword):\n",
    "    query = \"\"\"\n",
    "    prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "    prefix obo: <http://purl.obolibrary.org/obo/>\n",
    "    prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "    SELECT ?entity\n",
    "    WHERE {\n",
    "        # only resolve diseases by infectious agent\n",
    "        ?entity rdfs:subClassOf* obo:DOID_0050117\n",
    "        ; oboInOwl:hasNarrowSynonym|oboInOwl:hasRelatedSynonym|oboInOwl:hasExactSynonym|rdfs:label ?label\n",
    "        FILTER regex(?label, \"^(\"\"\" + str_escape(re.escape(keyword)) + str_escape(\"(\\s[\\[\\(].*[\\]\\)])*\") + \"\"\")$\", \"i\")\n",
    "    }\n",
    "    \"\"\"\n",
    "    qres = list(disease_ontology.query(query))\n",
    "    if len(qres) == 0:\n",
    "        print(\"no match for\", keyword.encode('ascii', 'xmlcharrefreplace'))\n",
    "    elif len(qres) > 1:\n",
    "        print(\"multiple matches for\", keyword.encode('ascii', 'xmlcharrefreplace'))\n",
    "        print(qres)\n",
    "    return qres"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def annotated_keywords_to_dict_list(keywords):\n",
    "    seen_keys = []\n",
    "    keyword_list = []\n",
    "    for keyword_entity in keywords:\n",
    "        keyword, uri = keyword_entity\n",
    "        if keyword in seen_keys:\n",
    "            continue\n",
    "        else:\n",
    "            keys.append(keyword)\n",
    "            keyword_dict = {\n",
    "                \"keyword\": keyword,\n",
    "                \"uri\": uri[0].entity.toPython()\n",
    "            }\n",
    "            keyword_list.append(keyword_dict)\n",
    "    return(keyword_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "def write_article_meta_to_mongo(article, collection):\n",
    "    pc_article = pubcrawler.Article(article)\n",
    "    anno_doc = AnnoDoc(pc_article.body)\n",
    "    anno_doc.add_tier(keyword_annotator)\n",
    "    infectious_diseases = [\n",
    "        (disease.text, resolve_keyword(disease.text))\n",
    "        for disease in anno_doc.tiers['keywords'].spans\n",
    "    ]\n",
    "    disease_ontology_keywords = None if len(infectious_diseases) == 0 else annotated_keywords_to_dict_list(infectious_diseases)\n",
    "    collection.update_one({'_id': article['_id']},\n",
    "        {\n",
    "        '$set':\n",
    "            {\n",
    "            'meta':\n",
    "                {\n",
    "                'article-ids': pc_article.pub_ids(),\n",
    "                'article-type': pc_article.article_type(),\n",
    "                # 'pub-dates': pc_article.pub_dates()\n",
    "                # Need to fix stuff with dates in Mongo\n",
    "                'keywords': pc_article.keywords()\n",
    "                },\n",
    "            'annotations':\n",
    "                {\n",
    "                'disease-ontology-keywords': disease_ontology_keywords\n",
    "                }\n",
    "            },\n",
    "        })"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def iterate_infectious_disease_articles(collection):\n",
    "    query = {}\n",
    "    if args.no_reannotation:\n",
    "        query = {'meta': {'$exists': False}}\n",
    "    total_articles = collection.count(query)\n",
    "    processed_articles = 0\n",
    "    for article in collection.find(query):\n",
    "        processed_articles += 1\n",
    "        print(\"Processing article {} of {} ({:.2}%)...\".format(processed_articles, total_articles, processed_articles / total_articles), end=\"\")\n",
    "        write_article_meta_to_mongo(article, collection=collection)\n",
    "        print(\" Done!\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "db = pymongo.MongoClient('localhost')['pmc']\n",
    "articles = db.articlesubset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "cursor = articles.find({'meta': {'$exists': False}})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "10001"
      ]
     },
     "execution_count": 60,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cursor.count()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "x = cursor.next()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'_id': 'PLoS_One_2016_Jan_25_11(1)_e0147962',\n",
       " 'nxml': '<!DOCTYPE article PUBLIC \"-//NLM//DTD JATS (Z39.96) Journal Archiving and Interchange DTD v1.0 20120330//EN\" \"JATS-archivearticle1.dtd\">\\n<article xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" article-type=\"research-article\"><?properties open_access?><front><journal-meta><journal-id journal-id-type=\"nlm-ta\">PLoS One</journal-id><journal-id journal-id-type=\"iso-abbrev\">PLoS ONE</journal-id><journal-id journal-id-type=\"publisher-id\">plos</journal-id><journal-id journal-id-type=\"pmc\">plosone</journal-id><journal-title-group><journal-title>PLoS ONE</journal-title></journal-title-group><issn pub-type=\"epub\">1932-6203</issn><publisher><publisher-name>Public Library of Science</publisher-name><publisher-loc>San Francisco, CA USA</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"pmid\">26807734</article-id><article-id pub-id-type=\"pmc\">4726549</article-id><article-id pub-id-type=\"doi\">10.1371/journal.pone.0147962</article-id><article-id pub-id-type=\"publisher-id\">PONE-D-15-43543</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Research Article</subject></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>RNA viruses</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>West Nile virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Microbiology</subject><subj-group><subject>Medical microbiology</subject><subj-group><subject>Microbial pathogens</subject><subj-group><subject>Viral pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>West Nile virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Medicine and health sciences</subject><subj-group><subject>Pathology and laboratory medicine</subject><subj-group><subject>Pathogens</subject><subj-group><subject>Microbial pathogens</subject><subj-group><subject>Viral pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>West Nile virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>Viral pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>West Nile virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>RNA viruses</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Western Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Microbiology</subject><subj-group><subject>Medical Microbiology</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Western Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Medicine and Health Sciences</subject><subj-group><subject>Pathology and Laboratory Medicine</subject><subj-group><subject>Pathogens</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Western Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Western Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Research and analysis methods</subject><subj-group><subject>Extraction techniques</subject><subj-group><subject>RNA extraction</subject></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>RNA viruses</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Eastern Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Microbiology</subject><subj-group><subject>Medical Microbiology</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Eastern Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Medicine and Health Sciences</subject><subj-group><subject>Pathology and Laboratory Medicine</subject><subj-group><subject>Pathogens</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Eastern Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Eastern Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>RNA viruses</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>Japanese Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Microbiology</subject><subj-group><subject>Medical Microbiology</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>Japanese Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Medicine and Health Sciences</subject><subj-group><subject>Pathology and Laboratory Medicine</subject><subj-group><subject>Pathogens</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>Japanese Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>Japanese Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Molecular Biology</subject><subj-group><subject>Molecular Biology Techniques</subject><subj-group><subject>Artificial Gene Amplification and Extension</subject><subj-group><subject>Polymerase Chain Reaction</subject></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Research and Analysis Methods</subject><subj-group><subject>Molecular Biology Techniques</subject><subj-group><subject>Artificial Gene Amplification and Extension</subject><subj-group><subject>Polymerase Chain Reaction</subject></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Molecular Biology</subject><subj-group><subject>Molecular Biology Techniques</subject><subj-group><subject>Sequencing Techniques</subject><subj-group><subject>Sequence Analysis</subject><subj-group><subject>Sequence Alignment</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Research and Analysis Methods</subject><subj-group><subject>Molecular Biology Techniques</subject><subj-group><subject>Sequencing Techniques</subject><subj-group><subject>Sequence Analysis</subject><subj-group><subject>Sequence Alignment</subject></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>RNA viruses</subject></subj-group></subj-group></subj-group></subj-group></article-categories><title-group><article-title>Surveillance for Western Equine Encephalitis, St. Louis Encephalitis, and West Nile Viruses Using Reverse Transcription Loop-Mediated Isothermal Amplification</article-title><alt-title alt-title-type=\"running-head\">RT-LAMP Surveillance for Western Equine Encephalitis, St. Louis Encephalitis, and West Nile Viruses</alt-title></title-group><contrib-group><contrib contrib-type=\"author\"><name><surname>Wheeler</surname><given-names>Sarah S.</given-names></name><xref ref-type=\"aff\" rid=\"aff001\"><sup>1</sup></xref><xref ref-type=\"author-notes\" rid=\"currentaff001\"><sup>&#x000a4;a</sup></xref></contrib><contrib contrib-type=\"author\"><name><surname>Ball</surname><given-names>Cameron S.</given-names></name><xref ref-type=\"aff\" rid=\"aff002\"><sup>2</sup></xref></contrib><contrib contrib-type=\"author\"><name><surname>Langevin</surname><given-names>Stanley A.</given-names></name><xref ref-type=\"aff\" rid=\"aff002\"><sup>2</sup></xref><xref ref-type=\"author-notes\" rid=\"currentaff002\"><sup>&#x000a4;b</sup></xref></contrib><contrib contrib-type=\"author\"><name><surname>Fang</surname><given-names>Ying</given-names></name><xref ref-type=\"aff\" rid=\"aff001\"><sup>1</sup></xref></contrib><contrib contrib-type=\"author\"><name><surname>Coffey</surname><given-names>Lark L.</given-names></name><xref ref-type=\"aff\" rid=\"aff001\"><sup>1</sup></xref></contrib><contrib contrib-type=\"author\"><name><surname>Meagher</surname><given-names>Robert J.</given-names></name><xref ref-type=\"aff\" rid=\"aff002\"><sup>2</sup></xref><xref ref-type=\"corresp\" rid=\"cor001\">*</xref></contrib></contrib-group><aff id=\"aff001\"><label>1</label><addr-line>University of California Davis, School of Veterinary Medicine, Department of Pathology, Microbiology and Immunology, Davis, California, United States of America</addr-line></aff><aff id=\"aff002\"><label>2</label><addr-line>Sandia National Laboratories, Biotechnology and Bioengineering Department, Livermore, California, United States of America</addr-line></aff><contrib-group><contrib contrib-type=\"editor\"><name><surname>Vasilakis</surname><given-names>Nikos</given-names></name><role>Editor</role><xref ref-type=\"aff\" rid=\"edit1\"/></contrib></contrib-group><aff id=\"edit1\"><addr-line>University of Texas Medical Branch, UNITED STATES</addr-line></aff><author-notes><fn fn-type=\"conflict\" id=\"coi001\"><p><bold>Competing Interests: </bold>The authors have read the journal\\'s policy and have the following competing interest: Authors CSB, RJM, and SAL are current or former employees of Sandia National Laboratories. The specific roles of these authors are articulated in the Author Contributions section. Sandia National Laboratories did not have any additional role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript. The authors declare no competing interests related to their employment with Sandia National Laboratories. This did not alter the authors\\' adherence to PLOS ONE policies on sharing data and materials.</p></fn><fn fn-type=\"con\" id=\"contrib001\"><p>Conceived and designed the experiments: SSW CSB SAL LLC RJM. Performed the experiments: SSW CSB YF LLC RJM. Analyzed the data: SSW CSB LLC RJM. Contributed reagents/materials/analysis tools: SSW YF LLC RJM. Wrote the paper: SSW CSB SAL LLC RJM.</p></fn><fn fn-type=\"current-aff\" id=\"currentaff001\"><label>&#x000a4;a</label><p>Current address: Sacramento-Yolo Mosquito and Vector Control District, Elk Grove, California, United States of America</p></fn><fn fn-type=\"current-aff\" id=\"currentaff002\"><label>&#x000a4;b</label><p>Current address: University of Washington, Department of Microbiology, Seattle, Washington, United States of America</p></fn><corresp id=\"cor001\">* E-mail: <email>rmeaghe@sandia.gov</email></corresp></author-notes><pub-date pub-type=\"epub\"><day>25</day><month>1</month><year>2016</year></pub-date><pub-date pub-type=\"collection\"><year>2016</year></pub-date><volume>11</volume><issue>1</issue><elocation-id>e0147962</elocation-id><history><date date-type=\"received\"><day>2</day><month>10</month><year>2015</year></date><date date-type=\"accepted\"><day>11</day><month>1</month><year>2016</year></date></history><permissions><copyright-statement>&#x000a9; 2016 Wheeler et al</copyright-statement><copyright-year>2016</copyright-year><copyright-holder>Wheeler et al</copyright-holder><license xlink:href=\"http://creativecommons.org/licenses/by/4.0/\"><license-p>This is an open access article distributed under the terms of the <ext-link ext-link-type=\"uri\" xlink:href=\"http://creativecommons.org/licenses/by/4.0/\">Creative Commons Attribution License</ext-link>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</license-p></license></permissions><self-uri content-type=\"pdf\" xlink:type=\"simple\" xlink:href=\"pone.0147962.pdf\"/><abstract><p>Collection of mosquitoes and testing for vector-borne viruses is a key surveillance activity that directly influences the vector control efforts of public health agencies, including determining when and where to apply insecticides. Vector control districts in California routinely monitor for three human pathogenic viruses including West Nile virus (WNV), Western equine encephalitis virus (WEEV), and St. Louis encephalitis virus (SLEV). Reverse transcription quantitative polymerase chain reaction (RT-qPCR) offers highly sensitive and specific detection of these three viruses in a single multiplex reaction, but this technique requires costly, specialized equipment that is generally only available in centralized public health laboratories. We report the use of reverse transcription loop-mediated isothermal amplification (RT-LAMP) to detect WNV, WEEV, and SLEV RNA extracted from pooled mosquito samples collected in California, including novel primer sets for specific detection of WEEV and SLEV, targeting the nonstructural protein 4 (nsP4) gene of WEEV and the 3&#x02019; untranslated region (3&#x02019;-UTR) of SLEV. Our WEEV and SLEV RT-LAMP primers allowed detection of &#x0003c;0.1 PFU/reaction of their respective targets in &#x0003c;30 minutes, and exhibited high specificity without cross reactivity when tested against a panel of alphaviruses and flaviviruses. Furthermore, the SLEV primers do not cross-react with WNV, despite both viruses being closely related members of the Japanese encephalitis virus complex. The SLEV and WEEV primers can also be combined in a single RT-LAMP reaction, with discrimination between amplicons by melt curve analysis. Although RT-qPCR is approximately one order of magnitude more sensitive than RT-LAMP for all three targets, the RT-LAMP technique is less instrumentally intensive than RT-qPCR and provides a more cost-effective method of vector-borne virus surveillance.</p></abstract><funding-group><funding-statement>This work was supported by a grant from the Defense Threat Reduction Agency (DTRA), contract DTRA1002710697. Sandia National Laboratories is a multi-program laboratory managed and operated by Sandia Corporation, a wholly owned subsidiary of Lockheed Martin Corporation, for the U.S. Department of Energy&#x02019;s National Nuclear Security Administration under Contract DE-AC04-94AL85000.</funding-statement></funding-group><counts><fig-count count=\"5\"/><table-count count=\"3\"/><page-count count=\"17\"/></counts><custom-meta-group><custom-meta id=\"data-availability\"><meta-name>Data Availability</meta-name><meta-value>All relevant data are within the paper and its Supporting Information files.</meta-value></custom-meta></custom-meta-group></article-meta><notes><title>Data Availability</title><p>All relevant data are within the paper and its Supporting Information files.</p></notes></front><body><sec sec-type=\"intro\" id=\"sec001\"><title>Introduction</title><p>Testing mosquito vectors for human pathogenic viruses like West Nile virus is a central feature of successful surveillance approaches and an early predictor of human epidemics. Most surveillance in the United States and increasingly in other regions uses molecular detection of viral RNAs, usually by reverse transcription polymerase chain reaction (RT-PCR) or quantitative real-time RT-PCR (RT-qPCR). The presence of vector-borne viruses in mosquitoes, together with other measures of activity including mosquito distribution and abundance, human cases, dead bird surveillance, meteorological parameters and (for some viruses) sentinel chicken seroconversions, are used to make vector control decisions including when and where to apply insecticides. Three medically important vector-borne viruses, West Nile virus (WNV), Western equine encephalitis virus (WEEV), and St. Louis encephalitis virus (SLEV) are endemic in California and cause sporadic seasonal outbreaks, primarily during the summer months.</p><p>WEEV incidence has declined in the last few decades [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>] and has not been detected in California since 2003. SLEV was detected in field-caught mosquitoes in California in 2015, for the first time since the arrival of WNV in 2003 [<xref rid=\"pone.0147962.ref002\" ref-type=\"bibr\">2</xref>, <xref rid=\"pone.0147962.ref003\" ref-type=\"bibr\">3</xref>]. SLEV also still occurs in Mexico and South America [<xref rid=\"pone.0147962.ref004\" ref-type=\"bibr\">4</xref>]. Despite the much higher prevalence of WNV in California since 2003, SLEV and WEEV are still included in routine mosquito surveillance since vector-borne viruses can exist in cryptic and sporadic transmission cycles without the detection of human or equine disease, as exemplified by the re-emergence of SLEV in 2015.</p><p>Vector-borne disease surveillance programs primarily rely on techniques such as virus isolation, standard RT-PCR, and RT-qPCR to detect the presence of vector-borne viruses in targeted mosquito vectors. While these methodologies are the gold standard for arbovirus disease surveillance, they require well-equipped laboratories and well-trained technicians, both of which are severely lacking in many low resource settings with high vector-borne disease burden. RT-qPCR is the most sensitive assay for detecting vector-borne viruses, but the technique generally requires expensive equipment, and high quality RNA purified from the mosquito samples. Loop-mediated isothermal amplification (LAMP) is an isothermal nucleic acid amplification technique that is a useful alternative to PCR for pathogen detection and diagnostics [<xref rid=\"pone.0147962.ref005\" ref-type=\"bibr\">5</xref>&#x02013;<xref rid=\"pone.0147962.ref008\" ref-type=\"bibr\">8</xref>]. When coupled with reverse transcription, the LAMP methodology can also be utilized for detecting RNA targets [<xref rid=\"pone.0147962.ref009\" ref-type=\"bibr\">9</xref>]. LAMP requires 4 or 6 primers that recognize 6 or 8 binding sites, providing a highly specific assay. A key advantage to the LAMP reaction is simplicity: the entire reaction (including reverse transcription) proceeds at a single temperature, eliminating the need for a thermal cycler. Reactions can be monitored through a variety of outputs including generation of turbidity [<xref rid=\"pone.0147962.ref010\" ref-type=\"bibr\">10</xref>], fluorescence indicators [<xref rid=\"pone.0147962.ref011\" ref-type=\"bibr\">11</xref>, <xref rid=\"pone.0147962.ref012\" ref-type=\"bibr\">12</xref>], or color change [<xref rid=\"pone.0147962.ref013\" ref-type=\"bibr\">13</xref>]. The technique is also robust for crude specimens, allowing successful amplification from minimally processed samples such as heat-treated blood [<xref rid=\"pone.0147962.ref012\" ref-type=\"bibr\">12</xref>, <xref rid=\"pone.0147962.ref014\" ref-type=\"bibr\">14</xref>&#x02013;<xref rid=\"pone.0147962.ref016\" ref-type=\"bibr\">16</xref>, <xref rid=\"pone.0147962.ref017\" ref-type=\"bibr\">17</xref>]. As such, LAMP can often be performed without DNA or RNA extraction. These advantages lessen the dependence upon highly skilled labor and well-equipped laboratories, which are prerequisites for RT-qPCR.</p><p>In this report, we describe use of RT-LAMP to detect WNV, WEEV, and SLEV, and apply the technique to RNA extracted from <italic>Culex</italic> mosquitoes collected in California. RT-LAMP was previously demonstrated for detection of WNV by Parida <italic>et al</italic> [<xref rid=\"pone.0147962.ref018\" ref-type=\"bibr\">18</xref>], and detection kits for this virus based on RT-LAMP are commercially available from several vendors. RT-LAMP primer sets have also been reported for numerous other important mosquito-borne viruses such as Dengue virus (DENV) [<xref rid=\"pone.0147962.ref019\" ref-type=\"bibr\">19</xref>, <xref rid=\"pone.0147962.ref020\" ref-type=\"bibr\">20</xref>], Japanese encephalitis virus (JEV) [<xref rid=\"pone.0147962.ref021\" ref-type=\"bibr\">21</xref>&#x02013;<xref rid=\"pone.0147962.ref023\" ref-type=\"bibr\">23</xref>], chikungunya virus (CHIKV) [<xref rid=\"pone.0147962.ref024\" ref-type=\"bibr\">24</xref>], Rift Valley fever virus (RVFV) [<xref rid=\"pone.0147962.ref025\" ref-type=\"bibr\">25</xref>, <xref rid=\"pone.0147962.ref026\" ref-type=\"bibr\">26</xref>], and yellow fever virus (YFV) [<xref rid=\"pone.0147962.ref027\" ref-type=\"bibr\">27</xref>]. Because RT-LAMP detection of SLEV and WEEV was not previously reported, RT-LAMP primer sets for these viruses were developed and tested for sensitivity and specificity. At the time of method development neither WEEV nor SLEV had been recently detected in California mosquito pools, and thus archived samples were used for RT-LAMP evaluation. WNV-positive mosquito pools collected in 2014 were used to verify the lack of cross-reactivity with new SLEV and WEEV primers. These WNV positive pools were also used to demonstrate the applicability of RT-LAMP for parallel detection of WNV, WEEV and SLEV, analogous to the RT-LAMP &#x0201c;detection array&#x0201d; previously reported for parallel detection of DENV serotypes 1&#x02013;4, JEV, and WNV [<xref rid=\"pone.0147962.ref028\" ref-type=\"bibr\">28</xref>]. In addition, we demonstrate duplex detection of WEEV and SLEV in a single tube using melt curve analysis.</p></sec><sec sec-type=\"materials|methods\" id=\"sec002\"><title>Materials and Methods</title><sec id=\"sec003\"><title>RT-LAMP primer design</title><p>Whole-genome sequence alignments for WEEV and SLEV were performed using ClustalX2. For WEEV, six North American strains were selected for which full-length or nearly full-length sequences were available in Genbank as of April, 2014 (a large number of additional WEEV genomes were deposited in Genbank after primers were designed [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>]). For SLEV, 6 strains were selected for which full-length or nearly full-length sequences were available, representing primarily North American strains from Clade II in the tree developed by Kramer and Chandler [<xref rid=\"pone.0147962.ref029\" ref-type=\"bibr\">29</xref>]. Accession numbers of WEEV and SLEV strains used for primer design are presented in Figs <xref ref-type=\"fig\" rid=\"pone.0147962.g001\">1</xref> and <xref ref-type=\"fig\" rid=\"pone.0147962.g002\">2</xref>.</p><fig id=\"pone.0147962.g001\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.g001</object-id><label>Fig 1</label><caption><title>Sequence alignment showing priming regions for the WEEV nsP4 primer set.</title><p>The arrow indicates the location of the <italic>Xba</italic>I restriction site. Genbank accession numbers refer to the following strains: GQ287644.1 = BFS2005; GQ287643.1 = Montana-64; GQ287645.1 = 71V1658; GQ287642.1 = Kern; GQ287641.1 = Imperial; GQ287647.1 = 85&#x02013;452NM.</p></caption><graphic xlink:href=\"pone.0147962.g001\"/></fig><fig id=\"pone.0147962.g002\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.g002</object-id><label>Fig 2</label><caption><title>Sequence alignment showing priming regions for the SLEV 3&#x02019; UTR primer set.</title><p>The arrow indicates the location of the <italic>Hin</italic>dIII restriction site.Genbank accession numbers refer to the following strains: DQ525916.1 = Kern 217; EU566860.1 = Hubbard; FJ753286.2 = CbaAr-4005; JF460774.1 = Imperial Valley; JQ957868.1 = Palenque-C475; JQ957869.1 = Palenque-A770.</p></caption><graphic xlink:href=\"pone.0147962.g002\"/></fig><p>Conserved regions of the viral genomes from the sequence alignments were identified. LAMP Designer v1.13 software (Premier Biosoft) was used to scan conserved regions for suitable LAMP primer sets using a single representative strain. Primer set candidates were then screened against the full set of aligned sequences to test for perfect or nearly perfect matches. Primer sets meeting this criterion were then analyzed by BLAST, first comparing to all WEEV or SLEV sequences (taxid 11039 and 11080, respectively) and then comparing to all viral sequences in Genbank, to determine likelihood of cross-reactivity with other viruses. Primer sets were also evaluated for hairpin formation, self-dimerization, and cross-dimerization.</p></sec><sec id=\"sec004\"><title>Viral culture</title><p>Selected viral strains used for sensitivity and specificity testing are listed in <xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>. These strains were pulled from a library of isolates archived at -80&#x000b0;C. From each isolate RNA was extracted and plaque assays were performed to obtain viral titers. RNA was extracted using a MagMax&#x02122; magnetic particle processor (Life Technologies, Grand Island, NY, USA), MagMAX&#x02122;-96 Viral RNA Isolation Kit (Life Technologies, Grand Island, NY, USA), and manufacturer provided protocols. Plaque assays were performed using Vero cell cultures grown to confluence in six well plates, cultured with Dulbecco&#x02019;s Modified Eagle Medium (DMEM) supplemented with 10% fetal bovine serum (FBS) and 500 U/mL penicillin and 0.5 mg/mL streptomycin, and maintained at 37&#x000b0;C and 5% CO<sub>2</sub>. Virus isolates were diluted in a 10-fold serial dilution, media was removed from each well of the 6-well plate and 200 uL of diluted virus was allowed to incubate on the monolayer for 60 min at 37&#x000b0;C. A double overlay system was used where the first overlay contained nutritive media and 5% agarose. The second overlay was applied and additionally contained 0.005% neutral red, the optimal time to count plaques occurred 1&#x02013;3 days after the second overlay.</p><table-wrap id=\"pone.0147962.t001\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.t001</object-id><label>Table 1</label><caption><title>Viral RNA used for testing sensitivity and specificity of RT-LAMP primers.</title></caption><alternatives><graphic id=\"pone.0147962.t001g\" xlink:href=\"pone.0147962.t001\"/><table frame=\"hsides\" rules=\"groups\"><colgroup span=\"1\"><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/></colgroup><thead><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\"/><th align=\"left\" rowspan=\"1\" colspan=\"1\"/><th align=\"center\" colspan=\"3\" rowspan=\"1\">RT-LAMP primer set (100 PFU target)</th></tr><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\">Virus</th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Strain or Isolate Designation</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019;-UTR</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WNV E gene</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WEEV nsP4</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>Flaviviruses</italic></td><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;St. Louis encephalitis virus (SLEV)<xref ref-type=\"table-fn\" rid=\"t001fn001\"><sup>1</sup></xref></td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Kern217</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">Bfs1750 (Ct 17.7)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">Ruls (Ct 12.9)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">Coav750 (Ct 23.9)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">69M1143 (Ct 13.7)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">BeAn246407 (Ct 20.4)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">CorAn9124 (PCR neg.)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">CorAn9275 (PCR neg.)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;<xref ref-type=\"table-fn\" rid=\"t001fn002\"><sup>2</sup></xref></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;West Nile virus (WNV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">L-CA-04 SAC-04-7168</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Yellow fever virus (YFV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">17D</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Rocio virus (ROCV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">SP H 34675</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Usutu virus (USV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">SA AR 1776</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Ilheus virus (ILHV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Ilheus B44532</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Dengue virus serotype 1 (DENV-1)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">BC-796</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Dengue virus serotype 2 (DENV-2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">BC-122-94</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Dengue virus serotype 3 (DENV-3)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">BC 156&#x02013;97</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>Alphaviruses</italic></td><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Western equine encephalitis virus (WEEV)<xref ref-type=\"table-fn\" rid=\"t001fn001\"><sup>1</sup></xref></td><td align=\"left\" rowspan=\"1\" colspan=\"1\">KERN 5547</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">Lake43 (Ct 14.9)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Sindbis virus (SINV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">EDS-14</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Ross River virus (RRV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">SW 38457</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Chikungunya virus (CHIKV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Ross</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Venezuelan equine encephalitis virus (VEEV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">TC-83</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;<xref ref-type=\"table-fn\" rid=\"t001fn002\"><sup>2</sup></xref></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Barmah Forest virus (BFV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Barmah Forest (TVP-4119)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Highlands J virus (HJV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">WC-431</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Eastern equine encephalitis virus (EEEV)<xref ref-type=\"table-fn\" rid=\"t001fn003\"><sup>3</sup></xref></td><td align=\"left\" rowspan=\"1\" colspan=\"1\">EEEV/X/USA/A15072/2003</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr></tbody></table></alternatives><table-wrap-foot><fn id=\"t001fn001\"><p><sup>1</sup> Isolatesof SLEV other than Kern 217, and WEEV isolate Lake43 were not quantitated by plaque assay. Extracted RNA was quantitated by RT-qPCR and used undiluted as a template in RT-LAMP. RT-qPCR Ct values are indicated. CorAn 9124 and CorAn9275 were not detected by RT-qPCR primers.</p></fn><fn id=\"t001fn002\"><p><sup>2</sup> A single reaction showed positive amplification within 45 minutes of incubation (1 of 11 replicates for SLEV CorAn 9275 with WNV primers; 1 of 18 replicates for VEEV TC-83 with SLEV primers). Melt curve analysis indicates that these are isolated occurrences of cross-contamination with positive-control RNA.</p></fn><fn id=\"t001fn003\"><p><sup>3</sup> Cross-reactivity testing for EEEV was performed with 10<sup>6</sup> copies of an <italic>in vitro</italic> transcribed fragment of the EEEV nsP4 gene corresponding to the WEEV RT-LAMP target site including an additional 46 nt upstream and 65 nt downstream of the outer primer (F3 and B3) binding sites. The <italic>in vitro</italic> transcribed RNA represents nucleotides 5980&#x02013;6329 (350 bases) of Genbank # KJ469613.1.</p></fn></table-wrap-foot></table-wrap><p>Due to select agent restrictions for work with Eastern equine encephalitis virus (EEEV), a segment of EEEV nsP4 RNA was purchased. The RNA was <italic>in vitro</italic> transcribed from a synthetic gene corresponding to nucleotides 5980&#x02013;6329 (350 bases) of Genbank accession number KJ469613.1 (Biosynthesis, Lewisville, TX, USA). The transcript was purified to remove traces of DNA, and verified DNA-free by PCR (performed by the supplier).</p></sec><sec id=\"sec005\"><title>Mosquito pools</title><p>Mosquito pools were collected as part of routine mosquito surveillance performed by California vector control agencies. RNA from selected pools was obtained through the UC Davis Center for Vectorborne Disease Surveillance Labortory (CVEC) with the permission of the collecting agency. Pools comprised up to 50 conspecific, female, field-caught mosquitoes. RNA was extracted by MagMax<sup>TM</sup> as described above.</p></sec><sec id=\"sec006\"><title>RT-qPCR</title><p>Detection of viral RNA in mosquito pools was performed at CVEC using a multiplex RT-qPCR for WNV, SLEV, and WEEV. This assay was performed using the primers and probes listed in <xref ref-type=\"supplementary-material\" rid=\"pone.0147962.s003\">S1 Table</xref>, except the probe for SLEV had a Quasar 670 reporter and Black Hole 2 Quencher (BHQ-2) and the probe for WEEV had a TAMRA reporter and BHQ-2. Reactions were completed on a ViiA 7 Real-Time PCR system (Life Technologies, Grand Island, NY, USA) using SensiFAST Probe Lo-ROX One-step Kits (Bioline, Taunton, MA, USA) and manufacturer recommended protocols. Additional RT-qPCR using the WNV, SLEV, and WEEV primer and probe sets (<xref ref-type=\"supplementary-material\" rid=\"pone.0147962.s003\">S1 Table</xref>) were performed on the RNA from selected virus isolates (<xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>) as part of the sensitivity and specificity testing at Sandia. These reactions were performed using a one-step RT-PCR kit (iTaq Universal Probes One-Step Kit, BioRad, Hercules, CA) and BioRad CFX96 instrument, with detection in the FAM channel. Total reaction volume was 10 &#x003bc;L and contained 500nM of each forward and reverse primer and 200nM of the probe. PCR was performed with an initial reverse transcription step at 50&#x000b0;C for 10 minutes, followed by an initial denaturation for 50 seconds, then 40 cycles of 95&#x000b0;C for 15 seconds and 60&#x000b0;C for 30 seconds followed by a plate read.</p></sec><sec id=\"sec007\"><title>RT-LAMP</title><p>RT-LAMP was performed in 10 &#x003bc;L reaction volumes in thin-walled PCR strip tubes or 96-well plates. The reaction mixture had a final composition (after adding water or template) of 1X Isothermal Amplification Buffer (New England Biolabs, NEB #B0537S) supplemented with an additional 6 mM MgSO4 (NEB #B1003S, final 8 mM MgSO<sub>4</sub>), 0.8 M Betaine (Sigma #B-0300), 1.4 mM each dNTP (NEB #N0447L), 0.32 units/&#x003bc;L Bst 2.0 WarmStart DNA polymerase (NEB #M0538M), 0.2 units/&#x003bc;L AMV reverse transcriptase (NEB #M0277T, or Life Science Advanced Technologies #AMVRTT-5), and 2 &#x003bc;M (or in some instances 4 &#x003bc;M) SYTO 82 detection dye (Life Technologies #S11363) [<xref rid=\"pone.0147962.ref011\" ref-type=\"bibr\">11</xref>]. Primers were used in the amounts typically recommended for LAMP: 0.2 &#x003bc;M each for outer primers F3 and B3; 1.6 &#x003bc;M each for inner primers FIP and BIP; and 0.8 &#x003bc;M each for loop primers LF and LB. RT-LAMP with real-time fluorescence monitoring was carried out in a BioRad CFX96, using detection channel 2 (HEX) for monitoring SYTO 82 dye. Reactions were incubated at a constant temperature of 63&#x000b0;C for 50&#x02013;70 minutes, with plate read steps at intervals of 1 minute (in the BioRad CFX96, this is accomplished with a 48-second single-temperature cycle followed by a plate read which takes approximately 12 seconds in all-channel mode). Incubation was typically followed by inactivation of the enzyme at 95&#x000b0;C for 2 minutes, followed by cooling to 65&#x000b0;C for 2 minutes, and a melt curve from 65&#x02013;95&#x000b0;C in 0.2&#x000b0;C increments. Time-to-positivity values were determined using the BioRad CFX Manager software, using baseline-subtracted curves, and a single threshold value auto-calculated by the CFX manager.</p><p>To demonstrate testing with simpler instrumentation, we also performed reactions using a portable isothermal fluorimeter (Optigene Genie III). The protocol was identical to that described above, except that SYTO 9 (Life Technologies #S34854) was used in place of SYTO 82, to ensure compatibility with the Genie III detection optics. A fluorescence photograph of the reaction endpoint (with SYTO 9) was taken using a blue LED flashlight (CREE WF-502B) and an amber-colored plastic filter (LEE Filters #158). Background fluorescence with SYTO 9 was minimized by photographing the tubes upon a hot plate set to 63&#x000b0;C.</p><p>Gel electrophoretic analysis of RT-LAMP products was performed using an Invitrogen e-Gel (2% agarose with ethidium bromide); 1 &#x003bc;L of RT-LAMP product was diluted with DI water to a total volume of 20 &#x003bc;L that was loaded onto the e-Gel for nucleic acid separation. In addition, to confirm specificity of amplification, 3 &#x003bc;L aliquots of products from positive control reactions were digested with 20 units of <italic>Xba</italic>I (for WEEV nsP4 RT-LAMP), or <italic>Hind</italic>III (for SLEV 3&#x02019;-UTR RT-LAMP) for 1 hour at 37&#x000b0;C in 20 &#x003bc;L total reaction volume, using buffer supplied by the manufacturer (New England Biolabs). Digested products were also run on an e-Gel as described above.</p></sec><sec id=\"sec008\"><title>Sensitivity and Specificity Testing</title><p>SLEV, WEEV and WNV LAMP primers were tested for sensitivity and cross-reactivity against multiple conspecific strains and closely related viruses (<xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>). Sensitivity testing was performed with a dilution series of a single reference isolate of each virus: WEEV Kern 5547 (Genbank KJ554975.1), SLEV Kern 217 (Genbank DQ525916.1), and WNV L-CA-04 SAC-04-7168 (Genbank DQ080059.1). Conspecific strains for SLEV were chosen amongst the lineages described by Kramer et al. [<xref rid=\"pone.0147962.ref027\" ref-type=\"bibr\">27</xref>]. We tested isolates falling within Clades IA (Bfs1750), IB (Coav750), IIA(69M143), VA (BeAn246407), and VII (CorAn9124 and CorAn9275). One conspecific strain of WEEV (Lake 43, Genbank KJ554985) was also tested. All other WEEV isolates within the collection at UC Davis were identical to the Kern 5547 reference isolate within the primer binding regions. A panel of flaviviruses and alphaviruses (<xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>) was developed to test for cross-reactivity. Each primer set (SLEV, WEEV, and WNV) was tested against the entire panel, using 100 PFU equivalents of viral RNA template per reaction. Because of the select agent status of EEEV, primers were tested against an in vitro transcribed portion of nsP4 gene instead of genomic RNA (methods described above), with 10<sup>6</sup> copies of RNA template per reaction.</p></sec><sec id=\"sec009\"><title>Multiplexed RT-LAMP with melt curve analysis</title><p>WNV, SLEV 3&#x02019;-UTR, and WEEV-nsP4 primer sets were tested with mixtures of WNV L-CA-04 SAC-04-7168, SLEV Kern 217, and WEEV Kern 5547 RNA templates. Individual primer sets were combined in equal amounts to create four multiplexed reaction types: WNV + SLEV + WEEV, WNV + SLEV, WNV + WEEV, and SLEV + WEEV. Corresponding viral RNA templates were combined in equal parts and added to matching reaction mixes. Positive control reactions contained a total of 100 PFU equivalents of viral RNA template per reaction. Multiplexed reactions were performed in triplicate with the RT-LAMP protocol described above.</p></sec></sec><sec id=\"sec010\"><title>Results and Discussion</title><sec id=\"sec011\"><title>WEEV and SLEV primer design</title><p>Our primer design protocol identified two candidate primer sets for WEEV (targeting nsP1 and nsP4), and two candidate primer sets for SLEV (targeting the 3&#x02019;-UTR and capsid C protein). According to the sequence alignment the 3&#x02019;-UTR is well conserved among sequenced isolates of SLEV, and yet sufficiently distinct from other flaviviruses to allow specific detection of SLEV by RT-LAMP. The flavivirus 3&#x02019;-UTR was previously targeted by LAMP primer sets designed for dengue serotypes 1&#x02013;4 [<xref rid=\"pone.0147962.ref019\" ref-type=\"bibr\">19</xref>, <xref rid=\"pone.0147962.ref020\" ref-type=\"bibr\">20</xref>]. Similarly, the nsP4 (RNA polymerase) gene of WEEV was among the most highly conserved region of that genome. The region targeted here shared 81% sequence identity with North American isolates of Eastern equine encephalitis virus (EEEV), but was &#x0003c;80% identical to any other alphaviruses.</p><p>In initial screening the WEEV-nsP4 and SLEV 3&#x02019;-UTR sets provided faster amplification and lower detection limits than the WEEV-nsP1 and SLEV-C sets; therefore, only the nsP4 and 3&#x02019;-UTR sets were evaluated further. The WEEV nsP4 and SLEV 3&#x02019;-UTR primer sequences are shown in <xref ref-type=\"table\" rid=\"pone.0147962.t002\">Table 2</xref>, and sequence alignments showing the positioning of the primers are presented in Figs <xref ref-type=\"fig\" rid=\"pone.0147962.g001\">1</xref> and <xref ref-type=\"fig\" rid=\"pone.0147962.g002\">2</xref>. The less efficient WEEV nsP1 and SLEV C primer sequences are provided in <xref ref-type=\"supplementary-material\" rid=\"pone.0147962.s004\">S2 Table</xref></p><table-wrap id=\"pone.0147962.t002\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.t002</object-id><label>Table 2</label><caption><title>RT-LAMP Primers used herein.</title></caption><alternatives><graphic id=\"pone.0147962.t002g\" xlink:href=\"pone.0147962.t002\"/><table frame=\"hsides\" rules=\"groups\"><colgroup span=\"1\"><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/></colgroup><thead><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\">Primer Name</th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Genome position<xref ref-type=\"table-fn\" rid=\"t002fn001\"><sup>1</sup></xref></th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Sequence<xref ref-type=\"table-fn\" rid=\"t002fn002\"><sup>2</sup></xref></th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Source</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; F3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10497&#x02013;10516</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GACTGGGTTAWCAAAGCCAA</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">New</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; B3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10736&#x02013;10718</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GGTCTCCTCTAACCTCTAG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; FIP(<underline>F1c</underline> + F2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10641&#x02013;10622;10583&#x02013;10600</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>GCATGTACAGACAGCACCTT</underline> +GCACAAGTCAGACCAGAA</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; BIP<xref ref-type=\"table-fn\" rid=\"t002fn003\"><sup>3</sup></xref>(<underline>B1c</underline> + B2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10644&#x02013;10665;10717&#x02013;10700</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>CAGGAGGACTGGGTTAACAAAG</underline> +TCCTTACGCCATGGTCAC</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; LF</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10618&#x02013;10601</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">CATGCTTTCAGGTGGCAT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; LB</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10681&#x02013;10688</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GGCCCAAACCATGGAGTG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 F3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">5949&#x02013;5958</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GTCAACGGTAATTAACAGGT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">New</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 B3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">6187&#x02013;6170</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GGACGGCTGATCTTATCT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 FIP(<underline>F1c</underline> + F2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">6087&#x02013;6070;6025&#x02013;6044</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>GCCGTCCACCATGTCAAG</underline> +CCTACAGTAGCCAGTTATTG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 BIP(<underline>B1c</underline> + B2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">6088&#x02013;6108;6156&#x02013;6138</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>GCATCGTGCTGTCTAGATACM</underline> +GCTATGCTTCTTTGGGTAG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 LF</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">6067&#x02013;6048</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">ACGCATCGTATTCATCTGTT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 LB</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">6116&#x02013;6134</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">TTTGTCCGGCTAAACTGAG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV F3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1028&#x02013;1046</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">TGGATTTGGTTCTCGAAGG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Parida [<xref rid=\"pone.0147962.ref018\" ref-type=\"bibr\">18</xref>]</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV B3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1228&#x02013;1210</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GGTCAGCACGTTTGTCATT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV FIP(F<underline>1c</underline> + TTTT + F2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1121&#x02013;1100;1050&#x02013;1069</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>TTGGCCGCCTCCATATTCATCA</underline> + TTTT +CAGCTGCGTGACTATCATGT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV BIP(<underline>B1c</underline> + TTTT + B2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1144&#x02013;1165;1208&#x02013;1190</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>TGCTATTTGGCTACCGTCAGCG</underline> + TTTT +TGAGCTTCTCCCATGGTCG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV LF</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1093&#x02013;1075</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">CATCGATGGTAGGCTTGTC</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV LB</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1169&#x02013;1186</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">TCTCCACCAAAGCTGCGT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr></tbody></table></alternatives><table-wrap-foot><fn id=\"t002fn001\"><p><sup>1</sup> Genome positions are based on SLEV strain Kern217 (Genbank DQ525916.1), WEEV strain Montana-64 (Genbank GQ287643.1), and (as per reference [<xref rid=\"pone.0147962.ref018\" ref-type=\"bibr\">18</xref>]) WNV strain NY99 (Genbank AF196835).</p></fn><fn id=\"t002fn002\"><p><sup>2</sup> Sequence for inner primers FIP and BIP is shown separating out the F1c or B1c region (underlined) followed by the F2 or B2 region (not underlined). The complete FIP primer sequence is obtained by concatenating the F1c and F2 sequences (FIP = F1c + F2), and likewise for BIP. Primers designed by Parida <italic>et al</italic> [<xref rid=\"pone.0147962.ref018\" ref-type=\"bibr\">18</xref>] also include a TTTT linker between the two segments of the inner primers.</p></fn><fn id=\"t002fn003\"><p><sup>3</sup> A second BIP primer with 3 single-base substitutions in the B1c region intended to cover additional SLEV strains was designed but not used for testing; the alternative sequence (BIP = <underline>B1c</underline> + B2) is <underline>CAGG<bold>C</bold>GGACTGGG<bold>T</bold>A<bold>A</bold>TCAAAG</underline> + TCCTTACGCCATGGTCAC. The alternative BIP sequence is a match for two Mexican isolates designated Palenque-C475 and Palenque-A770 (Genbank JQ957868.1 and JQ957869.1), which form a distinct phylogenetic clade [<xref rid=\"pone.0147962.ref030\" ref-type=\"bibr\">30</xref>].</p></fn></table-wrap-foot></table-wrap><p>Gel electrophoresis of the SLEV (3&#x02019;-UTR) and WEEV (nsP4) RT-LAMP reaction products resulted in a prototypical ladder-like banding pattern typical of LAMP reactions (<xref ref-type=\"fig\" rid=\"pone.0147962.g003\">Fig 3</xref>, lanes 1 and 5). An online tool for LAMP restriction digest fragment analysis (<ext-link ext-link-type=\"uri\" xlink:href=\"http://creisle.github.io/creisle.lamprflp/\">http://creisle.github.io/creisle.lamprflp/</ext-link>) was used to predict restriction digest banding patterns based on the LAMP amplicon structure proposed by Notomi <italic>et al</italic> [<xref rid=\"pone.0147962.ref006\" ref-type=\"bibr\">6</xref>]. A restriction digest of the WEEV nsP4 amplicon with <italic>XbaI</italic> was predicted to yield main bands at 67 and 115 bp, with fainter bands at 50, 63, 77, and 97 bp. This is consistent with the banding pattern in Lane 2, which has primary bands slightly larger than 50 and 100 bp. Digestion of the SLEV 3&#x02019;-UTR amplicon with <italic>Hin</italic>dIII was predicted to yield bright bands at 130, 185, and 240 bp, with fainter bands at 58, 62, 76, 80, 136, and 146 bp. This corresponds closely to the pattern of banding in Lane 6. Additional faint bands in these digests at higher molecular weight may be due to incomplete digestion, or the presence of other amplicon structures not proposed by Notomi <italic>et al</italic> [<xref rid=\"pone.0147962.ref005\" ref-type=\"bibr\">5</xref>, <xref rid=\"pone.0147962.ref006\" ref-type=\"bibr\">6</xref>].</p><fig id=\"pone.0147962.g003\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.g003</object-id><label>Fig 3</label><caption><title>Electrophoretic analysis of RT-LAMP amplification products with WEEV nsP4 and SLEV 3&#x02019; UTR primer sets.</title><p>The gel is a 2% agarose E-gel (Invitrogen) with ethidium bromide. The marker (&#x0201c;M&#x0201d;) is a Low Molecular Weight DNA Ladder (New England Biolabs). Lane 1: WEEV positive control reaction with 2 PFU equivalent WEEV RNA in 20 &#x003bc;L. Lane 2: WEEV positive control after digestion with <italic>Xba</italic>I. Lane 3: No-template control reaction with WEEV nsP4 primer set. Lane 4: no sample. Lane 5: SLEV positive control reaction with 2 PFU equivalent SLEV RNA in 20 &#x003bc;L. Lane 6: SLEV positive control after digestion with <italic>Hin</italic>dIII. Lane 7: No-template control reaction with SLEV 3&#x02019;-UTR primer set. Cutting sites within the amplicon regions are shown in Figs <xref ref-type=\"fig\" rid=\"pone.0147962.g001\">1</xref> and <xref ref-type=\"fig\" rid=\"pone.0147962.g002\">2</xref>.</p></caption><graphic xlink:href=\"pone.0147962.g003\"/></fig><p>A recent survey of 33 WEEV strains [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>] collected primarily in the United States indicated that all sampled strains were ~95% identical. The six strains used here for the sequence alignment were representative of three (Groups B1, B2, and B3) out of four of the WEEV lineages defined by Bergren et al. [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>]. The nsP4 regions of these isolates had a single polymorphic site falling within a LAMP primer binding region. This mismatch occurred at the 3&#x02019; end of the B1c priming region. Based upon LAMP primer design guidelines provided by Eiken Genomics (<ext-link ext-link-type=\"uri\" xlink:href=\"http://loopamp.eiken.co.jp/e/lamp/primer.html\">http://loopamp.eiken.co.jp/e/lamp/primer.html</ext-link>), we infer that mismatches at the 3&#x02019; end of the B2, and the 5&#x02019; end of the B1c are more detrimental than mismatches at the 3&#x02019; end of B1c. However, since the effect of mismatches on LAMP performance is not well characterized, the polymorphic site was accommodated with a degenerate base (M) which covers all isolates used in the sequence alignment. After primer development, we did identify a single isolate of WEEV within the collection at UC Davis (Lake43, Genbank #KJ554985) with a single mismatch to the primer set, at the 5&#x02019; end of the F1c priming region. RNA from this isolate was successfully amplified, despite the mismatch (which also occurs in one other closely related WEEV isolate within Genbank, #KJ554984). The fourth WEEV lineage (Group A) described by Bergren et al. [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>] comprised early WEEV isolates from 1930 and 1941. When we compared our primers to the &#x0201c;California&#x0201d; isolate (Genbank KJ554965.1, from the &#x0201c;A&#x0201d; lineage defined by Bergen et al. [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>]), two mismatches were identified, one each near the middle of the F1c and B1c priming regions. The tolerance of LAMP to mismatches in the middle of priming regions is not well described. However, due to the high conservation of the target site within the WEEV nsP4 gene, our LAMP primers would likely detect most naturally occurring WEEV isolates.SLEV has a greater geographic distribution than WEEV, and sequences in Genbank show significantly higher divergence than is observed for sequenced isolates of WEEV. A phylogenetic tree based on the SLEV E gene sequences identified 7 clades [<xref rid=\"pone.0147962.ref029\" ref-type=\"bibr\">29</xref>]. Clades I and II represent primarily North American isolates, and clades III-VI represent primarily Central and South American isolates. The E gene is more divergent than the 3&#x02019; UTR region targeted by our primers. However, a comprehensive tree was not available for the 3&#x02019; UTR region, due to limited published sequences for SLEV that include the 3&#x02019; UTR. Sequence alignment for primer design included only North American isolates, and sensitivity testing was performed primarily with a single strain (Kern 217, Clade IIC). To determine whether the primer set was effective for detecting diverse lineages of SLEV, additional isolates falling within Clades IA (Bfs1750), IB (Coav750), IIA(69M143), VA (BeAn246407), and VII (CorAn9124 and CorAn9275) were tested. SLEV LAMP primers successfully detected RNA from all tested isolates, originating from both North and South America. In contrast the RT-qPCR assay used for surveillance in California failed to detect the two Clade VII isolates from Argentina, as indicated in <xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref></p></sec><sec id=\"sec012\"><title>Sensitivity and specificity of RT-LAMP for WEEV, SLEV, and WNV</title><p>Real-time monitoring of RT-LAMP reactions for WEEV and SLEV primer sets was performed using serial dilutions of viral RNA, extracted from cultured virus. <xref ref-type=\"fig\" rid=\"pone.0147962.g004\">Fig 4</xref> shows real-time monitoring curves for RT-LAMP reactions for each virus, as well as standard curves generated for these viral RNA by both RT-LAMP and RT-qPCR. In addition, a standard curve was generated using the published WNV RT-LAMP primer set [<xref rid=\"pone.0147962.ref018\" ref-type=\"bibr\">18</xref>] and was used for comparison to RT-qPCR; the resulting standard curves for WNV are shown in <xref ref-type=\"supplementary-material\" rid=\"pone.0147962.s001\">S1 Fig</xref>. All three RT-LAMP primer sets reliably detect their corresponding targets at 0.1 PFU equivalent, with less reliable detection down to 0.01 PFU. For all three viruses, RT-qPCR produced linear calibration curves to Ct &#x0003e;35, corresponding to viral titers of ~0.01 PFU, and less reliable detection down to 0.001 PFU. Overall, RT-LAMP reactions were 10-fold less sensitive than the corresponding RT-qPCR assays. The threshold of reliable detection for RT-LAMP was 100 PFU/mL corresponding to Ct ~30&#x02013;32 by RT-qPCR. Positive detection generally occurred in 20&#x02013;30 minutes, and often in &#x0003c;10 minutes for samples containing &#x0003e;10<sup>4</sup> PFU/mL. Although RT-LAMP sensitivity was slightly lower than RT-qPCR, WNV RT-LAMP reactions were markedly more sensitive than commercially available antigen tests. Two common tests used for WNV surveillance are VecTest&#x000ae; WNV antigen assay (Medical Analysis Systems, Inc., Camarillo, CA) and Rapid Analyte Measurement Platform (RAMP&#x000ae;) WNV test (Response Biomedical, Corp., Burnaby British Columbia, Canada); these tests have minimum sensitivities of 10<sup>5</sup> and 10<sup>3</sup> PFU/mL, respectively.</p><fig id=\"pone.0147962.g004\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.g004</object-id><label>Fig 4</label><caption><title>Quantitative analysis of SLEV and WEEV RT-LAMP.</title><p>Panels A,B: Real-time RT-LAMP amplification curves for SLEV isolate Kern 217 (A) and WEEV isolate Kern 5547 (B). Reactions were performed in triplicate with viral RNA ranging from 10,000 PFU equivalents to 0.01 PFU equivalent per 10 &#x003bc;L reaction. Reactions were monitored in real-time using a BioRad CFX96 thermal cycler, using the HEX channel to monitor the signal from SYTO 82; the abbreviation &#x0201c;RFU&#x0201d; refers to Relative Fluorescence Units. Panels C,D: Standard curves for SLEV (C) and WEEV (D) generated by real-time monitoring of RT-LAMP (blue circles), versus RT-qPCR (red triangles). Time-to-positivity in RT-LAMP was determined by the BioRad CFX manager software, using baseline-subtracted quantitation data.</p></caption><graphic xlink:href=\"pone.0147962.g004\"/></fig><p>Although real-time monitoring of LAMP by means such as turbidity or fluorescent dyes is common [<xref rid=\"pone.0147962.ref010\" ref-type=\"bibr\">10</xref>, <xref rid=\"pone.0147962.ref011\" ref-type=\"bibr\">11</xref>], we note that real-time monitoring of RT-LAMP does not produce as precise a standard curve as RT-qPCR, particularly at lower viral RNA levels where time to positivity can vary substantially. We attribute this to the continuous nature of RT-LAMP, as opposed to the discrete cycling nature of RT-qPCR. In a well-designed RT-qPCR assay, each cycle in the exponential amplification regime leads to a reproducible doubling of the amount of amplicon. In RT-LAMP, however, slight variations in the early stages of the reaction can result in larger fluctuations in amplification time, particularly in the stochastic regime with few template copies, where multiple mechanisms may reduce the efficiency of the RT step in particular [<xref rid=\"pone.0147962.ref031\" ref-type=\"bibr\">31</xref>]. Real-time monitoring of RT-LAMP reactions can provide order-of-magnitude estimates of viral RNA concentration, but for precise quantitation RT-qPCR is preferable, particularly for low concentrations of viral RNA. The value of RT-LAMP for viral detection lies in the simplicity of the technique, particularly when applied in low-resource settings, where a yes-or-no answer determined by an endpoint measurement is sufficient.</p><p>Although the purpose of the current study was to develop and characterize new primer sets for SLEV and WEEV, we also demonstrated RT-LAMP detection of these targets using simpler instrumentation, as shown in <xref ref-type=\"supplementary-material\" rid=\"pone.0147962.s002\">S2 Fig</xref>. First, we carried out the RT-LAMP reactions for SLEV and WEEV using the Genie III, a portable isothermal fluorimeter with rechargeable battery and built-in wireless communication. Secondly, we took a photograph of the fluorescence endpoint using a handheld LED flashlight for illumination, and placing an amber-colored plastic film in front of the camera lens. In both cases, the fluorescence endpoint is clearly distinguishable between no-template control reactions and positive control reactions (tested down to 0.1 PFU equivalent of template).</p><p>For specificity testing, WEEV, SLEV, and WNV primer sets were tested against a panel of alphaviruses and flaviviruses, as indicated in <xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>. For WEEV, the closest relatives are other New World alphaviruses, Venezuelan equine encephalitis virus (VEEV) and Eastern equine encephalitis virus (EEEV). We found no cross-reactivity between any of the primer sets among non-target viruses. The only unexpected positive reactions were two isolated instances that we attribute to cross-contamination, where RNA from positive controls was inadvertently introduced into non-target cross-reactivity reactions. The specific instances (noted in <xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>) are a single positive reaction (out of 18 replicates) when VEEV TC-83 RNA template was tested with the SLEV primers, and a single positive reaction (out of 11 replicates) when SLEV strain CorAn 9275 was tested with WNV primers. In both cases, the single positive amplifications displayed identical melt curves to positive-control reactions performed on the same plate, and these isolated positive results could not be reproduced. Otherwise no evidence of cross-reactivity between primers sets and non-target viruses was observed. In addition, there was no evidence of non-specific amplification in no-template control reactions with the SLEV and WEEV primer sets incubated for up to 50 minutes. Previously, non-specific amplification was observed with the previously published WNV primer set, but most often developed after &#x0003e; 50 minutes of incubation.</p><p>We tested archived RNA from 12 pooled mosquito samples collected in California in 2002 or 2014 that were positive by RT-qPCR for WEEV, SLEV, or WNV, or negative for all three, to determine if our RT-LAMP assays could detect these viruses in mosquito samples. Results of these tests are presented in <xref ref-type=\"table\" rid=\"pone.0147962.t003\">Table 3</xref>. The RT-LAMP assay detected SLEV in four pools that had RT-qPCR Cts ranging from 24.4 to 27.7. The RT-LAMP assay also detected WEEV in 5 pools that had RT-qPCR Cts ranging from 22.8 to 26.5, but failed to detect WEEV in one sample with a Ct value of 32, which is below the sensitivity demonstrated for the WEEV RT-LAMP assays. Sample sizes were limited, but the new primer sets readily detected SLEV and WEEV in mosquito pool RNA. Although sensitivity was slightly reduced compared to RT-qPCR, the specificity of these new primers was absolute.</p><table-wrap id=\"pone.0147962.t003\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.t003</object-id><label>Table 3</label><caption><title>Analysis of mosquito pools by SLEV, WEEV, and WNV RT-LAMP.</title></caption><alternatives><graphic id=\"pone.0147962.t003g\" xlink:href=\"pone.0147962.t003\"/><table frame=\"hsides\" rules=\"groups\"><colgroup span=\"1\"><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/></colgroup><thead><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\"/><th align=\"center\" colspan=\"3\" rowspan=\"1\"><underline>RT-qPCR Ct</underline><xref ref-type=\"table-fn\" rid=\"t003fn001\"><sup>1</sup></xref></th><th align=\"center\" colspan=\"3\" rowspan=\"1\"><underline>RT-LAMP Time-to-positivity (min)</underline><xref ref-type=\"table-fn\" rid=\"t003fn002\"><sup>2</sup></xref></th></tr><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\">Sample<xref ref-type=\"table-fn\" rid=\"t003fn003\"><sup>3</sup></xref></th><th align=\"center\" rowspan=\"1\" colspan=\"1\">SLEV</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WEEV</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WNV</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">SLEV</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WEEV</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WNV</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 490</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">22.8</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">12.2, 12.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 512</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">26.5</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">14.0, 15.9</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 513</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">25.1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">13.0, 14.6</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 984</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">23.0</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">12.1, 12.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 942</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">24.5</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">12.2, 12.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 862</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">27.7</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">13.1, 13.9</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 IMPR 136</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">24.4</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">12.4, 12.5</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 IMPR 139</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">26.5</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">13.4, 14.0</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 IMPR 145</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">27.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">32.0</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">19.1, 21.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2014 KERN 14&#x02013;163</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">23.0</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">10.8, 11.1</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2014 MARN 14&#x02013;300</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">25.0</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">13.3, 13.8</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2014 SUYA 14&#x02013;144</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr></tbody></table></alternatives><table-wrap-foot><fn id=\"t003fn001\"><p><sup>1</sup>Triplex RT-qPCRfor SLEV, WEEV, and WNV surveillance samples performed by UC Davis Center for Vectorborne Diseases.</p></fn><fn id=\"t003fn002\"><p><sup>2</sup>Results of two replicates reported</p></fn><fn id=\"t003fn003\"><p><sup>3</sup>Sample designations indicate year and location of sample collection within California: COAV = Coachella Valley, IMPR = Imperial Valley; KERN = Kern County; MARN = Marin-Sonoma counties, SUYA = Sutter-Yuba counties.</p></fn></table-wrap-foot></table-wrap></sec><sec id=\"sec013\"><title>Multiplexing RT-LAMP by melt curve analysis</title><p>Multiplexed amplification by LAMP was consistent, and all positive control RT-LAMP reactions amplified target RNA in less than 30 minutes. No-template controls remained negative for 40 minutes, after which non-specific amplification was detected in multiplexes containing WNV and WEEV primer sets together. Amplification in WNV + SLEV + WEEV and WNV + WEEV no-template controls likely resulted from primer-dimer formation between the WNV BIP and the WEEV FIP primers. Negative control reactions containing the combination of WNV and WEEV primer sets showed left-skewed melting peaks at 86.3&#x000b0;C, corresponding to melting of amplified primer-dimers. The intensity of the negative control melt curves was lower in reactions with a lower initial concentration of WNV/WEEV primer sets. Positive controls were distinguishable from negative controls and produced LAMP products that were suitable for melt curve analysis. Examples are presented in <xref ref-type=\"fig\" rid=\"pone.0147962.g005\">Fig 5</xref>.</p><fig id=\"pone.0147962.g005\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.g005</object-id><label>Fig 5</label><caption><title>Melt derivative curves for singleplex and multiplex RT-LAMP.</title><p>Solid black lines show average data from triplicate runs. Colored shaded areas represent the contributions of individual RT-LAMP targets to the overall signals to aid visualization. Blue = WEEV. Yellow = SLEV. Red = WNV. A) Singleplex reactions. B) Three-way combination. Only two peaks are distinguishable by eye. C) WEEV + WNV no template control. A relatively small melting peak is present, corresponding to spurious amplification after 40 min. D-F) Pairwise combinations. A smaller WEEV peak is distinguishable from WNV and SLEV peaks in D and F. WNV and SLEV peaks are indistinguishable by eye in E.</p></caption><graphic xlink:href=\"pone.0147962.g005\"/></fig><p>The melting curves of multiplexed LAMP reactions contained up to two distinguishable peaks for target identification. The first peak (84.5&#x02013;84.9&#x000b0;C) corresponded to the melting of WEEV-specific amplicons. The second peak (86.5&#x02013;87&#x000b0;C) corresponds to the melting of WNV-specific amplicons, SLEV-specific amplicons, or their combination.</p><p>The center and width of amplicon melting distributions determined the ease of multiplexed target detection. For example, the melting point of WNV amplicons was 0.5&#x000b0;C higher than that of SLEV amplicons. Because this difference is close to the 0.2&#x000b0;C accuracy limit of the real-time PCR instrument, reactions containing both WNV and SLEV amplicons displayed a single melting curve&#x02014;with an intermediate melting temperature&#x02014;instead of two discrete curves. Thus, simple peak picking was insufficient to differentiate WNV from SLEV in reactions containing both WNV and SLEV primer sets.</p><p>Furthermore, the relative intensity of discrete melt peaks reflected the relative abundance of individual LAMP products. However, in reactions with equal starting amounts of template, the WEEV amplicon melt-curve intensity was less than that of either the WNV or SLEV amplicons. This suggests that amplification with the WEEV primer set is slower or less efficient than either the WNV or SLEV primer sets. We have not tested experimentally whether this lower efficiency of amplification for WEEV reduces the detection limit of WEEV when present together in the same sample with a larger amount of WNV or SLEV.</p><p>Given the tendency to non-specific amplification observed with the combination of the WNV and WEEV primers, and the overlap of the WNV and SLEV melt curves, we conclude that only the WEEV and SLEV assays could be multiplexed in a single reaction. A more complex curve fitting algorithm based on parameters such as peak width, skew and temperature may be capable of robust differentiation of WEEV, SLEV, and WNV amplicons in multiplexed LAMP reactions. However, to our knowledge such an approach has not been demonstrated for LAMP melt curve analysis.</p></sec></sec><sec sec-type=\"conclusions\" id=\"sec014\"><title>Conclusion</title><p>The newly developed RT-LAMP primer sets for SLEV and WEEV show sensitivity and specificity comparable to a previously reported primer set for WNV [<xref rid=\"pone.0147962.ref016\" ref-type=\"bibr\">16</xref>]. Although the RT-LAMP assays are approximately an order of magnitude less sensitive than the corresponding RT-qPCR assays, the assay consistently detected to 0.1 PFU/reaction equivalents without cross-reactivity between related viruses. In addition, the RT-LAMP method can be greatly simplified compared to standard RT-qPCR reactions. Here purified RNA targets and a conventional real-time PCR thermal cycler were used to characterize RT-LAMP reactions, but we also demonstrate compatibility with a low-cost portable fluorimeter for monitoring isothermal amplification reactions, as well as compatibility with non-instrumented detection Future work is directly at developing simplified RT-LAMP protocols for RNA viruses such as WNV, SLEV, and WEEV. New protocols will utilize the tolerance of the Bst DNA polymerase to potential &#x0201c;PCR inhibitors&#x0201d; that are often present in crude samples such as mosquito or tissue homogenates. The combination of simplified instrumentation, minimal sample prep, no RNA extraction, and low reagent cost may enable the development of RT-LAMP based assays that are as simple to use as commercially available antigen tests. This would enable highly sensitive and specific surveillance for vector-borne diseases in basic laboratories or field settings where RT-qPCR resources are unavailable.</p></sec><sec sec-type=\"supplementary-material\" id=\"sec015\"><title>Supporting Information</title><supplementary-material content-type=\"local-data\" id=\"pone.0147962.s001\"><label>S1 Fig</label><caption><title>Standard curves for WNV RT-LAMP and RT-qPCR.</title><p>RT-LAMP time to positivity (blue circles) versus RT-qPCR Ct (red triangles), for WNV isolate L-CA-04 SAC-04-7168 using the RT-LAMP primers published by Parida et al. [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>], and the RT-qPCR primers published by Lanciotti et al [<xref rid=\"pone.0147962.ref002\" ref-type=\"bibr\">2</xref>]. At least three technical replicates were performed for each template concentration.</p><p>(PDF)</p></caption><media xlink:href=\"pone.0147962.s001.pdf\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"pone.0147962.s002\"><label>S2 Fig</label><caption><title>Detection of SLEV and WEEV by RT-LAMP using simplified techniques.</title><p>(A) SLEV and WEEV RNA were detected by RT-LAMP using the OptiGene Genie III, a portable, battery-operated real-time fluorimeter. The screenshot in the inset of panel (A) shows real-time amplification curves generated for eight reactions, which are specified in Panel B. Reactions 1&#x02013;4 use the SLEV 3&#x02019; UTR primer set, with 10, 1, 0.1 PFU equivalents of SLEV RNA (Reaction 4 is an SLEV no-template control). Reactions 5&#x02013;8 use the WEEV nsP4 primer set, with 10, 1, 0.1 PFU equivalents of WEEV RNA (Reaction 8 is a WEEV no-template control). Panel (B) illustrates the endpoint of the same eight reactions, after incubation at 63&#x000b0;C for 30 minutes. To take this photograph, the strip of tubes was set upon a dark background, on a hotplate set at 63&#x000b0;C. Fluorescence was excited using a handheld blue LED flashlight, and a sheet of amber plastic film (LEE Filters #158) was placed in front of the lens of a digital camera. No contrast adjustment was applied. In Panel A, a barcode label affixed to the instrument has been pixelated to obscure identification.</p><p>(TIF)</p></caption><media xlink:href=\"pone.0147962.s002.tif\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"pone.0147962.s003\"><label>S1 Table</label><caption><title>RT-qPCR primers used for quantitation of Western equine encephalitis virus (WEEV), St. Louis encephalitis virus (SLEV), and West Nile virus (WNV) viral RNA.</title><p>IBFQ = Iowa Black FQ; BHQ1 = Black Hole Quencher 1 (The different quenchers have similar performance when paired with FAM, and do not affect the results of the PCR assays). The primer set names from the original literature references are provided in the table next to the reference numbers.</p><p>(PDF)</p></caption><media xlink:href=\"pone.0147962.s003.pdf\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"pone.0147962.s004\"><label>S2 Table</label><caption><title>Additional RT-LAMP primer sets for SLEV and WEEV.</title><p>(PDF)</p></caption><media xlink:href=\"pone.0147962.s004.pdf\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material></sec></body><back><ack><p>This work was supported by a grant from the Defense Threat Reduction Agency (DTRA), contract DTRA1002710697. Sandia National Laboratories is a multi-program laboratory managed and operated by Sandia Corporation, a wholly owned subsidiary of Lockheed Martin Corporation, for the U.S. Department of Energy&#x02019;s National Nuclear Security Administration under Contract DE-AC04-94AL85000.</p></ack><ref-list><title>References</title><ref id=\"pone.0147962.ref001\"><label>1</label><mixed-citation publication-type=\"journal\"><name><surname>Bergren</surname><given-names>NA</given-names></name>, <name><surname>Auguste</surname><given-names>AJ</given-names></name>, <name><surname>Forrester</surname><given-names>NL</given-names></name>, <name><surname>Negi</surname><given-names>SS</given-names></name>, <name><surname>Braun</surname><given-names>WA</given-names></name>, <name><surname>Weaver</surname><given-names>SC</given-names></name>. <article-title>Western equine encephalitis virus: evolutionary analysis of a declining alphavirus based on complete genome sequences</article-title>. <source>J Virol</source>. <year>2014</year>;<volume>88</volume>(<issue>16</issue>):<fpage>9260</fpage>&#x02013;<lpage>7</lpage>. Epub 2014/06/06. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jvi.01463-14\">10.1128/jvi.01463-14</ext-link></comment> ; PubMed Central PMCID: PMCPmc4136285.<pub-id pub-id-type=\"pmid\">24899192</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref002\"><label>2</label><mixed-citation publication-type=\"journal\"><name><surname>Reisen</surname><given-names>WK</given-names></name>, <name><surname>Lothrop</surname><given-names>HD</given-names></name>, <name><surname>Wheeler</surname><given-names>SS</given-names></name>, <name><surname>Kennsington</surname><given-names>M</given-names></name>, <name><surname>Gutierrez</surname><given-names>A</given-names></name>, <name><surname>Fang</surname><given-names>Y</given-names></name>, <etal>et al</etal>\\n<article-title>Persistent West Nile virus transmission and the apparent displacement St. Louis encephalitis virus in southeastern California, 2003&#x02013;2006</article-title>. <source>J Medical Entomol</source>. <year>2008</year>;<volume>45</volume>(<issue>3</issue>):<fpage>494</fpage>&#x02013;<lpage>508</lpage>. Epub 2008/06/07. ; PubMed Central PMCID: PMCPmc2435167.<pub-id pub-id-type=\"pmid\">18533445</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref003\"><label>3</label><mixed-citation publication-type=\"other\">Oviatt J. Saint Louis encephalitis virus detected in valley mosquitoes for the first time since 2003: Coachella Valley Mosquito and Vector Control District; 2015 [cited 2015 10/2/2015]. Press release]. Available: <ext-link ext-link-type=\"uri\" xlink:href=\"http://www.cvmvcd.org/press/releases/2015/engpressreleaseslewnv080515.pdf\">http://www.cvmvcd.org/press/releases/2015/engpressreleaseslewnv080515.pdf</ext-link>.</mixed-citation></ref><ref id=\"pone.0147962.ref004\"><label>4</label><mixed-citation publication-type=\"journal\"><name><surname>Vidal</surname><given-names>F</given-names></name>, <name><surname>Julia</surname><given-names>SA</given-names></name>, <name><surname>Carolina</surname><given-names>G</given-names></name>, <name><surname>Edna</surname><given-names>RC</given-names></name>, <name><surname>Jorge</surname><given-names>G</given-names></name>, <name><surname>Felix</surname><given-names>WS-M</given-names></name>, <etal>et al</etal>\\n<article-title>St. Louis Encephalitis virus infection in woman, Peru</article-title>. <source>Emerg Infect Dis</source>. <year>2014</year>;<volume>20</volume>(<issue>4</issue>):<fpage>730</fpage>\\n<comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.3201/eid2004.131735\">10.3201/eid2004.131735</ext-link></comment>\\n<pub-id pub-id-type=\"pmid\">24655897</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref005\"><label>5</label><mixed-citation publication-type=\"journal\"><name><surname>Nagamine</surname><given-names>K</given-names></name>, <name><surname>Hase</surname><given-names>T</given-names></name>, <name><surname>Notomi</surname><given-names>T</given-names></name>. <article-title>Accelerated reaction by loop-mediated isothermal amplification using loop primers</article-title>. <source>Mol Cell Probes</source>. <year>2002</year>;<volume>16</volume>(<issue>3</issue>):<fpage>223</fpage>&#x02013;<lpage>9</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1006/mcpr.2002.0415\">10.1006/mcpr.2002.0415</ext-link></comment> .<pub-id pub-id-type=\"pmid\">12144774</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref006\"><label>6</label><mixed-citation publication-type=\"journal\"><name><surname>Notomi</surname><given-names>T</given-names></name>, <name><surname>Okayama</surname><given-names>H</given-names></name>, <name><surname>Masubuchi</surname><given-names>H</given-names></name>, <name><surname>Yonekawa</surname><given-names>T</given-names></name>, <name><surname>Watanabe</surname><given-names>K</given-names></name>, <name><surname>Amino</surname><given-names>N</given-names></name>, <etal>et al</etal>\\n<article-title>Loop-mediated isothermal amplification of DNA</article-title>. <source>Nucleic Acids Res</source>. <year>2000</year>;<volume>28</volume>(<issue>12</issue>). <fpage>e63</fpage>\\n<comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1093/nar/28.12.e63\">10.1093/nar/28.12.e63</ext-link></comment> .<pub-id pub-id-type=\"pmid\">10871386</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref007\"><label>7</label><mixed-citation publication-type=\"journal\"><name><surname>Iwamoto</surname><given-names>T</given-names></name>, <name><surname>Sonobe</surname><given-names>T</given-names></name>, <name><surname>Hayashi</surname><given-names>K</given-names></name>. <article-title>Loop-mediated isothermal amplification for direct detection of Mycobacterium tuberculosis complex, M. avium, and M. intracellulare in sputum samples</article-title>. <source>J Clin Microbiol</source>. <year>2003</year>;<volume>41</volume>(<issue>6</issue>):<fpage>2616</fpage>&#x02013;<lpage>22</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/JCM.41.6.2616&#x02013;2622.2003\">10.1128/JCM.41.6.2616&#x02013;2622.2003</ext-link></comment> .<pub-id pub-id-type=\"pmid\">12791888</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref008\"><label>8</label><mixed-citation publication-type=\"journal\"><name><surname>Polley</surname><given-names>SD</given-names></name>, <name><surname>Gonz&#x000e1;lez</surname><given-names>IJ</given-names></name>, <name><surname>Mohamed</surname><given-names>D</given-names></name>, <name><surname>Daly</surname><given-names>R</given-names></name>, <name><surname>Bowers</surname><given-names>K</given-names></name>, <name><surname>Watson</surname><given-names>J</given-names></name>, <etal>et al</etal>\\n<article-title>Clinical evaluation of a LAMP test kit for diagnosis of imported malaria</article-title>. <source>J Infect Dis</source>. <year>2013</year>\\n<comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1093/infdis/jit183\">10.1093/infdis/jit183</ext-link></comment></mixed-citation></ref><ref id=\"pone.0147962.ref009\"><label>9</label><mixed-citation publication-type=\"journal\"><name><surname>Fukuta</surname><given-names>S</given-names></name>, <name><surname>Iida</surname><given-names>T</given-names></name>, <name><surname>Mizukami</surname><given-names>Y</given-names></name>, <name><surname>Ishida</surname><given-names>A</given-names></name>, <name><surname>Ueda</surname><given-names>J</given-names></name>, <name><surname>Kanbe</surname><given-names>M</given-names></name>, <etal>et al</etal>\\n<article-title>Detection of Japanese yam mosaic virus by RT-LAMP</article-title>. <source>Arch Virol</source>. <year>2003</year>;<volume>148</volume>(<issue>9</issue>):<fpage>1713</fpage>&#x02013;<lpage>20</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1007/s00705-003-0134-5\">10.1007/s00705-003-0134-5</ext-link></comment> .<pub-id pub-id-type=\"pmid\">14505084</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref010\"><label>10</label><mixed-citation publication-type=\"journal\"><name><surname>Mori</surname><given-names>Y</given-names></name>, <name><surname>Nagamine</surname><given-names>K</given-names></name>, <name><surname>Tomita</surname><given-names>N</given-names></name>, <name><surname>Notomi</surname><given-names>T</given-names></name>. <article-title>Detection of loop-mediated isothermal amplification reaction by turbidity derived from magnesium pyrophosphate formation</article-title>. <source>Biochem Biophys Res Commun</source>. <year>2001</year>;<volume>289</volume>(<issue>1</issue>):<fpage>150</fpage>&#x02013;<lpage>4</lpage>. Epub 2001/11/16. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1006/bbrc.2001.5921\">10.1006/bbrc.2001.5921</ext-link></comment> .<pub-id pub-id-type=\"pmid\">11708792</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref011\"><label>11</label><mixed-citation publication-type=\"journal\"><name><surname>Ahmad</surname><given-names>F</given-names></name>, <name><surname>Seyrig</surname><given-names>G</given-names></name>, <name><surname>Tourlousse</surname><given-names>DM</given-names></name>, <name><surname>Stedtfeld</surname><given-names>RD</given-names></name>, <name><surname>Tiedje</surname><given-names>JM</given-names></name>, <name><surname>Hashsham</surname><given-names>SA</given-names></name>. <article-title>A CCD-based fluorescence imaging system for real-time loop-mediated isothermal amplification-based rapid and sensitive detection of waterborne pathogens on microchips</article-title>. <source>Biomed Microdevices</source>. <year>2011</year>;<volume>13</volume>(<issue>5</issue>):<fpage>929</fpage>&#x02013;<lpage>37</lpage>. Epub 2011/07/02. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1007/s10544-011-9562-2\">10.1007/s10544-011-9562-2</ext-link></comment> .<pub-id pub-id-type=\"pmid\">21720851</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref012\"><label>12</label><mixed-citation publication-type=\"journal\"><name><surname>Nie</surname><given-names>K</given-names></name>, <name><surname>Qi</surname><given-names>SX</given-names></name>, <name><surname>Zhang</surname><given-names>Y</given-names></name>, <name><surname>Luo</surname><given-names>L</given-names></name>, <name><surname>Xie</surname><given-names>Y</given-names></name>, <name><surname>Yang</surname><given-names>MJ</given-names></name>, <etal>et al</etal>\\n<article-title>Evaluation of a direct reverse transcription loop-mediated isothermal amplification method without RNA extraction for the detection of human enterovirus 71 subgenotype C4 in nasopharyngeal swab specimens</article-title>. <source>PLOS One</source>. <year>2012</year>;<volume>7</volume>(<issue>12</issue>):<fpage>e52486</fpage> Epub 2012/12/29. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1371/journal.pone.0052486\">10.1371/journal.pone.0052486</ext-link></comment> ; PubMed Central PMCID: PMC3525532.<pub-id pub-id-type=\"pmid\">23272248</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref013\"><label>13</label><mixed-citation publication-type=\"journal\"><name><surname>Wang</surname><given-names>X</given-names></name>, <name><surname>Zhang</surname><given-names>Q</given-names></name>, <name><surname>Zhang</surname><given-names>F</given-names></name>, <name><surname>Ma</surname><given-names>F</given-names></name>, <name><surname>Zheng</surname><given-names>W</given-names></name>, <name><surname>Zhao</surname><given-names>Z</given-names></name>, <etal>et al</etal>\\n<article-title>Visual detection of the human metapneumovirus using reverse transcription loop-mediated isothermal amplification with hydroxynaphthol blue dye</article-title>. <source>Virol J</source>. <year>2012</year>;<volume>9</volume>:<fpage>138</fpage> Epub 2012/07/31. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1186/1743-422x-9-138\">10.1186/1743-422x-9-138</ext-link></comment> ; PubMed Central PMCID: PMC3487928.<pub-id pub-id-type=\"pmid\">22838725</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref014\"><label>14</label><mixed-citation publication-type=\"journal\"><name><surname>Poon</surname><given-names>LL</given-names></name>, <name><surname>Wong</surname><given-names>BW</given-names></name>, <name><surname>Ma</surname><given-names>EH</given-names></name>, <name><surname>Chan</surname><given-names>KH</given-names></name>, <name><surname>Chow</surname><given-names>LM</given-names></name>, <name><surname>Abeyewickreme</surname><given-names>W</given-names></name>, <etal>et al</etal>\\n<article-title>Sensitive and inexpensive molecular test for falciparum malaria: detecting Plasmodium falciparum DNA directly from heat-treated blood by loop-mediated isothermal amplification</article-title>. <source>Clin Chem</source>. <year>2006</year>;<volume>52</volume>(<issue>2</issue>):<fpage>303</fpage>&#x02013;<lpage>6</lpage>. Epub 2005/12/13. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1373/clinchem.2005.057901\">10.1373/clinchem.2005.057901</ext-link></comment> .<pub-id pub-id-type=\"pmid\">16339303</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref015\"><label>15</label><mixed-citation publication-type=\"journal\"><name><surname>Curtis</surname><given-names>KA</given-names></name>, <name><surname>Rudolph</surname><given-names>DL</given-names></name>, <name><surname>Nejad</surname><given-names>I</given-names></name>, <name><surname>Singleton</surname><given-names>J</given-names></name>, <name><surname>Beddoe</surname><given-names>A</given-names></name>, <name><surname>Weigl</surname><given-names>B</given-names></name>, <etal>et al</etal>\\n<article-title>Isothermal amplification using a chemical heating device for point-of-care detection of HIV-1</article-title>. <source>PLOS One</source>. <year>2012</year>;<volume>7</volume>(<issue>2</issue>):<fpage>e31432</fpage> Epub 2012/03/03. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1371/journal.pone.0031432\">10.1371/journal.pone.0031432</ext-link></comment> ; PubMed Central PMCID: PMCPmc3285652.<pub-id pub-id-type=\"pmid\">22384022</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref016\"><label>16</label><mixed-citation publication-type=\"journal\"><name><surname>Curtis</surname><given-names>KA</given-names></name>, <name><surname>Rudolph</surname><given-names>DL</given-names></name>, <name><surname>Owen</surname><given-names>SM</given-names></name>. <article-title>Rapid detection of HIV-1 by reverse-transcription, loop-mediated isothermal amplification (RT-LAMP)</article-title>. <source>J Virol Meth</source>. <year>2008</year>;<volume>151</volume>(<issue>2</issue>):<fpage>264</fpage>&#x02013;<lpage>70</lpage>. Epub 2008/06/06. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1016/j.jviromet.2008.04.011\">10.1016/j.jviromet.2008.04.011</ext-link></comment> .<pub-id pub-id-type=\"pmid\">18524393</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref017\"><label>17</label><mixed-citation publication-type=\"journal\"><name><surname>Francois</surname><given-names>P</given-names></name>, <name><surname>Tangomo</surname><given-names>M</given-names></name>, <name><surname>Hibbs</surname><given-names>J</given-names></name>, <name><surname>Bonetti</surname><given-names>EJ</given-names></name>, <name><surname>Boehme</surname><given-names>CC</given-names></name>, <name><surname>Notomi</surname><given-names>T</given-names></name>, <etal>et al</etal>\\n<article-title>Robustness of a loop-mediated isothermal amplification reaction for diagnostic applications</article-title>. <source>FEMS Immunol Med Microbiol</source>. <year>2011</year>;<volume>62</volume>(<issue>1</issue>):<fpage>41</fpage>&#x02013;<lpage>8</lpage>. Epub 2011/02/01. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1111/j.1574-695X.2011.00785.x\">10.1111/j.1574-695X.2011.00785.x</ext-link></comment> .<pub-id pub-id-type=\"pmid\">21276085</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref018\"><label>18</label><mixed-citation publication-type=\"journal\"><name><surname>Parida</surname><given-names>M</given-names></name>, <name><surname>Posadas</surname><given-names>G</given-names></name>, <name><surname>Inoue</surname><given-names>S</given-names></name>, <name><surname>Hasebe</surname><given-names>F</given-names></name>, <name><surname>Morita</surname><given-names>K</given-names></name>. <article-title>Real-time reverse transcription loop-mediated isothermal amplification for rapid detection of West Nile virus</article-title>. <source>J Clin Microbiol</source>. 2005. <year>2004</year>;<volume>42</volume>(<issue>1</issue>):<fpage>257</fpage>&#x02013;<lpage>63</lpage>. Epub 2004/01/13. ; PubMed Central PMCID: PMC321710.<pub-id pub-id-type=\"pmid\">14715762</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref019\"><label>19</label><mixed-citation publication-type=\"journal\"><name><surname>Parida</surname><given-names>M</given-names></name>, <name><surname>Horioke</surname><given-names>K</given-names></name>, <name><surname>Ishida</surname><given-names>H</given-names></name>, <name><surname>Dash</surname><given-names>PK</given-names></name>, <name><surname>Saxena</surname><given-names>P</given-names></name>, <name><surname>Jana</surname><given-names>AM</given-names></name>, <etal>et al</etal>\\n<article-title>Rapid detection and differentiation of dengue virus serotypes by a real-time reverse transcription-loop-mediated isothermal amplification assay</article-title>. <source>J Clin Microbiol</source>. <year>2005</year>;<volume>43</volume>(<issue>6</issue>):<fpage>2895</fpage>&#x02013;<lpage>903</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jcm.43.6.2895&#x02013;2903.2005.\">10.1128/jcm.43.6.2895&#x02013;2903.2005.</ext-link></comment> .<pub-id pub-id-type=\"pmid\">15956414</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref020\"><label>20</label><mixed-citation publication-type=\"journal\"><name><surname>Teoh</surname><given-names>BT</given-names></name>, <name><surname>Sam</surname><given-names>SS</given-names></name>, <name><surname>Tan</surname><given-names>KK</given-names></name>, <name><surname>Johari</surname><given-names>J</given-names></name>, <name><surname>Danlami</surname><given-names>MB</given-names></name>, <name><surname>Hooi</surname><given-names>PS</given-names></name>, <etal>et al</etal>\\n<article-title>Detection of dengue viruses using reverse transcription-loop-mediated isothermal amplification</article-title>. <source>BMC Infect Dis</source>. <year>2013</year>;<volume>13</volume>:<fpage>387</fpage> Epub 2013/08/24. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1186/1471-2334-13-387\">10.1186/1471-2334-13-387</ext-link></comment> ; PubMed Central PMCID: PMC3846474.<pub-id pub-id-type=\"pmid\">23964963</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref021\"><label>21</label><mixed-citation publication-type=\"journal\"><name><surname>Liu</surname><given-names>H</given-names></name>, <name><surname>Liu</surname><given-names>ZJ</given-names></name>, <name><surname>Jing</surname><given-names>J</given-names></name>, <name><surname>Ren</surname><given-names>JQ</given-names></name>, <name><surname>Liu</surname><given-names>YY</given-names></name>, <name><surname>Guo</surname><given-names>HH</given-names></name>, <etal>et al</etal>\\n<article-title>Reverse transcription loop-mediated isothermal amplification for rapid detection of Japanese encephalitis virus in swine and mosquitoes</article-title>. <source>Vector Borne Zoonotic Dis</source>. <year>2012</year>;<volume>12</volume>(<issue>12</issue>):<fpage>1042</fpage>&#x02013;<lpage>52</lpage>. Epub 2012/11/28. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1089/vbz.2012.0991\">10.1089/vbz.2012.0991</ext-link></comment> ; PubMed Central PMCID: PMCPmc3525893.<pub-id pub-id-type=\"pmid\">23176446</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref022\"><label>22</label><mixed-citation publication-type=\"journal\"><name><surname>Parida</surname><given-names>MM</given-names></name>, <name><surname>Santhosh</surname><given-names>SR</given-names></name>, <name><surname>Dash</surname><given-names>PK</given-names></name>, <name><surname>Tripathi</surname><given-names>NK</given-names></name>, <name><surname>Saxena</surname><given-names>P</given-names></name>, <name><surname>Ambuj</surname><given-names>S</given-names></name>, <etal>et al</etal>\\n<article-title>Development and evaluation of reverse transcription-loop-mediated isothermal amplification assay for rapid and real-time detection of Japanese encephalitis virus</article-title>. <source>J Clin Microbiol</source>. <year>2006</year>;<volume>44</volume>(<issue>11</issue>):<fpage>4172</fpage>&#x02013;<lpage>8</lpage>. Epub 2006/09/29. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jcm.01487-06\">10.1128/jcm.01487-06</ext-link></comment> ; PubMed Central PMCID: PMCPmc1698363.<pub-id pub-id-type=\"pmid\">17005741</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref023\"><label>23</label><mixed-citation publication-type=\"journal\"><name><surname>Toriniwa</surname><given-names>H</given-names></name>, <name><surname>Komiya</surname><given-names>T</given-names></name>. <article-title>Rapid detection and quantification of Japanese encephalitis virus by real-time reverse transcription loop-mediated isothermal amplification</article-title>. <source>Microbiol Immunol</source>. <year>2006</year>;<volume>50</volume>(<issue>5</issue>):<fpage>379</fpage>&#x02013;<lpage>87</lpage>. Epub 2006/05/23. .<pub-id pub-id-type=\"pmid\">16714845</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref024\"><label>24</label><mixed-citation publication-type=\"journal\"><name><surname>Parida</surname><given-names>MM</given-names></name>, <name><surname>Santhosh</surname><given-names>SR</given-names></name>, <name><surname>Dash</surname><given-names>PK</given-names></name>, <name><surname>Tripathi</surname><given-names>NK</given-names></name>, <name><surname>Lakshmi</surname><given-names>V</given-names></name>, <name><surname>Mamidi</surname><given-names>N</given-names></name>, <etal>et al</etal>\\n<article-title>Rapid and real-time detection of chikungunya virus by reverse transcription loop-mediated isothermal amplification assay</article-title>. <source>J Clin Microbiol</source>. <year>2007</year>;<volume>45</volume>(<issue>2</issue>):<fpage>351</fpage>&#x02013;<lpage>7</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jcm.01734-06\">10.1128/jcm.01734-06</ext-link></comment> .<pub-id pub-id-type=\"pmid\">17135444</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref025\"><label>25</label><mixed-citation publication-type=\"journal\"><name><surname>Le Roux</surname><given-names>CA</given-names></name>, <name><surname>Kubo</surname><given-names>T</given-names></name>, <name><surname>Grobbelaar</surname><given-names>AA</given-names></name>, <name><surname>van Vuren</surname><given-names>PJ</given-names></name>, <name><surname>Weyer</surname><given-names>J</given-names></name>, <name><surname>Nel</surname><given-names>LH</given-names></name>, <etal>et al</etal>\\n<article-title>Development and evaluation of a real-time reverse transcription-loop-mediated isothermal amplification assay for rapid detection of Rift Valley fever virus in clinical specimens</article-title>. <source>J Clin Microbiol</source>. <year>2009</year>;<volume>47</volume>(<issue>3</issue>):<fpage>645</fpage>&#x02013;<lpage>51</lpage>. Epub 2008/12/26. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jcm.01412-08\">10.1128/jcm.01412-08</ext-link></comment> ; PubMed Central PMCID: PMCPmc2650915.<pub-id pub-id-type=\"pmid\">19109471</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref026\"><label>26</label><mixed-citation publication-type=\"journal\"><name><surname>Peyrefitte</surname><given-names>CN</given-names></name>, <name><surname>Boubis</surname><given-names>L</given-names></name>, <name><surname>Coudrier</surname><given-names>D</given-names></name>, <name><surname>Bouloy</surname><given-names>M</given-names></name>, <name><surname>Grandadam</surname><given-names>M</given-names></name>, <name><surname>Tolou</surname><given-names>HJ</given-names></name>, <etal>et al</etal>\\n<article-title>Real-time reverse-transcription loop-mediated isothermal amplification for rapid detection of Rift Valley fever virus</article-title>. <source>J Clin Microbiol</source>. <year>2008</year>;<volume>46</volume>(<issue>11</issue>):<fpage>3653</fpage>&#x02013;<lpage>9</lpage>. Epub 2008/09/19. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jcm.01188-08\">10.1128/jcm.01188-08</ext-link></comment> ; PubMed Central PMCID: PMCPmc2576582.<pub-id pub-id-type=\"pmid\">18799705</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref027\"><label>27</label><mixed-citation publication-type=\"journal\"><name><surname>Kwallah</surname><given-names>A</given-names></name>, <name><surname>Inoue</surname><given-names>S</given-names></name>, <name><surname>Muigai</surname><given-names>AW</given-names></name>, <name><surname>Kubo</surname><given-names>T</given-names></name>, <name><surname>Sang</surname><given-names>R</given-names></name>, <name><surname>Morita</surname><given-names>K</given-names></name>, <etal>et al</etal>\\n<article-title>A real-time reverse transcription loop-mediated isothermal amplification assay for the rapid detection of yellow fever virus</article-title>. <source>J Virol Meth</source>. <year>2013</year>;<volume>193</volume>(<issue>1</issue>):<fpage>23</fpage>&#x02013;<lpage>7</lpage>. Epub 2013/05/23. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1016/j.jviromet.2013.05.004\">10.1016/j.jviromet.2013.05.004</ext-link></comment> .<pub-id pub-id-type=\"pmid\">23692685</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref028\"><label>28</label><mixed-citation publication-type=\"journal\"><name><surname>Li</surname><given-names>S</given-names></name>, <name><surname>Fang</surname><given-names>M</given-names></name>, <name><surname>Zhou</surname><given-names>B</given-names></name>, <name><surname>Ni</surname><given-names>H</given-names></name>, <name><surname>Shen</surname><given-names>Q</given-names></name>, <name><surname>Zhang</surname><given-names>H</given-names></name>, <etal>et al</etal>\\n<article-title>Simultaneous detection and differentiation of dengue virus serotypes 1&#x02013;4, Japanese encephalitis virus, and West Nile virus by a combined reverse-transcription loop-mediated isothermal amplification assay</article-title>. <source>Virol J</source>. <year>2011</year>;<volume>8</volume>:<fpage>360</fpage> Epub 2011/07/23. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1186/1743-422x-8-360\">10.1186/1743-422x-8-360</ext-link></comment> ; PubMed Central PMCID: PMCPmc3149006.<pub-id pub-id-type=\"pmid\">21777455</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref029\"><label>29</label><mixed-citation publication-type=\"journal\"><name><surname>Kramer</surname><given-names>LD</given-names></name>, <name><surname>Chandler</surname><given-names>LJ</given-names></name>. <article-title>Phylogenetic analysis of the envelope gene of St. Louis encephalitis virus</article-title>. <source>Arch Virol</source>. <year>2001</year>;<volume>146</volume>(<issue>12</issue>):<fpage>2341</fpage>&#x02013;<lpage>55</lpage>. Epub 2002/01/29. .<pub-id pub-id-type=\"pmid\">11811684</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref030\"><label>30</label><mixed-citation publication-type=\"journal\"><name><surname>Kopp</surname><given-names>A</given-names></name>, <name><surname>Gillespie</surname><given-names>TR</given-names></name>, <name><surname>Hobelsberger</surname><given-names>D</given-names></name>, <name><surname>Estrada</surname><given-names>A</given-names></name>, <name><surname>Harper</surname><given-names>JM</given-names></name>, <name><surname>Miller</surname><given-names>RA</given-names></name>, <etal>et al</etal>\\n<article-title>Provenance and geographic spread of St. Louis encephalitis virus</article-title>. <source>mBio</source>. <year>2013</year>;<volume>4</volume>(<issue>3</issue>):<fpage>e00322</fpage>&#x02013;<lpage>13</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/mBio.00322-13\">10.1128/mBio.00322-13</ext-link></comment> .<pub-id pub-id-type=\"pmid\">23760463</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref031\"><label>31</label><mixed-citation publication-type=\"journal\"><name><surname>Sun</surname><given-names>B</given-names></name>, <name><surname>Shen</surname><given-names>F</given-names></name>, <name><surname>McCalla</surname><given-names>SE</given-names></name>, <name><surname>Kreutz</surname><given-names>JE</given-names></name>, <name><surname>Karymov</surname><given-names>MA</given-names></name>, <name><surname>Ismagilov</surname><given-names>RF</given-names></name>. <article-title>Mechanistic evaluation of the pros and cons of digital RT-LAMP for HIV-1 viral load quantification on a microfluidic device and improved efficiency via a two-step digital protocol</article-title>. <source>Anal Chem</source>. <year>2013</year>;<volume>85</volume>(<issue>3</issue>):<fpage>1540</fpage>&#x02013;<lpage>6</lpage>. Epub 2013/01/18. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1021/ac3037206\">10.1021/ac3037206</ext-link></comment> ; PubMed Central PMCID: PMCPmc3578705.<pub-id pub-id-type=\"pmid\">23324061</pub-id></mixed-citation></ref></ref-list></back></article>'}"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "x"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<pymongo.results.InsertOneResult at 0x111c873a8>"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "x['_id'] = \"test\"\n",
    "articles.delete_one({'_id': \"test\"})\n",
    "articles.insert_one(x)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'_id': 'test',\n",
       " 'nxml': '<!DOCTYPE article PUBLIC \"-//NLM//DTD JATS (Z39.96) Journal Archiving and Interchange DTD v1.0 20120330//EN\" \"JATS-archivearticle1.dtd\">\\n<article xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" article-type=\"research-article\"><?properties open_access?><front><journal-meta><journal-id journal-id-type=\"nlm-ta\">PLoS One</journal-id><journal-id journal-id-type=\"iso-abbrev\">PLoS ONE</journal-id><journal-id journal-id-type=\"publisher-id\">plos</journal-id><journal-id journal-id-type=\"pmc\">plosone</journal-id><journal-title-group><journal-title>PLoS ONE</journal-title></journal-title-group><issn pub-type=\"epub\">1932-6203</issn><publisher><publisher-name>Public Library of Science</publisher-name><publisher-loc>San Francisco, CA USA</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"pmid\">26807734</article-id><article-id pub-id-type=\"pmc\">4726549</article-id><article-id pub-id-type=\"doi\">10.1371/journal.pone.0147962</article-id><article-id pub-id-type=\"publisher-id\">PONE-D-15-43543</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Research Article</subject></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>RNA viruses</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>West Nile virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Microbiology</subject><subj-group><subject>Medical microbiology</subject><subj-group><subject>Microbial pathogens</subject><subj-group><subject>Viral pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>West Nile virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Medicine and health sciences</subject><subj-group><subject>Pathology and laboratory medicine</subject><subj-group><subject>Pathogens</subject><subj-group><subject>Microbial pathogens</subject><subj-group><subject>Viral pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>West Nile virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>Viral pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>West Nile virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>RNA viruses</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Western Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Microbiology</subject><subj-group><subject>Medical Microbiology</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Western Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Medicine and Health Sciences</subject><subj-group><subject>Pathology and Laboratory Medicine</subject><subj-group><subject>Pathogens</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Western Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Western Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Research and analysis methods</subject><subj-group><subject>Extraction techniques</subject><subj-group><subject>RNA extraction</subject></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>RNA viruses</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Eastern Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Microbiology</subject><subj-group><subject>Medical Microbiology</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Eastern Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Medicine and Health Sciences</subject><subj-group><subject>Pathology and Laboratory Medicine</subject><subj-group><subject>Pathogens</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Eastern Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Eastern Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>RNA viruses</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>Japanese Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Microbiology</subject><subj-group><subject>Medical Microbiology</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>Japanese Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Medicine and Health Sciences</subject><subj-group><subject>Pathology and Laboratory Medicine</subject><subj-group><subject>Pathogens</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>Japanese Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>Japanese Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Molecular Biology</subject><subj-group><subject>Molecular Biology Techniques</subject><subj-group><subject>Artificial Gene Amplification and Extension</subject><subj-group><subject>Polymerase Chain Reaction</subject></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Research and Analysis Methods</subject><subj-group><subject>Molecular Biology Techniques</subject><subj-group><subject>Artificial Gene Amplification and Extension</subject><subj-group><subject>Polymerase Chain Reaction</subject></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Molecular Biology</subject><subj-group><subject>Molecular Biology Techniques</subject><subj-group><subject>Sequencing Techniques</subject><subj-group><subject>Sequence Analysis</subject><subj-group><subject>Sequence Alignment</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Research and Analysis Methods</subject><subj-group><subject>Molecular Biology Techniques</subject><subj-group><subject>Sequencing Techniques</subject><subj-group><subject>Sequence Analysis</subject><subj-group><subject>Sequence Alignment</subject></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>RNA viruses</subject></subj-group></subj-group></subj-group></subj-group></article-categories><title-group><article-title>Surveillance for Western Equine Encephalitis, St. Louis Encephalitis, and West Nile Viruses Using Reverse Transcription Loop-Mediated Isothermal Amplification</article-title><alt-title alt-title-type=\"running-head\">RT-LAMP Surveillance for Western Equine Encephalitis, St. Louis Encephalitis, and West Nile Viruses</alt-title></title-group><contrib-group><contrib contrib-type=\"author\"><name><surname>Wheeler</surname><given-names>Sarah S.</given-names></name><xref ref-type=\"aff\" rid=\"aff001\"><sup>1</sup></xref><xref ref-type=\"author-notes\" rid=\"currentaff001\"><sup>&#x000a4;a</sup></xref></contrib><contrib contrib-type=\"author\"><name><surname>Ball</surname><given-names>Cameron S.</given-names></name><xref ref-type=\"aff\" rid=\"aff002\"><sup>2</sup></xref></contrib><contrib contrib-type=\"author\"><name><surname>Langevin</surname><given-names>Stanley A.</given-names></name><xref ref-type=\"aff\" rid=\"aff002\"><sup>2</sup></xref><xref ref-type=\"author-notes\" rid=\"currentaff002\"><sup>&#x000a4;b</sup></xref></contrib><contrib contrib-type=\"author\"><name><surname>Fang</surname><given-names>Ying</given-names></name><xref ref-type=\"aff\" rid=\"aff001\"><sup>1</sup></xref></contrib><contrib contrib-type=\"author\"><name><surname>Coffey</surname><given-names>Lark L.</given-names></name><xref ref-type=\"aff\" rid=\"aff001\"><sup>1</sup></xref></contrib><contrib contrib-type=\"author\"><name><surname>Meagher</surname><given-names>Robert J.</given-names></name><xref ref-type=\"aff\" rid=\"aff002\"><sup>2</sup></xref><xref ref-type=\"corresp\" rid=\"cor001\">*</xref></contrib></contrib-group><aff id=\"aff001\"><label>1</label><addr-line>University of California Davis, School of Veterinary Medicine, Department of Pathology, Microbiology and Immunology, Davis, California, United States of America</addr-line></aff><aff id=\"aff002\"><label>2</label><addr-line>Sandia National Laboratories, Biotechnology and Bioengineering Department, Livermore, California, United States of America</addr-line></aff><contrib-group><contrib contrib-type=\"editor\"><name><surname>Vasilakis</surname><given-names>Nikos</given-names></name><role>Editor</role><xref ref-type=\"aff\" rid=\"edit1\"/></contrib></contrib-group><aff id=\"edit1\"><addr-line>University of Texas Medical Branch, UNITED STATES</addr-line></aff><author-notes><fn fn-type=\"conflict\" id=\"coi001\"><p><bold>Competing Interests: </bold>The authors have read the journal\\'s policy and have the following competing interest: Authors CSB, RJM, and SAL are current or former employees of Sandia National Laboratories. The specific roles of these authors are articulated in the Author Contributions section. Sandia National Laboratories did not have any additional role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript. The authors declare no competing interests related to their employment with Sandia National Laboratories. This did not alter the authors\\' adherence to PLOS ONE policies on sharing data and materials.</p></fn><fn fn-type=\"con\" id=\"contrib001\"><p>Conceived and designed the experiments: SSW CSB SAL LLC RJM. Performed the experiments: SSW CSB YF LLC RJM. Analyzed the data: SSW CSB LLC RJM. Contributed reagents/materials/analysis tools: SSW YF LLC RJM. Wrote the paper: SSW CSB SAL LLC RJM.</p></fn><fn fn-type=\"current-aff\" id=\"currentaff001\"><label>&#x000a4;a</label><p>Current address: Sacramento-Yolo Mosquito and Vector Control District, Elk Grove, California, United States of America</p></fn><fn fn-type=\"current-aff\" id=\"currentaff002\"><label>&#x000a4;b</label><p>Current address: University of Washington, Department of Microbiology, Seattle, Washington, United States of America</p></fn><corresp id=\"cor001\">* E-mail: <email>rmeaghe@sandia.gov</email></corresp></author-notes><pub-date pub-type=\"epub\"><day>25</day><month>1</month><year>2016</year></pub-date><pub-date pub-type=\"collection\"><year>2016</year></pub-date><volume>11</volume><issue>1</issue><elocation-id>e0147962</elocation-id><history><date date-type=\"received\"><day>2</day><month>10</month><year>2015</year></date><date date-type=\"accepted\"><day>11</day><month>1</month><year>2016</year></date></history><permissions><copyright-statement>&#x000a9; 2016 Wheeler et al</copyright-statement><copyright-year>2016</copyright-year><copyright-holder>Wheeler et al</copyright-holder><license xlink:href=\"http://creativecommons.org/licenses/by/4.0/\"><license-p>This is an open access article distributed under the terms of the <ext-link ext-link-type=\"uri\" xlink:href=\"http://creativecommons.org/licenses/by/4.0/\">Creative Commons Attribution License</ext-link>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</license-p></license></permissions><self-uri content-type=\"pdf\" xlink:type=\"simple\" xlink:href=\"pone.0147962.pdf\"/><abstract><p>Collection of mosquitoes and testing for vector-borne viruses is a key surveillance activity that directly influences the vector control efforts of public health agencies, including determining when and where to apply insecticides. Vector control districts in California routinely monitor for three human pathogenic viruses including West Nile virus (WNV), Western equine encephalitis virus (WEEV), and St. Louis encephalitis virus (SLEV). Reverse transcription quantitative polymerase chain reaction (RT-qPCR) offers highly sensitive and specific detection of these three viruses in a single multiplex reaction, but this technique requires costly, specialized equipment that is generally only available in centralized public health laboratories. We report the use of reverse transcription loop-mediated isothermal amplification (RT-LAMP) to detect WNV, WEEV, and SLEV RNA extracted from pooled mosquito samples collected in California, including novel primer sets for specific detection of WEEV and SLEV, targeting the nonstructural protein 4 (nsP4) gene of WEEV and the 3&#x02019; untranslated region (3&#x02019;-UTR) of SLEV. Our WEEV and SLEV RT-LAMP primers allowed detection of &#x0003c;0.1 PFU/reaction of their respective targets in &#x0003c;30 minutes, and exhibited high specificity without cross reactivity when tested against a panel of alphaviruses and flaviviruses. Furthermore, the SLEV primers do not cross-react with WNV, despite both viruses being closely related members of the Japanese encephalitis virus complex. The SLEV and WEEV primers can also be combined in a single RT-LAMP reaction, with discrimination between amplicons by melt curve analysis. Although RT-qPCR is approximately one order of magnitude more sensitive than RT-LAMP for all three targets, the RT-LAMP technique is less instrumentally intensive than RT-qPCR and provides a more cost-effective method of vector-borne virus surveillance.</p></abstract><funding-group><funding-statement>This work was supported by a grant from the Defense Threat Reduction Agency (DTRA), contract DTRA1002710697. Sandia National Laboratories is a multi-program laboratory managed and operated by Sandia Corporation, a wholly owned subsidiary of Lockheed Martin Corporation, for the U.S. Department of Energy&#x02019;s National Nuclear Security Administration under Contract DE-AC04-94AL85000.</funding-statement></funding-group><counts><fig-count count=\"5\"/><table-count count=\"3\"/><page-count count=\"17\"/></counts><custom-meta-group><custom-meta id=\"data-availability\"><meta-name>Data Availability</meta-name><meta-value>All relevant data are within the paper and its Supporting Information files.</meta-value></custom-meta></custom-meta-group></article-meta><notes><title>Data Availability</title><p>All relevant data are within the paper and its Supporting Information files.</p></notes></front><body><sec sec-type=\"intro\" id=\"sec001\"><title>Introduction</title><p>Testing mosquito vectors for human pathogenic viruses like West Nile virus is a central feature of successful surveillance approaches and an early predictor of human epidemics. Most surveillance in the United States and increasingly in other regions uses molecular detection of viral RNAs, usually by reverse transcription polymerase chain reaction (RT-PCR) or quantitative real-time RT-PCR (RT-qPCR). The presence of vector-borne viruses in mosquitoes, together with other measures of activity including mosquito distribution and abundance, human cases, dead bird surveillance, meteorological parameters and (for some viruses) sentinel chicken seroconversions, are used to make vector control decisions including when and where to apply insecticides. Three medically important vector-borne viruses, West Nile virus (WNV), Western equine encephalitis virus (WEEV), and St. Louis encephalitis virus (SLEV) are endemic in California and cause sporadic seasonal outbreaks, primarily during the summer months.</p><p>WEEV incidence has declined in the last few decades [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>] and has not been detected in California since 2003. SLEV was detected in field-caught mosquitoes in California in 2015, for the first time since the arrival of WNV in 2003 [<xref rid=\"pone.0147962.ref002\" ref-type=\"bibr\">2</xref>, <xref rid=\"pone.0147962.ref003\" ref-type=\"bibr\">3</xref>]. SLEV also still occurs in Mexico and South America [<xref rid=\"pone.0147962.ref004\" ref-type=\"bibr\">4</xref>]. Despite the much higher prevalence of WNV in California since 2003, SLEV and WEEV are still included in routine mosquito surveillance since vector-borne viruses can exist in cryptic and sporadic transmission cycles without the detection of human or equine disease, as exemplified by the re-emergence of SLEV in 2015.</p><p>Vector-borne disease surveillance programs primarily rely on techniques such as virus isolation, standard RT-PCR, and RT-qPCR to detect the presence of vector-borne viruses in targeted mosquito vectors. While these methodologies are the gold standard for arbovirus disease surveillance, they require well-equipped laboratories and well-trained technicians, both of which are severely lacking in many low resource settings with high vector-borne disease burden. RT-qPCR is the most sensitive assay for detecting vector-borne viruses, but the technique generally requires expensive equipment, and high quality RNA purified from the mosquito samples. Loop-mediated isothermal amplification (LAMP) is an isothermal nucleic acid amplification technique that is a useful alternative to PCR for pathogen detection and diagnostics [<xref rid=\"pone.0147962.ref005\" ref-type=\"bibr\">5</xref>&#x02013;<xref rid=\"pone.0147962.ref008\" ref-type=\"bibr\">8</xref>]. When coupled with reverse transcription, the LAMP methodology can also be utilized for detecting RNA targets [<xref rid=\"pone.0147962.ref009\" ref-type=\"bibr\">9</xref>]. LAMP requires 4 or 6 primers that recognize 6 or 8 binding sites, providing a highly specific assay. A key advantage to the LAMP reaction is simplicity: the entire reaction (including reverse transcription) proceeds at a single temperature, eliminating the need for a thermal cycler. Reactions can be monitored through a variety of outputs including generation of turbidity [<xref rid=\"pone.0147962.ref010\" ref-type=\"bibr\">10</xref>], fluorescence indicators [<xref rid=\"pone.0147962.ref011\" ref-type=\"bibr\">11</xref>, <xref rid=\"pone.0147962.ref012\" ref-type=\"bibr\">12</xref>], or color change [<xref rid=\"pone.0147962.ref013\" ref-type=\"bibr\">13</xref>]. The technique is also robust for crude specimens, allowing successful amplification from minimally processed samples such as heat-treated blood [<xref rid=\"pone.0147962.ref012\" ref-type=\"bibr\">12</xref>, <xref rid=\"pone.0147962.ref014\" ref-type=\"bibr\">14</xref>&#x02013;<xref rid=\"pone.0147962.ref016\" ref-type=\"bibr\">16</xref>, <xref rid=\"pone.0147962.ref017\" ref-type=\"bibr\">17</xref>]. As such, LAMP can often be performed without DNA or RNA extraction. These advantages lessen the dependence upon highly skilled labor and well-equipped laboratories, which are prerequisites for RT-qPCR.</p><p>In this report, we describe use of RT-LAMP to detect WNV, WEEV, and SLEV, and apply the technique to RNA extracted from <italic>Culex</italic> mosquitoes collected in California. RT-LAMP was previously demonstrated for detection of WNV by Parida <italic>et al</italic> [<xref rid=\"pone.0147962.ref018\" ref-type=\"bibr\">18</xref>], and detection kits for this virus based on RT-LAMP are commercially available from several vendors. RT-LAMP primer sets have also been reported for numerous other important mosquito-borne viruses such as Dengue virus (DENV) [<xref rid=\"pone.0147962.ref019\" ref-type=\"bibr\">19</xref>, <xref rid=\"pone.0147962.ref020\" ref-type=\"bibr\">20</xref>], Japanese encephalitis virus (JEV) [<xref rid=\"pone.0147962.ref021\" ref-type=\"bibr\">21</xref>&#x02013;<xref rid=\"pone.0147962.ref023\" ref-type=\"bibr\">23</xref>], chikungunya virus (CHIKV) [<xref rid=\"pone.0147962.ref024\" ref-type=\"bibr\">24</xref>], Rift Valley fever virus (RVFV) [<xref rid=\"pone.0147962.ref025\" ref-type=\"bibr\">25</xref>, <xref rid=\"pone.0147962.ref026\" ref-type=\"bibr\">26</xref>], and yellow fever virus (YFV) [<xref rid=\"pone.0147962.ref027\" ref-type=\"bibr\">27</xref>]. Because RT-LAMP detection of SLEV and WEEV was not previously reported, RT-LAMP primer sets for these viruses were developed and tested for sensitivity and specificity. At the time of method development neither WEEV nor SLEV had been recently detected in California mosquito pools, and thus archived samples were used for RT-LAMP evaluation. WNV-positive mosquito pools collected in 2014 were used to verify the lack of cross-reactivity with new SLEV and WEEV primers. These WNV positive pools were also used to demonstrate the applicability of RT-LAMP for parallel detection of WNV, WEEV and SLEV, analogous to the RT-LAMP &#x0201c;detection array&#x0201d; previously reported for parallel detection of DENV serotypes 1&#x02013;4, JEV, and WNV [<xref rid=\"pone.0147962.ref028\" ref-type=\"bibr\">28</xref>]. In addition, we demonstrate duplex detection of WEEV and SLEV in a single tube using melt curve analysis.</p></sec><sec sec-type=\"materials|methods\" id=\"sec002\"><title>Materials and Methods</title><sec id=\"sec003\"><title>RT-LAMP primer design</title><p>Whole-genome sequence alignments for WEEV and SLEV were performed using ClustalX2. For WEEV, six North American strains were selected for which full-length or nearly full-length sequences were available in Genbank as of April, 2014 (a large number of additional WEEV genomes were deposited in Genbank after primers were designed [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>]). For SLEV, 6 strains were selected for which full-length or nearly full-length sequences were available, representing primarily North American strains from Clade II in the tree developed by Kramer and Chandler [<xref rid=\"pone.0147962.ref029\" ref-type=\"bibr\">29</xref>]. Accession numbers of WEEV and SLEV strains used for primer design are presented in Figs <xref ref-type=\"fig\" rid=\"pone.0147962.g001\">1</xref> and <xref ref-type=\"fig\" rid=\"pone.0147962.g002\">2</xref>.</p><fig id=\"pone.0147962.g001\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.g001</object-id><label>Fig 1</label><caption><title>Sequence alignment showing priming regions for the WEEV nsP4 primer set.</title><p>The arrow indicates the location of the <italic>Xba</italic>I restriction site. Genbank accession numbers refer to the following strains: GQ287644.1 = BFS2005; GQ287643.1 = Montana-64; GQ287645.1 = 71V1658; GQ287642.1 = Kern; GQ287641.1 = Imperial; GQ287647.1 = 85&#x02013;452NM.</p></caption><graphic xlink:href=\"pone.0147962.g001\"/></fig><fig id=\"pone.0147962.g002\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.g002</object-id><label>Fig 2</label><caption><title>Sequence alignment showing priming regions for the SLEV 3&#x02019; UTR primer set.</title><p>The arrow indicates the location of the <italic>Hin</italic>dIII restriction site.Genbank accession numbers refer to the following strains: DQ525916.1 = Kern 217; EU566860.1 = Hubbard; FJ753286.2 = CbaAr-4005; JF460774.1 = Imperial Valley; JQ957868.1 = Palenque-C475; JQ957869.1 = Palenque-A770.</p></caption><graphic xlink:href=\"pone.0147962.g002\"/></fig><p>Conserved regions of the viral genomes from the sequence alignments were identified. LAMP Designer v1.13 software (Premier Biosoft) was used to scan conserved regions for suitable LAMP primer sets using a single representative strain. Primer set candidates were then screened against the full set of aligned sequences to test for perfect or nearly perfect matches. Primer sets meeting this criterion were then analyzed by BLAST, first comparing to all WEEV or SLEV sequences (taxid 11039 and 11080, respectively) and then comparing to all viral sequences in Genbank, to determine likelihood of cross-reactivity with other viruses. Primer sets were also evaluated for hairpin formation, self-dimerization, and cross-dimerization.</p></sec><sec id=\"sec004\"><title>Viral culture</title><p>Selected viral strains used for sensitivity and specificity testing are listed in <xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>. These strains were pulled from a library of isolates archived at -80&#x000b0;C. From each isolate RNA was extracted and plaque assays were performed to obtain viral titers. RNA was extracted using a MagMax&#x02122; magnetic particle processor (Life Technologies, Grand Island, NY, USA), MagMAX&#x02122;-96 Viral RNA Isolation Kit (Life Technologies, Grand Island, NY, USA), and manufacturer provided protocols. Plaque assays were performed using Vero cell cultures grown to confluence in six well plates, cultured with Dulbecco&#x02019;s Modified Eagle Medium (DMEM) supplemented with 10% fetal bovine serum (FBS) and 500 U/mL penicillin and 0.5 mg/mL streptomycin, and maintained at 37&#x000b0;C and 5% CO<sub>2</sub>. Virus isolates were diluted in a 10-fold serial dilution, media was removed from each well of the 6-well plate and 200 uL of diluted virus was allowed to incubate on the monolayer for 60 min at 37&#x000b0;C. A double overlay system was used where the first overlay contained nutritive media and 5% agarose. The second overlay was applied and additionally contained 0.005% neutral red, the optimal time to count plaques occurred 1&#x02013;3 days after the second overlay.</p><table-wrap id=\"pone.0147962.t001\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.t001</object-id><label>Table 1</label><caption><title>Viral RNA used for testing sensitivity and specificity of RT-LAMP primers.</title></caption><alternatives><graphic id=\"pone.0147962.t001g\" xlink:href=\"pone.0147962.t001\"/><table frame=\"hsides\" rules=\"groups\"><colgroup span=\"1\"><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/></colgroup><thead><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\"/><th align=\"left\" rowspan=\"1\" colspan=\"1\"/><th align=\"center\" colspan=\"3\" rowspan=\"1\">RT-LAMP primer set (100 PFU target)</th></tr><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\">Virus</th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Strain or Isolate Designation</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019;-UTR</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WNV E gene</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WEEV nsP4</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>Flaviviruses</italic></td><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;St. Louis encephalitis virus (SLEV)<xref ref-type=\"table-fn\" rid=\"t001fn001\"><sup>1</sup></xref></td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Kern217</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">Bfs1750 (Ct 17.7)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">Ruls (Ct 12.9)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">Coav750 (Ct 23.9)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">69M1143 (Ct 13.7)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">BeAn246407 (Ct 20.4)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">CorAn9124 (PCR neg.)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">CorAn9275 (PCR neg.)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;<xref ref-type=\"table-fn\" rid=\"t001fn002\"><sup>2</sup></xref></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;West Nile virus (WNV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">L-CA-04 SAC-04-7168</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Yellow fever virus (YFV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">17D</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Rocio virus (ROCV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">SP H 34675</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Usutu virus (USV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">SA AR 1776</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Ilheus virus (ILHV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Ilheus B44532</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Dengue virus serotype 1 (DENV-1)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">BC-796</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Dengue virus serotype 2 (DENV-2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">BC-122-94</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Dengue virus serotype 3 (DENV-3)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">BC 156&#x02013;97</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>Alphaviruses</italic></td><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Western equine encephalitis virus (WEEV)<xref ref-type=\"table-fn\" rid=\"t001fn001\"><sup>1</sup></xref></td><td align=\"left\" rowspan=\"1\" colspan=\"1\">KERN 5547</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">Lake43 (Ct 14.9)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Sindbis virus (SINV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">EDS-14</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Ross River virus (RRV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">SW 38457</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Chikungunya virus (CHIKV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Ross</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Venezuelan equine encephalitis virus (VEEV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">TC-83</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;<xref ref-type=\"table-fn\" rid=\"t001fn002\"><sup>2</sup></xref></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Barmah Forest virus (BFV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Barmah Forest (TVP-4119)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Highlands J virus (HJV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">WC-431</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Eastern equine encephalitis virus (EEEV)<xref ref-type=\"table-fn\" rid=\"t001fn003\"><sup>3</sup></xref></td><td align=\"left\" rowspan=\"1\" colspan=\"1\">EEEV/X/USA/A15072/2003</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr></tbody></table></alternatives><table-wrap-foot><fn id=\"t001fn001\"><p><sup>1</sup> Isolatesof SLEV other than Kern 217, and WEEV isolate Lake43 were not quantitated by plaque assay. Extracted RNA was quantitated by RT-qPCR and used undiluted as a template in RT-LAMP. RT-qPCR Ct values are indicated. CorAn 9124 and CorAn9275 were not detected by RT-qPCR primers.</p></fn><fn id=\"t001fn002\"><p><sup>2</sup> A single reaction showed positive amplification within 45 minutes of incubation (1 of 11 replicates for SLEV CorAn 9275 with WNV primers; 1 of 18 replicates for VEEV TC-83 with SLEV primers). Melt curve analysis indicates that these are isolated occurrences of cross-contamination with positive-control RNA.</p></fn><fn id=\"t001fn003\"><p><sup>3</sup> Cross-reactivity testing for EEEV was performed with 10<sup>6</sup> copies of an <italic>in vitro</italic> transcribed fragment of the EEEV nsP4 gene corresponding to the WEEV RT-LAMP target site including an additional 46 nt upstream and 65 nt downstream of the outer primer (F3 and B3) binding sites. The <italic>in vitro</italic> transcribed RNA represents nucleotides 5980&#x02013;6329 (350 bases) of Genbank # KJ469613.1.</p></fn></table-wrap-foot></table-wrap><p>Due to select agent restrictions for work with Eastern equine encephalitis virus (EEEV), a segment of EEEV nsP4 RNA was purchased. The RNA was <italic>in vitro</italic> transcribed from a synthetic gene corresponding to nucleotides 5980&#x02013;6329 (350 bases) of Genbank accession number KJ469613.1 (Biosynthesis, Lewisville, TX, USA). The transcript was purified to remove traces of DNA, and verified DNA-free by PCR (performed by the supplier).</p></sec><sec id=\"sec005\"><title>Mosquito pools</title><p>Mosquito pools were collected as part of routine mosquito surveillance performed by California vector control agencies. RNA from selected pools was obtained through the UC Davis Center for Vectorborne Disease Surveillance Labortory (CVEC) with the permission of the collecting agency. Pools comprised up to 50 conspecific, female, field-caught mosquitoes. RNA was extracted by MagMax<sup>TM</sup> as described above.</p></sec><sec id=\"sec006\"><title>RT-qPCR</title><p>Detection of viral RNA in mosquito pools was performed at CVEC using a multiplex RT-qPCR for WNV, SLEV, and WEEV. This assay was performed using the primers and probes listed in <xref ref-type=\"supplementary-material\" rid=\"pone.0147962.s003\">S1 Table</xref>, except the probe for SLEV had a Quasar 670 reporter and Black Hole 2 Quencher (BHQ-2) and the probe for WEEV had a TAMRA reporter and BHQ-2. Reactions were completed on a ViiA 7 Real-Time PCR system (Life Technologies, Grand Island, NY, USA) using SensiFAST Probe Lo-ROX One-step Kits (Bioline, Taunton, MA, USA) and manufacturer recommended protocols. Additional RT-qPCR using the WNV, SLEV, and WEEV primer and probe sets (<xref ref-type=\"supplementary-material\" rid=\"pone.0147962.s003\">S1 Table</xref>) were performed on the RNA from selected virus isolates (<xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>) as part of the sensitivity and specificity testing at Sandia. These reactions were performed using a one-step RT-PCR kit (iTaq Universal Probes One-Step Kit, BioRad, Hercules, CA) and BioRad CFX96 instrument, with detection in the FAM channel. Total reaction volume was 10 &#x003bc;L and contained 500nM of each forward and reverse primer and 200nM of the probe. PCR was performed with an initial reverse transcription step at 50&#x000b0;C for 10 minutes, followed by an initial denaturation for 50 seconds, then 40 cycles of 95&#x000b0;C for 15 seconds and 60&#x000b0;C for 30 seconds followed by a plate read.</p></sec><sec id=\"sec007\"><title>RT-LAMP</title><p>RT-LAMP was performed in 10 &#x003bc;L reaction volumes in thin-walled PCR strip tubes or 96-well plates. The reaction mixture had a final composition (after adding water or template) of 1X Isothermal Amplification Buffer (New England Biolabs, NEB #B0537S) supplemented with an additional 6 mM MgSO4 (NEB #B1003S, final 8 mM MgSO<sub>4</sub>), 0.8 M Betaine (Sigma #B-0300), 1.4 mM each dNTP (NEB #N0447L), 0.32 units/&#x003bc;L Bst 2.0 WarmStart DNA polymerase (NEB #M0538M), 0.2 units/&#x003bc;L AMV reverse transcriptase (NEB #M0277T, or Life Science Advanced Technologies #AMVRTT-5), and 2 &#x003bc;M (or in some instances 4 &#x003bc;M) SYTO 82 detection dye (Life Technologies #S11363) [<xref rid=\"pone.0147962.ref011\" ref-type=\"bibr\">11</xref>]. Primers were used in the amounts typically recommended for LAMP: 0.2 &#x003bc;M each for outer primers F3 and B3; 1.6 &#x003bc;M each for inner primers FIP and BIP; and 0.8 &#x003bc;M each for loop primers LF and LB. RT-LAMP with real-time fluorescence monitoring was carried out in a BioRad CFX96, using detection channel 2 (HEX) for monitoring SYTO 82 dye. Reactions were incubated at a constant temperature of 63&#x000b0;C for 50&#x02013;70 minutes, with plate read steps at intervals of 1 minute (in the BioRad CFX96, this is accomplished with a 48-second single-temperature cycle followed by a plate read which takes approximately 12 seconds in all-channel mode). Incubation was typically followed by inactivation of the enzyme at 95&#x000b0;C for 2 minutes, followed by cooling to 65&#x000b0;C for 2 minutes, and a melt curve from 65&#x02013;95&#x000b0;C in 0.2&#x000b0;C increments. Time-to-positivity values were determined using the BioRad CFX Manager software, using baseline-subtracted curves, and a single threshold value auto-calculated by the CFX manager.</p><p>To demonstrate testing with simpler instrumentation, we also performed reactions using a portable isothermal fluorimeter (Optigene Genie III). The protocol was identical to that described above, except that SYTO 9 (Life Technologies #S34854) was used in place of SYTO 82, to ensure compatibility with the Genie III detection optics. A fluorescence photograph of the reaction endpoint (with SYTO 9) was taken using a blue LED flashlight (CREE WF-502B) and an amber-colored plastic filter (LEE Filters #158). Background fluorescence with SYTO 9 was minimized by photographing the tubes upon a hot plate set to 63&#x000b0;C.</p><p>Gel electrophoretic analysis of RT-LAMP products was performed using an Invitrogen e-Gel (2% agarose with ethidium bromide); 1 &#x003bc;L of RT-LAMP product was diluted with DI water to a total volume of 20 &#x003bc;L that was loaded onto the e-Gel for nucleic acid separation. In addition, to confirm specificity of amplification, 3 &#x003bc;L aliquots of products from positive control reactions were digested with 20 units of <italic>Xba</italic>I (for WEEV nsP4 RT-LAMP), or <italic>Hind</italic>III (for SLEV 3&#x02019;-UTR RT-LAMP) for 1 hour at 37&#x000b0;C in 20 &#x003bc;L total reaction volume, using buffer supplied by the manufacturer (New England Biolabs). Digested products were also run on an e-Gel as described above.</p></sec><sec id=\"sec008\"><title>Sensitivity and Specificity Testing</title><p>SLEV, WEEV and WNV LAMP primers were tested for sensitivity and cross-reactivity against multiple conspecific strains and closely related viruses (<xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>). Sensitivity testing was performed with a dilution series of a single reference isolate of each virus: WEEV Kern 5547 (Genbank KJ554975.1), SLEV Kern 217 (Genbank DQ525916.1), and WNV L-CA-04 SAC-04-7168 (Genbank DQ080059.1). Conspecific strains for SLEV were chosen amongst the lineages described by Kramer et al. [<xref rid=\"pone.0147962.ref027\" ref-type=\"bibr\">27</xref>]. We tested isolates falling within Clades IA (Bfs1750), IB (Coav750), IIA(69M143), VA (BeAn246407), and VII (CorAn9124 and CorAn9275). One conspecific strain of WEEV (Lake 43, Genbank KJ554985) was also tested. All other WEEV isolates within the collection at UC Davis were identical to the Kern 5547 reference isolate within the primer binding regions. A panel of flaviviruses and alphaviruses (<xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>) was developed to test for cross-reactivity. Each primer set (SLEV, WEEV, and WNV) was tested against the entire panel, using 100 PFU equivalents of viral RNA template per reaction. Because of the select agent status of EEEV, primers were tested against an in vitro transcribed portion of nsP4 gene instead of genomic RNA (methods described above), with 10<sup>6</sup> copies of RNA template per reaction.</p></sec><sec id=\"sec009\"><title>Multiplexed RT-LAMP with melt curve analysis</title><p>WNV, SLEV 3&#x02019;-UTR, and WEEV-nsP4 primer sets were tested with mixtures of WNV L-CA-04 SAC-04-7168, SLEV Kern 217, and WEEV Kern 5547 RNA templates. Individual primer sets were combined in equal amounts to create four multiplexed reaction types: WNV + SLEV + WEEV, WNV + SLEV, WNV + WEEV, and SLEV + WEEV. Corresponding viral RNA templates were combined in equal parts and added to matching reaction mixes. Positive control reactions contained a total of 100 PFU equivalents of viral RNA template per reaction. Multiplexed reactions were performed in triplicate with the RT-LAMP protocol described above.</p></sec></sec><sec id=\"sec010\"><title>Results and Discussion</title><sec id=\"sec011\"><title>WEEV and SLEV primer design</title><p>Our primer design protocol identified two candidate primer sets for WEEV (targeting nsP1 and nsP4), and two candidate primer sets for SLEV (targeting the 3&#x02019;-UTR and capsid C protein). According to the sequence alignment the 3&#x02019;-UTR is well conserved among sequenced isolates of SLEV, and yet sufficiently distinct from other flaviviruses to allow specific detection of SLEV by RT-LAMP. The flavivirus 3&#x02019;-UTR was previously targeted by LAMP primer sets designed for dengue serotypes 1&#x02013;4 [<xref rid=\"pone.0147962.ref019\" ref-type=\"bibr\">19</xref>, <xref rid=\"pone.0147962.ref020\" ref-type=\"bibr\">20</xref>]. Similarly, the nsP4 (RNA polymerase) gene of WEEV was among the most highly conserved region of that genome. The region targeted here shared 81% sequence identity with North American isolates of Eastern equine encephalitis virus (EEEV), but was &#x0003c;80% identical to any other alphaviruses.</p><p>In initial screening the WEEV-nsP4 and SLEV 3&#x02019;-UTR sets provided faster amplification and lower detection limits than the WEEV-nsP1 and SLEV-C sets; therefore, only the nsP4 and 3&#x02019;-UTR sets were evaluated further. The WEEV nsP4 and SLEV 3&#x02019;-UTR primer sequences are shown in <xref ref-type=\"table\" rid=\"pone.0147962.t002\">Table 2</xref>, and sequence alignments showing the positioning of the primers are presented in Figs <xref ref-type=\"fig\" rid=\"pone.0147962.g001\">1</xref> and <xref ref-type=\"fig\" rid=\"pone.0147962.g002\">2</xref>. The less efficient WEEV nsP1 and SLEV C primer sequences are provided in <xref ref-type=\"supplementary-material\" rid=\"pone.0147962.s004\">S2 Table</xref></p><table-wrap id=\"pone.0147962.t002\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.t002</object-id><label>Table 2</label><caption><title>RT-LAMP Primers used herein.</title></caption><alternatives><graphic id=\"pone.0147962.t002g\" xlink:href=\"pone.0147962.t002\"/><table frame=\"hsides\" rules=\"groups\"><colgroup span=\"1\"><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/></colgroup><thead><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\">Primer Name</th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Genome position<xref ref-type=\"table-fn\" rid=\"t002fn001\"><sup>1</sup></xref></th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Sequence<xref ref-type=\"table-fn\" rid=\"t002fn002\"><sup>2</sup></xref></th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Source</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; F3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10497&#x02013;10516</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GACTGGGTTAWCAAAGCCAA</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">New</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; B3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10736&#x02013;10718</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GGTCTCCTCTAACCTCTAG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; FIP(<underline>F1c</underline> + F2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10641&#x02013;10622;10583&#x02013;10600</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>GCATGTACAGACAGCACCTT</underline> +GCACAAGTCAGACCAGAA</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; BIP<xref ref-type=\"table-fn\" rid=\"t002fn003\"><sup>3</sup></xref>(<underline>B1c</underline> + B2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10644&#x02013;10665;10717&#x02013;10700</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>CAGGAGGACTGGGTTAACAAAG</underline> +TCCTTACGCCATGGTCAC</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; LF</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10618&#x02013;10601</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">CATGCTTTCAGGTGGCAT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; LB</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10681&#x02013;10688</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GGCCCAAACCATGGAGTG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 F3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">5949&#x02013;5958</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GTCAACGGTAATTAACAGGT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">New</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 B3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">6187&#x02013;6170</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GGACGGCTGATCTTATCT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 FIP(<underline>F1c</underline> + F2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">6087&#x02013;6070;6025&#x02013;6044</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>GCCGTCCACCATGTCAAG</underline> +CCTACAGTAGCCAGTTATTG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 BIP(<underline>B1c</underline> + B2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">6088&#x02013;6108;6156&#x02013;6138</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>GCATCGTGCTGTCTAGATACM</underline> +GCTATGCTTCTTTGGGTAG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 LF</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">6067&#x02013;6048</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">ACGCATCGTATTCATCTGTT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 LB</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">6116&#x02013;6134</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">TTTGTCCGGCTAAACTGAG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV F3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1028&#x02013;1046</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">TGGATTTGGTTCTCGAAGG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Parida [<xref rid=\"pone.0147962.ref018\" ref-type=\"bibr\">18</xref>]</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV B3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1228&#x02013;1210</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GGTCAGCACGTTTGTCATT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV FIP(F<underline>1c</underline> + TTTT + F2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1121&#x02013;1100;1050&#x02013;1069</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>TTGGCCGCCTCCATATTCATCA</underline> + TTTT +CAGCTGCGTGACTATCATGT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV BIP(<underline>B1c</underline> + TTTT + B2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1144&#x02013;1165;1208&#x02013;1190</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>TGCTATTTGGCTACCGTCAGCG</underline> + TTTT +TGAGCTTCTCCCATGGTCG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV LF</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1093&#x02013;1075</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">CATCGATGGTAGGCTTGTC</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV LB</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1169&#x02013;1186</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">TCTCCACCAAAGCTGCGT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr></tbody></table></alternatives><table-wrap-foot><fn id=\"t002fn001\"><p><sup>1</sup> Genome positions are based on SLEV strain Kern217 (Genbank DQ525916.1), WEEV strain Montana-64 (Genbank GQ287643.1), and (as per reference [<xref rid=\"pone.0147962.ref018\" ref-type=\"bibr\">18</xref>]) WNV strain NY99 (Genbank AF196835).</p></fn><fn id=\"t002fn002\"><p><sup>2</sup> Sequence for inner primers FIP and BIP is shown separating out the F1c or B1c region (underlined) followed by the F2 or B2 region (not underlined). The complete FIP primer sequence is obtained by concatenating the F1c and F2 sequences (FIP = F1c + F2), and likewise for BIP. Primers designed by Parida <italic>et al</italic> [<xref rid=\"pone.0147962.ref018\" ref-type=\"bibr\">18</xref>] also include a TTTT linker between the two segments of the inner primers.</p></fn><fn id=\"t002fn003\"><p><sup>3</sup> A second BIP primer with 3 single-base substitutions in the B1c region intended to cover additional SLEV strains was designed but not used for testing; the alternative sequence (BIP = <underline>B1c</underline> + B2) is <underline>CAGG<bold>C</bold>GGACTGGG<bold>T</bold>A<bold>A</bold>TCAAAG</underline> + TCCTTACGCCATGGTCAC. The alternative BIP sequence is a match for two Mexican isolates designated Palenque-C475 and Palenque-A770 (Genbank JQ957868.1 and JQ957869.1), which form a distinct phylogenetic clade [<xref rid=\"pone.0147962.ref030\" ref-type=\"bibr\">30</xref>].</p></fn></table-wrap-foot></table-wrap><p>Gel electrophoresis of the SLEV (3&#x02019;-UTR) and WEEV (nsP4) RT-LAMP reaction products resulted in a prototypical ladder-like banding pattern typical of LAMP reactions (<xref ref-type=\"fig\" rid=\"pone.0147962.g003\">Fig 3</xref>, lanes 1 and 5). An online tool for LAMP restriction digest fragment analysis (<ext-link ext-link-type=\"uri\" xlink:href=\"http://creisle.github.io/creisle.lamprflp/\">http://creisle.github.io/creisle.lamprflp/</ext-link>) was used to predict restriction digest banding patterns based on the LAMP amplicon structure proposed by Notomi <italic>et al</italic> [<xref rid=\"pone.0147962.ref006\" ref-type=\"bibr\">6</xref>]. A restriction digest of the WEEV nsP4 amplicon with <italic>XbaI</italic> was predicted to yield main bands at 67 and 115 bp, with fainter bands at 50, 63, 77, and 97 bp. This is consistent with the banding pattern in Lane 2, which has primary bands slightly larger than 50 and 100 bp. Digestion of the SLEV 3&#x02019;-UTR amplicon with <italic>Hin</italic>dIII was predicted to yield bright bands at 130, 185, and 240 bp, with fainter bands at 58, 62, 76, 80, 136, and 146 bp. This corresponds closely to the pattern of banding in Lane 6. Additional faint bands in these digests at higher molecular weight may be due to incomplete digestion, or the presence of other amplicon structures not proposed by Notomi <italic>et al</italic> [<xref rid=\"pone.0147962.ref005\" ref-type=\"bibr\">5</xref>, <xref rid=\"pone.0147962.ref006\" ref-type=\"bibr\">6</xref>].</p><fig id=\"pone.0147962.g003\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.g003</object-id><label>Fig 3</label><caption><title>Electrophoretic analysis of RT-LAMP amplification products with WEEV nsP4 and SLEV 3&#x02019; UTR primer sets.</title><p>The gel is a 2% agarose E-gel (Invitrogen) with ethidium bromide. The marker (&#x0201c;M&#x0201d;) is a Low Molecular Weight DNA Ladder (New England Biolabs). Lane 1: WEEV positive control reaction with 2 PFU equivalent WEEV RNA in 20 &#x003bc;L. Lane 2: WEEV positive control after digestion with <italic>Xba</italic>I. Lane 3: No-template control reaction with WEEV nsP4 primer set. Lane 4: no sample. Lane 5: SLEV positive control reaction with 2 PFU equivalent SLEV RNA in 20 &#x003bc;L. Lane 6: SLEV positive control after digestion with <italic>Hin</italic>dIII. Lane 7: No-template control reaction with SLEV 3&#x02019;-UTR primer set. Cutting sites within the amplicon regions are shown in Figs <xref ref-type=\"fig\" rid=\"pone.0147962.g001\">1</xref> and <xref ref-type=\"fig\" rid=\"pone.0147962.g002\">2</xref>.</p></caption><graphic xlink:href=\"pone.0147962.g003\"/></fig><p>A recent survey of 33 WEEV strains [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>] collected primarily in the United States indicated that all sampled strains were ~95% identical. The six strains used here for the sequence alignment were representative of three (Groups B1, B2, and B3) out of four of the WEEV lineages defined by Bergren et al. [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>]. The nsP4 regions of these isolates had a single polymorphic site falling within a LAMP primer binding region. This mismatch occurred at the 3&#x02019; end of the B1c priming region. Based upon LAMP primer design guidelines provided by Eiken Genomics (<ext-link ext-link-type=\"uri\" xlink:href=\"http://loopamp.eiken.co.jp/e/lamp/primer.html\">http://loopamp.eiken.co.jp/e/lamp/primer.html</ext-link>), we infer that mismatches at the 3&#x02019; end of the B2, and the 5&#x02019; end of the B1c are more detrimental than mismatches at the 3&#x02019; end of B1c. However, since the effect of mismatches on LAMP performance is not well characterized, the polymorphic site was accommodated with a degenerate base (M) which covers all isolates used in the sequence alignment. After primer development, we did identify a single isolate of WEEV within the collection at UC Davis (Lake43, Genbank #KJ554985) with a single mismatch to the primer set, at the 5&#x02019; end of the F1c priming region. RNA from this isolate was successfully amplified, despite the mismatch (which also occurs in one other closely related WEEV isolate within Genbank, #KJ554984). The fourth WEEV lineage (Group A) described by Bergren et al. [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>] comprised early WEEV isolates from 1930 and 1941. When we compared our primers to the &#x0201c;California&#x0201d; isolate (Genbank KJ554965.1, from the &#x0201c;A&#x0201d; lineage defined by Bergen et al. [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>]), two mismatches were identified, one each near the middle of the F1c and B1c priming regions. The tolerance of LAMP to mismatches in the middle of priming regions is not well described. However, due to the high conservation of the target site within the WEEV nsP4 gene, our LAMP primers would likely detect most naturally occurring WEEV isolates.SLEV has a greater geographic distribution than WEEV, and sequences in Genbank show significantly higher divergence than is observed for sequenced isolates of WEEV. A phylogenetic tree based on the SLEV E gene sequences identified 7 clades [<xref rid=\"pone.0147962.ref029\" ref-type=\"bibr\">29</xref>]. Clades I and II represent primarily North American isolates, and clades III-VI represent primarily Central and South American isolates. The E gene is more divergent than the 3&#x02019; UTR region targeted by our primers. However, a comprehensive tree was not available for the 3&#x02019; UTR region, due to limited published sequences for SLEV that include the 3&#x02019; UTR. Sequence alignment for primer design included only North American isolates, and sensitivity testing was performed primarily with a single strain (Kern 217, Clade IIC). To determine whether the primer set was effective for detecting diverse lineages of SLEV, additional isolates falling within Clades IA (Bfs1750), IB (Coav750), IIA(69M143), VA (BeAn246407), and VII (CorAn9124 and CorAn9275) were tested. SLEV LAMP primers successfully detected RNA from all tested isolates, originating from both North and South America. In contrast the RT-qPCR assay used for surveillance in California failed to detect the two Clade VII isolates from Argentina, as indicated in <xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref></p></sec><sec id=\"sec012\"><title>Sensitivity and specificity of RT-LAMP for WEEV, SLEV, and WNV</title><p>Real-time monitoring of RT-LAMP reactions for WEEV and SLEV primer sets was performed using serial dilutions of viral RNA, extracted from cultured virus. <xref ref-type=\"fig\" rid=\"pone.0147962.g004\">Fig 4</xref> shows real-time monitoring curves for RT-LAMP reactions for each virus, as well as standard curves generated for these viral RNA by both RT-LAMP and RT-qPCR. In addition, a standard curve was generated using the published WNV RT-LAMP primer set [<xref rid=\"pone.0147962.ref018\" ref-type=\"bibr\">18</xref>] and was used for comparison to RT-qPCR; the resulting standard curves for WNV are shown in <xref ref-type=\"supplementary-material\" rid=\"pone.0147962.s001\">S1 Fig</xref>. All three RT-LAMP primer sets reliably detect their corresponding targets at 0.1 PFU equivalent, with less reliable detection down to 0.01 PFU. For all three viruses, RT-qPCR produced linear calibration curves to Ct &#x0003e;35, corresponding to viral titers of ~0.01 PFU, and less reliable detection down to 0.001 PFU. Overall, RT-LAMP reactions were 10-fold less sensitive than the corresponding RT-qPCR assays. The threshold of reliable detection for RT-LAMP was 100 PFU/mL corresponding to Ct ~30&#x02013;32 by RT-qPCR. Positive detection generally occurred in 20&#x02013;30 minutes, and often in &#x0003c;10 minutes for samples containing &#x0003e;10<sup>4</sup> PFU/mL. Although RT-LAMP sensitivity was slightly lower than RT-qPCR, WNV RT-LAMP reactions were markedly more sensitive than commercially available antigen tests. Two common tests used for WNV surveillance are VecTest&#x000ae; WNV antigen assay (Medical Analysis Systems, Inc., Camarillo, CA) and Rapid Analyte Measurement Platform (RAMP&#x000ae;) WNV test (Response Biomedical, Corp., Burnaby British Columbia, Canada); these tests have minimum sensitivities of 10<sup>5</sup> and 10<sup>3</sup> PFU/mL, respectively.</p><fig id=\"pone.0147962.g004\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.g004</object-id><label>Fig 4</label><caption><title>Quantitative analysis of SLEV and WEEV RT-LAMP.</title><p>Panels A,B: Real-time RT-LAMP amplification curves for SLEV isolate Kern 217 (A) and WEEV isolate Kern 5547 (B). Reactions were performed in triplicate with viral RNA ranging from 10,000 PFU equivalents to 0.01 PFU equivalent per 10 &#x003bc;L reaction. Reactions were monitored in real-time using a BioRad CFX96 thermal cycler, using the HEX channel to monitor the signal from SYTO 82; the abbreviation &#x0201c;RFU&#x0201d; refers to Relative Fluorescence Units. Panels C,D: Standard curves for SLEV (C) and WEEV (D) generated by real-time monitoring of RT-LAMP (blue circles), versus RT-qPCR (red triangles). Time-to-positivity in RT-LAMP was determined by the BioRad CFX manager software, using baseline-subtracted quantitation data.</p></caption><graphic xlink:href=\"pone.0147962.g004\"/></fig><p>Although real-time monitoring of LAMP by means such as turbidity or fluorescent dyes is common [<xref rid=\"pone.0147962.ref010\" ref-type=\"bibr\">10</xref>, <xref rid=\"pone.0147962.ref011\" ref-type=\"bibr\">11</xref>], we note that real-time monitoring of RT-LAMP does not produce as precise a standard curve as RT-qPCR, particularly at lower viral RNA levels where time to positivity can vary substantially. We attribute this to the continuous nature of RT-LAMP, as opposed to the discrete cycling nature of RT-qPCR. In a well-designed RT-qPCR assay, each cycle in the exponential amplification regime leads to a reproducible doubling of the amount of amplicon. In RT-LAMP, however, slight variations in the early stages of the reaction can result in larger fluctuations in amplification time, particularly in the stochastic regime with few template copies, where multiple mechanisms may reduce the efficiency of the RT step in particular [<xref rid=\"pone.0147962.ref031\" ref-type=\"bibr\">31</xref>]. Real-time monitoring of RT-LAMP reactions can provide order-of-magnitude estimates of viral RNA concentration, but for precise quantitation RT-qPCR is preferable, particularly for low concentrations of viral RNA. The value of RT-LAMP for viral detection lies in the simplicity of the technique, particularly when applied in low-resource settings, where a yes-or-no answer determined by an endpoint measurement is sufficient.</p><p>Although the purpose of the current study was to develop and characterize new primer sets for SLEV and WEEV, we also demonstrated RT-LAMP detection of these targets using simpler instrumentation, as shown in <xref ref-type=\"supplementary-material\" rid=\"pone.0147962.s002\">S2 Fig</xref>. First, we carried out the RT-LAMP reactions for SLEV and WEEV using the Genie III, a portable isothermal fluorimeter with rechargeable battery and built-in wireless communication. Secondly, we took a photograph of the fluorescence endpoint using a handheld LED flashlight for illumination, and placing an amber-colored plastic film in front of the camera lens. In both cases, the fluorescence endpoint is clearly distinguishable between no-template control reactions and positive control reactions (tested down to 0.1 PFU equivalent of template).</p><p>For specificity testing, WEEV, SLEV, and WNV primer sets were tested against a panel of alphaviruses and flaviviruses, as indicated in <xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>. For WEEV, the closest relatives are other New World alphaviruses, Venezuelan equine encephalitis virus (VEEV) and Eastern equine encephalitis virus (EEEV). We found no cross-reactivity between any of the primer sets among non-target viruses. The only unexpected positive reactions were two isolated instances that we attribute to cross-contamination, where RNA from positive controls was inadvertently introduced into non-target cross-reactivity reactions. The specific instances (noted in <xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>) are a single positive reaction (out of 18 replicates) when VEEV TC-83 RNA template was tested with the SLEV primers, and a single positive reaction (out of 11 replicates) when SLEV strain CorAn 9275 was tested with WNV primers. In both cases, the single positive amplifications displayed identical melt curves to positive-control reactions performed on the same plate, and these isolated positive results could not be reproduced. Otherwise no evidence of cross-reactivity between primers sets and non-target viruses was observed. In addition, there was no evidence of non-specific amplification in no-template control reactions with the SLEV and WEEV primer sets incubated for up to 50 minutes. Previously, non-specific amplification was observed with the previously published WNV primer set, but most often developed after &#x0003e; 50 minutes of incubation.</p><p>We tested archived RNA from 12 pooled mosquito samples collected in California in 2002 or 2014 that were positive by RT-qPCR for WEEV, SLEV, or WNV, or negative for all three, to determine if our RT-LAMP assays could detect these viruses in mosquito samples. Results of these tests are presented in <xref ref-type=\"table\" rid=\"pone.0147962.t003\">Table 3</xref>. The RT-LAMP assay detected SLEV in four pools that had RT-qPCR Cts ranging from 24.4 to 27.7. The RT-LAMP assay also detected WEEV in 5 pools that had RT-qPCR Cts ranging from 22.8 to 26.5, but failed to detect WEEV in one sample with a Ct value of 32, which is below the sensitivity demonstrated for the WEEV RT-LAMP assays. Sample sizes were limited, but the new primer sets readily detected SLEV and WEEV in mosquito pool RNA. Although sensitivity was slightly reduced compared to RT-qPCR, the specificity of these new primers was absolute.</p><table-wrap id=\"pone.0147962.t003\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.t003</object-id><label>Table 3</label><caption><title>Analysis of mosquito pools by SLEV, WEEV, and WNV RT-LAMP.</title></caption><alternatives><graphic id=\"pone.0147962.t003g\" xlink:href=\"pone.0147962.t003\"/><table frame=\"hsides\" rules=\"groups\"><colgroup span=\"1\"><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/></colgroup><thead><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\"/><th align=\"center\" colspan=\"3\" rowspan=\"1\"><underline>RT-qPCR Ct</underline><xref ref-type=\"table-fn\" rid=\"t003fn001\"><sup>1</sup></xref></th><th align=\"center\" colspan=\"3\" rowspan=\"1\"><underline>RT-LAMP Time-to-positivity (min)</underline><xref ref-type=\"table-fn\" rid=\"t003fn002\"><sup>2</sup></xref></th></tr><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\">Sample<xref ref-type=\"table-fn\" rid=\"t003fn003\"><sup>3</sup></xref></th><th align=\"center\" rowspan=\"1\" colspan=\"1\">SLEV</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WEEV</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WNV</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">SLEV</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WEEV</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WNV</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 490</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">22.8</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">12.2, 12.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 512</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">26.5</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">14.0, 15.9</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 513</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">25.1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">13.0, 14.6</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 984</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">23.0</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">12.1, 12.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 942</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">24.5</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">12.2, 12.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 862</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">27.7</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">13.1, 13.9</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 IMPR 136</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">24.4</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">12.4, 12.5</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 IMPR 139</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">26.5</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">13.4, 14.0</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 IMPR 145</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">27.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">32.0</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">19.1, 21.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2014 KERN 14&#x02013;163</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">23.0</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">10.8, 11.1</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2014 MARN 14&#x02013;300</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">25.0</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">13.3, 13.8</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2014 SUYA 14&#x02013;144</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr></tbody></table></alternatives><table-wrap-foot><fn id=\"t003fn001\"><p><sup>1</sup>Triplex RT-qPCRfor SLEV, WEEV, and WNV surveillance samples performed by UC Davis Center for Vectorborne Diseases.</p></fn><fn id=\"t003fn002\"><p><sup>2</sup>Results of two replicates reported</p></fn><fn id=\"t003fn003\"><p><sup>3</sup>Sample designations indicate year and location of sample collection within California: COAV = Coachella Valley, IMPR = Imperial Valley; KERN = Kern County; MARN = Marin-Sonoma counties, SUYA = Sutter-Yuba counties.</p></fn></table-wrap-foot></table-wrap></sec><sec id=\"sec013\"><title>Multiplexing RT-LAMP by melt curve analysis</title><p>Multiplexed amplification by LAMP was consistent, and all positive control RT-LAMP reactions amplified target RNA in less than 30 minutes. No-template controls remained negative for 40 minutes, after which non-specific amplification was detected in multiplexes containing WNV and WEEV primer sets together. Amplification in WNV + SLEV + WEEV and WNV + WEEV no-template controls likely resulted from primer-dimer formation between the WNV BIP and the WEEV FIP primers. Negative control reactions containing the combination of WNV and WEEV primer sets showed left-skewed melting peaks at 86.3&#x000b0;C, corresponding to melting of amplified primer-dimers. The intensity of the negative control melt curves was lower in reactions with a lower initial concentration of WNV/WEEV primer sets. Positive controls were distinguishable from negative controls and produced LAMP products that were suitable for melt curve analysis. Examples are presented in <xref ref-type=\"fig\" rid=\"pone.0147962.g005\">Fig 5</xref>.</p><fig id=\"pone.0147962.g005\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.g005</object-id><label>Fig 5</label><caption><title>Melt derivative curves for singleplex and multiplex RT-LAMP.</title><p>Solid black lines show average data from triplicate runs. Colored shaded areas represent the contributions of individual RT-LAMP targets to the overall signals to aid visualization. Blue = WEEV. Yellow = SLEV. Red = WNV. A) Singleplex reactions. B) Three-way combination. Only two peaks are distinguishable by eye. C) WEEV + WNV no template control. A relatively small melting peak is present, corresponding to spurious amplification after 40 min. D-F) Pairwise combinations. A smaller WEEV peak is distinguishable from WNV and SLEV peaks in D and F. WNV and SLEV peaks are indistinguishable by eye in E.</p></caption><graphic xlink:href=\"pone.0147962.g005\"/></fig><p>The melting curves of multiplexed LAMP reactions contained up to two distinguishable peaks for target identification. The first peak (84.5&#x02013;84.9&#x000b0;C) corresponded to the melting of WEEV-specific amplicons. The second peak (86.5&#x02013;87&#x000b0;C) corresponds to the melting of WNV-specific amplicons, SLEV-specific amplicons, or their combination.</p><p>The center and width of amplicon melting distributions determined the ease of multiplexed target detection. For example, the melting point of WNV amplicons was 0.5&#x000b0;C higher than that of SLEV amplicons. Because this difference is close to the 0.2&#x000b0;C accuracy limit of the real-time PCR instrument, reactions containing both WNV and SLEV amplicons displayed a single melting curve&#x02014;with an intermediate melting temperature&#x02014;instead of two discrete curves. Thus, simple peak picking was insufficient to differentiate WNV from SLEV in reactions containing both WNV and SLEV primer sets.</p><p>Furthermore, the relative intensity of discrete melt peaks reflected the relative abundance of individual LAMP products. However, in reactions with equal starting amounts of template, the WEEV amplicon melt-curve intensity was less than that of either the WNV or SLEV amplicons. This suggests that amplification with the WEEV primer set is slower or less efficient than either the WNV or SLEV primer sets. We have not tested experimentally whether this lower efficiency of amplification for WEEV reduces the detection limit of WEEV when present together in the same sample with a larger amount of WNV or SLEV.</p><p>Given the tendency to non-specific amplification observed with the combination of the WNV and WEEV primers, and the overlap of the WNV and SLEV melt curves, we conclude that only the WEEV and SLEV assays could be multiplexed in a single reaction. A more complex curve fitting algorithm based on parameters such as peak width, skew and temperature may be capable of robust differentiation of WEEV, SLEV, and WNV amplicons in multiplexed LAMP reactions. However, to our knowledge such an approach has not been demonstrated for LAMP melt curve analysis.</p></sec></sec><sec sec-type=\"conclusions\" id=\"sec014\"><title>Conclusion</title><p>The newly developed RT-LAMP primer sets for SLEV and WEEV show sensitivity and specificity comparable to a previously reported primer set for WNV [<xref rid=\"pone.0147962.ref016\" ref-type=\"bibr\">16</xref>]. Although the RT-LAMP assays are approximately an order of magnitude less sensitive than the corresponding RT-qPCR assays, the assay consistently detected to 0.1 PFU/reaction equivalents without cross-reactivity between related viruses. In addition, the RT-LAMP method can be greatly simplified compared to standard RT-qPCR reactions. Here purified RNA targets and a conventional real-time PCR thermal cycler were used to characterize RT-LAMP reactions, but we also demonstrate compatibility with a low-cost portable fluorimeter for monitoring isothermal amplification reactions, as well as compatibility with non-instrumented detection Future work is directly at developing simplified RT-LAMP protocols for RNA viruses such as WNV, SLEV, and WEEV. New protocols will utilize the tolerance of the Bst DNA polymerase to potential &#x0201c;PCR inhibitors&#x0201d; that are often present in crude samples such as mosquito or tissue homogenates. The combination of simplified instrumentation, minimal sample prep, no RNA extraction, and low reagent cost may enable the development of RT-LAMP based assays that are as simple to use as commercially available antigen tests. This would enable highly sensitive and specific surveillance for vector-borne diseases in basic laboratories or field settings where RT-qPCR resources are unavailable.</p></sec><sec sec-type=\"supplementary-material\" id=\"sec015\"><title>Supporting Information</title><supplementary-material content-type=\"local-data\" id=\"pone.0147962.s001\"><label>S1 Fig</label><caption><title>Standard curves for WNV RT-LAMP and RT-qPCR.</title><p>RT-LAMP time to positivity (blue circles) versus RT-qPCR Ct (red triangles), for WNV isolate L-CA-04 SAC-04-7168 using the RT-LAMP primers published by Parida et al. [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>], and the RT-qPCR primers published by Lanciotti et al [<xref rid=\"pone.0147962.ref002\" ref-type=\"bibr\">2</xref>]. At least three technical replicates were performed for each template concentration.</p><p>(PDF)</p></caption><media xlink:href=\"pone.0147962.s001.pdf\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"pone.0147962.s002\"><label>S2 Fig</label><caption><title>Detection of SLEV and WEEV by RT-LAMP using simplified techniques.</title><p>(A) SLEV and WEEV RNA were detected by RT-LAMP using the OptiGene Genie III, a portable, battery-operated real-time fluorimeter. The screenshot in the inset of panel (A) shows real-time amplification curves generated for eight reactions, which are specified in Panel B. Reactions 1&#x02013;4 use the SLEV 3&#x02019; UTR primer set, with 10, 1, 0.1 PFU equivalents of SLEV RNA (Reaction 4 is an SLEV no-template control). Reactions 5&#x02013;8 use the WEEV nsP4 primer set, with 10, 1, 0.1 PFU equivalents of WEEV RNA (Reaction 8 is a WEEV no-template control). Panel (B) illustrates the endpoint of the same eight reactions, after incubation at 63&#x000b0;C for 30 minutes. To take this photograph, the strip of tubes was set upon a dark background, on a hotplate set at 63&#x000b0;C. Fluorescence was excited using a handheld blue LED flashlight, and a sheet of amber plastic film (LEE Filters #158) was placed in front of the lens of a digital camera. No contrast adjustment was applied. In Panel A, a barcode label affixed to the instrument has been pixelated to obscure identification.</p><p>(TIF)</p></caption><media xlink:href=\"pone.0147962.s002.tif\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"pone.0147962.s003\"><label>S1 Table</label><caption><title>RT-qPCR primers used for quantitation of Western equine encephalitis virus (WEEV), St. Louis encephalitis virus (SLEV), and West Nile virus (WNV) viral RNA.</title><p>IBFQ = Iowa Black FQ; BHQ1 = Black Hole Quencher 1 (The different quenchers have similar performance when paired with FAM, and do not affect the results of the PCR assays). The primer set names from the original literature references are provided in the table next to the reference numbers.</p><p>(PDF)</p></caption><media xlink:href=\"pone.0147962.s003.pdf\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"pone.0147962.s004\"><label>S2 Table</label><caption><title>Additional RT-LAMP primer sets for SLEV and WEEV.</title><p>(PDF)</p></caption><media xlink:href=\"pone.0147962.s004.pdf\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material></sec></body><back><ack><p>This work was supported by a grant from the Defense Threat Reduction Agency (DTRA), contract DTRA1002710697. Sandia National Laboratories is a multi-program laboratory managed and operated by Sandia Corporation, a wholly owned subsidiary of Lockheed Martin Corporation, for the U.S. Department of Energy&#x02019;s National Nuclear Security Administration under Contract DE-AC04-94AL85000.</p></ack><ref-list><title>References</title><ref id=\"pone.0147962.ref001\"><label>1</label><mixed-citation publication-type=\"journal\"><name><surname>Bergren</surname><given-names>NA</given-names></name>, <name><surname>Auguste</surname><given-names>AJ</given-names></name>, <name><surname>Forrester</surname><given-names>NL</given-names></name>, <name><surname>Negi</surname><given-names>SS</given-names></name>, <name><surname>Braun</surname><given-names>WA</given-names></name>, <name><surname>Weaver</surname><given-names>SC</given-names></name>. <article-title>Western equine encephalitis virus: evolutionary analysis of a declining alphavirus based on complete genome sequences</article-title>. <source>J Virol</source>. <year>2014</year>;<volume>88</volume>(<issue>16</issue>):<fpage>9260</fpage>&#x02013;<lpage>7</lpage>. Epub 2014/06/06. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jvi.01463-14\">10.1128/jvi.01463-14</ext-link></comment> ; PubMed Central PMCID: PMCPmc4136285.<pub-id pub-id-type=\"pmid\">24899192</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref002\"><label>2</label><mixed-citation publication-type=\"journal\"><name><surname>Reisen</surname><given-names>WK</given-names></name>, <name><surname>Lothrop</surname><given-names>HD</given-names></name>, <name><surname>Wheeler</surname><given-names>SS</given-names></name>, <name><surname>Kennsington</surname><given-names>M</given-names></name>, <name><surname>Gutierrez</surname><given-names>A</given-names></name>, <name><surname>Fang</surname><given-names>Y</given-names></name>, <etal>et al</etal>\\n<article-title>Persistent West Nile virus transmission and the apparent displacement St. Louis encephalitis virus in southeastern California, 2003&#x02013;2006</article-title>. <source>J Medical Entomol</source>. <year>2008</year>;<volume>45</volume>(<issue>3</issue>):<fpage>494</fpage>&#x02013;<lpage>508</lpage>. Epub 2008/06/07. ; PubMed Central PMCID: PMCPmc2435167.<pub-id pub-id-type=\"pmid\">18533445</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref003\"><label>3</label><mixed-citation publication-type=\"other\">Oviatt J. Saint Louis encephalitis virus detected in valley mosquitoes for the first time since 2003: Coachella Valley Mosquito and Vector Control District; 2015 [cited 2015 10/2/2015]. Press release]. Available: <ext-link ext-link-type=\"uri\" xlink:href=\"http://www.cvmvcd.org/press/releases/2015/engpressreleaseslewnv080515.pdf\">http://www.cvmvcd.org/press/releases/2015/engpressreleaseslewnv080515.pdf</ext-link>.</mixed-citation></ref><ref id=\"pone.0147962.ref004\"><label>4</label><mixed-citation publication-type=\"journal\"><name><surname>Vidal</surname><given-names>F</given-names></name>, <name><surname>Julia</surname><given-names>SA</given-names></name>, <name><surname>Carolina</surname><given-names>G</given-names></name>, <name><surname>Edna</surname><given-names>RC</given-names></name>, <name><surname>Jorge</surname><given-names>G</given-names></name>, <name><surname>Felix</surname><given-names>WS-M</given-names></name>, <etal>et al</etal>\\n<article-title>St. Louis Encephalitis virus infection in woman, Peru</article-title>. <source>Emerg Infect Dis</source>. <year>2014</year>;<volume>20</volume>(<issue>4</issue>):<fpage>730</fpage>\\n<comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.3201/eid2004.131735\">10.3201/eid2004.131735</ext-link></comment>\\n<pub-id pub-id-type=\"pmid\">24655897</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref005\"><label>5</label><mixed-citation publication-type=\"journal\"><name><surname>Nagamine</surname><given-names>K</given-names></name>, <name><surname>Hase</surname><given-names>T</given-names></name>, <name><surname>Notomi</surname><given-names>T</given-names></name>. <article-title>Accelerated reaction by loop-mediated isothermal amplification using loop primers</article-title>. <source>Mol Cell Probes</source>. <year>2002</year>;<volume>16</volume>(<issue>3</issue>):<fpage>223</fpage>&#x02013;<lpage>9</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1006/mcpr.2002.0415\">10.1006/mcpr.2002.0415</ext-link></comment> .<pub-id pub-id-type=\"pmid\">12144774</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref006\"><label>6</label><mixed-citation publication-type=\"journal\"><name><surname>Notomi</surname><given-names>T</given-names></name>, <name><surname>Okayama</surname><given-names>H</given-names></name>, <name><surname>Masubuchi</surname><given-names>H</given-names></name>, <name><surname>Yonekawa</surname><given-names>T</given-names></name>, <name><surname>Watanabe</surname><given-names>K</given-names></name>, <name><surname>Amino</surname><given-names>N</given-names></name>, <etal>et al</etal>\\n<article-title>Loop-mediated isothermal amplification of DNA</article-title>. <source>Nucleic Acids Res</source>. <year>2000</year>;<volume>28</volume>(<issue>12</issue>). <fpage>e63</fpage>\\n<comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1093/nar/28.12.e63\">10.1093/nar/28.12.e63</ext-link></comment> .<pub-id pub-id-type=\"pmid\">10871386</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref007\"><label>7</label><mixed-citation publication-type=\"journal\"><name><surname>Iwamoto</surname><given-names>T</given-names></name>, <name><surname>Sonobe</surname><given-names>T</given-names></name>, <name><surname>Hayashi</surname><given-names>K</given-names></name>. <article-title>Loop-mediated isothermal amplification for direct detection of Mycobacterium tuberculosis complex, M. avium, and M. intracellulare in sputum samples</article-title>. <source>J Clin Microbiol</source>. <year>2003</year>;<volume>41</volume>(<issue>6</issue>):<fpage>2616</fpage>&#x02013;<lpage>22</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/JCM.41.6.2616&#x02013;2622.2003\">10.1128/JCM.41.6.2616&#x02013;2622.2003</ext-link></comment> .<pub-id pub-id-type=\"pmid\">12791888</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref008\"><label>8</label><mixed-citation publication-type=\"journal\"><name><surname>Polley</surname><given-names>SD</given-names></name>, <name><surname>Gonz&#x000e1;lez</surname><given-names>IJ</given-names></name>, <name><surname>Mohamed</surname><given-names>D</given-names></name>, <name><surname>Daly</surname><given-names>R</given-names></name>, <name><surname>Bowers</surname><given-names>K</given-names></name>, <name><surname>Watson</surname><given-names>J</given-names></name>, <etal>et al</etal>\\n<article-title>Clinical evaluation of a LAMP test kit for diagnosis of imported malaria</article-title>. <source>J Infect Dis</source>. <year>2013</year>\\n<comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1093/infdis/jit183\">10.1093/infdis/jit183</ext-link></comment></mixed-citation></ref><ref id=\"pone.0147962.ref009\"><label>9</label><mixed-citation publication-type=\"journal\"><name><surname>Fukuta</surname><given-names>S</given-names></name>, <name><surname>Iida</surname><given-names>T</given-names></name>, <name><surname>Mizukami</surname><given-names>Y</given-names></name>, <name><surname>Ishida</surname><given-names>A</given-names></name>, <name><surname>Ueda</surname><given-names>J</given-names></name>, <name><surname>Kanbe</surname><given-names>M</given-names></name>, <etal>et al</etal>\\n<article-title>Detection of Japanese yam mosaic virus by RT-LAMP</article-title>. <source>Arch Virol</source>. <year>2003</year>;<volume>148</volume>(<issue>9</issue>):<fpage>1713</fpage>&#x02013;<lpage>20</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1007/s00705-003-0134-5\">10.1007/s00705-003-0134-5</ext-link></comment> .<pub-id pub-id-type=\"pmid\">14505084</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref010\"><label>10</label><mixed-citation publication-type=\"journal\"><name><surname>Mori</surname><given-names>Y</given-names></name>, <name><surname>Nagamine</surname><given-names>K</given-names></name>, <name><surname>Tomita</surname><given-names>N</given-names></name>, <name><surname>Notomi</surname><given-names>T</given-names></name>. <article-title>Detection of loop-mediated isothermal amplification reaction by turbidity derived from magnesium pyrophosphate formation</article-title>. <source>Biochem Biophys Res Commun</source>. <year>2001</year>;<volume>289</volume>(<issue>1</issue>):<fpage>150</fpage>&#x02013;<lpage>4</lpage>. Epub 2001/11/16. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1006/bbrc.2001.5921\">10.1006/bbrc.2001.5921</ext-link></comment> .<pub-id pub-id-type=\"pmid\">11708792</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref011\"><label>11</label><mixed-citation publication-type=\"journal\"><name><surname>Ahmad</surname><given-names>F</given-names></name>, <name><surname>Seyrig</surname><given-names>G</given-names></name>, <name><surname>Tourlousse</surname><given-names>DM</given-names></name>, <name><surname>Stedtfeld</surname><given-names>RD</given-names></name>, <name><surname>Tiedje</surname><given-names>JM</given-names></name>, <name><surname>Hashsham</surname><given-names>SA</given-names></name>. <article-title>A CCD-based fluorescence imaging system for real-time loop-mediated isothermal amplification-based rapid and sensitive detection of waterborne pathogens on microchips</article-title>. <source>Biomed Microdevices</source>. <year>2011</year>;<volume>13</volume>(<issue>5</issue>):<fpage>929</fpage>&#x02013;<lpage>37</lpage>. Epub 2011/07/02. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1007/s10544-011-9562-2\">10.1007/s10544-011-9562-2</ext-link></comment> .<pub-id pub-id-type=\"pmid\">21720851</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref012\"><label>12</label><mixed-citation publication-type=\"journal\"><name><surname>Nie</surname><given-names>K</given-names></name>, <name><surname>Qi</surname><given-names>SX</given-names></name>, <name><surname>Zhang</surname><given-names>Y</given-names></name>, <name><surname>Luo</surname><given-names>L</given-names></name>, <name><surname>Xie</surname><given-names>Y</given-names></name>, <name><surname>Yang</surname><given-names>MJ</given-names></name>, <etal>et al</etal>\\n<article-title>Evaluation of a direct reverse transcription loop-mediated isothermal amplification method without RNA extraction for the detection of human enterovirus 71 subgenotype C4 in nasopharyngeal swab specimens</article-title>. <source>PLOS One</source>. <year>2012</year>;<volume>7</volume>(<issue>12</issue>):<fpage>e52486</fpage> Epub 2012/12/29. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1371/journal.pone.0052486\">10.1371/journal.pone.0052486</ext-link></comment> ; PubMed Central PMCID: PMC3525532.<pub-id pub-id-type=\"pmid\">23272248</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref013\"><label>13</label><mixed-citation publication-type=\"journal\"><name><surname>Wang</surname><given-names>X</given-names></name>, <name><surname>Zhang</surname><given-names>Q</given-names></name>, <name><surname>Zhang</surname><given-names>F</given-names></name>, <name><surname>Ma</surname><given-names>F</given-names></name>, <name><surname>Zheng</surname><given-names>W</given-names></name>, <name><surname>Zhao</surname><given-names>Z</given-names></name>, <etal>et al</etal>\\n<article-title>Visual detection of the human metapneumovirus using reverse transcription loop-mediated isothermal amplification with hydroxynaphthol blue dye</article-title>. <source>Virol J</source>. <year>2012</year>;<volume>9</volume>:<fpage>138</fpage> Epub 2012/07/31. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1186/1743-422x-9-138\">10.1186/1743-422x-9-138</ext-link></comment> ; PubMed Central PMCID: PMC3487928.<pub-id pub-id-type=\"pmid\">22838725</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref014\"><label>14</label><mixed-citation publication-type=\"journal\"><name><surname>Poon</surname><given-names>LL</given-names></name>, <name><surname>Wong</surname><given-names>BW</given-names></name>, <name><surname>Ma</surname><given-names>EH</given-names></name>, <name><surname>Chan</surname><given-names>KH</given-names></name>, <name><surname>Chow</surname><given-names>LM</given-names></name>, <name><surname>Abeyewickreme</surname><given-names>W</given-names></name>, <etal>et al</etal>\\n<article-title>Sensitive and inexpensive molecular test for falciparum malaria: detecting Plasmodium falciparum DNA directly from heat-treated blood by loop-mediated isothermal amplification</article-title>. <source>Clin Chem</source>. <year>2006</year>;<volume>52</volume>(<issue>2</issue>):<fpage>303</fpage>&#x02013;<lpage>6</lpage>. Epub 2005/12/13. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1373/clinchem.2005.057901\">10.1373/clinchem.2005.057901</ext-link></comment> .<pub-id pub-id-type=\"pmid\">16339303</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref015\"><label>15</label><mixed-citation publication-type=\"journal\"><name><surname>Curtis</surname><given-names>KA</given-names></name>, <name><surname>Rudolph</surname><given-names>DL</given-names></name>, <name><surname>Nejad</surname><given-names>I</given-names></name>, <name><surname>Singleton</surname><given-names>J</given-names></name>, <name><surname>Beddoe</surname><given-names>A</given-names></name>, <name><surname>Weigl</surname><given-names>B</given-names></name>, <etal>et al</etal>\\n<article-title>Isothermal amplification using a chemical heating device for point-of-care detection of HIV-1</article-title>. <source>PLOS One</source>. <year>2012</year>;<volume>7</volume>(<issue>2</issue>):<fpage>e31432</fpage> Epub 2012/03/03. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1371/journal.pone.0031432\">10.1371/journal.pone.0031432</ext-link></comment> ; PubMed Central PMCID: PMCPmc3285652.<pub-id pub-id-type=\"pmid\">22384022</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref016\"><label>16</label><mixed-citation publication-type=\"journal\"><name><surname>Curtis</surname><given-names>KA</given-names></name>, <name><surname>Rudolph</surname><given-names>DL</given-names></name>, <name><surname>Owen</surname><given-names>SM</given-names></name>. <article-title>Rapid detection of HIV-1 by reverse-transcription, loop-mediated isothermal amplification (RT-LAMP)</article-title>. <source>J Virol Meth</source>. <year>2008</year>;<volume>151</volume>(<issue>2</issue>):<fpage>264</fpage>&#x02013;<lpage>70</lpage>. Epub 2008/06/06. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1016/j.jviromet.2008.04.011\">10.1016/j.jviromet.2008.04.011</ext-link></comment> .<pub-id pub-id-type=\"pmid\">18524393</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref017\"><label>17</label><mixed-citation publication-type=\"journal\"><name><surname>Francois</surname><given-names>P</given-names></name>, <name><surname>Tangomo</surname><given-names>M</given-names></name>, <name><surname>Hibbs</surname><given-names>J</given-names></name>, <name><surname>Bonetti</surname><given-names>EJ</given-names></name>, <name><surname>Boehme</surname><given-names>CC</given-names></name>, <name><surname>Notomi</surname><given-names>T</given-names></name>, <etal>et al</etal>\\n<article-title>Robustness of a loop-mediated isothermal amplification reaction for diagnostic applications</article-title>. <source>FEMS Immunol Med Microbiol</source>. <year>2011</year>;<volume>62</volume>(<issue>1</issue>):<fpage>41</fpage>&#x02013;<lpage>8</lpage>. Epub 2011/02/01. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1111/j.1574-695X.2011.00785.x\">10.1111/j.1574-695X.2011.00785.x</ext-link></comment> .<pub-id pub-id-type=\"pmid\">21276085</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref018\"><label>18</label><mixed-citation publication-type=\"journal\"><name><surname>Parida</surname><given-names>M</given-names></name>, <name><surname>Posadas</surname><given-names>G</given-names></name>, <name><surname>Inoue</surname><given-names>S</given-names></name>, <name><surname>Hasebe</surname><given-names>F</given-names></name>, <name><surname>Morita</surname><given-names>K</given-names></name>. <article-title>Real-time reverse transcription loop-mediated isothermal amplification for rapid detection of West Nile virus</article-title>. <source>J Clin Microbiol</source>. 2005. <year>2004</year>;<volume>42</volume>(<issue>1</issue>):<fpage>257</fpage>&#x02013;<lpage>63</lpage>. Epub 2004/01/13. ; PubMed Central PMCID: PMC321710.<pub-id pub-id-type=\"pmid\">14715762</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref019\"><label>19</label><mixed-citation publication-type=\"journal\"><name><surname>Parida</surname><given-names>M</given-names></name>, <name><surname>Horioke</surname><given-names>K</given-names></name>, <name><surname>Ishida</surname><given-names>H</given-names></name>, <name><surname>Dash</surname><given-names>PK</given-names></name>, <name><surname>Saxena</surname><given-names>P</given-names></name>, <name><surname>Jana</surname><given-names>AM</given-names></name>, <etal>et al</etal>\\n<article-title>Rapid detection and differentiation of dengue virus serotypes by a real-time reverse transcription-loop-mediated isothermal amplification assay</article-title>. <source>J Clin Microbiol</source>. <year>2005</year>;<volume>43</volume>(<issue>6</issue>):<fpage>2895</fpage>&#x02013;<lpage>903</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jcm.43.6.2895&#x02013;2903.2005.\">10.1128/jcm.43.6.2895&#x02013;2903.2005.</ext-link></comment> .<pub-id pub-id-type=\"pmid\">15956414</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref020\"><label>20</label><mixed-citation publication-type=\"journal\"><name><surname>Teoh</surname><given-names>BT</given-names></name>, <name><surname>Sam</surname><given-names>SS</given-names></name>, <name><surname>Tan</surname><given-names>KK</given-names></name>, <name><surname>Johari</surname><given-names>J</given-names></name>, <name><surname>Danlami</surname><given-names>MB</given-names></name>, <name><surname>Hooi</surname><given-names>PS</given-names></name>, <etal>et al</etal>\\n<article-title>Detection of dengue viruses using reverse transcription-loop-mediated isothermal amplification</article-title>. <source>BMC Infect Dis</source>. <year>2013</year>;<volume>13</volume>:<fpage>387</fpage> Epub 2013/08/24. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1186/1471-2334-13-387\">10.1186/1471-2334-13-387</ext-link></comment> ; PubMed Central PMCID: PMC3846474.<pub-id pub-id-type=\"pmid\">23964963</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref021\"><label>21</label><mixed-citation publication-type=\"journal\"><name><surname>Liu</surname><given-names>H</given-names></name>, <name><surname>Liu</surname><given-names>ZJ</given-names></name>, <name><surname>Jing</surname><given-names>J</given-names></name>, <name><surname>Ren</surname><given-names>JQ</given-names></name>, <name><surname>Liu</surname><given-names>YY</given-names></name>, <name><surname>Guo</surname><given-names>HH</given-names></name>, <etal>et al</etal>\\n<article-title>Reverse transcription loop-mediated isothermal amplification for rapid detection of Japanese encephalitis virus in swine and mosquitoes</article-title>. <source>Vector Borne Zoonotic Dis</source>. <year>2012</year>;<volume>12</volume>(<issue>12</issue>):<fpage>1042</fpage>&#x02013;<lpage>52</lpage>. Epub 2012/11/28. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1089/vbz.2012.0991\">10.1089/vbz.2012.0991</ext-link></comment> ; PubMed Central PMCID: PMCPmc3525893.<pub-id pub-id-type=\"pmid\">23176446</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref022\"><label>22</label><mixed-citation publication-type=\"journal\"><name><surname>Parida</surname><given-names>MM</given-names></name>, <name><surname>Santhosh</surname><given-names>SR</given-names></name>, <name><surname>Dash</surname><given-names>PK</given-names></name>, <name><surname>Tripathi</surname><given-names>NK</given-names></name>, <name><surname>Saxena</surname><given-names>P</given-names></name>, <name><surname>Ambuj</surname><given-names>S</given-names></name>, <etal>et al</etal>\\n<article-title>Development and evaluation of reverse transcription-loop-mediated isothermal amplification assay for rapid and real-time detection of Japanese encephalitis virus</article-title>. <source>J Clin Microbiol</source>. <year>2006</year>;<volume>44</volume>(<issue>11</issue>):<fpage>4172</fpage>&#x02013;<lpage>8</lpage>. Epub 2006/09/29. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jcm.01487-06\">10.1128/jcm.01487-06</ext-link></comment> ; PubMed Central PMCID: PMCPmc1698363.<pub-id pub-id-type=\"pmid\">17005741</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref023\"><label>23</label><mixed-citation publication-type=\"journal\"><name><surname>Toriniwa</surname><given-names>H</given-names></name>, <name><surname>Komiya</surname><given-names>T</given-names></name>. <article-title>Rapid detection and quantification of Japanese encephalitis virus by real-time reverse transcription loop-mediated isothermal amplification</article-title>. <source>Microbiol Immunol</source>. <year>2006</year>;<volume>50</volume>(<issue>5</issue>):<fpage>379</fpage>&#x02013;<lpage>87</lpage>. Epub 2006/05/23. .<pub-id pub-id-type=\"pmid\">16714845</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref024\"><label>24</label><mixed-citation publication-type=\"journal\"><name><surname>Parida</surname><given-names>MM</given-names></name>, <name><surname>Santhosh</surname><given-names>SR</given-names></name>, <name><surname>Dash</surname><given-names>PK</given-names></name>, <name><surname>Tripathi</surname><given-names>NK</given-names></name>, <name><surname>Lakshmi</surname><given-names>V</given-names></name>, <name><surname>Mamidi</surname><given-names>N</given-names></name>, <etal>et al</etal>\\n<article-title>Rapid and real-time detection of chikungunya virus by reverse transcription loop-mediated isothermal amplification assay</article-title>. <source>J Clin Microbiol</source>. <year>2007</year>;<volume>45</volume>(<issue>2</issue>):<fpage>351</fpage>&#x02013;<lpage>7</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jcm.01734-06\">10.1128/jcm.01734-06</ext-link></comment> .<pub-id pub-id-type=\"pmid\">17135444</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref025\"><label>25</label><mixed-citation publication-type=\"journal\"><name><surname>Le Roux</surname><given-names>CA</given-names></name>, <name><surname>Kubo</surname><given-names>T</given-names></name>, <name><surname>Grobbelaar</surname><given-names>AA</given-names></name>, <name><surname>van Vuren</surname><given-names>PJ</given-names></name>, <name><surname>Weyer</surname><given-names>J</given-names></name>, <name><surname>Nel</surname><given-names>LH</given-names></name>, <etal>et al</etal>\\n<article-title>Development and evaluation of a real-time reverse transcription-loop-mediated isothermal amplification assay for rapid detection of Rift Valley fever virus in clinical specimens</article-title>. <source>J Clin Microbiol</source>. <year>2009</year>;<volume>47</volume>(<issue>3</issue>):<fpage>645</fpage>&#x02013;<lpage>51</lpage>. Epub 2008/12/26. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jcm.01412-08\">10.1128/jcm.01412-08</ext-link></comment> ; PubMed Central PMCID: PMCPmc2650915.<pub-id pub-id-type=\"pmid\">19109471</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref026\"><label>26</label><mixed-citation publication-type=\"journal\"><name><surname>Peyrefitte</surname><given-names>CN</given-names></name>, <name><surname>Boubis</surname><given-names>L</given-names></name>, <name><surname>Coudrier</surname><given-names>D</given-names></name>, <name><surname>Bouloy</surname><given-names>M</given-names></name>, <name><surname>Grandadam</surname><given-names>M</given-names></name>, <name><surname>Tolou</surname><given-names>HJ</given-names></name>, <etal>et al</etal>\\n<article-title>Real-time reverse-transcription loop-mediated isothermal amplification for rapid detection of Rift Valley fever virus</article-title>. <source>J Clin Microbiol</source>. <year>2008</year>;<volume>46</volume>(<issue>11</issue>):<fpage>3653</fpage>&#x02013;<lpage>9</lpage>. Epub 2008/09/19. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jcm.01188-08\">10.1128/jcm.01188-08</ext-link></comment> ; PubMed Central PMCID: PMCPmc2576582.<pub-id pub-id-type=\"pmid\">18799705</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref027\"><label>27</label><mixed-citation publication-type=\"journal\"><name><surname>Kwallah</surname><given-names>A</given-names></name>, <name><surname>Inoue</surname><given-names>S</given-names></name>, <name><surname>Muigai</surname><given-names>AW</given-names></name>, <name><surname>Kubo</surname><given-names>T</given-names></name>, <name><surname>Sang</surname><given-names>R</given-names></name>, <name><surname>Morita</surname><given-names>K</given-names></name>, <etal>et al</etal>\\n<article-title>A real-time reverse transcription loop-mediated isothermal amplification assay for the rapid detection of yellow fever virus</article-title>. <source>J Virol Meth</source>. <year>2013</year>;<volume>193</volume>(<issue>1</issue>):<fpage>23</fpage>&#x02013;<lpage>7</lpage>. Epub 2013/05/23. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1016/j.jviromet.2013.05.004\">10.1016/j.jviromet.2013.05.004</ext-link></comment> .<pub-id pub-id-type=\"pmid\">23692685</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref028\"><label>28</label><mixed-citation publication-type=\"journal\"><name><surname>Li</surname><given-names>S</given-names></name>, <name><surname>Fang</surname><given-names>M</given-names></name>, <name><surname>Zhou</surname><given-names>B</given-names></name>, <name><surname>Ni</surname><given-names>H</given-names></name>, <name><surname>Shen</surname><given-names>Q</given-names></name>, <name><surname>Zhang</surname><given-names>H</given-names></name>, <etal>et al</etal>\\n<article-title>Simultaneous detection and differentiation of dengue virus serotypes 1&#x02013;4, Japanese encephalitis virus, and West Nile virus by a combined reverse-transcription loop-mediated isothermal amplification assay</article-title>. <source>Virol J</source>. <year>2011</year>;<volume>8</volume>:<fpage>360</fpage> Epub 2011/07/23. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1186/1743-422x-8-360\">10.1186/1743-422x-8-360</ext-link></comment> ; PubMed Central PMCID: PMCPmc3149006.<pub-id pub-id-type=\"pmid\">21777455</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref029\"><label>29</label><mixed-citation publication-type=\"journal\"><name><surname>Kramer</surname><given-names>LD</given-names></name>, <name><surname>Chandler</surname><given-names>LJ</given-names></name>. <article-title>Phylogenetic analysis of the envelope gene of St. Louis encephalitis virus</article-title>. <source>Arch Virol</source>. <year>2001</year>;<volume>146</volume>(<issue>12</issue>):<fpage>2341</fpage>&#x02013;<lpage>55</lpage>. Epub 2002/01/29. .<pub-id pub-id-type=\"pmid\">11811684</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref030\"><label>30</label><mixed-citation publication-type=\"journal\"><name><surname>Kopp</surname><given-names>A</given-names></name>, <name><surname>Gillespie</surname><given-names>TR</given-names></name>, <name><surname>Hobelsberger</surname><given-names>D</given-names></name>, <name><surname>Estrada</surname><given-names>A</given-names></name>, <name><surname>Harper</surname><given-names>JM</given-names></name>, <name><surname>Miller</surname><given-names>RA</given-names></name>, <etal>et al</etal>\\n<article-title>Provenance and geographic spread of St. Louis encephalitis virus</article-title>. <source>mBio</source>. <year>2013</year>;<volume>4</volume>(<issue>3</issue>):<fpage>e00322</fpage>&#x02013;<lpage>13</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/mBio.00322-13\">10.1128/mBio.00322-13</ext-link></comment> .<pub-id pub-id-type=\"pmid\">23760463</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref031\"><label>31</label><mixed-citation publication-type=\"journal\"><name><surname>Sun</surname><given-names>B</given-names></name>, <name><surname>Shen</surname><given-names>F</given-names></name>, <name><surname>McCalla</surname><given-names>SE</given-names></name>, <name><surname>Kreutz</surname><given-names>JE</given-names></name>, <name><surname>Karymov</surname><given-names>MA</given-names></name>, <name><surname>Ismagilov</surname><given-names>RF</given-names></name>. <article-title>Mechanistic evaluation of the pros and cons of digital RT-LAMP for HIV-1 viral load quantification on a microfluidic device and improved efficiency via a two-step digital protocol</article-title>. <source>Anal Chem</source>. <year>2013</year>;<volume>85</volume>(<issue>3</issue>):<fpage>1540</fpage>&#x02013;<lpage>6</lpage>. Epub 2013/01/18. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1021/ac3037206\">10.1021/ac3037206</ext-link></comment> ; PubMed Central PMCID: PMCPmc3578705.<pub-id pub-id-type=\"pmid\">23324061</pub-id></mixed-citation></ref></ref-list></back></article>'}"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "x = articles.find_one({'_id': 'test'})\n",
    "x"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "keyword_annotator = KeywordAnnotator(keywords=get_annotation_keywords())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "write_article_meta_to_mongo(x, articles)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'_id': 'test',\n",
       " 'annotations': {'disease-ontology-keywords': [{'keyword': 'Western equine encephalitis',\n",
       "    'uri': 'http://purl.obolibrary.org/obo/DOID_10843'},\n",
       "   {'keyword': 'St. Louis encephalitis',\n",
       "    'uri': 'http://purl.obolibrary.org/obo/DOID_10845'},\n",
       "   {'keyword': 'Japanese encephalitis',\n",
       "    'uri': 'http://purl.obolibrary.org/obo/DOID_10844'},\n",
       "   {'keyword': 'chikungunya',\n",
       "    'uri': 'http://purl.obolibrary.org/obo/DOID_0050012'},\n",
       "   {'keyword': 'Rift Valley fever',\n",
       "    'uri': 'http://purl.obolibrary.org/obo/DOID_1328'},\n",
       "   {'keyword': 'yellow fever',\n",
       "    'uri': 'http://purl.obolibrary.org/obo/DOID_9682'},\n",
       "   {'keyword': 'St. Louis encephalitis',\n",
       "    'uri': 'http://purl.obolibrary.org/obo/DOID_10845'},\n",
       "   {'keyword': 'Western equine encephalitis',\n",
       "    'uri': 'http://purl.obolibrary.org/obo/DOID_10843'},\n",
       "   {'keyword': 'Venezuelan equine encephalitis',\n",
       "    'uri': 'http://purl.obolibrary.org/obo/DOID_9584'},\n",
       "   {'keyword': 'Eastern equine encephalitis',\n",
       "    'uri': 'http://purl.obolibrary.org/obo/DOID_10841'},\n",
       "   {'keyword': 'Eastern equine encephalitis',\n",
       "    'uri': 'http://purl.obolibrary.org/obo/DOID_10841'},\n",
       "   {'keyword': 'Eastern equine encephalitis',\n",
       "    'uri': 'http://purl.obolibrary.org/obo/DOID_10841'},\n",
       "   {'keyword': 'Venezuelan equine encephalitis',\n",
       "    'uri': 'http://purl.obolibrary.org/obo/DOID_9584'},\n",
       "   {'keyword': 'Eastern equine encephalitis',\n",
       "    'uri': 'http://purl.obolibrary.org/obo/DOID_10841'},\n",
       "   {'keyword': 'Western equine encephalitis',\n",
       "    'uri': 'http://purl.obolibrary.org/obo/DOID_10843'},\n",
       "   {'keyword': 'St. Louis encephalitis',\n",
       "    'uri': 'http://purl.obolibrary.org/obo/DOID_10845'}]},\n",
       " 'meta': {'article-ids': {'doi': '10.1371/journal.pone.0147962',\n",
       "   'pmc': '4726549',\n",
       "   'pmid': '26807734',\n",
       "   'publisher-id': 'PONE-D-15-43543'},\n",
       "  'article-type': 'research-article',\n",
       "  'keywords': []},\n",
       " 'nxml': '<!DOCTYPE article PUBLIC \"-//NLM//DTD JATS (Z39.96) Journal Archiving and Interchange DTD v1.0 20120330//EN\" \"JATS-archivearticle1.dtd\">\\n<article xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" article-type=\"research-article\"><?properties open_access?><front><journal-meta><journal-id journal-id-type=\"nlm-ta\">PLoS One</journal-id><journal-id journal-id-type=\"iso-abbrev\">PLoS ONE</journal-id><journal-id journal-id-type=\"publisher-id\">plos</journal-id><journal-id journal-id-type=\"pmc\">plosone</journal-id><journal-title-group><journal-title>PLoS ONE</journal-title></journal-title-group><issn pub-type=\"epub\">1932-6203</issn><publisher><publisher-name>Public Library of Science</publisher-name><publisher-loc>San Francisco, CA USA</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"pmid\">26807734</article-id><article-id pub-id-type=\"pmc\">4726549</article-id><article-id pub-id-type=\"doi\">10.1371/journal.pone.0147962</article-id><article-id pub-id-type=\"publisher-id\">PONE-D-15-43543</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Research Article</subject></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>RNA viruses</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>West Nile virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Microbiology</subject><subj-group><subject>Medical microbiology</subject><subj-group><subject>Microbial pathogens</subject><subj-group><subject>Viral pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>West Nile virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Medicine and health sciences</subject><subj-group><subject>Pathology and laboratory medicine</subject><subj-group><subject>Pathogens</subject><subj-group><subject>Microbial pathogens</subject><subj-group><subject>Viral pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>West Nile virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>Viral pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>West Nile virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>RNA viruses</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Western Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Microbiology</subject><subj-group><subject>Medical Microbiology</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Western Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Medicine and Health Sciences</subject><subj-group><subject>Pathology and Laboratory Medicine</subject><subj-group><subject>Pathogens</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Western Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Western Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Research and analysis methods</subject><subj-group><subject>Extraction techniques</subject><subj-group><subject>RNA extraction</subject></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>RNA viruses</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Eastern Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Microbiology</subject><subj-group><subject>Medical Microbiology</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Eastern Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Medicine and Health Sciences</subject><subj-group><subject>Pathology and Laboratory Medicine</subject><subj-group><subject>Pathogens</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Eastern Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Togaviruses</subject><subj-group><subject>Alphaviruses</subject><subj-group><subject>Eastern Equine Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>RNA viruses</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>Japanese Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Microbiology</subject><subj-group><subject>Medical Microbiology</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>Japanese Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Medicine and Health Sciences</subject><subj-group><subject>Pathology and Laboratory Medicine</subject><subj-group><subject>Pathogens</subject><subj-group><subject>Microbial Pathogens</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>Japanese Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>Viral Pathogens</subject><subj-group><subject>Flaviviruses</subject><subj-group><subject>Japanese Encephalitis Virus</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Molecular Biology</subject><subj-group><subject>Molecular Biology Techniques</subject><subj-group><subject>Artificial Gene Amplification and Extension</subject><subj-group><subject>Polymerase Chain Reaction</subject></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Research and Analysis Methods</subject><subj-group><subject>Molecular Biology Techniques</subject><subj-group><subject>Artificial Gene Amplification and Extension</subject><subj-group><subject>Polymerase Chain Reaction</subject></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and Life Sciences</subject><subj-group><subject>Molecular Biology</subject><subj-group><subject>Molecular Biology Techniques</subject><subj-group><subject>Sequencing Techniques</subject><subj-group><subject>Sequence Analysis</subject><subj-group><subject>Sequence Alignment</subject></subj-group></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Research and Analysis Methods</subject><subj-group><subject>Molecular Biology Techniques</subject><subj-group><subject>Sequencing Techniques</subject><subj-group><subject>Sequence Analysis</subject><subj-group><subject>Sequence Alignment</subject></subj-group></subj-group></subj-group></subj-group></subj-group><subj-group subj-group-type=\"Discipline-v3\"><subject>Biology and life sciences</subject><subj-group><subject>Organisms</subject><subj-group><subject>Viruses</subject><subj-group><subject>RNA viruses</subject></subj-group></subj-group></subj-group></subj-group></article-categories><title-group><article-title>Surveillance for Western Equine Encephalitis, St. Louis Encephalitis, and West Nile Viruses Using Reverse Transcription Loop-Mediated Isothermal Amplification</article-title><alt-title alt-title-type=\"running-head\">RT-LAMP Surveillance for Western Equine Encephalitis, St. Louis Encephalitis, and West Nile Viruses</alt-title></title-group><contrib-group><contrib contrib-type=\"author\"><name><surname>Wheeler</surname><given-names>Sarah S.</given-names></name><xref ref-type=\"aff\" rid=\"aff001\"><sup>1</sup></xref><xref ref-type=\"author-notes\" rid=\"currentaff001\"><sup>&#x000a4;a</sup></xref></contrib><contrib contrib-type=\"author\"><name><surname>Ball</surname><given-names>Cameron S.</given-names></name><xref ref-type=\"aff\" rid=\"aff002\"><sup>2</sup></xref></contrib><contrib contrib-type=\"author\"><name><surname>Langevin</surname><given-names>Stanley A.</given-names></name><xref ref-type=\"aff\" rid=\"aff002\"><sup>2</sup></xref><xref ref-type=\"author-notes\" rid=\"currentaff002\"><sup>&#x000a4;b</sup></xref></contrib><contrib contrib-type=\"author\"><name><surname>Fang</surname><given-names>Ying</given-names></name><xref ref-type=\"aff\" rid=\"aff001\"><sup>1</sup></xref></contrib><contrib contrib-type=\"author\"><name><surname>Coffey</surname><given-names>Lark L.</given-names></name><xref ref-type=\"aff\" rid=\"aff001\"><sup>1</sup></xref></contrib><contrib contrib-type=\"author\"><name><surname>Meagher</surname><given-names>Robert J.</given-names></name><xref ref-type=\"aff\" rid=\"aff002\"><sup>2</sup></xref><xref ref-type=\"corresp\" rid=\"cor001\">*</xref></contrib></contrib-group><aff id=\"aff001\"><label>1</label><addr-line>University of California Davis, School of Veterinary Medicine, Department of Pathology, Microbiology and Immunology, Davis, California, United States of America</addr-line></aff><aff id=\"aff002\"><label>2</label><addr-line>Sandia National Laboratories, Biotechnology and Bioengineering Department, Livermore, California, United States of America</addr-line></aff><contrib-group><contrib contrib-type=\"editor\"><name><surname>Vasilakis</surname><given-names>Nikos</given-names></name><role>Editor</role><xref ref-type=\"aff\" rid=\"edit1\"/></contrib></contrib-group><aff id=\"edit1\"><addr-line>University of Texas Medical Branch, UNITED STATES</addr-line></aff><author-notes><fn fn-type=\"conflict\" id=\"coi001\"><p><bold>Competing Interests: </bold>The authors have read the journal\\'s policy and have the following competing interest: Authors CSB, RJM, and SAL are current or former employees of Sandia National Laboratories. The specific roles of these authors are articulated in the Author Contributions section. Sandia National Laboratories did not have any additional role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript. The authors declare no competing interests related to their employment with Sandia National Laboratories. This did not alter the authors\\' adherence to PLOS ONE policies on sharing data and materials.</p></fn><fn fn-type=\"con\" id=\"contrib001\"><p>Conceived and designed the experiments: SSW CSB SAL LLC RJM. Performed the experiments: SSW CSB YF LLC RJM. Analyzed the data: SSW CSB LLC RJM. Contributed reagents/materials/analysis tools: SSW YF LLC RJM. Wrote the paper: SSW CSB SAL LLC RJM.</p></fn><fn fn-type=\"current-aff\" id=\"currentaff001\"><label>&#x000a4;a</label><p>Current address: Sacramento-Yolo Mosquito and Vector Control District, Elk Grove, California, United States of America</p></fn><fn fn-type=\"current-aff\" id=\"currentaff002\"><label>&#x000a4;b</label><p>Current address: University of Washington, Department of Microbiology, Seattle, Washington, United States of America</p></fn><corresp id=\"cor001\">* E-mail: <email>rmeaghe@sandia.gov</email></corresp></author-notes><pub-date pub-type=\"epub\"><day>25</day><month>1</month><year>2016</year></pub-date><pub-date pub-type=\"collection\"><year>2016</year></pub-date><volume>11</volume><issue>1</issue><elocation-id>e0147962</elocation-id><history><date date-type=\"received\"><day>2</day><month>10</month><year>2015</year></date><date date-type=\"accepted\"><day>11</day><month>1</month><year>2016</year></date></history><permissions><copyright-statement>&#x000a9; 2016 Wheeler et al</copyright-statement><copyright-year>2016</copyright-year><copyright-holder>Wheeler et al</copyright-holder><license xlink:href=\"http://creativecommons.org/licenses/by/4.0/\"><license-p>This is an open access article distributed under the terms of the <ext-link ext-link-type=\"uri\" xlink:href=\"http://creativecommons.org/licenses/by/4.0/\">Creative Commons Attribution License</ext-link>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</license-p></license></permissions><self-uri content-type=\"pdf\" xlink:type=\"simple\" xlink:href=\"pone.0147962.pdf\"/><abstract><p>Collection of mosquitoes and testing for vector-borne viruses is a key surveillance activity that directly influences the vector control efforts of public health agencies, including determining when and where to apply insecticides. Vector control districts in California routinely monitor for three human pathogenic viruses including West Nile virus (WNV), Western equine encephalitis virus (WEEV), and St. Louis encephalitis virus (SLEV). Reverse transcription quantitative polymerase chain reaction (RT-qPCR) offers highly sensitive and specific detection of these three viruses in a single multiplex reaction, but this technique requires costly, specialized equipment that is generally only available in centralized public health laboratories. We report the use of reverse transcription loop-mediated isothermal amplification (RT-LAMP) to detect WNV, WEEV, and SLEV RNA extracted from pooled mosquito samples collected in California, including novel primer sets for specific detection of WEEV and SLEV, targeting the nonstructural protein 4 (nsP4) gene of WEEV and the 3&#x02019; untranslated region (3&#x02019;-UTR) of SLEV. Our WEEV and SLEV RT-LAMP primers allowed detection of &#x0003c;0.1 PFU/reaction of their respective targets in &#x0003c;30 minutes, and exhibited high specificity without cross reactivity when tested against a panel of alphaviruses and flaviviruses. Furthermore, the SLEV primers do not cross-react with WNV, despite both viruses being closely related members of the Japanese encephalitis virus complex. The SLEV and WEEV primers can also be combined in a single RT-LAMP reaction, with discrimination between amplicons by melt curve analysis. Although RT-qPCR is approximately one order of magnitude more sensitive than RT-LAMP for all three targets, the RT-LAMP technique is less instrumentally intensive than RT-qPCR and provides a more cost-effective method of vector-borne virus surveillance.</p></abstract><funding-group><funding-statement>This work was supported by a grant from the Defense Threat Reduction Agency (DTRA), contract DTRA1002710697. Sandia National Laboratories is a multi-program laboratory managed and operated by Sandia Corporation, a wholly owned subsidiary of Lockheed Martin Corporation, for the U.S. Department of Energy&#x02019;s National Nuclear Security Administration under Contract DE-AC04-94AL85000.</funding-statement></funding-group><counts><fig-count count=\"5\"/><table-count count=\"3\"/><page-count count=\"17\"/></counts><custom-meta-group><custom-meta id=\"data-availability\"><meta-name>Data Availability</meta-name><meta-value>All relevant data are within the paper and its Supporting Information files.</meta-value></custom-meta></custom-meta-group></article-meta><notes><title>Data Availability</title><p>All relevant data are within the paper and its Supporting Information files.</p></notes></front><body><sec sec-type=\"intro\" id=\"sec001\"><title>Introduction</title><p>Testing mosquito vectors for human pathogenic viruses like West Nile virus is a central feature of successful surveillance approaches and an early predictor of human epidemics. Most surveillance in the United States and increasingly in other regions uses molecular detection of viral RNAs, usually by reverse transcription polymerase chain reaction (RT-PCR) or quantitative real-time RT-PCR (RT-qPCR). The presence of vector-borne viruses in mosquitoes, together with other measures of activity including mosquito distribution and abundance, human cases, dead bird surveillance, meteorological parameters and (for some viruses) sentinel chicken seroconversions, are used to make vector control decisions including when and where to apply insecticides. Three medically important vector-borne viruses, West Nile virus (WNV), Western equine encephalitis virus (WEEV), and St. Louis encephalitis virus (SLEV) are endemic in California and cause sporadic seasonal outbreaks, primarily during the summer months.</p><p>WEEV incidence has declined in the last few decades [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>] and has not been detected in California since 2003. SLEV was detected in field-caught mosquitoes in California in 2015, for the first time since the arrival of WNV in 2003 [<xref rid=\"pone.0147962.ref002\" ref-type=\"bibr\">2</xref>, <xref rid=\"pone.0147962.ref003\" ref-type=\"bibr\">3</xref>]. SLEV also still occurs in Mexico and South America [<xref rid=\"pone.0147962.ref004\" ref-type=\"bibr\">4</xref>]. Despite the much higher prevalence of WNV in California since 2003, SLEV and WEEV are still included in routine mosquito surveillance since vector-borne viruses can exist in cryptic and sporadic transmission cycles without the detection of human or equine disease, as exemplified by the re-emergence of SLEV in 2015.</p><p>Vector-borne disease surveillance programs primarily rely on techniques such as virus isolation, standard RT-PCR, and RT-qPCR to detect the presence of vector-borne viruses in targeted mosquito vectors. While these methodologies are the gold standard for arbovirus disease surveillance, they require well-equipped laboratories and well-trained technicians, both of which are severely lacking in many low resource settings with high vector-borne disease burden. RT-qPCR is the most sensitive assay for detecting vector-borne viruses, but the technique generally requires expensive equipment, and high quality RNA purified from the mosquito samples. Loop-mediated isothermal amplification (LAMP) is an isothermal nucleic acid amplification technique that is a useful alternative to PCR for pathogen detection and diagnostics [<xref rid=\"pone.0147962.ref005\" ref-type=\"bibr\">5</xref>&#x02013;<xref rid=\"pone.0147962.ref008\" ref-type=\"bibr\">8</xref>]. When coupled with reverse transcription, the LAMP methodology can also be utilized for detecting RNA targets [<xref rid=\"pone.0147962.ref009\" ref-type=\"bibr\">9</xref>]. LAMP requires 4 or 6 primers that recognize 6 or 8 binding sites, providing a highly specific assay. A key advantage to the LAMP reaction is simplicity: the entire reaction (including reverse transcription) proceeds at a single temperature, eliminating the need for a thermal cycler. Reactions can be monitored through a variety of outputs including generation of turbidity [<xref rid=\"pone.0147962.ref010\" ref-type=\"bibr\">10</xref>], fluorescence indicators [<xref rid=\"pone.0147962.ref011\" ref-type=\"bibr\">11</xref>, <xref rid=\"pone.0147962.ref012\" ref-type=\"bibr\">12</xref>], or color change [<xref rid=\"pone.0147962.ref013\" ref-type=\"bibr\">13</xref>]. The technique is also robust for crude specimens, allowing successful amplification from minimally processed samples such as heat-treated blood [<xref rid=\"pone.0147962.ref012\" ref-type=\"bibr\">12</xref>, <xref rid=\"pone.0147962.ref014\" ref-type=\"bibr\">14</xref>&#x02013;<xref rid=\"pone.0147962.ref016\" ref-type=\"bibr\">16</xref>, <xref rid=\"pone.0147962.ref017\" ref-type=\"bibr\">17</xref>]. As such, LAMP can often be performed without DNA or RNA extraction. These advantages lessen the dependence upon highly skilled labor and well-equipped laboratories, which are prerequisites for RT-qPCR.</p><p>In this report, we describe use of RT-LAMP to detect WNV, WEEV, and SLEV, and apply the technique to RNA extracted from <italic>Culex</italic> mosquitoes collected in California. RT-LAMP was previously demonstrated for detection of WNV by Parida <italic>et al</italic> [<xref rid=\"pone.0147962.ref018\" ref-type=\"bibr\">18</xref>], and detection kits for this virus based on RT-LAMP are commercially available from several vendors. RT-LAMP primer sets have also been reported for numerous other important mosquito-borne viruses such as Dengue virus (DENV) [<xref rid=\"pone.0147962.ref019\" ref-type=\"bibr\">19</xref>, <xref rid=\"pone.0147962.ref020\" ref-type=\"bibr\">20</xref>], Japanese encephalitis virus (JEV) [<xref rid=\"pone.0147962.ref021\" ref-type=\"bibr\">21</xref>&#x02013;<xref rid=\"pone.0147962.ref023\" ref-type=\"bibr\">23</xref>], chikungunya virus (CHIKV) [<xref rid=\"pone.0147962.ref024\" ref-type=\"bibr\">24</xref>], Rift Valley fever virus (RVFV) [<xref rid=\"pone.0147962.ref025\" ref-type=\"bibr\">25</xref>, <xref rid=\"pone.0147962.ref026\" ref-type=\"bibr\">26</xref>], and yellow fever virus (YFV) [<xref rid=\"pone.0147962.ref027\" ref-type=\"bibr\">27</xref>]. Because RT-LAMP detection of SLEV and WEEV was not previously reported, RT-LAMP primer sets for these viruses were developed and tested for sensitivity and specificity. At the time of method development neither WEEV nor SLEV had been recently detected in California mosquito pools, and thus archived samples were used for RT-LAMP evaluation. WNV-positive mosquito pools collected in 2014 were used to verify the lack of cross-reactivity with new SLEV and WEEV primers. These WNV positive pools were also used to demonstrate the applicability of RT-LAMP for parallel detection of WNV, WEEV and SLEV, analogous to the RT-LAMP &#x0201c;detection array&#x0201d; previously reported for parallel detection of DENV serotypes 1&#x02013;4, JEV, and WNV [<xref rid=\"pone.0147962.ref028\" ref-type=\"bibr\">28</xref>]. In addition, we demonstrate duplex detection of WEEV and SLEV in a single tube using melt curve analysis.</p></sec><sec sec-type=\"materials|methods\" id=\"sec002\"><title>Materials and Methods</title><sec id=\"sec003\"><title>RT-LAMP primer design</title><p>Whole-genome sequence alignments for WEEV and SLEV were performed using ClustalX2. For WEEV, six North American strains were selected for which full-length or nearly full-length sequences were available in Genbank as of April, 2014 (a large number of additional WEEV genomes were deposited in Genbank after primers were designed [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>]). For SLEV, 6 strains were selected for which full-length or nearly full-length sequences were available, representing primarily North American strains from Clade II in the tree developed by Kramer and Chandler [<xref rid=\"pone.0147962.ref029\" ref-type=\"bibr\">29</xref>]. Accession numbers of WEEV and SLEV strains used for primer design are presented in Figs <xref ref-type=\"fig\" rid=\"pone.0147962.g001\">1</xref> and <xref ref-type=\"fig\" rid=\"pone.0147962.g002\">2</xref>.</p><fig id=\"pone.0147962.g001\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.g001</object-id><label>Fig 1</label><caption><title>Sequence alignment showing priming regions for the WEEV nsP4 primer set.</title><p>The arrow indicates the location of the <italic>Xba</italic>I restriction site. Genbank accession numbers refer to the following strains: GQ287644.1 = BFS2005; GQ287643.1 = Montana-64; GQ287645.1 = 71V1658; GQ287642.1 = Kern; GQ287641.1 = Imperial; GQ287647.1 = 85&#x02013;452NM.</p></caption><graphic xlink:href=\"pone.0147962.g001\"/></fig><fig id=\"pone.0147962.g002\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.g002</object-id><label>Fig 2</label><caption><title>Sequence alignment showing priming regions for the SLEV 3&#x02019; UTR primer set.</title><p>The arrow indicates the location of the <italic>Hin</italic>dIII restriction site.Genbank accession numbers refer to the following strains: DQ525916.1 = Kern 217; EU566860.1 = Hubbard; FJ753286.2 = CbaAr-4005; JF460774.1 = Imperial Valley; JQ957868.1 = Palenque-C475; JQ957869.1 = Palenque-A770.</p></caption><graphic xlink:href=\"pone.0147962.g002\"/></fig><p>Conserved regions of the viral genomes from the sequence alignments were identified. LAMP Designer v1.13 software (Premier Biosoft) was used to scan conserved regions for suitable LAMP primer sets using a single representative strain. Primer set candidates were then screened against the full set of aligned sequences to test for perfect or nearly perfect matches. Primer sets meeting this criterion were then analyzed by BLAST, first comparing to all WEEV or SLEV sequences (taxid 11039 and 11080, respectively) and then comparing to all viral sequences in Genbank, to determine likelihood of cross-reactivity with other viruses. Primer sets were also evaluated for hairpin formation, self-dimerization, and cross-dimerization.</p></sec><sec id=\"sec004\"><title>Viral culture</title><p>Selected viral strains used for sensitivity and specificity testing are listed in <xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>. These strains were pulled from a library of isolates archived at -80&#x000b0;C. From each isolate RNA was extracted and plaque assays were performed to obtain viral titers. RNA was extracted using a MagMax&#x02122; magnetic particle processor (Life Technologies, Grand Island, NY, USA), MagMAX&#x02122;-96 Viral RNA Isolation Kit (Life Technologies, Grand Island, NY, USA), and manufacturer provided protocols. Plaque assays were performed using Vero cell cultures grown to confluence in six well plates, cultured with Dulbecco&#x02019;s Modified Eagle Medium (DMEM) supplemented with 10% fetal bovine serum (FBS) and 500 U/mL penicillin and 0.5 mg/mL streptomycin, and maintained at 37&#x000b0;C and 5% CO<sub>2</sub>. Virus isolates were diluted in a 10-fold serial dilution, media was removed from each well of the 6-well plate and 200 uL of diluted virus was allowed to incubate on the monolayer for 60 min at 37&#x000b0;C. A double overlay system was used where the first overlay contained nutritive media and 5% agarose. The second overlay was applied and additionally contained 0.005% neutral red, the optimal time to count plaques occurred 1&#x02013;3 days after the second overlay.</p><table-wrap id=\"pone.0147962.t001\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.t001</object-id><label>Table 1</label><caption><title>Viral RNA used for testing sensitivity and specificity of RT-LAMP primers.</title></caption><alternatives><graphic id=\"pone.0147962.t001g\" xlink:href=\"pone.0147962.t001\"/><table frame=\"hsides\" rules=\"groups\"><colgroup span=\"1\"><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/></colgroup><thead><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\"/><th align=\"left\" rowspan=\"1\" colspan=\"1\"/><th align=\"center\" colspan=\"3\" rowspan=\"1\">RT-LAMP primer set (100 PFU target)</th></tr><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\">Virus</th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Strain or Isolate Designation</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019;-UTR</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WNV E gene</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WEEV nsP4</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>Flaviviruses</italic></td><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;St. Louis encephalitis virus (SLEV)<xref ref-type=\"table-fn\" rid=\"t001fn001\"><sup>1</sup></xref></td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Kern217</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">Bfs1750 (Ct 17.7)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">Ruls (Ct 12.9)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">Coav750 (Ct 23.9)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">69M1143 (Ct 13.7)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">BeAn246407 (Ct 20.4)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">CorAn9124 (PCR neg.)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">CorAn9275 (PCR neg.)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;<xref ref-type=\"table-fn\" rid=\"t001fn002\"><sup>2</sup></xref></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;West Nile virus (WNV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">L-CA-04 SAC-04-7168</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Yellow fever virus (YFV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">17D</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Rocio virus (ROCV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">SP H 34675</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Usutu virus (USV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">SA AR 1776</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Ilheus virus (ILHV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Ilheus B44532</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Dengue virus serotype 1 (DENV-1)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">BC-796</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Dengue virus serotype 2 (DENV-2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">BC-122-94</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Dengue virus serotype 3 (DENV-3)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">BC 156&#x02013;97</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>Alphaviruses</italic></td><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Western equine encephalitis virus (WEEV)<xref ref-type=\"table-fn\" rid=\"t001fn001\"><sup>1</sup></xref></td><td align=\"left\" rowspan=\"1\" colspan=\"1\">KERN 5547</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"/><td align=\"left\" rowspan=\"1\" colspan=\"1\">Lake43 (Ct 14.9)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">+</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Sindbis virus (SINV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">EDS-14</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Ross River virus (RRV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">SW 38457</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Chikungunya virus (CHIKV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Ross</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Venezuelan equine encephalitis virus (VEEV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">TC-83</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;<xref ref-type=\"table-fn\" rid=\"t001fn002\"><sup>2</sup></xref></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Barmah Forest virus (BFV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Barmah Forest (TVP-4119)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Highlands J virus (HJV)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">WC-431</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x000a0;&#x000a0;&#x000a0;&#x000a0;Eastern equine encephalitis virus (EEEV)<xref ref-type=\"table-fn\" rid=\"t001fn003\"><sup>3</sup></xref></td><td align=\"left\" rowspan=\"1\" colspan=\"1\">EEEV/X/USA/A15072/2003</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr></tbody></table></alternatives><table-wrap-foot><fn id=\"t001fn001\"><p><sup>1</sup> Isolatesof SLEV other than Kern 217, and WEEV isolate Lake43 were not quantitated by plaque assay. Extracted RNA was quantitated by RT-qPCR and used undiluted as a template in RT-LAMP. RT-qPCR Ct values are indicated. CorAn 9124 and CorAn9275 were not detected by RT-qPCR primers.</p></fn><fn id=\"t001fn002\"><p><sup>2</sup> A single reaction showed positive amplification within 45 minutes of incubation (1 of 11 replicates for SLEV CorAn 9275 with WNV primers; 1 of 18 replicates for VEEV TC-83 with SLEV primers). Melt curve analysis indicates that these are isolated occurrences of cross-contamination with positive-control RNA.</p></fn><fn id=\"t001fn003\"><p><sup>3</sup> Cross-reactivity testing for EEEV was performed with 10<sup>6</sup> copies of an <italic>in vitro</italic> transcribed fragment of the EEEV nsP4 gene corresponding to the WEEV RT-LAMP target site including an additional 46 nt upstream and 65 nt downstream of the outer primer (F3 and B3) binding sites. The <italic>in vitro</italic> transcribed RNA represents nucleotides 5980&#x02013;6329 (350 bases) of Genbank # KJ469613.1.</p></fn></table-wrap-foot></table-wrap><p>Due to select agent restrictions for work with Eastern equine encephalitis virus (EEEV), a segment of EEEV nsP4 RNA was purchased. The RNA was <italic>in vitro</italic> transcribed from a synthetic gene corresponding to nucleotides 5980&#x02013;6329 (350 bases) of Genbank accession number KJ469613.1 (Biosynthesis, Lewisville, TX, USA). The transcript was purified to remove traces of DNA, and verified DNA-free by PCR (performed by the supplier).</p></sec><sec id=\"sec005\"><title>Mosquito pools</title><p>Mosquito pools were collected as part of routine mosquito surveillance performed by California vector control agencies. RNA from selected pools was obtained through the UC Davis Center for Vectorborne Disease Surveillance Labortory (CVEC) with the permission of the collecting agency. Pools comprised up to 50 conspecific, female, field-caught mosquitoes. RNA was extracted by MagMax<sup>TM</sup> as described above.</p></sec><sec id=\"sec006\"><title>RT-qPCR</title><p>Detection of viral RNA in mosquito pools was performed at CVEC using a multiplex RT-qPCR for WNV, SLEV, and WEEV. This assay was performed using the primers and probes listed in <xref ref-type=\"supplementary-material\" rid=\"pone.0147962.s003\">S1 Table</xref>, except the probe for SLEV had a Quasar 670 reporter and Black Hole 2 Quencher (BHQ-2) and the probe for WEEV had a TAMRA reporter and BHQ-2. Reactions were completed on a ViiA 7 Real-Time PCR system (Life Technologies, Grand Island, NY, USA) using SensiFAST Probe Lo-ROX One-step Kits (Bioline, Taunton, MA, USA) and manufacturer recommended protocols. Additional RT-qPCR using the WNV, SLEV, and WEEV primer and probe sets (<xref ref-type=\"supplementary-material\" rid=\"pone.0147962.s003\">S1 Table</xref>) were performed on the RNA from selected virus isolates (<xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>) as part of the sensitivity and specificity testing at Sandia. These reactions were performed using a one-step RT-PCR kit (iTaq Universal Probes One-Step Kit, BioRad, Hercules, CA) and BioRad CFX96 instrument, with detection in the FAM channel. Total reaction volume was 10 &#x003bc;L and contained 500nM of each forward and reverse primer and 200nM of the probe. PCR was performed with an initial reverse transcription step at 50&#x000b0;C for 10 minutes, followed by an initial denaturation for 50 seconds, then 40 cycles of 95&#x000b0;C for 15 seconds and 60&#x000b0;C for 30 seconds followed by a plate read.</p></sec><sec id=\"sec007\"><title>RT-LAMP</title><p>RT-LAMP was performed in 10 &#x003bc;L reaction volumes in thin-walled PCR strip tubes or 96-well plates. The reaction mixture had a final composition (after adding water or template) of 1X Isothermal Amplification Buffer (New England Biolabs, NEB #B0537S) supplemented with an additional 6 mM MgSO4 (NEB #B1003S, final 8 mM MgSO<sub>4</sub>), 0.8 M Betaine (Sigma #B-0300), 1.4 mM each dNTP (NEB #N0447L), 0.32 units/&#x003bc;L Bst 2.0 WarmStart DNA polymerase (NEB #M0538M), 0.2 units/&#x003bc;L AMV reverse transcriptase (NEB #M0277T, or Life Science Advanced Technologies #AMVRTT-5), and 2 &#x003bc;M (or in some instances 4 &#x003bc;M) SYTO 82 detection dye (Life Technologies #S11363) [<xref rid=\"pone.0147962.ref011\" ref-type=\"bibr\">11</xref>]. Primers were used in the amounts typically recommended for LAMP: 0.2 &#x003bc;M each for outer primers F3 and B3; 1.6 &#x003bc;M each for inner primers FIP and BIP; and 0.8 &#x003bc;M each for loop primers LF and LB. RT-LAMP with real-time fluorescence monitoring was carried out in a BioRad CFX96, using detection channel 2 (HEX) for monitoring SYTO 82 dye. Reactions were incubated at a constant temperature of 63&#x000b0;C for 50&#x02013;70 minutes, with plate read steps at intervals of 1 minute (in the BioRad CFX96, this is accomplished with a 48-second single-temperature cycle followed by a plate read which takes approximately 12 seconds in all-channel mode). Incubation was typically followed by inactivation of the enzyme at 95&#x000b0;C for 2 minutes, followed by cooling to 65&#x000b0;C for 2 minutes, and a melt curve from 65&#x02013;95&#x000b0;C in 0.2&#x000b0;C increments. Time-to-positivity values were determined using the BioRad CFX Manager software, using baseline-subtracted curves, and a single threshold value auto-calculated by the CFX manager.</p><p>To demonstrate testing with simpler instrumentation, we also performed reactions using a portable isothermal fluorimeter (Optigene Genie III). The protocol was identical to that described above, except that SYTO 9 (Life Technologies #S34854) was used in place of SYTO 82, to ensure compatibility with the Genie III detection optics. A fluorescence photograph of the reaction endpoint (with SYTO 9) was taken using a blue LED flashlight (CREE WF-502B) and an amber-colored plastic filter (LEE Filters #158). Background fluorescence with SYTO 9 was minimized by photographing the tubes upon a hot plate set to 63&#x000b0;C.</p><p>Gel electrophoretic analysis of RT-LAMP products was performed using an Invitrogen e-Gel (2% agarose with ethidium bromide); 1 &#x003bc;L of RT-LAMP product was diluted with DI water to a total volume of 20 &#x003bc;L that was loaded onto the e-Gel for nucleic acid separation. In addition, to confirm specificity of amplification, 3 &#x003bc;L aliquots of products from positive control reactions were digested with 20 units of <italic>Xba</italic>I (for WEEV nsP4 RT-LAMP), or <italic>Hind</italic>III (for SLEV 3&#x02019;-UTR RT-LAMP) for 1 hour at 37&#x000b0;C in 20 &#x003bc;L total reaction volume, using buffer supplied by the manufacturer (New England Biolabs). Digested products were also run on an e-Gel as described above.</p></sec><sec id=\"sec008\"><title>Sensitivity and Specificity Testing</title><p>SLEV, WEEV and WNV LAMP primers were tested for sensitivity and cross-reactivity against multiple conspecific strains and closely related viruses (<xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>). Sensitivity testing was performed with a dilution series of a single reference isolate of each virus: WEEV Kern 5547 (Genbank KJ554975.1), SLEV Kern 217 (Genbank DQ525916.1), and WNV L-CA-04 SAC-04-7168 (Genbank DQ080059.1). Conspecific strains for SLEV were chosen amongst the lineages described by Kramer et al. [<xref rid=\"pone.0147962.ref027\" ref-type=\"bibr\">27</xref>]. We tested isolates falling within Clades IA (Bfs1750), IB (Coav750), IIA(69M143), VA (BeAn246407), and VII (CorAn9124 and CorAn9275). One conspecific strain of WEEV (Lake 43, Genbank KJ554985) was also tested. All other WEEV isolates within the collection at UC Davis were identical to the Kern 5547 reference isolate within the primer binding regions. A panel of flaviviruses and alphaviruses (<xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>) was developed to test for cross-reactivity. Each primer set (SLEV, WEEV, and WNV) was tested against the entire panel, using 100 PFU equivalents of viral RNA template per reaction. Because of the select agent status of EEEV, primers were tested against an in vitro transcribed portion of nsP4 gene instead of genomic RNA (methods described above), with 10<sup>6</sup> copies of RNA template per reaction.</p></sec><sec id=\"sec009\"><title>Multiplexed RT-LAMP with melt curve analysis</title><p>WNV, SLEV 3&#x02019;-UTR, and WEEV-nsP4 primer sets were tested with mixtures of WNV L-CA-04 SAC-04-7168, SLEV Kern 217, and WEEV Kern 5547 RNA templates. Individual primer sets were combined in equal amounts to create four multiplexed reaction types: WNV + SLEV + WEEV, WNV + SLEV, WNV + WEEV, and SLEV + WEEV. Corresponding viral RNA templates were combined in equal parts and added to matching reaction mixes. Positive control reactions contained a total of 100 PFU equivalents of viral RNA template per reaction. Multiplexed reactions were performed in triplicate with the RT-LAMP protocol described above.</p></sec></sec><sec id=\"sec010\"><title>Results and Discussion</title><sec id=\"sec011\"><title>WEEV and SLEV primer design</title><p>Our primer design protocol identified two candidate primer sets for WEEV (targeting nsP1 and nsP4), and two candidate primer sets for SLEV (targeting the 3&#x02019;-UTR and capsid C protein). According to the sequence alignment the 3&#x02019;-UTR is well conserved among sequenced isolates of SLEV, and yet sufficiently distinct from other flaviviruses to allow specific detection of SLEV by RT-LAMP. The flavivirus 3&#x02019;-UTR was previously targeted by LAMP primer sets designed for dengue serotypes 1&#x02013;4 [<xref rid=\"pone.0147962.ref019\" ref-type=\"bibr\">19</xref>, <xref rid=\"pone.0147962.ref020\" ref-type=\"bibr\">20</xref>]. Similarly, the nsP4 (RNA polymerase) gene of WEEV was among the most highly conserved region of that genome. The region targeted here shared 81% sequence identity with North American isolates of Eastern equine encephalitis virus (EEEV), but was &#x0003c;80% identical to any other alphaviruses.</p><p>In initial screening the WEEV-nsP4 and SLEV 3&#x02019;-UTR sets provided faster amplification and lower detection limits than the WEEV-nsP1 and SLEV-C sets; therefore, only the nsP4 and 3&#x02019;-UTR sets were evaluated further. The WEEV nsP4 and SLEV 3&#x02019;-UTR primer sequences are shown in <xref ref-type=\"table\" rid=\"pone.0147962.t002\">Table 2</xref>, and sequence alignments showing the positioning of the primers are presented in Figs <xref ref-type=\"fig\" rid=\"pone.0147962.g001\">1</xref> and <xref ref-type=\"fig\" rid=\"pone.0147962.g002\">2</xref>. The less efficient WEEV nsP1 and SLEV C primer sequences are provided in <xref ref-type=\"supplementary-material\" rid=\"pone.0147962.s004\">S2 Table</xref></p><table-wrap id=\"pone.0147962.t002\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.t002</object-id><label>Table 2</label><caption><title>RT-LAMP Primers used herein.</title></caption><alternatives><graphic id=\"pone.0147962.t002g\" xlink:href=\"pone.0147962.t002\"/><table frame=\"hsides\" rules=\"groups\"><colgroup span=\"1\"><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/></colgroup><thead><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\">Primer Name</th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Genome position<xref ref-type=\"table-fn\" rid=\"t002fn001\"><sup>1</sup></xref></th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Sequence<xref ref-type=\"table-fn\" rid=\"t002fn002\"><sup>2</sup></xref></th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Source</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; F3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10497&#x02013;10516</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GACTGGGTTAWCAAAGCCAA</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">New</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; B3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10736&#x02013;10718</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GGTCTCCTCTAACCTCTAG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; FIP(<underline>F1c</underline> + F2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10641&#x02013;10622;10583&#x02013;10600</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>GCATGTACAGACAGCACCTT</underline> +GCACAAGTCAGACCAGAA</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; BIP<xref ref-type=\"table-fn\" rid=\"t002fn003\"><sup>3</sup></xref>(<underline>B1c</underline> + B2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10644&#x02013;10665;10717&#x02013;10700</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>CAGGAGGACTGGGTTAACAAAG</underline> +TCCTTACGCCATGGTCAC</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; LF</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10618&#x02013;10601</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">CATGCTTTCAGGTGGCAT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">SLEV 3&#x02019; LB</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">10681&#x02013;10688</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GGCCCAAACCATGGAGTG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 F3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">5949&#x02013;5958</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GTCAACGGTAATTAACAGGT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">New</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 B3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">6187&#x02013;6170</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GGACGGCTGATCTTATCT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 FIP(<underline>F1c</underline> + F2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">6087&#x02013;6070;6025&#x02013;6044</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>GCCGTCCACCATGTCAAG</underline> +CCTACAGTAGCCAGTTATTG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 BIP(<underline>B1c</underline> + B2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">6088&#x02013;6108;6156&#x02013;6138</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>GCATCGTGCTGTCTAGATACM</underline> +GCTATGCTTCTTTGGGTAG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 LF</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">6067&#x02013;6048</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">ACGCATCGTATTCATCTGTT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WEEV nsP4 LB</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">6116&#x02013;6134</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">TTTGTCCGGCTAAACTGAG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV F3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1028&#x02013;1046</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">TGGATTTGGTTCTCGAAGG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Parida [<xref rid=\"pone.0147962.ref018\" ref-type=\"bibr\">18</xref>]</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV B3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1228&#x02013;1210</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">GGTCAGCACGTTTGTCATT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV FIP(F<underline>1c</underline> + TTTT + F2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1121&#x02013;1100;1050&#x02013;1069</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>TTGGCCGCCTCCATATTCATCA</underline> + TTTT +CAGCTGCGTGACTATCATGT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV BIP(<underline>B1c</underline> + TTTT + B2)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1144&#x02013;1165;1208&#x02013;1190</td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><underline>TGCTATTTGGCTACCGTCAGCG</underline> + TTTT +TGAGCTTCTCCCATGGTCG</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV LF</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1093&#x02013;1075</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">CATCGATGGTAGGCTTGTC</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">WNV LB</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">1169&#x02013;1186</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">TCTCCACCAAAGCTGCGT</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"/></tr></tbody></table></alternatives><table-wrap-foot><fn id=\"t002fn001\"><p><sup>1</sup> Genome positions are based on SLEV strain Kern217 (Genbank DQ525916.1), WEEV strain Montana-64 (Genbank GQ287643.1), and (as per reference [<xref rid=\"pone.0147962.ref018\" ref-type=\"bibr\">18</xref>]) WNV strain NY99 (Genbank AF196835).</p></fn><fn id=\"t002fn002\"><p><sup>2</sup> Sequence for inner primers FIP and BIP is shown separating out the F1c or B1c region (underlined) followed by the F2 or B2 region (not underlined). The complete FIP primer sequence is obtained by concatenating the F1c and F2 sequences (FIP = F1c + F2), and likewise for BIP. Primers designed by Parida <italic>et al</italic> [<xref rid=\"pone.0147962.ref018\" ref-type=\"bibr\">18</xref>] also include a TTTT linker between the two segments of the inner primers.</p></fn><fn id=\"t002fn003\"><p><sup>3</sup> A second BIP primer with 3 single-base substitutions in the B1c region intended to cover additional SLEV strains was designed but not used for testing; the alternative sequence (BIP = <underline>B1c</underline> + B2) is <underline>CAGG<bold>C</bold>GGACTGGG<bold>T</bold>A<bold>A</bold>TCAAAG</underline> + TCCTTACGCCATGGTCAC. The alternative BIP sequence is a match for two Mexican isolates designated Palenque-C475 and Palenque-A770 (Genbank JQ957868.1 and JQ957869.1), which form a distinct phylogenetic clade [<xref rid=\"pone.0147962.ref030\" ref-type=\"bibr\">30</xref>].</p></fn></table-wrap-foot></table-wrap><p>Gel electrophoresis of the SLEV (3&#x02019;-UTR) and WEEV (nsP4) RT-LAMP reaction products resulted in a prototypical ladder-like banding pattern typical of LAMP reactions (<xref ref-type=\"fig\" rid=\"pone.0147962.g003\">Fig 3</xref>, lanes 1 and 5). An online tool for LAMP restriction digest fragment analysis (<ext-link ext-link-type=\"uri\" xlink:href=\"http://creisle.github.io/creisle.lamprflp/\">http://creisle.github.io/creisle.lamprflp/</ext-link>) was used to predict restriction digest banding patterns based on the LAMP amplicon structure proposed by Notomi <italic>et al</italic> [<xref rid=\"pone.0147962.ref006\" ref-type=\"bibr\">6</xref>]. A restriction digest of the WEEV nsP4 amplicon with <italic>XbaI</italic> was predicted to yield main bands at 67 and 115 bp, with fainter bands at 50, 63, 77, and 97 bp. This is consistent with the banding pattern in Lane 2, which has primary bands slightly larger than 50 and 100 bp. Digestion of the SLEV 3&#x02019;-UTR amplicon with <italic>Hin</italic>dIII was predicted to yield bright bands at 130, 185, and 240 bp, with fainter bands at 58, 62, 76, 80, 136, and 146 bp. This corresponds closely to the pattern of banding in Lane 6. Additional faint bands in these digests at higher molecular weight may be due to incomplete digestion, or the presence of other amplicon structures not proposed by Notomi <italic>et al</italic> [<xref rid=\"pone.0147962.ref005\" ref-type=\"bibr\">5</xref>, <xref rid=\"pone.0147962.ref006\" ref-type=\"bibr\">6</xref>].</p><fig id=\"pone.0147962.g003\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.g003</object-id><label>Fig 3</label><caption><title>Electrophoretic analysis of RT-LAMP amplification products with WEEV nsP4 and SLEV 3&#x02019; UTR primer sets.</title><p>The gel is a 2% agarose E-gel (Invitrogen) with ethidium bromide. The marker (&#x0201c;M&#x0201d;) is a Low Molecular Weight DNA Ladder (New England Biolabs). Lane 1: WEEV positive control reaction with 2 PFU equivalent WEEV RNA in 20 &#x003bc;L. Lane 2: WEEV positive control after digestion with <italic>Xba</italic>I. Lane 3: No-template control reaction with WEEV nsP4 primer set. Lane 4: no sample. Lane 5: SLEV positive control reaction with 2 PFU equivalent SLEV RNA in 20 &#x003bc;L. Lane 6: SLEV positive control after digestion with <italic>Hin</italic>dIII. Lane 7: No-template control reaction with SLEV 3&#x02019;-UTR primer set. Cutting sites within the amplicon regions are shown in Figs <xref ref-type=\"fig\" rid=\"pone.0147962.g001\">1</xref> and <xref ref-type=\"fig\" rid=\"pone.0147962.g002\">2</xref>.</p></caption><graphic xlink:href=\"pone.0147962.g003\"/></fig><p>A recent survey of 33 WEEV strains [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>] collected primarily in the United States indicated that all sampled strains were ~95% identical. The six strains used here for the sequence alignment were representative of three (Groups B1, B2, and B3) out of four of the WEEV lineages defined by Bergren et al. [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>]. The nsP4 regions of these isolates had a single polymorphic site falling within a LAMP primer binding region. This mismatch occurred at the 3&#x02019; end of the B1c priming region. Based upon LAMP primer design guidelines provided by Eiken Genomics (<ext-link ext-link-type=\"uri\" xlink:href=\"http://loopamp.eiken.co.jp/e/lamp/primer.html\">http://loopamp.eiken.co.jp/e/lamp/primer.html</ext-link>), we infer that mismatches at the 3&#x02019; end of the B2, and the 5&#x02019; end of the B1c are more detrimental than mismatches at the 3&#x02019; end of B1c. However, since the effect of mismatches on LAMP performance is not well characterized, the polymorphic site was accommodated with a degenerate base (M) which covers all isolates used in the sequence alignment. After primer development, we did identify a single isolate of WEEV within the collection at UC Davis (Lake43, Genbank #KJ554985) with a single mismatch to the primer set, at the 5&#x02019; end of the F1c priming region. RNA from this isolate was successfully amplified, despite the mismatch (which also occurs in one other closely related WEEV isolate within Genbank, #KJ554984). The fourth WEEV lineage (Group A) described by Bergren et al. [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>] comprised early WEEV isolates from 1930 and 1941. When we compared our primers to the &#x0201c;California&#x0201d; isolate (Genbank KJ554965.1, from the &#x0201c;A&#x0201d; lineage defined by Bergen et al. [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>]), two mismatches were identified, one each near the middle of the F1c and B1c priming regions. The tolerance of LAMP to mismatches in the middle of priming regions is not well described. However, due to the high conservation of the target site within the WEEV nsP4 gene, our LAMP primers would likely detect most naturally occurring WEEV isolates.SLEV has a greater geographic distribution than WEEV, and sequences in Genbank show significantly higher divergence than is observed for sequenced isolates of WEEV. A phylogenetic tree based on the SLEV E gene sequences identified 7 clades [<xref rid=\"pone.0147962.ref029\" ref-type=\"bibr\">29</xref>]. Clades I and II represent primarily North American isolates, and clades III-VI represent primarily Central and South American isolates. The E gene is more divergent than the 3&#x02019; UTR region targeted by our primers. However, a comprehensive tree was not available for the 3&#x02019; UTR region, due to limited published sequences for SLEV that include the 3&#x02019; UTR. Sequence alignment for primer design included only North American isolates, and sensitivity testing was performed primarily with a single strain (Kern 217, Clade IIC). To determine whether the primer set was effective for detecting diverse lineages of SLEV, additional isolates falling within Clades IA (Bfs1750), IB (Coav750), IIA(69M143), VA (BeAn246407), and VII (CorAn9124 and CorAn9275) were tested. SLEV LAMP primers successfully detected RNA from all tested isolates, originating from both North and South America. In contrast the RT-qPCR assay used for surveillance in California failed to detect the two Clade VII isolates from Argentina, as indicated in <xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref></p></sec><sec id=\"sec012\"><title>Sensitivity and specificity of RT-LAMP for WEEV, SLEV, and WNV</title><p>Real-time monitoring of RT-LAMP reactions for WEEV and SLEV primer sets was performed using serial dilutions of viral RNA, extracted from cultured virus. <xref ref-type=\"fig\" rid=\"pone.0147962.g004\">Fig 4</xref> shows real-time monitoring curves for RT-LAMP reactions for each virus, as well as standard curves generated for these viral RNA by both RT-LAMP and RT-qPCR. In addition, a standard curve was generated using the published WNV RT-LAMP primer set [<xref rid=\"pone.0147962.ref018\" ref-type=\"bibr\">18</xref>] and was used for comparison to RT-qPCR; the resulting standard curves for WNV are shown in <xref ref-type=\"supplementary-material\" rid=\"pone.0147962.s001\">S1 Fig</xref>. All three RT-LAMP primer sets reliably detect their corresponding targets at 0.1 PFU equivalent, with less reliable detection down to 0.01 PFU. For all three viruses, RT-qPCR produced linear calibration curves to Ct &#x0003e;35, corresponding to viral titers of ~0.01 PFU, and less reliable detection down to 0.001 PFU. Overall, RT-LAMP reactions were 10-fold less sensitive than the corresponding RT-qPCR assays. The threshold of reliable detection for RT-LAMP was 100 PFU/mL corresponding to Ct ~30&#x02013;32 by RT-qPCR. Positive detection generally occurred in 20&#x02013;30 minutes, and often in &#x0003c;10 minutes for samples containing &#x0003e;10<sup>4</sup> PFU/mL. Although RT-LAMP sensitivity was slightly lower than RT-qPCR, WNV RT-LAMP reactions were markedly more sensitive than commercially available antigen tests. Two common tests used for WNV surveillance are VecTest&#x000ae; WNV antigen assay (Medical Analysis Systems, Inc., Camarillo, CA) and Rapid Analyte Measurement Platform (RAMP&#x000ae;) WNV test (Response Biomedical, Corp., Burnaby British Columbia, Canada); these tests have minimum sensitivities of 10<sup>5</sup> and 10<sup>3</sup> PFU/mL, respectively.</p><fig id=\"pone.0147962.g004\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.g004</object-id><label>Fig 4</label><caption><title>Quantitative analysis of SLEV and WEEV RT-LAMP.</title><p>Panels A,B: Real-time RT-LAMP amplification curves for SLEV isolate Kern 217 (A) and WEEV isolate Kern 5547 (B). Reactions were performed in triplicate with viral RNA ranging from 10,000 PFU equivalents to 0.01 PFU equivalent per 10 &#x003bc;L reaction. Reactions were monitored in real-time using a BioRad CFX96 thermal cycler, using the HEX channel to monitor the signal from SYTO 82; the abbreviation &#x0201c;RFU&#x0201d; refers to Relative Fluorescence Units. Panels C,D: Standard curves for SLEV (C) and WEEV (D) generated by real-time monitoring of RT-LAMP (blue circles), versus RT-qPCR (red triangles). Time-to-positivity in RT-LAMP was determined by the BioRad CFX manager software, using baseline-subtracted quantitation data.</p></caption><graphic xlink:href=\"pone.0147962.g004\"/></fig><p>Although real-time monitoring of LAMP by means such as turbidity or fluorescent dyes is common [<xref rid=\"pone.0147962.ref010\" ref-type=\"bibr\">10</xref>, <xref rid=\"pone.0147962.ref011\" ref-type=\"bibr\">11</xref>], we note that real-time monitoring of RT-LAMP does not produce as precise a standard curve as RT-qPCR, particularly at lower viral RNA levels where time to positivity can vary substantially. We attribute this to the continuous nature of RT-LAMP, as opposed to the discrete cycling nature of RT-qPCR. In a well-designed RT-qPCR assay, each cycle in the exponential amplification regime leads to a reproducible doubling of the amount of amplicon. In RT-LAMP, however, slight variations in the early stages of the reaction can result in larger fluctuations in amplification time, particularly in the stochastic regime with few template copies, where multiple mechanisms may reduce the efficiency of the RT step in particular [<xref rid=\"pone.0147962.ref031\" ref-type=\"bibr\">31</xref>]. Real-time monitoring of RT-LAMP reactions can provide order-of-magnitude estimates of viral RNA concentration, but for precise quantitation RT-qPCR is preferable, particularly for low concentrations of viral RNA. The value of RT-LAMP for viral detection lies in the simplicity of the technique, particularly when applied in low-resource settings, where a yes-or-no answer determined by an endpoint measurement is sufficient.</p><p>Although the purpose of the current study was to develop and characterize new primer sets for SLEV and WEEV, we also demonstrated RT-LAMP detection of these targets using simpler instrumentation, as shown in <xref ref-type=\"supplementary-material\" rid=\"pone.0147962.s002\">S2 Fig</xref>. First, we carried out the RT-LAMP reactions for SLEV and WEEV using the Genie III, a portable isothermal fluorimeter with rechargeable battery and built-in wireless communication. Secondly, we took a photograph of the fluorescence endpoint using a handheld LED flashlight for illumination, and placing an amber-colored plastic film in front of the camera lens. In both cases, the fluorescence endpoint is clearly distinguishable between no-template control reactions and positive control reactions (tested down to 0.1 PFU equivalent of template).</p><p>For specificity testing, WEEV, SLEV, and WNV primer sets were tested against a panel of alphaviruses and flaviviruses, as indicated in <xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>. For WEEV, the closest relatives are other New World alphaviruses, Venezuelan equine encephalitis virus (VEEV) and Eastern equine encephalitis virus (EEEV). We found no cross-reactivity between any of the primer sets among non-target viruses. The only unexpected positive reactions were two isolated instances that we attribute to cross-contamination, where RNA from positive controls was inadvertently introduced into non-target cross-reactivity reactions. The specific instances (noted in <xref ref-type=\"table\" rid=\"pone.0147962.t001\">Table 1</xref>) are a single positive reaction (out of 18 replicates) when VEEV TC-83 RNA template was tested with the SLEV primers, and a single positive reaction (out of 11 replicates) when SLEV strain CorAn 9275 was tested with WNV primers. In both cases, the single positive amplifications displayed identical melt curves to positive-control reactions performed on the same plate, and these isolated positive results could not be reproduced. Otherwise no evidence of cross-reactivity between primers sets and non-target viruses was observed. In addition, there was no evidence of non-specific amplification in no-template control reactions with the SLEV and WEEV primer sets incubated for up to 50 minutes. Previously, non-specific amplification was observed with the previously published WNV primer set, but most often developed after &#x0003e; 50 minutes of incubation.</p><p>We tested archived RNA from 12 pooled mosquito samples collected in California in 2002 or 2014 that were positive by RT-qPCR for WEEV, SLEV, or WNV, or negative for all three, to determine if our RT-LAMP assays could detect these viruses in mosquito samples. Results of these tests are presented in <xref ref-type=\"table\" rid=\"pone.0147962.t003\">Table 3</xref>. The RT-LAMP assay detected SLEV in four pools that had RT-qPCR Cts ranging from 24.4 to 27.7. The RT-LAMP assay also detected WEEV in 5 pools that had RT-qPCR Cts ranging from 22.8 to 26.5, but failed to detect WEEV in one sample with a Ct value of 32, which is below the sensitivity demonstrated for the WEEV RT-LAMP assays. Sample sizes were limited, but the new primer sets readily detected SLEV and WEEV in mosquito pool RNA. Although sensitivity was slightly reduced compared to RT-qPCR, the specificity of these new primers was absolute.</p><table-wrap id=\"pone.0147962.t003\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.t003</object-id><label>Table 3</label><caption><title>Analysis of mosquito pools by SLEV, WEEV, and WNV RT-LAMP.</title></caption><alternatives><graphic id=\"pone.0147962.t003g\" xlink:href=\"pone.0147962.t003\"/><table frame=\"hsides\" rules=\"groups\"><colgroup span=\"1\"><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/><col align=\"left\" valign=\"middle\" span=\"1\"/></colgroup><thead><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\"/><th align=\"center\" colspan=\"3\" rowspan=\"1\"><underline>RT-qPCR Ct</underline><xref ref-type=\"table-fn\" rid=\"t003fn001\"><sup>1</sup></xref></th><th align=\"center\" colspan=\"3\" rowspan=\"1\"><underline>RT-LAMP Time-to-positivity (min)</underline><xref ref-type=\"table-fn\" rid=\"t003fn002\"><sup>2</sup></xref></th></tr><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\">Sample<xref ref-type=\"table-fn\" rid=\"t003fn003\"><sup>3</sup></xref></th><th align=\"center\" rowspan=\"1\" colspan=\"1\">SLEV</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WEEV</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WNV</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">SLEV</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WEEV</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">WNV</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 490</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">22.8</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">12.2, 12.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 512</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">26.5</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">14.0, 15.9</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 513</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">25.1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">13.0, 14.6</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 984</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">23.0</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">12.1, 12.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 942</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">24.5</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">12.2, 12.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 COAV 862</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">27.7</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">13.1, 13.9</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 IMPR 136</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">24.4</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">12.4, 12.5</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 IMPR 139</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">26.5</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">13.4, 14.0</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2002 IMPR 145</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">27.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">32.0</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">19.1, 21.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2014 KERN 14&#x02013;163</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">23.0</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">10.8, 11.1</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2014 MARN 14&#x02013;300</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">25.0</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">13.3, 13.8</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2014 SUYA 14&#x02013;144</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02013;</td></tr></tbody></table></alternatives><table-wrap-foot><fn id=\"t003fn001\"><p><sup>1</sup>Triplex RT-qPCRfor SLEV, WEEV, and WNV surveillance samples performed by UC Davis Center for Vectorborne Diseases.</p></fn><fn id=\"t003fn002\"><p><sup>2</sup>Results of two replicates reported</p></fn><fn id=\"t003fn003\"><p><sup>3</sup>Sample designations indicate year and location of sample collection within California: COAV = Coachella Valley, IMPR = Imperial Valley; KERN = Kern County; MARN = Marin-Sonoma counties, SUYA = Sutter-Yuba counties.</p></fn></table-wrap-foot></table-wrap></sec><sec id=\"sec013\"><title>Multiplexing RT-LAMP by melt curve analysis</title><p>Multiplexed amplification by LAMP was consistent, and all positive control RT-LAMP reactions amplified target RNA in less than 30 minutes. No-template controls remained negative for 40 minutes, after which non-specific amplification was detected in multiplexes containing WNV and WEEV primer sets together. Amplification in WNV + SLEV + WEEV and WNV + WEEV no-template controls likely resulted from primer-dimer formation between the WNV BIP and the WEEV FIP primers. Negative control reactions containing the combination of WNV and WEEV primer sets showed left-skewed melting peaks at 86.3&#x000b0;C, corresponding to melting of amplified primer-dimers. The intensity of the negative control melt curves was lower in reactions with a lower initial concentration of WNV/WEEV primer sets. Positive controls were distinguishable from negative controls and produced LAMP products that were suitable for melt curve analysis. Examples are presented in <xref ref-type=\"fig\" rid=\"pone.0147962.g005\">Fig 5</xref>.</p><fig id=\"pone.0147962.g005\" orientation=\"portrait\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.pone.0147962.g005</object-id><label>Fig 5</label><caption><title>Melt derivative curves for singleplex and multiplex RT-LAMP.</title><p>Solid black lines show average data from triplicate runs. Colored shaded areas represent the contributions of individual RT-LAMP targets to the overall signals to aid visualization. Blue = WEEV. Yellow = SLEV. Red = WNV. A) Singleplex reactions. B) Three-way combination. Only two peaks are distinguishable by eye. C) WEEV + WNV no template control. A relatively small melting peak is present, corresponding to spurious amplification after 40 min. D-F) Pairwise combinations. A smaller WEEV peak is distinguishable from WNV and SLEV peaks in D and F. WNV and SLEV peaks are indistinguishable by eye in E.</p></caption><graphic xlink:href=\"pone.0147962.g005\"/></fig><p>The melting curves of multiplexed LAMP reactions contained up to two distinguishable peaks for target identification. The first peak (84.5&#x02013;84.9&#x000b0;C) corresponded to the melting of WEEV-specific amplicons. The second peak (86.5&#x02013;87&#x000b0;C) corresponds to the melting of WNV-specific amplicons, SLEV-specific amplicons, or their combination.</p><p>The center and width of amplicon melting distributions determined the ease of multiplexed target detection. For example, the melting point of WNV amplicons was 0.5&#x000b0;C higher than that of SLEV amplicons. Because this difference is close to the 0.2&#x000b0;C accuracy limit of the real-time PCR instrument, reactions containing both WNV and SLEV amplicons displayed a single melting curve&#x02014;with an intermediate melting temperature&#x02014;instead of two discrete curves. Thus, simple peak picking was insufficient to differentiate WNV from SLEV in reactions containing both WNV and SLEV primer sets.</p><p>Furthermore, the relative intensity of discrete melt peaks reflected the relative abundance of individual LAMP products. However, in reactions with equal starting amounts of template, the WEEV amplicon melt-curve intensity was less than that of either the WNV or SLEV amplicons. This suggests that amplification with the WEEV primer set is slower or less efficient than either the WNV or SLEV primer sets. We have not tested experimentally whether this lower efficiency of amplification for WEEV reduces the detection limit of WEEV when present together in the same sample with a larger amount of WNV or SLEV.</p><p>Given the tendency to non-specific amplification observed with the combination of the WNV and WEEV primers, and the overlap of the WNV and SLEV melt curves, we conclude that only the WEEV and SLEV assays could be multiplexed in a single reaction. A more complex curve fitting algorithm based on parameters such as peak width, skew and temperature may be capable of robust differentiation of WEEV, SLEV, and WNV amplicons in multiplexed LAMP reactions. However, to our knowledge such an approach has not been demonstrated for LAMP melt curve analysis.</p></sec></sec><sec sec-type=\"conclusions\" id=\"sec014\"><title>Conclusion</title><p>The newly developed RT-LAMP primer sets for SLEV and WEEV show sensitivity and specificity comparable to a previously reported primer set for WNV [<xref rid=\"pone.0147962.ref016\" ref-type=\"bibr\">16</xref>]. Although the RT-LAMP assays are approximately an order of magnitude less sensitive than the corresponding RT-qPCR assays, the assay consistently detected to 0.1 PFU/reaction equivalents without cross-reactivity between related viruses. In addition, the RT-LAMP method can be greatly simplified compared to standard RT-qPCR reactions. Here purified RNA targets and a conventional real-time PCR thermal cycler were used to characterize RT-LAMP reactions, but we also demonstrate compatibility with a low-cost portable fluorimeter for monitoring isothermal amplification reactions, as well as compatibility with non-instrumented detection Future work is directly at developing simplified RT-LAMP protocols for RNA viruses such as WNV, SLEV, and WEEV. New protocols will utilize the tolerance of the Bst DNA polymerase to potential &#x0201c;PCR inhibitors&#x0201d; that are often present in crude samples such as mosquito or tissue homogenates. The combination of simplified instrumentation, minimal sample prep, no RNA extraction, and low reagent cost may enable the development of RT-LAMP based assays that are as simple to use as commercially available antigen tests. This would enable highly sensitive and specific surveillance for vector-borne diseases in basic laboratories or field settings where RT-qPCR resources are unavailable.</p></sec><sec sec-type=\"supplementary-material\" id=\"sec015\"><title>Supporting Information</title><supplementary-material content-type=\"local-data\" id=\"pone.0147962.s001\"><label>S1 Fig</label><caption><title>Standard curves for WNV RT-LAMP and RT-qPCR.</title><p>RT-LAMP time to positivity (blue circles) versus RT-qPCR Ct (red triangles), for WNV isolate L-CA-04 SAC-04-7168 using the RT-LAMP primers published by Parida et al. [<xref rid=\"pone.0147962.ref001\" ref-type=\"bibr\">1</xref>], and the RT-qPCR primers published by Lanciotti et al [<xref rid=\"pone.0147962.ref002\" ref-type=\"bibr\">2</xref>]. At least three technical replicates were performed for each template concentration.</p><p>(PDF)</p></caption><media xlink:href=\"pone.0147962.s001.pdf\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"pone.0147962.s002\"><label>S2 Fig</label><caption><title>Detection of SLEV and WEEV by RT-LAMP using simplified techniques.</title><p>(A) SLEV and WEEV RNA were detected by RT-LAMP using the OptiGene Genie III, a portable, battery-operated real-time fluorimeter. The screenshot in the inset of panel (A) shows real-time amplification curves generated for eight reactions, which are specified in Panel B. Reactions 1&#x02013;4 use the SLEV 3&#x02019; UTR primer set, with 10, 1, 0.1 PFU equivalents of SLEV RNA (Reaction 4 is an SLEV no-template control). Reactions 5&#x02013;8 use the WEEV nsP4 primer set, with 10, 1, 0.1 PFU equivalents of WEEV RNA (Reaction 8 is a WEEV no-template control). Panel (B) illustrates the endpoint of the same eight reactions, after incubation at 63&#x000b0;C for 30 minutes. To take this photograph, the strip of tubes was set upon a dark background, on a hotplate set at 63&#x000b0;C. Fluorescence was excited using a handheld blue LED flashlight, and a sheet of amber plastic film (LEE Filters #158) was placed in front of the lens of a digital camera. No contrast adjustment was applied. In Panel A, a barcode label affixed to the instrument has been pixelated to obscure identification.</p><p>(TIF)</p></caption><media xlink:href=\"pone.0147962.s002.tif\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"pone.0147962.s003\"><label>S1 Table</label><caption><title>RT-qPCR primers used for quantitation of Western equine encephalitis virus (WEEV), St. Louis encephalitis virus (SLEV), and West Nile virus (WNV) viral RNA.</title><p>IBFQ = Iowa Black FQ; BHQ1 = Black Hole Quencher 1 (The different quenchers have similar performance when paired with FAM, and do not affect the results of the PCR assays). The primer set names from the original literature references are provided in the table next to the reference numbers.</p><p>(PDF)</p></caption><media xlink:href=\"pone.0147962.s003.pdf\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"pone.0147962.s004\"><label>S2 Table</label><caption><title>Additional RT-LAMP primer sets for SLEV and WEEV.</title><p>(PDF)</p></caption><media xlink:href=\"pone.0147962.s004.pdf\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material></sec></body><back><ack><p>This work was supported by a grant from the Defense Threat Reduction Agency (DTRA), contract DTRA1002710697. Sandia National Laboratories is a multi-program laboratory managed and operated by Sandia Corporation, a wholly owned subsidiary of Lockheed Martin Corporation, for the U.S. Department of Energy&#x02019;s National Nuclear Security Administration under Contract DE-AC04-94AL85000.</p></ack><ref-list><title>References</title><ref id=\"pone.0147962.ref001\"><label>1</label><mixed-citation publication-type=\"journal\"><name><surname>Bergren</surname><given-names>NA</given-names></name>, <name><surname>Auguste</surname><given-names>AJ</given-names></name>, <name><surname>Forrester</surname><given-names>NL</given-names></name>, <name><surname>Negi</surname><given-names>SS</given-names></name>, <name><surname>Braun</surname><given-names>WA</given-names></name>, <name><surname>Weaver</surname><given-names>SC</given-names></name>. <article-title>Western equine encephalitis virus: evolutionary analysis of a declining alphavirus based on complete genome sequences</article-title>. <source>J Virol</source>. <year>2014</year>;<volume>88</volume>(<issue>16</issue>):<fpage>9260</fpage>&#x02013;<lpage>7</lpage>. Epub 2014/06/06. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jvi.01463-14\">10.1128/jvi.01463-14</ext-link></comment> ; PubMed Central PMCID: PMCPmc4136285.<pub-id pub-id-type=\"pmid\">24899192</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref002\"><label>2</label><mixed-citation publication-type=\"journal\"><name><surname>Reisen</surname><given-names>WK</given-names></name>, <name><surname>Lothrop</surname><given-names>HD</given-names></name>, <name><surname>Wheeler</surname><given-names>SS</given-names></name>, <name><surname>Kennsington</surname><given-names>M</given-names></name>, <name><surname>Gutierrez</surname><given-names>A</given-names></name>, <name><surname>Fang</surname><given-names>Y</given-names></name>, <etal>et al</etal>\\n<article-title>Persistent West Nile virus transmission and the apparent displacement St. Louis encephalitis virus in southeastern California, 2003&#x02013;2006</article-title>. <source>J Medical Entomol</source>. <year>2008</year>;<volume>45</volume>(<issue>3</issue>):<fpage>494</fpage>&#x02013;<lpage>508</lpage>. Epub 2008/06/07. ; PubMed Central PMCID: PMCPmc2435167.<pub-id pub-id-type=\"pmid\">18533445</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref003\"><label>3</label><mixed-citation publication-type=\"other\">Oviatt J. Saint Louis encephalitis virus detected in valley mosquitoes for the first time since 2003: Coachella Valley Mosquito and Vector Control District; 2015 [cited 2015 10/2/2015]. Press release]. Available: <ext-link ext-link-type=\"uri\" xlink:href=\"http://www.cvmvcd.org/press/releases/2015/engpressreleaseslewnv080515.pdf\">http://www.cvmvcd.org/press/releases/2015/engpressreleaseslewnv080515.pdf</ext-link>.</mixed-citation></ref><ref id=\"pone.0147962.ref004\"><label>4</label><mixed-citation publication-type=\"journal\"><name><surname>Vidal</surname><given-names>F</given-names></name>, <name><surname>Julia</surname><given-names>SA</given-names></name>, <name><surname>Carolina</surname><given-names>G</given-names></name>, <name><surname>Edna</surname><given-names>RC</given-names></name>, <name><surname>Jorge</surname><given-names>G</given-names></name>, <name><surname>Felix</surname><given-names>WS-M</given-names></name>, <etal>et al</etal>\\n<article-title>St. Louis Encephalitis virus infection in woman, Peru</article-title>. <source>Emerg Infect Dis</source>. <year>2014</year>;<volume>20</volume>(<issue>4</issue>):<fpage>730</fpage>\\n<comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.3201/eid2004.131735\">10.3201/eid2004.131735</ext-link></comment>\\n<pub-id pub-id-type=\"pmid\">24655897</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref005\"><label>5</label><mixed-citation publication-type=\"journal\"><name><surname>Nagamine</surname><given-names>K</given-names></name>, <name><surname>Hase</surname><given-names>T</given-names></name>, <name><surname>Notomi</surname><given-names>T</given-names></name>. <article-title>Accelerated reaction by loop-mediated isothermal amplification using loop primers</article-title>. <source>Mol Cell Probes</source>. <year>2002</year>;<volume>16</volume>(<issue>3</issue>):<fpage>223</fpage>&#x02013;<lpage>9</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1006/mcpr.2002.0415\">10.1006/mcpr.2002.0415</ext-link></comment> .<pub-id pub-id-type=\"pmid\">12144774</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref006\"><label>6</label><mixed-citation publication-type=\"journal\"><name><surname>Notomi</surname><given-names>T</given-names></name>, <name><surname>Okayama</surname><given-names>H</given-names></name>, <name><surname>Masubuchi</surname><given-names>H</given-names></name>, <name><surname>Yonekawa</surname><given-names>T</given-names></name>, <name><surname>Watanabe</surname><given-names>K</given-names></name>, <name><surname>Amino</surname><given-names>N</given-names></name>, <etal>et al</etal>\\n<article-title>Loop-mediated isothermal amplification of DNA</article-title>. <source>Nucleic Acids Res</source>. <year>2000</year>;<volume>28</volume>(<issue>12</issue>). <fpage>e63</fpage>\\n<comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1093/nar/28.12.e63\">10.1093/nar/28.12.e63</ext-link></comment> .<pub-id pub-id-type=\"pmid\">10871386</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref007\"><label>7</label><mixed-citation publication-type=\"journal\"><name><surname>Iwamoto</surname><given-names>T</given-names></name>, <name><surname>Sonobe</surname><given-names>T</given-names></name>, <name><surname>Hayashi</surname><given-names>K</given-names></name>. <article-title>Loop-mediated isothermal amplification for direct detection of Mycobacterium tuberculosis complex, M. avium, and M. intracellulare in sputum samples</article-title>. <source>J Clin Microbiol</source>. <year>2003</year>;<volume>41</volume>(<issue>6</issue>):<fpage>2616</fpage>&#x02013;<lpage>22</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/JCM.41.6.2616&#x02013;2622.2003\">10.1128/JCM.41.6.2616&#x02013;2622.2003</ext-link></comment> .<pub-id pub-id-type=\"pmid\">12791888</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref008\"><label>8</label><mixed-citation publication-type=\"journal\"><name><surname>Polley</surname><given-names>SD</given-names></name>, <name><surname>Gonz&#x000e1;lez</surname><given-names>IJ</given-names></name>, <name><surname>Mohamed</surname><given-names>D</given-names></name>, <name><surname>Daly</surname><given-names>R</given-names></name>, <name><surname>Bowers</surname><given-names>K</given-names></name>, <name><surname>Watson</surname><given-names>J</given-names></name>, <etal>et al</etal>\\n<article-title>Clinical evaluation of a LAMP test kit for diagnosis of imported malaria</article-title>. <source>J Infect Dis</source>. <year>2013</year>\\n<comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1093/infdis/jit183\">10.1093/infdis/jit183</ext-link></comment></mixed-citation></ref><ref id=\"pone.0147962.ref009\"><label>9</label><mixed-citation publication-type=\"journal\"><name><surname>Fukuta</surname><given-names>S</given-names></name>, <name><surname>Iida</surname><given-names>T</given-names></name>, <name><surname>Mizukami</surname><given-names>Y</given-names></name>, <name><surname>Ishida</surname><given-names>A</given-names></name>, <name><surname>Ueda</surname><given-names>J</given-names></name>, <name><surname>Kanbe</surname><given-names>M</given-names></name>, <etal>et al</etal>\\n<article-title>Detection of Japanese yam mosaic virus by RT-LAMP</article-title>. <source>Arch Virol</source>. <year>2003</year>;<volume>148</volume>(<issue>9</issue>):<fpage>1713</fpage>&#x02013;<lpage>20</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1007/s00705-003-0134-5\">10.1007/s00705-003-0134-5</ext-link></comment> .<pub-id pub-id-type=\"pmid\">14505084</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref010\"><label>10</label><mixed-citation publication-type=\"journal\"><name><surname>Mori</surname><given-names>Y</given-names></name>, <name><surname>Nagamine</surname><given-names>K</given-names></name>, <name><surname>Tomita</surname><given-names>N</given-names></name>, <name><surname>Notomi</surname><given-names>T</given-names></name>. <article-title>Detection of loop-mediated isothermal amplification reaction by turbidity derived from magnesium pyrophosphate formation</article-title>. <source>Biochem Biophys Res Commun</source>. <year>2001</year>;<volume>289</volume>(<issue>1</issue>):<fpage>150</fpage>&#x02013;<lpage>4</lpage>. Epub 2001/11/16. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1006/bbrc.2001.5921\">10.1006/bbrc.2001.5921</ext-link></comment> .<pub-id pub-id-type=\"pmid\">11708792</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref011\"><label>11</label><mixed-citation publication-type=\"journal\"><name><surname>Ahmad</surname><given-names>F</given-names></name>, <name><surname>Seyrig</surname><given-names>G</given-names></name>, <name><surname>Tourlousse</surname><given-names>DM</given-names></name>, <name><surname>Stedtfeld</surname><given-names>RD</given-names></name>, <name><surname>Tiedje</surname><given-names>JM</given-names></name>, <name><surname>Hashsham</surname><given-names>SA</given-names></name>. <article-title>A CCD-based fluorescence imaging system for real-time loop-mediated isothermal amplification-based rapid and sensitive detection of waterborne pathogens on microchips</article-title>. <source>Biomed Microdevices</source>. <year>2011</year>;<volume>13</volume>(<issue>5</issue>):<fpage>929</fpage>&#x02013;<lpage>37</lpage>. Epub 2011/07/02. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1007/s10544-011-9562-2\">10.1007/s10544-011-9562-2</ext-link></comment> .<pub-id pub-id-type=\"pmid\">21720851</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref012\"><label>12</label><mixed-citation publication-type=\"journal\"><name><surname>Nie</surname><given-names>K</given-names></name>, <name><surname>Qi</surname><given-names>SX</given-names></name>, <name><surname>Zhang</surname><given-names>Y</given-names></name>, <name><surname>Luo</surname><given-names>L</given-names></name>, <name><surname>Xie</surname><given-names>Y</given-names></name>, <name><surname>Yang</surname><given-names>MJ</given-names></name>, <etal>et al</etal>\\n<article-title>Evaluation of a direct reverse transcription loop-mediated isothermal amplification method without RNA extraction for the detection of human enterovirus 71 subgenotype C4 in nasopharyngeal swab specimens</article-title>. <source>PLOS One</source>. <year>2012</year>;<volume>7</volume>(<issue>12</issue>):<fpage>e52486</fpage> Epub 2012/12/29. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1371/journal.pone.0052486\">10.1371/journal.pone.0052486</ext-link></comment> ; PubMed Central PMCID: PMC3525532.<pub-id pub-id-type=\"pmid\">23272248</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref013\"><label>13</label><mixed-citation publication-type=\"journal\"><name><surname>Wang</surname><given-names>X</given-names></name>, <name><surname>Zhang</surname><given-names>Q</given-names></name>, <name><surname>Zhang</surname><given-names>F</given-names></name>, <name><surname>Ma</surname><given-names>F</given-names></name>, <name><surname>Zheng</surname><given-names>W</given-names></name>, <name><surname>Zhao</surname><given-names>Z</given-names></name>, <etal>et al</etal>\\n<article-title>Visual detection of the human metapneumovirus using reverse transcription loop-mediated isothermal amplification with hydroxynaphthol blue dye</article-title>. <source>Virol J</source>. <year>2012</year>;<volume>9</volume>:<fpage>138</fpage> Epub 2012/07/31. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1186/1743-422x-9-138\">10.1186/1743-422x-9-138</ext-link></comment> ; PubMed Central PMCID: PMC3487928.<pub-id pub-id-type=\"pmid\">22838725</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref014\"><label>14</label><mixed-citation publication-type=\"journal\"><name><surname>Poon</surname><given-names>LL</given-names></name>, <name><surname>Wong</surname><given-names>BW</given-names></name>, <name><surname>Ma</surname><given-names>EH</given-names></name>, <name><surname>Chan</surname><given-names>KH</given-names></name>, <name><surname>Chow</surname><given-names>LM</given-names></name>, <name><surname>Abeyewickreme</surname><given-names>W</given-names></name>, <etal>et al</etal>\\n<article-title>Sensitive and inexpensive molecular test for falciparum malaria: detecting Plasmodium falciparum DNA directly from heat-treated blood by loop-mediated isothermal amplification</article-title>. <source>Clin Chem</source>. <year>2006</year>;<volume>52</volume>(<issue>2</issue>):<fpage>303</fpage>&#x02013;<lpage>6</lpage>. Epub 2005/12/13. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1373/clinchem.2005.057901\">10.1373/clinchem.2005.057901</ext-link></comment> .<pub-id pub-id-type=\"pmid\">16339303</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref015\"><label>15</label><mixed-citation publication-type=\"journal\"><name><surname>Curtis</surname><given-names>KA</given-names></name>, <name><surname>Rudolph</surname><given-names>DL</given-names></name>, <name><surname>Nejad</surname><given-names>I</given-names></name>, <name><surname>Singleton</surname><given-names>J</given-names></name>, <name><surname>Beddoe</surname><given-names>A</given-names></name>, <name><surname>Weigl</surname><given-names>B</given-names></name>, <etal>et al</etal>\\n<article-title>Isothermal amplification using a chemical heating device for point-of-care detection of HIV-1</article-title>. <source>PLOS One</source>. <year>2012</year>;<volume>7</volume>(<issue>2</issue>):<fpage>e31432</fpage> Epub 2012/03/03. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1371/journal.pone.0031432\">10.1371/journal.pone.0031432</ext-link></comment> ; PubMed Central PMCID: PMCPmc3285652.<pub-id pub-id-type=\"pmid\">22384022</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref016\"><label>16</label><mixed-citation publication-type=\"journal\"><name><surname>Curtis</surname><given-names>KA</given-names></name>, <name><surname>Rudolph</surname><given-names>DL</given-names></name>, <name><surname>Owen</surname><given-names>SM</given-names></name>. <article-title>Rapid detection of HIV-1 by reverse-transcription, loop-mediated isothermal amplification (RT-LAMP)</article-title>. <source>J Virol Meth</source>. <year>2008</year>;<volume>151</volume>(<issue>2</issue>):<fpage>264</fpage>&#x02013;<lpage>70</lpage>. Epub 2008/06/06. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1016/j.jviromet.2008.04.011\">10.1016/j.jviromet.2008.04.011</ext-link></comment> .<pub-id pub-id-type=\"pmid\">18524393</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref017\"><label>17</label><mixed-citation publication-type=\"journal\"><name><surname>Francois</surname><given-names>P</given-names></name>, <name><surname>Tangomo</surname><given-names>M</given-names></name>, <name><surname>Hibbs</surname><given-names>J</given-names></name>, <name><surname>Bonetti</surname><given-names>EJ</given-names></name>, <name><surname>Boehme</surname><given-names>CC</given-names></name>, <name><surname>Notomi</surname><given-names>T</given-names></name>, <etal>et al</etal>\\n<article-title>Robustness of a loop-mediated isothermal amplification reaction for diagnostic applications</article-title>. <source>FEMS Immunol Med Microbiol</source>. <year>2011</year>;<volume>62</volume>(<issue>1</issue>):<fpage>41</fpage>&#x02013;<lpage>8</lpage>. Epub 2011/02/01. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1111/j.1574-695X.2011.00785.x\">10.1111/j.1574-695X.2011.00785.x</ext-link></comment> .<pub-id pub-id-type=\"pmid\">21276085</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref018\"><label>18</label><mixed-citation publication-type=\"journal\"><name><surname>Parida</surname><given-names>M</given-names></name>, <name><surname>Posadas</surname><given-names>G</given-names></name>, <name><surname>Inoue</surname><given-names>S</given-names></name>, <name><surname>Hasebe</surname><given-names>F</given-names></name>, <name><surname>Morita</surname><given-names>K</given-names></name>. <article-title>Real-time reverse transcription loop-mediated isothermal amplification for rapid detection of West Nile virus</article-title>. <source>J Clin Microbiol</source>. 2005. <year>2004</year>;<volume>42</volume>(<issue>1</issue>):<fpage>257</fpage>&#x02013;<lpage>63</lpage>. Epub 2004/01/13. ; PubMed Central PMCID: PMC321710.<pub-id pub-id-type=\"pmid\">14715762</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref019\"><label>19</label><mixed-citation publication-type=\"journal\"><name><surname>Parida</surname><given-names>M</given-names></name>, <name><surname>Horioke</surname><given-names>K</given-names></name>, <name><surname>Ishida</surname><given-names>H</given-names></name>, <name><surname>Dash</surname><given-names>PK</given-names></name>, <name><surname>Saxena</surname><given-names>P</given-names></name>, <name><surname>Jana</surname><given-names>AM</given-names></name>, <etal>et al</etal>\\n<article-title>Rapid detection and differentiation of dengue virus serotypes by a real-time reverse transcription-loop-mediated isothermal amplification assay</article-title>. <source>J Clin Microbiol</source>. <year>2005</year>;<volume>43</volume>(<issue>6</issue>):<fpage>2895</fpage>&#x02013;<lpage>903</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jcm.43.6.2895&#x02013;2903.2005.\">10.1128/jcm.43.6.2895&#x02013;2903.2005.</ext-link></comment> .<pub-id pub-id-type=\"pmid\">15956414</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref020\"><label>20</label><mixed-citation publication-type=\"journal\"><name><surname>Teoh</surname><given-names>BT</given-names></name>, <name><surname>Sam</surname><given-names>SS</given-names></name>, <name><surname>Tan</surname><given-names>KK</given-names></name>, <name><surname>Johari</surname><given-names>J</given-names></name>, <name><surname>Danlami</surname><given-names>MB</given-names></name>, <name><surname>Hooi</surname><given-names>PS</given-names></name>, <etal>et al</etal>\\n<article-title>Detection of dengue viruses using reverse transcription-loop-mediated isothermal amplification</article-title>. <source>BMC Infect Dis</source>. <year>2013</year>;<volume>13</volume>:<fpage>387</fpage> Epub 2013/08/24. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1186/1471-2334-13-387\">10.1186/1471-2334-13-387</ext-link></comment> ; PubMed Central PMCID: PMC3846474.<pub-id pub-id-type=\"pmid\">23964963</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref021\"><label>21</label><mixed-citation publication-type=\"journal\"><name><surname>Liu</surname><given-names>H</given-names></name>, <name><surname>Liu</surname><given-names>ZJ</given-names></name>, <name><surname>Jing</surname><given-names>J</given-names></name>, <name><surname>Ren</surname><given-names>JQ</given-names></name>, <name><surname>Liu</surname><given-names>YY</given-names></name>, <name><surname>Guo</surname><given-names>HH</given-names></name>, <etal>et al</etal>\\n<article-title>Reverse transcription loop-mediated isothermal amplification for rapid detection of Japanese encephalitis virus in swine and mosquitoes</article-title>. <source>Vector Borne Zoonotic Dis</source>. <year>2012</year>;<volume>12</volume>(<issue>12</issue>):<fpage>1042</fpage>&#x02013;<lpage>52</lpage>. Epub 2012/11/28. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1089/vbz.2012.0991\">10.1089/vbz.2012.0991</ext-link></comment> ; PubMed Central PMCID: PMCPmc3525893.<pub-id pub-id-type=\"pmid\">23176446</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref022\"><label>22</label><mixed-citation publication-type=\"journal\"><name><surname>Parida</surname><given-names>MM</given-names></name>, <name><surname>Santhosh</surname><given-names>SR</given-names></name>, <name><surname>Dash</surname><given-names>PK</given-names></name>, <name><surname>Tripathi</surname><given-names>NK</given-names></name>, <name><surname>Saxena</surname><given-names>P</given-names></name>, <name><surname>Ambuj</surname><given-names>S</given-names></name>, <etal>et al</etal>\\n<article-title>Development and evaluation of reverse transcription-loop-mediated isothermal amplification assay for rapid and real-time detection of Japanese encephalitis virus</article-title>. <source>J Clin Microbiol</source>. <year>2006</year>;<volume>44</volume>(<issue>11</issue>):<fpage>4172</fpage>&#x02013;<lpage>8</lpage>. Epub 2006/09/29. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jcm.01487-06\">10.1128/jcm.01487-06</ext-link></comment> ; PubMed Central PMCID: PMCPmc1698363.<pub-id pub-id-type=\"pmid\">17005741</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref023\"><label>23</label><mixed-citation publication-type=\"journal\"><name><surname>Toriniwa</surname><given-names>H</given-names></name>, <name><surname>Komiya</surname><given-names>T</given-names></name>. <article-title>Rapid detection and quantification of Japanese encephalitis virus by real-time reverse transcription loop-mediated isothermal amplification</article-title>. <source>Microbiol Immunol</source>. <year>2006</year>;<volume>50</volume>(<issue>5</issue>):<fpage>379</fpage>&#x02013;<lpage>87</lpage>. Epub 2006/05/23. .<pub-id pub-id-type=\"pmid\">16714845</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref024\"><label>24</label><mixed-citation publication-type=\"journal\"><name><surname>Parida</surname><given-names>MM</given-names></name>, <name><surname>Santhosh</surname><given-names>SR</given-names></name>, <name><surname>Dash</surname><given-names>PK</given-names></name>, <name><surname>Tripathi</surname><given-names>NK</given-names></name>, <name><surname>Lakshmi</surname><given-names>V</given-names></name>, <name><surname>Mamidi</surname><given-names>N</given-names></name>, <etal>et al</etal>\\n<article-title>Rapid and real-time detection of chikungunya virus by reverse transcription loop-mediated isothermal amplification assay</article-title>. <source>J Clin Microbiol</source>. <year>2007</year>;<volume>45</volume>(<issue>2</issue>):<fpage>351</fpage>&#x02013;<lpage>7</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jcm.01734-06\">10.1128/jcm.01734-06</ext-link></comment> .<pub-id pub-id-type=\"pmid\">17135444</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref025\"><label>25</label><mixed-citation publication-type=\"journal\"><name><surname>Le Roux</surname><given-names>CA</given-names></name>, <name><surname>Kubo</surname><given-names>T</given-names></name>, <name><surname>Grobbelaar</surname><given-names>AA</given-names></name>, <name><surname>van Vuren</surname><given-names>PJ</given-names></name>, <name><surname>Weyer</surname><given-names>J</given-names></name>, <name><surname>Nel</surname><given-names>LH</given-names></name>, <etal>et al</etal>\\n<article-title>Development and evaluation of a real-time reverse transcription-loop-mediated isothermal amplification assay for rapid detection of Rift Valley fever virus in clinical specimens</article-title>. <source>J Clin Microbiol</source>. <year>2009</year>;<volume>47</volume>(<issue>3</issue>):<fpage>645</fpage>&#x02013;<lpage>51</lpage>. Epub 2008/12/26. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jcm.01412-08\">10.1128/jcm.01412-08</ext-link></comment> ; PubMed Central PMCID: PMCPmc2650915.<pub-id pub-id-type=\"pmid\">19109471</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref026\"><label>26</label><mixed-citation publication-type=\"journal\"><name><surname>Peyrefitte</surname><given-names>CN</given-names></name>, <name><surname>Boubis</surname><given-names>L</given-names></name>, <name><surname>Coudrier</surname><given-names>D</given-names></name>, <name><surname>Bouloy</surname><given-names>M</given-names></name>, <name><surname>Grandadam</surname><given-names>M</given-names></name>, <name><surname>Tolou</surname><given-names>HJ</given-names></name>, <etal>et al</etal>\\n<article-title>Real-time reverse-transcription loop-mediated isothermal amplification for rapid detection of Rift Valley fever virus</article-title>. <source>J Clin Microbiol</source>. <year>2008</year>;<volume>46</volume>(<issue>11</issue>):<fpage>3653</fpage>&#x02013;<lpage>9</lpage>. Epub 2008/09/19. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/jcm.01188-08\">10.1128/jcm.01188-08</ext-link></comment> ; PubMed Central PMCID: PMCPmc2576582.<pub-id pub-id-type=\"pmid\">18799705</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref027\"><label>27</label><mixed-citation publication-type=\"journal\"><name><surname>Kwallah</surname><given-names>A</given-names></name>, <name><surname>Inoue</surname><given-names>S</given-names></name>, <name><surname>Muigai</surname><given-names>AW</given-names></name>, <name><surname>Kubo</surname><given-names>T</given-names></name>, <name><surname>Sang</surname><given-names>R</given-names></name>, <name><surname>Morita</surname><given-names>K</given-names></name>, <etal>et al</etal>\\n<article-title>A real-time reverse transcription loop-mediated isothermal amplification assay for the rapid detection of yellow fever virus</article-title>. <source>J Virol Meth</source>. <year>2013</year>;<volume>193</volume>(<issue>1</issue>):<fpage>23</fpage>&#x02013;<lpage>7</lpage>. Epub 2013/05/23. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1016/j.jviromet.2013.05.004\">10.1016/j.jviromet.2013.05.004</ext-link></comment> .<pub-id pub-id-type=\"pmid\">23692685</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref028\"><label>28</label><mixed-citation publication-type=\"journal\"><name><surname>Li</surname><given-names>S</given-names></name>, <name><surname>Fang</surname><given-names>M</given-names></name>, <name><surname>Zhou</surname><given-names>B</given-names></name>, <name><surname>Ni</surname><given-names>H</given-names></name>, <name><surname>Shen</surname><given-names>Q</given-names></name>, <name><surname>Zhang</surname><given-names>H</given-names></name>, <etal>et al</etal>\\n<article-title>Simultaneous detection and differentiation of dengue virus serotypes 1&#x02013;4, Japanese encephalitis virus, and West Nile virus by a combined reverse-transcription loop-mediated isothermal amplification assay</article-title>. <source>Virol J</source>. <year>2011</year>;<volume>8</volume>:<fpage>360</fpage> Epub 2011/07/23. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1186/1743-422x-8-360\">10.1186/1743-422x-8-360</ext-link></comment> ; PubMed Central PMCID: PMCPmc3149006.<pub-id pub-id-type=\"pmid\">21777455</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref029\"><label>29</label><mixed-citation publication-type=\"journal\"><name><surname>Kramer</surname><given-names>LD</given-names></name>, <name><surname>Chandler</surname><given-names>LJ</given-names></name>. <article-title>Phylogenetic analysis of the envelope gene of St. Louis encephalitis virus</article-title>. <source>Arch Virol</source>. <year>2001</year>;<volume>146</volume>(<issue>12</issue>):<fpage>2341</fpage>&#x02013;<lpage>55</lpage>. Epub 2002/01/29. .<pub-id pub-id-type=\"pmid\">11811684</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref030\"><label>30</label><mixed-citation publication-type=\"journal\"><name><surname>Kopp</surname><given-names>A</given-names></name>, <name><surname>Gillespie</surname><given-names>TR</given-names></name>, <name><surname>Hobelsberger</surname><given-names>D</given-names></name>, <name><surname>Estrada</surname><given-names>A</given-names></name>, <name><surname>Harper</surname><given-names>JM</given-names></name>, <name><surname>Miller</surname><given-names>RA</given-names></name>, <etal>et al</etal>\\n<article-title>Provenance and geographic spread of St. Louis encephalitis virus</article-title>. <source>mBio</source>. <year>2013</year>;<volume>4</volume>(<issue>3</issue>):<fpage>e00322</fpage>&#x02013;<lpage>13</lpage>. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1128/mBio.00322-13\">10.1128/mBio.00322-13</ext-link></comment> .<pub-id pub-id-type=\"pmid\">23760463</pub-id></mixed-citation></ref><ref id=\"pone.0147962.ref031\"><label>31</label><mixed-citation publication-type=\"journal\"><name><surname>Sun</surname><given-names>B</given-names></name>, <name><surname>Shen</surname><given-names>F</given-names></name>, <name><surname>McCalla</surname><given-names>SE</given-names></name>, <name><surname>Kreutz</surname><given-names>JE</given-names></name>, <name><surname>Karymov</surname><given-names>MA</given-names></name>, <name><surname>Ismagilov</surname><given-names>RF</given-names></name>. <article-title>Mechanistic evaluation of the pros and cons of digital RT-LAMP for HIV-1 viral load quantification on a microfluidic device and improved efficiency via a two-step digital protocol</article-title>. <source>Anal Chem</source>. <year>2013</year>;<volume>85</volume>(<issue>3</issue>):<fpage>1540</fpage>&#x02013;<lpage>6</lpage>. Epub 2013/01/18. <comment>doi: <ext-link ext-link-type=\"uri\" xlink:href=\"http://dx.doi.org/10.1021/ac3037206\">10.1021/ac3037206</ext-link></comment> ; PubMed Central PMCID: PMCPmc3578705.<pub-id pub-id-type=\"pmid\">23324061</pub-id></mixed-citation></ref></ref-list></back></article>'}"
      ]
     },
     "execution_count": 52,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "x = articles.find_one({'_id': 'test', 'annotations.disease-ontology-keywords': {'$exists': True}})\n",
    "x"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<pymongo.results.DeleteResult at 0x1145052d0>"
      ]
     },
     "execution_count": 61,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "articles.delete_one({'_id': \"test\"})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'iterate_infectious_disease_articles' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-16-5a5cc4bd510a>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0;32mfor\u001b[0m \u001b[0marticle\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0minfectious_diseases\u001b[0m \u001b[0;32min\u001b[0m \u001b[0miterate_infectious_disease_articles\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdb\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0marticlesubset\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0marticle\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'_id'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0minfectious_diseases\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mNameError\u001b[0m: name 'iterate_infectious_disease_articles' is not defined"
     ]
    }
   ],
   "source": [
    "for article, infectious_diseases in iterate_infectious_disease_articles(db.articlesubset):\n",
    "    print(article['_id'], infectious_diseases)\n",
    "    print(\"\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'total_article_count' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-10-c28412185261>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mtotal_article_count\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m: name 'total_article_count' is not defined"
     ]
    }
   ],
   "source": [
    "total_article_count"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'article_with_body_count' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-11-b7972d653aab>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0marticle_with_body_count\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m: name 'article_with_body_count' is not defined"
     ]
    }
   ],
   "source": [
    "article_with_body_count"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'infectious_disease_article_count' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-12-db9743c64222>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0minfectious_disease_article_count\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m: name 'infectious_disease_article_count' is not defined"
     ]
    }
   ],
   "source": [
    "infectious_disease_article_count"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def strip_article_meta(collection):\n",
    "    collection.update_many({'meta': {'$exists': True}},\n",
    "        {\n",
    "        '$unset':\n",
    "            {\n",
    "            'meta': \"\",\n",
    "            'annotations': \"\"\n",
    "            }\n",
    "        })"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "strip_article_meta(articles)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
