{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import os\n",
    "os.chdir(\"..\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "\"\"\"\n",
    "Iterate over the PubMED articles that mention infecious diseases from the\n",
    "disease ontology.\n",
    "\"\"\"\n",
    "import rdflib\n",
    "from pylru import lrudecorator\n",
    "import pubcrawler.article as pubcrawler\n",
    "from annotator.keyword_annotator import KeywordAnnotator\n",
    "from annotator.annotator import AnnoDoc\n",
    "import re\n",
    "import json\n",
    "import pymongo"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Loading disease ontology...\n",
      "disease ontology loaded\n"
     ]
    }
   ],
   "source": [
    "print(\"Loading disease ontology...\")\n",
    "disease_ontology = rdflib.Graph()\n",
    "disease_ontology.parse(\n",
    "    \"http://purl.obolibrary.org/obo/doid.owl\",\n",
    "    format=\"xml\"\n",
    ")\n",
    "print(\"disease ontology loaded\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<Graph identifier=Nbda20d900ed74f759fe6397bf8111869 (<class 'rdflib.graph.Graph'>)>"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "disease_ontology"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def get_annotation_keywords():\n",
    "    qres = disease_ontology.query(\"\"\"\n",
    "    prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "    prefix obo: <http://purl.obolibrary.org/obo/>\n",
    "    prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "    SELECT ?entity ?label\n",
    "    WHERE {\n",
    "        # only resolve diseases by infectious agent\n",
    "        ?entity rdfs:subClassOf* obo:DOID_0050117\n",
    "        ; oboInOwl:hasNarrowSynonym|oboInOwl:hasRelatedSynonym|oboInOwl:hasExactSynonym|rdfs:label ?label\n",
    "    }\n",
    "    \"\"\")\n",
    "    def remove_parenthetical_notes(label):\n",
    "        label = re.sub(r\"\\s\\(.*\\)\",\"\", label)\n",
    "        label = re.sub(r\"\\s\\[.*\\]\",\"\", label)\n",
    "        assert(len(label) > 0)\n",
    "        return label\n",
    "    return list(set([remove_parenthetical_notes(str(r[1])) for r in qres]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "with open(\"annotation_keywords\", \"w+\") as f:\n",
    "    for item in get_annotation_keywords():\n",
    "        f.write(\"{}\\n\".format(item))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def str_escape(s):\n",
    "    return json.dumps(s)[1:-1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "@lrudecorator(500)\n",
    "def resolve_keyword(keyword):\n",
    "    query = \"\"\"\n",
    "    prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "    prefix obo: <http://purl.obolibrary.org/obo/>\n",
    "    prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "    SELECT ?entity\n",
    "    WHERE {\n",
    "        # only resolve diseases by infectious agent\n",
    "        ?entity rdfs:subClassOf* obo:DOID_0050117\n",
    "        ; oboInOwl:hasNarrowSynonym|oboInOwl:hasRelatedSynonym|oboInOwl:hasExactSynonym|rdfs:label ?label\n",
    "        FILTER regex(?label, \"^(\"\"\" + str_escape(re.escape(keyword)) + str_escape(\"(\\s[\\[\\(].*[\\]\\)])*\") + \"\"\")$\", \"i\")\n",
    "    }\n",
    "    \"\"\"\n",
    "    qres = list(disease_ontology.query(query))\n",
    "    if len(qres) == 0:\n",
    "        print(\"no match for\", keyword.encode('ascii', 'xmlcharrefreplace'))\n",
    "    elif len(qres) > 1:\n",
    "        print(\"multiple matches for\", keyword.encode('ascii', 'xmlcharrefreplace'))\n",
    "        print(qres)\n",
    "    return qres"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[(rdflib.term.URIRef('http://purl.obolibrary.org/obo/DOID_4325'))]"
      ]
     },
     "execution_count": 62,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "resolve_keyword(\"ebola virus disease\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def iterate_infectious_disease_articles(collection, update_collection = False, no_reannotation = False):\n",
    "    keyword_annotator = KeywordAnnotator(keywords=get_annotation_keywords())\n",
    "    total_article_count = 0\n",
    "    article_with_body_count = 0\n",
    "    infectious_disease_article_count = 0\n",
    "    query = {}\n",
    "    if no_reannotation == False:\n",
    "        query = {\n",
    "            \"articles.\"\n",
    "        }\n",
    "    for article in collection.find():\n",
    "        total_article_count += 1\n",
    "        pc_article = pubcrawler.Article(article)\n",
    "        if pc_article.article_type() == \"research-article\":\n",
    "            body = pc_article.get_text_from_tags('body')\n",
    "            if len(body) > 0:\n",
    "                article_with_body_count += 1\n",
    "                anno_doc = AnnoDoc(body)\n",
    "                anno_doc.add_tier(keyword_annotator)\n",
    "                infectious_diseases = [\n",
    "                    (disease.text, resolve_keyword(disease.text))\n",
    "                    for disease in anno_doc.tiers['keywords'].spans\n",
    "                ]\n",
    "                if len(infectious_diseases) > 0:\n",
    "                    infectious_disease_article_count += 1\n",
    "                    #print(infectious_disease_article_count, \"/\", total_article_count, \",\", article_with_body_count)\n",
    "                    yield article, infectious_diseases"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "db = pymongo.MongoClient('localhost')['pmc']\n",
    "articles = db.articlesubset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 373,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "cursor = articles.find()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 376,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "10000"
      ]
     },
     "execution_count": 376,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "articles.count()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "keyword_annotator = KeywordAnnotator(keywords=get_annotation_keywords())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 305,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def annotated_keyword_list_to_dict(keyword_list):\n",
    "    keyword_dict = {}\n",
    "    for keyword_entity in keyword_list:\n",
    "        keyword, uri = keyword_entity\n",
    "        if keyword in keyword_dict:\n",
    "            continue\n",
    "        else:\n",
    "            keyword_dict[keyword] = uri[0].entity.toPython()\n",
    "    return(keyword_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 356,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def write_article_meta_to_mongo(article):\n",
    "    pc_article = pubcrawler.Article(article)\n",
    "    anno_doc = AnnoDoc(pc_article.body)\n",
    "    anno_doc.add_tier(keyword_annotator)\n",
    "    infectious_diseases = [\n",
    "        (disease.text, resolve_keyword(disease.text))\n",
    "        for disease in anno_doc.tiers['keywords'].spans\n",
    "    ]\n",
    "    disease_ontology_keywords = None if len(infectious_diseases) == 0 else annotated_keyword_list_to_dict(infectious_diseases)\n",
    "    print(disease_ontology_keywords)\n",
    "    articles.update_one({'_id': 'test'},\n",
    "                        {\n",
    "                        '$set':\n",
    "                            {\n",
    "                            'meta':\n",
    "                                {\n",
    "                                'article-ids': pc_article.pub_ids(),\n",
    "                                'article-type': pc_article.article_type(),\n",
    "                                # 'pub-dates': pc_article.pub_dates()\n",
    "                                # Need to fix stuff with dates in Mongo\n",
    "                                'keywords': pc_article.keywords()\n",
    "                                },\n",
    "                            'annotations':\n",
    "                                {\n",
    "                                'disease-ontology-keywords': disease_ontology_keywords\n",
    "                                }\n",
    "                            },\n",
    "                        })"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 364,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<pymongo.results.InsertOneResult at 0x11af04bd0>"
      ]
     },
     "execution_count": 364,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "x = cursor.next()\n",
    "x['_id'] = \"test\"\n",
    "articles.delete_one({'_id': \"test\"})\n",
    "articles.insert_one(x)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 365,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'_id': 'test',\n",
       " 'nxml': '<!DOCTYPE article PUBLIC \"-//NLM//DTD JATS (Z39.96) Journal Archiving and Interchange DTD v1.0 20120330//EN\" \"JATS-archivearticle1.dtd\">\\n<article xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" article-type=\"research-article\"><?properties open_access?><front><journal-meta><journal-id journal-id-type=\"nlm-ta\">J Pathol</journal-id><journal-id journal-id-type=\"publisher-id\">path</journal-id><journal-title-group><journal-title>The Journal of Pathology</journal-title></journal-title-group><issn pub-type=\"ppub\">0022-3417</issn><issn pub-type=\"epub\">1096-9896</issn><publisher><publisher-name>John Wiley &#x00026; Sons, Ltd.</publisher-name></publisher></journal-meta><article-meta><article-id pub-id-type=\"pmid\">20229506</article-id><article-id pub-id-type=\"pmc\">3262968</article-id><article-id pub-id-type=\"doi\">10.1002/path.2696</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Original Paper</subject></subj-group></article-categories><title-group><article-title>Driver mutations in <italic>TP53</italic> are ubiquitous in high grade serous carcinoma of the ovary</article-title></title-group><contrib-group><contrib contrib-type=\"author\"><name><surname>Ahmed</surname><given-names>Ahmed Ashour</given-names></name><xref ref-type=\"aff\" rid=\"au1\">1</xref><xref ref-type=\"aff\" rid=\"au2\">2</xref></contrib><contrib contrib-type=\"author\"><name><surname>Etemadmoghadam</surname><given-names>Dariush</given-names></name><xref ref-type=\"aff\" rid=\"au3\">3</xref></contrib><contrib contrib-type=\"author\"><name><surname>Temple</surname><given-names>Jillian</given-names></name><xref ref-type=\"aff\" rid=\"au1\">1</xref><xref ref-type=\"aff\" rid=\"au2\">2</xref></contrib><contrib contrib-type=\"author\"><name><surname>Lynch</surname><given-names>Andy G</given-names></name><xref ref-type=\"aff\" rid=\"au2\">2</xref></contrib><contrib contrib-type=\"author\"><name><surname>Riad</surname><given-names>Mohamed</given-names></name><xref ref-type=\"aff\" rid=\"au4\">4</xref></contrib><contrib contrib-type=\"author\"><name><surname>Sharma</surname><given-names>Raghwa</given-names></name><xref ref-type=\"aff\" rid=\"au5\">5</xref></contrib><contrib contrib-type=\"author\"><name><surname>Stewart</surname><given-names>Colin</given-names></name><xref ref-type=\"aff\" rid=\"au6\">6</xref></contrib><contrib contrib-type=\"author\"><name><surname>Fereday</surname><given-names>Sian</given-names></name><xref ref-type=\"aff\" rid=\"au3\">3</xref></contrib><contrib contrib-type=\"author\"><name><surname>Caldas</surname><given-names>Carlos</given-names></name><xref ref-type=\"aff\" rid=\"au2\">2</xref><xref ref-type=\"aff\" rid=\"au7\">7</xref></contrib><contrib contrib-type=\"author\"><name><surname>deFazio</surname><given-names>Anna</given-names></name><xref ref-type=\"aff\" rid=\"au8\">8</xref></contrib><contrib contrib-type=\"author\"><name><surname>Bowtell</surname><given-names>David</given-names></name><xref ref-type=\"aff\" rid=\"au3\">3</xref><xref ref-type=\"aff\" rid=\"au9\">9</xref><xref ref-type=\"corresp\" rid=\"cor1\">*</xref></contrib><contrib contrib-type=\"author\"><name><surname>Brenton</surname><given-names>James D</given-names></name><xref ref-type=\"aff\" rid=\"au1\">1</xref><xref ref-type=\"aff\" rid=\"au2\">2</xref><xref ref-type=\"corresp\" rid=\"cor2\">*</xref></contrib></contrib-group><aff id=\"au1\"><label>1</label><institution>Functional Genomics of Ovarian Cancer Laboratory, Cancer Research UK Cambridge Research Institute, Li Ka Shing Centre</institution><addr-line>Robinson Way, Cambridge, CB2 0RE, UK</addr-line></aff><aff id=\"au2\"><label>2</label><institution>Department of Oncology, University of Cambridge, Hutchison/MRC Research Centre</institution><addr-line>Hills Road, Cambridge, CB2 0XZ, UK</addr-line></aff><aff id=\"au3\"><label>3</label><institution>Peter MacCallum Cancer Centre</institution><addr-line>Melbourne, Victoria, Australia</addr-line></aff><aff id=\"au4\"><label>4</label><institution>Department of Obstetrics and Gynaecology, El-Galaa Maternity Teaching Hospital</institution><addr-line>41 July 26th Street, Cairo 11511, Egypt</addr-line></aff><aff id=\"au5\"><label>5</label><institution>Department of Anatomical Pathology, University of Sydney and University of Western Sydney at Westmead Hospital</institution><addr-line>Sydney, Australia</addr-line></aff><aff id=\"au6\"><label>6</label><institution>Department of Pathology, King Edward Memorial Hospital</institution><addr-line>Western Australia, Australia</addr-line></aff><aff id=\"au7\"><label>7</label><institution>Functional Genomics of Breast Cancer Laboratory, Cancer Research UK Cambridge Research Institute, Li Ka Shing Centre</institution><addr-line>Robinson Way, Cambridge, CB2 0RE, UK</addr-line></aff><aff id=\"au8\"><label>8</label><institution>Westmead Institute for Cancer Research, University of Sydney at Westmead Millennium Institute, Westmead Hospital</institution><addr-line>New South Wales, Australia</addr-line></aff><aff id=\"au9\"><label>9</label><institution>Department of Biochemistry and Molecular Biology, University of Melbourne</institution><addr-line>Parkville, Victoria, Australia</addr-line></aff><author-notes><corresp id=\"cor1\">*Correspondence to: Professor David Bowtell, Research Division, Peter MacCallum Cancer Centre, Locked Bag 1, A\\'Beckett Street, Melbourne, 8006, Victoria, Australia. e-mail: <email>d.bowtell@petermac.org</email></corresp><corresp id=\"cor2\">*Correspondence to: Dr James D Brenton, Functional Genomics of Ovarian Cancer Laboratory, Cancer Research UK Cambridge Research Institute, Li Ka Shing Centre, Robinson Way, Cambridge, CB2 0RE, UK. e-mail: <email>james.brenton@cancer.org.uk</email></corresp><fn><p>No conflicts of interest were declared.</p></fn></author-notes><pub-date pub-type=\"ppub\"><month>5</month><year>2010</year></pub-date><volume>221</volume><issue>1</issue><fpage>49</fpage><lpage>56</lpage><history><date date-type=\"received\"><day>09</day><month>12</month><year>2009</year></date><date date-type=\"rev-recd\"><day>24</day><month>1</month><year>2010</year></date><date date-type=\"accepted\"><day>26</day><month>1</month><year>2010</year></date></history><permissions><copyright-statement>Copyright &#x000a9; 2010 Pathological Society of Great Britain and Ireland. Published by John Wiley &#x00026; Sons, Ltd.</copyright-statement><copyright-year>2010</copyright-year><license license-type=\"open-access\" xlink:href=\"http://creativecommons.org/licenses/by/2.5/\"><license-p>Re-use of this article is permitted in accordance with the Creative Commons Deed, Attribution 2.5, which does not permit commercial exploitation.</license-p></license></permissions><abstract><p>Numerous studies have tested the association between <italic>TP53</italic> mutations in ovarian cancer and prognosis but these have been consistently confounded by limitations in study design, methodology, and/or heterogeneity in the sample cohort. High-grade serous (HGS) carcinoma is the most clinically important histological subtype of ovarian cancer. As these tumours may arise from the ovary, Fallopian tube or peritoneum, they are collectively referred to as high-grade pelvic serous carcinoma (HGPSC). To identify the true prevalence of <italic>TP53</italic> mutations in HGPSC, we sequenced exons 2&#x02013;11 and intron&#x02013;exon boundaries in tumour DNA from 145 patients. HGPSC cases were defined as having histological grade 2 or 3 and FIGO stage III or IV. Surprisingly, pathogenic <italic>TP53</italic> mutations were identified in 96.7% (<italic>n</italic> = 119/123) of HGPSC cases. Molecular and pathological review of mutation-negative cases showed evidence of p53 dysfunction associated with copy number gain of <italic>MDM2</italic> or <italic>MDM4</italic>, or indicated the exclusion of samples as being low-grade serous tumours or carcinoma of uncertain primary site. Overall, p53 dysfunction rate approached 100% of confirmed HGPSCs. No association between <italic>TP53</italic> mutation and progression-free or overall survival was found. From this first comprehensive mapping of <italic>TP53</italic> mutation rate in a homogeneous group of HGPSC patients, we conclude that mutant <italic>TP53</italic> is a driver mutation in the pathogenesis of HGPSC cancers. Because <italic>TP53</italic> mutation is almost invariably present in HGPSC, it is not of substantial prognostic or predictive significance. Copyright &#x000a9; 2010 Pathological Society of Great Britain and Ireland. Published by John Wiley &#x00026; Sons, Ltd.</p></abstract><kwd-group><kwd>p53</kwd><kwd>high-grade pelvic serous carcinoma</kwd><kwd>ovarian cancer</kwd><kwd>DNA sequence analysis</kwd><kwd>array-based genomic hybridization</kwd><kwd>histopathology</kwd><kwd>clinical outcome</kwd><kwd>BRCA</kwd></kwd-group></article-meta></front><body><sec><title>Introduction</title><p>The gene <italic>TP53</italic> that encodes the tumour suppressor protein p53 is amongst the most commonly mutated genes in human cancer [<xref ref-type=\"bibr\" rid=\"b1\">1</xref>]. The frequent presence of <italic>TP53</italic> mutations in ovarian cancer has been suggested for almost two decades [<xref ref-type=\"bibr\" rid=\"b2\">2</xref>,<xref ref-type=\"bibr\" rid=\"b3\">3</xref>] and during this time, approximately 70 publications have described the relationship between <italic>TP53</italic> mutation status and clinical outcome [<xref ref-type=\"bibr\" rid=\"b4\">4</xref>]. For such an intensively studied question, there is a surprising degree of inconsistency in the published data for both the prevalence of mutation and association with prognosis or chemotherapy response.</p><p>A recent analysis of 64 publications reporting the relationship of <italic>TP53</italic> mutation to clinical outcome found that only six studies fulfilled minimum quality criteria for the method of detecting mutations or assignment of clinical response [<xref ref-type=\"bibr\" rid=\"b5\">5</xref>]. One of the most consistent limitations of previous p53 studies is a reliance on p53 immunostaining as a surrogate marker for <italic>TP53</italic> mutation [<xref ref-type=\"bibr\" rid=\"b4\">4</xref>], even though this leads to an unacceptable number of misclassifications [<xref ref-type=\"bibr\" rid=\"b6\">6</xref>]. Sequencing of tumour DNA is the gold standard to detect different types of mutation and relate these to clinical outcome. However, a recent review of 12 studies that sequenced <italic>TP53</italic> in ovarian cancer found variable mutation frequencies of 15&#x02013;80% [<xref ref-type=\"bibr\" rid=\"b5\">5</xref>]. Inconsistencies between these studies may relate to whether the whole gene was or only the most commonly mutated exons were sequenced. Additionally, most studies were confounded by the inclusion of cases with different histological types, stages, and grades of tumours [<xref ref-type=\"bibr\" rid=\"b4\">4</xref>].</p><p>Determining the true prevalence of <italic>TP53</italic> mutation is critical for understanding the pathogenesis of high-grade serous cancers arising as ovarian, Fallopian tube or primary peritoneal cancer. Although previously assumed to arise from the ovary, pathological examination of the Fallopian tube from carriers with <italic>BRCA1</italic> or <italic>BRCA2</italic> mutation has demonstrated that tubal intraepithelial carcinomas (TICs) arising from secretory epithelial cells in the fimbria are the early invasive lesion in high-grade serous (HGS) carcinoma [<xref ref-type=\"bibr\" rid=\"b7\">7</xref>,<xref ref-type=\"bibr\" rid=\"b8\">8</xref>]. In addition, TICs are found in patients without evidence of germline <italic>BRCA1</italic> or <italic>BRCA2</italic> lesions and show concomitant <italic>TP53</italic> mutations in the tube and in pelvic metastases, further supporting the model that TIC represents the origin of HGS carcinoma. HGS cancers may also arise from the peritoneum and although these may have different epidemiological risk factor profiles [<xref ref-type=\"bibr\" rid=\"b9\">9</xref>], they exist in a clinical continuum with ovarian and Fallopian tube cancers. Hence we use the term high-grade pelvic serous carcinoma (HGPSC) [<xref ref-type=\"bibr\" rid=\"b10\">10</xref>] to recognize the frequent lack of clarity regarding primary site in ovarian/Fallopian tube/peritoneal cancers and their overall biological and clinical similarity.</p><p>To address the limitations of previous studies and determine accurately the relationship between <italic>TP53</italic> mutation and HGPSC, we designed pilot and validation studies to measure the mutation frequency in a homogeneous group of advanced stage HGPSC cancers from a large population-based cohort.</p></sec><sec sec-type=\"materials|methods\"><title>Materials and methods</title><sec><title>Samples and study design</title><p>The results of this study are presented in accordance with reporting recommendations for tumour marker prognostic studies (REMARK) criteria [<xref ref-type=\"bibr\" rid=\"b11\">11</xref>]. Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD3\">Figure 1</xref> (Supporting information) summarizes the flow of patients through the pilot and validation study, including the number of patients included in each stage of the analysis and reasons for exclusion. Patients (<italic>n</italic> = 123) were selected from the Australian Ovarian Cancer Study (AOCS), a population-based, multicentre translational study that comprised prospective collection of bio-specimens and clinical and epidemiological data from patients with primary epithelial ovarian, primary peritoneal, and Fallopian tube cancer diagnosed between 2001 and 2005. Patients treated at Westmead Hospital, Sydney between 1992 and 2002 (<italic>n</italic> = 22) were also included in the pilot cohort, making an overall sample size of 145 HGPSCs. All women in the pilot cohort received first-line platinum-based chemotherapy (cisplatin or carboplatin) with (<italic>n</italic> = 39) or without (<italic>n</italic> = 6) the taxane paclitaxel. Similarly, the majority of patients in the validation set received combination platinum&#x02013;taxane treatment (<italic>n</italic> = 93) or platinum monotherapy (<italic>n</italic> = 4). Three patients (<italic>n</italic> = 3) did not receive any chemotherapy, including one patient with stage I cancer who declined treatment and two patients who were too ill to receive chemotherapy. All patients were prospectively consented using a protocol approved by the human research ethics committees at multiple participating clinical and research centres. Further details of the AOCS cohort can be found at <ext-link ext-link-type=\"uri\" xlink:href=\"http://www.aocstudy.org\">http://www.aocstudy.org</ext-link>. Statistical design and power calculations are described in the Supporting information, Supplementary methods.</p></sec><sec><title><italic>TP53</italic> sequencing</title><p>Coding <italic>TP53</italic> exons 2&#x02013;11 were amplified using primers that encompassed the entire exon and exon&#x02013;intron boundaries. Purified PCR products were se- quenced using an ABI 3100 genetic analyser (Applied Biosystems, Warrington, UK). Sequencing reactions were performed in forward and reverse directions. Mutational analysis was performed using SeqScape&#x000ae; Software v2.6 (Applied Biosystems). Sequence traces were aligned against the <italic>TP53</italic> reference sequence NC_000017. Independent, blinded sequencing of cases from the pilot study was performed by the CR-UK Mutation Detection Facility, Clinical Sciences Building, Level 6, St James\\'s University Hospital, Leeds, UK.</p></sec><sec><title>Characterization of mutation-negative HGPSC cancers</title><p>For single nucleotide polymorphism (SNP) mapping assay, SNP microarray data analysis, quantitative PCR (qPCR), and immunohistochemistry, the reader is referred to the Supporting information, Supplementary methods.</p></sec><sec><title>aCGH data</title><p>Raw data files from aCGH experiments are available from the Gene Expression Omnibus <ext-link ext-link-type=\"uri\" xlink:href=\"http://www.ncbi.nlm.nih.gov/geo/\">http://www.ncbi.nlm.nih.gov/geo/</ext-link> (Accession number GSE19416).</p></sec></sec><sec><title>Results</title><sec><title>Both treatment-responsive and primary resistant HGPSCs have a high frequency of <italic>TP53</italic> mutation</title><p>To estimate the number of samples required to detect a significant association between <italic>TP53</italic> mutations and survival, we performed a pilot study with 45 HGPSCs (ovarian, Fallopian tube, and primary peritoneal). The cohort included 20 chemotherapy-resistant patients who had tumour progression within 6 months of completion of treatment and 25 responsive patients who did not progress for at least 9 months. The two groups showed no statistically significant differences in age, tumour stage, residual disease or type of chemotherapy (<xref ref-type=\"table\" rid=\"tbl1\">Table 1</xref>). Review of pathology reports showed one patient with low-grade serous (LGS) carcinoma (case 450) and this case was excluded from further analysis. Unexpectedly, pathogenic <italic>TP53</italic> mutations were identified in 43 out of 44 patients included in the study (Supporting information, Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD1\">Table 1</xref> and Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD3\">Figure 1</xref>). Mutations were compared to Release 13 of the IARC <italic>TP53</italic> mutation database [<xref ref-type=\"bibr\" rid=\"b12\">12</xref>] and 20 cases had rare <italic>TP53</italic> mutations, defined as less than ten occurrences (Supporting information, Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD1\">Table 1</xref>). From these, 12 were randomly selected and independently confirmed by a clinical reference laboratory blinded to our results (data not shown). Further analysis of mutations is described below.</p><table-wrap id=\"tbl1\" position=\"float\"><label>Table 1</label><caption><p>A comparison between the clinical characteristics of the chemotherapy-resistant and -responsive patients in the pilot study</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th rowspan=\"1\" colspan=\"1\"/><th align=\"center\" rowspan=\"1\" colspan=\"1\">Resistant (<italic>n</italic> = 20)</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Responsive (<italic>n</italic> = 25)</th><th align=\"center\" rowspan=\"1\" colspan=\"1\"><italic>p</italic> (test)</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">PFS (months)</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Median</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">10.2</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">23.3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x0003c; 0.001 (log rank)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">OS (months)</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Median</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">21.4</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">57.3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x0003c; 0.001 (log rank)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Age (years)</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Median</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">58.26</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">58.7</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">0.7 (Wilcoxon rank sum)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Range</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">22.9&#x02013;78</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">38.1&#x02013;77.7</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Stage (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;III</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">23</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">0.5 (Fisher)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;IV</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02014;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">2</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Chemotherapy (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Adjuvant</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">25</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">0.4 (Fisher)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Neo-adjuvant</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Residual disease (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;No residual disease</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">0.2 (Fisher)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;&#x02264;1 cm</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">8</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;&#x0003e; 1 cm and &#x02264;2 cm</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">5</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;&#x0003e; 2 cm</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">6</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Unknown</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">TP53 (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Wild type</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">0.4 (Fisher)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Mutant</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">25</td><td rowspan=\"1\" colspan=\"1\"/></tr></tbody></table><table-wrap-foot><fn><p>OS = overall survival; PFS = progression-free survival, defined as the period from diagnosis to the date of documented progression. Six samples (2 resistant, 4 responsive) did not have precise values for size &#x0003c; 2 cm.</p></fn></table-wrap-foot></table-wrap></sec><sec><title>Confirmation of high frequency of <italic>TP53</italic> mutation in an unselected population-based cohort of serous cancers</title><p>To exclude the possibility that the high rate of <italic>TP53</italic> mutations was biased by the selection criteria used in the pilot study, we obtained additional serous cancer cases that were a representative subset of a large population-based series (<xref ref-type=\"table\" rid=\"tbl2\">Table 2</xref>). Mutations were detected in 76/82 advanced-stage HGPSCs from these cases, with no significant difference between the percentage of mutations in the matched validation study compared with the pilot study (<italic>p</italic> = 0.3; Fisher exact test).</p><table-wrap id=\"tbl2\" position=\"float\"><label>Table 2</label><caption><p>Clinical characteristics of the validation study patients</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th rowspan=\"1\" colspan=\"1\"/><th align=\"center\" rowspan=\"1\" colspan=\"1\">Validation set (<italic>n</italic> = 82</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">PFS (months)</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Median</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">13.3</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">OS (months)</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Median</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">34</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Age (years)</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Median</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">61.3</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Range</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">40.4&#x02013;79.3</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Stage (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;III</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">73</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;IV</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">9</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Grade (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;2</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">18</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">63</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Unknown</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Chemotherapy (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Adjuvant</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">71</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Neo-adjuvant</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">11</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Residual disease (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;No residual disease</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">16</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;&#x02264;1 cm</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">26</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;&#x0003e; 1 cm and &#x02264;2 cm</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">5</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;&#x0003e; 2 cm</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">25</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Unknown</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">10</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">TP53 (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Wild type</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">6</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Mutant</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">76</td></tr></tbody></table><table-wrap-foot><fn><p>OS = overall survival; PFS = progression-free survival.</p></fn></table-wrap-foot></table-wrap></sec><sec><title><italic>TP53</italic> mutations arise at high frequency in exons 2&#x02013;4 and 9&#x02013;11 in HGPSC carcinoma</title><p>We analysed the 119 mutations found in 126 HGPSC cases from the pilot and validation set (Supporting information, Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD1\">Table 1</xref>). As expected, the majority of <italic>TP53</italic> mutations were missense mutations (56.3%, <italic>n</italic> = 67/119) occurring in exons 4&#x02013;8 (Supporting information, Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD1\">Table 1</xref>). The most frequently mutated codon was 273 (<italic>n</italic> = 9/67 cases, 13.4%) comprising R273C, R273H, and R273L mutants. Analysis of exon and splice sites for exons 2&#x02013;4 and 9&#x02013;11, outside the DNA binding domain, showed 12 (10.1%) and 10 (8.4%) mutations, respectively, which taken together comprised 18.5% (<italic>n</italic> = 22/119) of all mutations in our series. Of note, a total of eight mutations affected exon 10 (6.7%) in the tetramerization domain (codons 324&#x02013;356) and all were predicted to cause truncation of the protein. Four of these mutations occurred at g.16915 (codon 342). In 717 serous sub-type ovarian, Fallopian, and primary peritoneal cancer cases in Release 13 of the IARC <italic>TP53</italic> database, 30 (4.2%) had mutations in exons 2&#x02013;4, 18 had mutations affecting exon 10 alone (2.5%), and 20 (2.8%) had mutations in exons 9&#x02013;11. Twenty-two mutations from our series (<italic>n</italic> = 22/119, 18.5%) were not present in the IARC database. With the exception of one, all were insertions or deletions (Supporting information, Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD1\">Table 1</xref>). The remaining novel mutation in case 434 was at position g.11602_11603GC &#x0003e; AA, creating a unique tandem nonsense mutation.</p></sec><sec><title>Low-stage HGPSC cases have a high frequency of <italic>TP53</italic> mutations</title><p>The very high prevalence of mutation in high-stage cases suggested that <italic>TP53</italic> was an early event in the pathogenesis of HGPSC. From the validation set, 15 cases were stage 1 or 2 HGPSC and 13/15 (86.7%) had <italic>TP53</italic> mutation (Supporting information, Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD1\">Table 1</xref>). The negative cases 41307 and 60111 had mixed endometrioid/HGS or grade 1 pathology, respectively. As for high-stage cases, the majority of mutations (<italic>n</italic> = 10/13; 76.9%) were missense mutations in exons 5&#x02013;8. There was one nonsense mutation in exon 10 (7.7%) and two splice junction mutations (15.4%).</p></sec><sec><title>Mutation-negative cases have alternative mechanisms of p53 dysfunction</title><p>We considered the possibility that <italic>TP53</italic> mutations in the seven mutation-negative samples (pilot <italic>n</italic> = 1, validation <italic>n</italic> = 6) were missed because of contamination with normal DNA. Although all samples with significant stroma were needle-dissected prior to sequencing, pathological review of the mutation-negative cases indicated that they did not have increased stroma compared with mutation-positive cases (data not shown). Additionally, array-based comparative genomic hybridization (aCGH) showed that all mutation-negative cases had DNA copy number abnormalities (DCNAs), and the least aberrant, case 533, had distinct loss of chromosome arm 1p (<xref ref-type=\"fig\" rid=\"fig01\">Figure 1A</xref>). No samples showed evidence of homozygous deletion of the <italic>TP53</italic> locus (<xref ref-type=\"fig\" rid=\"fig01\">Figure 1</xref>B).</p><fig id=\"fig01\" position=\"float\"><label>Figure 1</label><caption><p>Characterization of <italic>TP53</italic> mutation-negative cancer samples. (A) Summary plots of whole genome DNA copy number data for seven mutation-negative samples. Red indicates chromosomal gain and blue shows regions of chromosomal loss. (B) High-resolution DNA copy number analysis of <italic>MDM2, MDM4</italic>, and <italic>TP53</italic> loci in each sample. The dotted line indicates the position of the gene within the region. Samples with chromosomal gain or loss are marked with a red or blue asterisk, respectively. (C) Confirmation of <italic>MDM2</italic> and <italic>MDM4</italic> copy number gain using quantitative PCR of tumour DNA. Samples with gain are indicated in red. (D) Immunohistochemical staining of p53 protein in the following selected mutation-negative cases: case 41358; case 60258 (<italic>MDM2</italic> gain); case 22029 (<italic>MDM4</italic> gain); and case 60049 (<italic>MDM4</italic> gain).</p></caption><graphic xlink:href=\"path0221-0049-f1\"/></fig><p>Pathology review of the mutation-negative cases resulted in the re-classification of case 60214 as LGS carcinoma, case 23221 as high-grade carcinoma of uncertain primary site, and case 533 as mixed LGS/HGS type (Supporting information, Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD2\">Table 2</xref>). Furthermore, aCGH analysis of cases 533 and 60214 did not show high-frequency DCNAs that are typical of HGPSC [<xref ref-type=\"bibr\" rid=\"b11\">11</xref>]. The original diagnosis of HGPSC was confirmed in the remaining four cases (22029, 41358, 60049, and 60258). To further exclude the possibility of pathological misclassification of endometrioid, clear cell or LGS tumours in the mutation-negative cases, we also tested for mutations commonly associated with these subtypes in <italic>KRAS</italic> (exon 2), <italic>BRAF</italic> (exon 15), <italic>CTNNB1</italic> (exon 3), and <italic>PIK3CA</italic> (exons 10 and 21). Case 60214 had a missense mutation (G12C) in exon 2 of <italic>KRAS</italic>, consistent with LGS origin. No other mutations were detected (data not shown).</p><p>Other mutation-independent mechanisms of inactivation of the p53 pathway were investigated. p53 is targeted for degradation through the activity of MDM2 protein and is also regulated by the related MDM4 protein [<xref ref-type=\"bibr\" rid=\"b13\">13</xref>,<xref ref-type=\"bibr\" rid=\"b14\">14</xref>]. Amplification of <italic>MDM2</italic> is associated with loss of p53 activity in some solid tumours [<xref ref-type=\"bibr\" rid=\"b15\">15</xref>,<xref ref-type=\"bibr\" rid=\"b16\">16</xref>]. aCGH analysis of the remaining four cases showed copy number gain of <italic>MDM4</italic> in cases 22029 and 60049, and of <italic>MDM2</italic> in case 60258 (<xref ref-type=\"fig\" rid=\"fig01\">Figure 1B</xref>). These observations were confirmed by qPCR (<xref ref-type=\"fig\" rid=\"fig01\">Figure 1C</xref>). Interestingly, both cases with <italic>MDM4</italic> gain showed heterogeneous p53 immunostaining, including cytoplasmic staining (<xref ref-type=\"fig\" rid=\"fig01\">Figure 1D</xref>). Case 60258 with <italic>MDM2</italic> gain showed strong nuclear staining for p53 protein (<xref ref-type=\"fig\" rid=\"fig01\">Figure 1D</xref>). Interestingly, the remaining case with no MDM2 or MDM4 gain (41358) showed strong nuclear p53 immunostaining (<xref ref-type=\"fig\" rid=\"fig01\">Figure 1D</xref>).</p><p>In summary, the overall <italic>TP53</italic> mutation rate for selected HGPSC cases in the pilot and validation cohorts was 94.4% (<italic>n</italic> = 119/126). After removing the three mutation-negative cases with atypical or LGS histology, the adjusted overall rate was 96.7% (<italic>n</italic> = 119/123). In the remaining four mutation-negative HGPSC cases, three showed evidence of p53 dysfunction with increased p53 nuclear staining or cytoplasmic mislocalization that was correlated with gains at the <italic>MDM2/4</italic> loci.</p></sec><sec><title><italic>TP53</italic> mutation type is not prognostic in HGPSC</title><p>It has been previously reported that the type of <italic>TP53</italic> mutation may predict outcome in ovarian cancer [<xref ref-type=\"bibr\" rid=\"b17\">17</xref>]. Although our study was not adequately powered to detect small effects, we tested for trends in the data. For this analysis, we used all presumed HGPSC cases from the pilot study and the validation cohort (<italic>n</italic> = 127, <italic>n</italic> = 120 had <italic>TP53</italic> mutation; <xref ref-type=\"table\" rid=\"tbl3\">Tables 3</xref> and <xref ref-type=\"table\" rid=\"tbl4\">4</xref>). There was no indication of significant association between the frequency of <italic>TP53</italic> mutation or the type of mutation (missense versus not) and progression-free survival or overall survival. Similarly, whether or not the missense mutation affected the DNA binding motifs formed by the L2 and L3 loops (codons 164&#x02013;194 and 237&#x02013;250, respectively) or the LSH motifs (codons 119&#x02013;135 and 272&#x02013;287) [<xref ref-type=\"bibr\" rid=\"b18\">18</xref>] had no significant effect on progression-free survival or overall survival (<xref ref-type=\"table\" rid=\"tbl3\">Tables 3</xref> and <xref ref-type=\"table\" rid=\"tbl4\">4</xref>). There was also no effect of codon 72 polymorphism (Arg, Pro or heterozygous) on prognosis in any of the HGPSCs with mutations (data not shown) or in tumours with missense mutations (<xref ref-type=\"table\" rid=\"tbl3\">Tables 3</xref> and <xref ref-type=\"table\" rid=\"tbl4\">4</xref>). There was no significant interaction between the type of missense mutation (DNA binding or not) and codon 72 polymorphism (data not shown).</p><table-wrap id=\"tbl3\" position=\"float\"><label>Table 3</label><caption><p>The type of <italic>TP53</italic> mutation does not influence progression-free survival</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\">Index category (<italic>n</italic>)</th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Comparison (<italic>n</italic>)</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Hazard ratio</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">CI</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Log-rank <italic>p</italic> value</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Mutant (120)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Wild type (7)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1.4</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.6&#x02013;3.1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.5</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Not missense (52)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Missense (68)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.9</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.6&#x02013;1.4</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.8</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">DBM (44)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">NDBM (24)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.6&#x02013;1.7</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td></tr></tbody></table><table-wrap-foot><fn><p>DBM = DNA binding mutations; NDBM = non-DNA binding mutations.</p></fn></table-wrap-foot></table-wrap><table-wrap id=\"tbl4\" position=\"float\"><label>Table 4</label><caption><p>The type of <italic>TP53</italic> mutation does not influence overall survival</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\">Index category (<italic>n</italic>)</th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Comparison (<italic>n</italic>)</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Hazard ratio</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">CI</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Log-rank <italic>p</italic> value</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Mutant (120)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Wild type (7)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.6&#x02013;4.6</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.3</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Not missense (52)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Missense (68)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.7&#x02013;1.6</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">DBM (44)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">NDBM (24)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.7&#x02013;2.4</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.4</td></tr></tbody></table><table-wrap-foot><fn><p>DBM = DNA binding mutations; NDBM = non-DNA binding mutations.</p></fn></table-wrap-foot></table-wrap></sec></sec><sec><title>Discussion</title><p>By addressing the limitations of numerous other studies of <italic>TP53</italic> mutation in ovarian cancer, we have demonstrated for the first time that HGPSCs have the highest frequency of p53 mutation of any solid cancer. Approximately two-thirds of mutations occurred in exons 5&#x02013;8, known to be mutation hotspots for <italic>TP53</italic>. By sequencing exons 2&#x02013;4 and 9&#x02013;11, we identified 22 mutations (<italic>n</italic> = 22/126, 17.5%) that would not have otherwise been identified in our series and are poorly represented in previous reports. The previously unreported mutations that we have discovered are mostly insertions/deletions predicted to result in truncated proteins, but it is notable that HGPSC cases have frequent involvement of exon 10 and specifically involvement of position g.16915 (codon 342) in the oligomerization domain of p53. This region also contains a putative nuclear export signal for p53 and a recent study of a novel missense mutation at codon 351 has shown that K351N results in significant loss of p53 tetramerization, reduced BAX activation, and reduced nuclear export of p53 [<xref ref-type=\"bibr\" rid=\"b19\">19</xref>]. Cytosolic export of p53 is required for cisplatin-induced apoptosis in A2780 cells, suggesting that combined loss of tetramerization and nuclear export is critical for platinum resistance.</p><p>Of the mutation-negative cases, approximately half were reclassified on pathological review as unrepresentative of HGPSC, including two tumours that were LGS and associated with borderline tumour. Serous cancers may follow two distinct routes to malignancy [<xref ref-type=\"bibr\" rid=\"b20\">20</xref>]. A minority of so-called type I tumours progress from borderline tumours, whereas the majority of HGS (type II) tumours arise with no evidence of LGS or borderline tumours. Borderline serous tumours and their type I invasive counterparts are generally low grade, have relatively limited genomic DNA copy number change, have frequent activating RAS pathway mutations, and have substantially lower rates of <italic>TP53</italic> mutation compared with type II tumours [<xref ref-type=\"bibr\" rid=\"b20\">20</xref>&#x02013;<xref ref-type=\"bibr\" rid=\"b23\">23</xref>]. Detailed analysis of six cases of HGPSC arising from LGS and borderline tumours showed no <italic>TP53</italic> mutation [<xref ref-type=\"bibr\" rid=\"b24\">24</xref>]. Consistent with this model, mutation-negative case 533 had both LGS and HGS components and a bland aCGH profile.</p><p>Exclusion of LGS cases and mixed high-grade serous cases increased the <italic>TP53</italic> mutation frequency to approximately 97% of HGPSCs. Of the remaining mutation-negative cases, three may have had inactivation of the p53 pathway through chromosomal gain at the <italic>MDM2</italic> or <italic>MDM4</italic> loci or potentially other genes that alter p53 function in <italic>trans</italic> such as <italic>RFWD2</italic> and <italic>RCHY1</italic> [<xref ref-type=\"bibr\" rid=\"b25\">25</xref>,<xref ref-type=\"bibr\" rid=\"b26\">26</xref>].</p><p>We are aware of over 70 publications that have sought to relate <italic>TP53</italic> mutation with clinical outcome in ovarian cancer, with conflicting conclusions about the importance of <italic>TP53</italic> mutations for tumour aggressiveness, response to treatment, and survival. There is no sign that a conclusion is at hand: since 2008, there have been a number of papers relating <italic>TP53</italic> mutation and clinical outcome [<xref ref-type=\"bibr\" rid=\"b27\">27</xref>&#x02013;<xref ref-type=\"bibr\" rid=\"b30\">30</xref>], with evidence presented for [<xref ref-type=\"bibr\" rid=\"b29\">29</xref>] and against [<xref ref-type=\"bibr\" rid=\"b28\">28</xref>] mutation being a significant prognostic factor. A recent meta-analysis of 62 studies concluded that <italic>TP53</italic> mutation had a modest impact on survival in ovarian cancer but the effect was insufficient to support clinical application [<xref ref-type=\"bibr\" rid=\"b4\">4</xref>]. Whilst meta-analyses such as that of de Graeff <italic>et al</italic> [<xref ref-type=\"bibr\" rid=\"b4\">4</xref>] have the potential to detect subtle trends that may be missed in all but the largest individual studies, they are confounded if there is a substantial false-negative rate, as our findings indicate. Given the near ubiquitous occurrence of <italic>TP53</italic> mutation, it difficult to conclude that it can be of significant prognostic or predictive significance in HGPSC. We have not, however, excluded the possibility that specific mutations can influence prognosis or response to therapy, an important consideration given the mounting evidence that some <italic>TP53</italic> mutations have specific neomorphic functions [<xref ref-type=\"bibr\" rid=\"b1\">1</xref>]. It is thought that inactivation of the p53 pathway is a common, perhaps mandatory, event in all solid cancers [<xref ref-type=\"bibr\" rid=\"b31\">31</xref>]. What is particularly unusual about HGPSC is an almost total reliance on mutation <italic>TP53</italic> itself for pathway inactivation, perhaps reflecting the requirement of certain tissues for neomorphic mutations for their transformation [<xref ref-type=\"bibr\" rid=\"b1\">1</xref>]. Future work will need to address interactions between specific mutations and other loci such as <italic>PTEN</italic> and <italic>CDKN2A</italic> [<xref ref-type=\"bibr\" rid=\"b32\">32</xref>,<xref ref-type=\"bibr\" rid=\"b33\">33</xref>].</p><p>Our findings have important implications for the new understanding of the pathogenesis of HGPSC. p53 immunostaining and mutation appear to be a feature of early-stage Fallopian tube lesions from <italic>BRCA1/2</italic> mutation carriers [<xref ref-type=\"bibr\" rid=\"b7\">7</xref>], suggesting that p53 dysfunction is essential for early tumourigenesis of HGPSC. The notion that <italic>TP53</italic> mutation is required to allow survival of <italic>BRCA</italic>-deficient ovarian precursor lesions is consistent with the extremely high rates of <italic>TP53</italic> mutation in breast cancers arising in <italic>BRCA1</italic> mutant women [<xref ref-type=\"bibr\" rid=\"b34\">34</xref>,<xref ref-type=\"bibr\" rid=\"b35\">35</xref>]. BRCA dysfunction may also occur in sporadic HGPSC via a variety of inactivating mechanisms but the very high rate of <italic>TP53</italic> mutation in our series of sporadic HGPSC cases suggests that most are &#x02018;BRCA-like&#x02019; with deficiencies for <italic>BRCA</italic> or closely related DNA repair pathways [<xref ref-type=\"bibr\" rid=\"b36\">36</xref>].</p><p>It has become increasingly clear that ovarian cancer is a series of distinctly different diseases with different aetiologies [<xref ref-type=\"bibr\" rid=\"b23\">23</xref>,<xref ref-type=\"bibr\" rid=\"b37\">37</xref>] and that the development of reliable biomarkers must be subtype-specific [<xref ref-type=\"bibr\" rid=\"b38\">38</xref>]. For example, several biomarkers that appeared to be prognostic in a cohort of all major subtypes of ovarian cancer were not informative within homogeneous subtypes. Our findings provide another example of the importance of refining the analysis of ovarian biomarkers by focusing on the most clinically significant group, HGPSC.</p><p>The key oncogenic and tumour suppressor genes for HGPSC have not been identified and as it has high rates of genomic instability, many of the alterations described may be passenger mutations. Our results for the prevalence of <italic>TP53</italic> mutation may be conservative as our prospectively determined selection criteria excluded low-stage HGPSC cases (<italic>n</italic> = 15). This was because true low-stage HGPSC is a rare entity and often suggests misdiagnosis of LGS histiotype. However, 13/15 of these cases had <italic>TP53</italic> mutation. Our data are therefore consistent with mutant <italic>TP53</italic> being an essential driver mutation early in the pathogenesis of HGPSC. Future studies will need to focus on studying the biological significance of the different types of <italic>TP53</italic> mutation in HGPSC and developing p53 synthetic-lethal therapies for patients with this disease.</p></sec></body><back><ack><p>We thank Sarah Hyland, Philip Chambers, Christian Bourne, and Nik Matthews for expert technical assistance. We acknowledge the support of the University of Cambridge, NIHR Cambridge Biomedical Research Centre, the Cambridge Experimental Cancer Medicine Centre, and Hutchison Whampoa Ltd. AAA holds a CR-UK Clinician Scientist Fellowship. D Bowtell, G Chenevix-Trench, A deFazio, A Green, D Gertig, and P Webb&#x02014;The AOCS Management Group&#x02014;gratefully acknowledge the contributions of the full AOCS Study Group, listed on <ext-link ext-link-type=\"uri\" xlink:href=\"http://www.aocstudy.org\">http://www.aocstudy.org</ext-link>, and express their gratitude to all women who participated in the study. The AOCS was supported by the US Army Medical Research and Material Command under DAMD17-01-1-0729, The Cancer Council Victoria, Queensland Cancer Fund, The Cancer Council New South Wales, The Cancer Council South Australia, The Cancer Foundation of Western Australia, The Cancer Council Tasmania, and the NH&#x00026;MRC. We also gratefully acknowledge Catherine Emmanuel, Catherine Kennedy, and Yoke-Eng Chiew for samples, and data from the Westmead Gynaecological Oncology Tissue Bank, a member of the Australasian Biospecimens Network-Oncology group funded by the NH&#x00026;MRC. This work was funded by Cancer Research UK (CR-UK), the Medical Research Council (MRC), and the National Health and Medical Research Council of Australia (NHMRC).</p></ack><ref-list><title>References</title><p>Note: References 39&#x02013;50 are cited in the Supporting information to this article.</p><ref id=\"b1\"><label>1</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Brosh</surname><given-names>R</given-names></name><name><surname>Rotter</surname><given-names>V</given-names></name></person-group><article-title>When mutants gain new powers: news from the mutant p53 field</article-title><source>Nature Rev Cancer</source><year>2009</year><volume>9</volume><fpage>701</fpage><lpage>713</lpage><pub-id pub-id-type=\"pmid\">19693097</pub-id></element-citation></ref><ref id=\"b2\"><label>2</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Marks</surname><given-names>JR</given-names></name><name><surname>Davidoff</surname><given-names>AM</given-names></name><name><surname>Kerns</surname><given-names>BJ</given-names></name><name><surname>Humphrey</surname><given-names>PA</given-names></name><name><surname>Pence</surname><given-names>JC</given-names></name><name><surname>Dodge</surname><given-names>RK</given-names></name><etal/></person-group><article-title>Overexpression and mutation of p53 in epithelial ovarian cancer</article-title><source>Cancer Res</source><year>1991</year><volume>51</volume><fpage>2979</fpage><lpage>2984</lpage><pub-id pub-id-type=\"pmid\">2032235</pub-id></element-citation></ref><ref id=\"b3\"><label>3</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Okamoto</surname><given-names>A</given-names></name><name><surname>Sameshima</surname><given-names>Y</given-names></name><name><surname>Yokoyama</surname><given-names>S</given-names></name><name><surname>Terashima</surname><given-names>Y</given-names></name><name><surname>Sugimura</surname><given-names>T</given-names></name><name><surname>Terada</surname><given-names>M</given-names></name><etal/></person-group><article-title>Frequent allelic losses and mutations of the p53 gene in human ovarian cancer</article-title><source>Cancer Res</source><year>1991</year><volume>51</volume><fpage>5171</fpage><lpage>5176</lpage><pub-id pub-id-type=\"pmid\">1680546</pub-id></element-citation></ref><ref id=\"b4\"><label>4</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>de Graeff</surname><given-names>P</given-names></name><name><surname>Crijns</surname><given-names>AP</given-names></name><name><surname>de Jong</surname><given-names>S</given-names></name><name><surname>Boezen</surname><given-names>M</given-names></name><name><surname>Post</surname><given-names>WJ</given-names></name><name><surname>de Vries</surname><given-names>EG</given-names></name><etal/></person-group><article-title>Modest effect of p53, EGFR and HER-2/neu on prognosis in epithelial ovarian cancer: a meta-analysis</article-title><source>Br J Cancer</source><year>2009</year><volume>101</volume><fpage>149</fpage><lpage>159</lpage><pub-id pub-id-type=\"pmid\">19513073</pub-id></element-citation></ref><ref id=\"b5\"><label>5</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hall</surname><given-names>J</given-names></name><name><surname>Paul</surname><given-names>J</given-names></name><name><surname>Brown</surname><given-names>R</given-names></name></person-group><article-title>Critical evaluation of p53 as a prognostic marker in ovarian cancer</article-title><source>Expert Rev Mol Med</source><year>2004</year><volume>6</volume><fpage>1</fpage><lpage>20</lpage><pub-id pub-id-type=\"pmid\">15147608</pub-id></element-citation></ref><ref id=\"b6\"><label>6</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Soussi</surname><given-names>T</given-names></name><name><surname>Ishioka</surname><given-names>C</given-names></name><name><surname>Claustres</surname><given-names>M</given-names></name><name><surname>Beroud</surname><given-names>C</given-names></name></person-group><article-title>Locus-specific mutation databases: pitfalls and good practice based on the p53 experience</article-title><source>Nature Rev Cancer</source><year>2006</year><volume>6</volume><fpage>83</fpage><lpage>90</lpage><pub-id pub-id-type=\"pmid\">16397528</pub-id></element-citation></ref><ref id=\"b7\"><label>7</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>Y</given-names></name><name><surname>Miron</surname><given-names>A</given-names></name><name><surname>Drapkin</surname><given-names>R</given-names></name><name><surname>Nucci</surname><given-names>MR</given-names></name><name><surname>Medeiros</surname><given-names>F</given-names></name><name><surname>Saleemuddin</surname><given-names>A</given-names></name><etal/></person-group><article-title>A candidate precursor to serous carcinoma that originates in the distal Fallopian tube</article-title><source>J Pathol</source><year>2007</year><volume>211</volume><fpage>26</fpage><lpage>35</lpage><pub-id pub-id-type=\"pmid\">17117391</pub-id></element-citation></ref><ref id=\"b8\"><label>8</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Crum</surname><given-names>CP</given-names></name><name><surname>Drapkin</surname><given-names>R</given-names></name><name><surname>Miron</surname><given-names>A</given-names></name><name><surname>Ince</surname><given-names>TA</given-names></name><name><surname>Muto</surname><given-names>M</given-names></name><name><surname>Kindelberger</surname><given-names>DW</given-names></name><etal/></person-group><article-title>The distal Fallopian tube: a new model for pelvic serous carcinogenesis</article-title><source>Curr Opin Obstet Gynecol</source><year>2007</year><volume>19</volume><fpage>3</fpage><lpage>9</lpage><pub-id pub-id-type=\"pmid\">17218844</pub-id></element-citation></ref><ref id=\"b9\"><label>9</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jordan</surname><given-names>SJ</given-names></name><name><surname>Green</surname><given-names>AC</given-names></name><name><surname>Whiteman</surname><given-names>DC</given-names></name><name><surname>Moore</surname><given-names>SP</given-names></name><name><surname>Bain</surname><given-names>CJ</given-names></name><name><surname>Gertig</surname><given-names>DM</given-names></name><etal/></person-group><article-title>Serous ovarian, Fallopian tube and primary peritoneal cancers: a comparative epidemiological analysis</article-title><source>Int J Cancer</source><year>2008</year><volume>122</volume><fpage>1598</fpage><lpage>1603</lpage><pub-id pub-id-type=\"pmid\">18058817</pub-id></element-citation></ref><ref id=\"b10\"><label>10</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Herrington</surname><given-names>CS</given-names></name><name><surname>McCluggage</surname><given-names>WG</given-names></name></person-group><article-title>The emerging role of the distal Fallopian tube and p53 in pelvic serous carcinogenesis</article-title><source>J Pathol</source><year>2007</year><volume>220</volume><fpage>5</fpage><lpage>6</lpage><pub-id pub-id-type=\"pmid\">19882674</pub-id></element-citation></ref><ref id=\"b11\"><label>11</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>McShane</surname><given-names>LM</given-names></name><name><surname>Altman</surname><given-names>DG</given-names></name><name><surname>Sauerbrei</surname><given-names>W</given-names></name><name><surname>Taube</surname><given-names>SE</given-names></name><name><surname>Gion</surname><given-names>M</given-names></name><name><surname>Clark</surname><given-names>GM</given-names></name></person-group><article-title>Reporting recommendations for tumor marker prognostic studies (REMARK)</article-title><source>J Natl Cancer Inst</source><year>2005</year><volume>97</volume><fpage>1180</fpage><lpage>1184</lpage><pub-id pub-id-type=\"pmid\">16106022</pub-id></element-citation></ref><ref id=\"b12\"><label>12</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Olivier</surname><given-names>M</given-names></name><name><surname>Petitjean</surname><given-names>A</given-names></name><name><surname>Marcel</surname><given-names>V</given-names></name><name><surname>Petre</surname><given-names>A</given-names></name><name><surname>Mounawar</surname><given-names>M</given-names></name><name><surname>Plymoth</surname><given-names>A</given-names></name><etal/></person-group><article-title>Recent advances in p53 research: an interdisciplinary perspective</article-title><source>Cancer Gene Ther</source><year>2009</year><volume>16</volume><fpage>1</fpage><lpage>12</lpage><pub-id pub-id-type=\"pmid\">18802452</pub-id></element-citation></ref><ref id=\"b13\"><label>13</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Montes de Oca Luna</surname><given-names>R</given-names></name><name><surname>Wagner</surname><given-names>DS</given-names></name><name><surname>Lozano</surname><given-names>G</given-names></name></person-group><article-title>Rescue of early embryonic lethality in mdm2-deficient mice by deletion of p53</article-title><source>Nature</source><year>1995</year><volume>378</volume><fpage>203</fpage><lpage>206</lpage><pub-id pub-id-type=\"pmid\">7477326</pub-id></element-citation></ref><ref id=\"b14\"><label>14</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Parant</surname><given-names>J</given-names></name><name><surname>Chavez-Reyes</surname><given-names>A</given-names></name><name><surname>Little</surname><given-names>NA</given-names></name><name><surname>Yan</surname><given-names>W</given-names></name><name><surname>Reinke</surname><given-names>V</given-names></name><name><surname>Jochemsen</surname><given-names>AG</given-names></name><etal/></person-group><article-title>Rescue of embryonic lethality in Mdm4-null mice by loss of Trp53 suggests a nonoverlapping pathway with MDM2 to regulate p53</article-title><source>Nature Genet</source><year>2001</year><volume>29</volume><fpage>92</fpage><lpage>95</lpage><pub-id pub-id-type=\"pmid\">11528400</pub-id></element-citation></ref><ref id=\"b15\"><label>15</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Oliner</surname><given-names>JD</given-names></name><name><surname>Kinzler</surname><given-names>KW</given-names></name><name><surname>Meltzer</surname><given-names>PS</given-names></name><name><surname>George</surname><given-names>DL</given-names></name><name><surname>Vogelstein</surname><given-names>B</given-names></name></person-group><article-title>Amplification of a gene encoding a p53-associated protein in human sarcomas</article-title><source>Nature</source><year>1992</year><volume>358</volume><fpage>80</fpage><lpage>83</lpage><pub-id pub-id-type=\"pmid\">1614537</pub-id></element-citation></ref><ref id=\"b16\"><label>16</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>McCann</surname><given-names>AH</given-names></name><name><surname>Kirley</surname><given-names>A</given-names></name><name><surname>Carney</surname><given-names>DN</given-names></name><name><surname>Corbally</surname><given-names>N</given-names></name><name><surname>Magee</surname><given-names>HM</given-names></name><name><surname>Keating</surname><given-names>G</given-names></name><etal/></person-group><article-title>Amplification of the MDM2 gene in human breast cancer and its association with MDM2 and p53 protein status</article-title><source>Br J Cancer</source><year>1995</year><volume>71</volume><fpage>981</fpage><lpage>985</lpage><pub-id pub-id-type=\"pmid\">7734324</pub-id></element-citation></ref><ref id=\"b17\"><label>17</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shahin</surname><given-names>MS</given-names></name><name><surname>Hughes</surname><given-names>JH</given-names></name><name><surname>Sood</surname><given-names>AK</given-names></name><name><surname>Buller</surname><given-names>RE</given-names></name></person-group><article-title>The prognostic significance of p53 tumor suppressor gene alterations in ovarian carcinoma</article-title><source>Cancer</source><year>2000</year><volume>89</volume><fpage>2006</fpage><lpage>2017</lpage><pub-id pub-id-type=\"pmid\">11064359</pub-id></element-citation></ref><ref id=\"b18\"><label>18</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Olivier</surname><given-names>M</given-names></name><name><surname>Langerod</surname><given-names>A</given-names></name><name><surname>Carrieri</surname><given-names>P</given-names></name><name><surname>Bergh</surname><given-names>J</given-names></name><name><surname>Klaar</surname><given-names>S</given-names></name><name><surname>Eyfjord</surname><given-names>J</given-names></name><etal/></person-group><article-title>The clinical value of somatic TP53 gene mutations in 1794 patients with breast cancer</article-title><source>Clin Cancer Res</source><year>2006</year><volume>12</volume><fpage>1157</fpage><lpage>1167</lpage><pub-id pub-id-type=\"pmid\">16489069</pub-id></element-citation></ref><ref id=\"b19\"><label>19</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Muscolini</surname><given-names>M</given-names></name><name><surname>Montagni</surname><given-names>E</given-names></name><name><surname>Caristi</surname><given-names>S</given-names></name><name><surname>Nomura</surname><given-names>T</given-names></name><name><surname>Kamada</surname><given-names>R</given-names></name><name><surname>Di Agostino</surname><given-names>S</given-names></name><etal/></person-group><article-title>Characterization of a new cancer-associated mutant of p53 with a missense mutation (K351N) in the tetramerization domain</article-title><source>Cell Cycle</source><year>2009</year><volume>8</volume><fpage>3396</fpage><lpage>3405</lpage><pub-id pub-id-type=\"pmid\">19806023</pub-id></element-citation></ref><ref id=\"b20\"><label>20</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Vang</surname><given-names>R</given-names></name><name><surname>Shih Ie</surname><given-names>M</given-names></name><name><surname>Kurman</surname><given-names>RJ</given-names></name></person-group><article-title>Ovarian low-grade and high-grade serous carcinoma: pathogenesis, clinicopathologic and molecular biologic features, and diagnostic problems</article-title><source>Adv Anat Pathol</source><year>2009</year><volume>16</volume><fpage>267</fpage><lpage>282</lpage><pub-id pub-id-type=\"pmid\">19700937</pub-id></element-citation></ref><ref id=\"b21\"><label>21</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shih Ie</surname><given-names>M</given-names></name><name><surname>Kurman</surname><given-names>RJ</given-names></name></person-group><article-title>Molecular pathogenesis of ovarian borderline tumors: new insights and old challenges</article-title><source>Clin Cancer Res</source><year>2005</year><volume>11</volume><fpage>7273</fpage><lpage>7279</lpage><pub-id pub-id-type=\"pmid\">16243797</pub-id></element-citation></ref><ref id=\"b22\"><label>22</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Meinhold-Heerlein</surname><given-names>I</given-names></name><name><surname>Bauerschlag</surname><given-names>D</given-names></name><name><surname>Hilpert</surname><given-names>F</given-names></name><name><surname>Dimitrov</surname><given-names>P</given-names></name><name><surname>Sapinoso</surname><given-names>LM</given-names></name><name><surname>Orlowska-Volk</surname><given-names>M</given-names></name><etal/></person-group><article-title>Molecular and prognostic distinction between serous ovarian carcinomas of varying grade and malignant potential</article-title><source>Oncogene</source><year>2005</year><volume>24</volume><fpage>1053</fpage><lpage>1065</lpage><pub-id pub-id-type=\"pmid\">15558012</pub-id></element-citation></ref><ref id=\"b23\"><label>23</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Salani</surname><given-names>R</given-names></name><name><surname>Kurman</surname><given-names>RJ</given-names></name><name><surname>Giuntoli</surname><given-names>R</given-names><suffix>2nd</suffix></name><name><surname>Gardner</surname><given-names>G</given-names></name><name><surname>Bristow</surname><given-names>R</given-names></name><name><surname>Wang</surname><given-names>TL</given-names></name><etal/></person-group><article-title>Assessment of TP53 mutation using purified tissue samples of ovarian serous carcinomas reveals a higher mutation rate than previously reported and does not correlate with drug resistance</article-title><source>Int J Gynecol Cancer</source><year>2008</year><volume>18</volume><fpage>487</fpage><lpage>491</lpage><pub-id pub-id-type=\"pmid\">17692090</pub-id></element-citation></ref><ref id=\"b24\"><label>24</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Dehari</surname><given-names>R</given-names></name><name><surname>Kurman</surname><given-names>RJ</given-names></name><name><surname>Logani</surname><given-names>S</given-names></name><name><surname>Shih</surname><given-names>IeM</given-names></name></person-group><article-title>The development of high-grade serous carcinoma from atypical proliferative (borderline) serous tumors and low-grade micropapillary serous carcinoma: a morphologic and molecular genetic analysis</article-title><source>Am J Surg Pathol</source><year>2007</year><volume>31</volume><fpage>1007</fpage><lpage>1012</lpage><pub-id pub-id-type=\"pmid\">17592266</pub-id></element-citation></ref><ref id=\"b25\"><label>25</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Leng</surname><given-names>RP</given-names></name><name><surname>Lin</surname><given-names>Y</given-names></name><name><surname>Ma</surname><given-names>W</given-names></name><name><surname>Wu</surname><given-names>H</given-names></name><name><surname>Lemmers</surname><given-names>B</given-names></name><name><surname>Chung</surname><given-names>S</given-names></name><etal/></person-group><article-title>Pirh2, a p53-induced ubiquitin-protein ligase, promotes p53 degradation</article-title><source>Cell</source><year>2003</year><volume>112</volume><fpage>779</fpage><lpage>791</lpage><pub-id pub-id-type=\"pmid\">12654245</pub-id></element-citation></ref><ref id=\"b26\"><label>26</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Dornan</surname><given-names>D</given-names></name><name><surname>Wertz</surname><given-names>I</given-names></name><name><surname>Shimizu</surname><given-names>H</given-names></name><name><surname>Arnott</surname><given-names>D</given-names></name><name><surname>Frantz</surname><given-names>GD</given-names></name><name><surname>Dowd</surname><given-names>P</given-names></name><etal/></person-group><article-title>The ubiquitin ligase COP1 is a critical negative regulator of p53</article-title><source>Nature</source><year>2004</year><volume>429</volume><fpage>86</fpage><lpage>92</lpage><pub-id pub-id-type=\"pmid\">15103385</pub-id></element-citation></ref><ref id=\"b27\"><label>27</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bartel</surname><given-names>F</given-names></name><name><surname>Jung</surname><given-names>J</given-names></name><name><surname>Bohnke</surname><given-names>A</given-names></name><name><surname>Gradhand</surname><given-names>E</given-names></name><name><surname>Zeng</surname><given-names>K</given-names></name><name><surname>Thomssen</surname><given-names>C</given-names></name><etal/></person-group><article-title>Both germ line and somatic genetics of the p53 pathway affect ovarian cancer incidence and survival</article-title><source>Clin Cancer Res</source><year>2008</year><volume>14</volume><fpage>89</fpage><lpage>96</lpage><pub-id pub-id-type=\"pmid\">18172257</pub-id></element-citation></ref><ref id=\"b28\"><label>28</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Darcy</surname><given-names>KM</given-names></name><name><surname>Brady</surname><given-names>WE</given-names></name><name><surname>McBroom</surname><given-names>JW</given-names></name><name><surname>Bell</surname><given-names>JG</given-names></name><name><surname>Young</surname><given-names>RC</given-names></name><name><surname>McGuire</surname><given-names>WP</given-names></name><etal/></person-group><article-title>Associations between p53 overexpression and multiple measures of clinical outcome in high-risk, early stage or suboptimally-resected, advanced stage epithelial ovarian cancers. A Gynecologic Oncology Group study</article-title><source>Gynecol Oncol</source><year>2008</year><volume>111</volume><fpage>487</fpage><lpage>495</lpage><pub-id pub-id-type=\"pmid\">18834621</pub-id></element-citation></ref><ref id=\"b29\"><label>29</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hogdall</surname><given-names>EV</given-names></name><name><surname>Christensen</surname><given-names>L</given-names></name><name><surname>Hogdall</surname><given-names>CK</given-names></name><name><surname>Frederiksen</surname><given-names>K</given-names></name><name><surname>Gayther</surname><given-names>S</given-names></name><name><surname>Blaakaer</surname><given-names>J</given-names></name><etal/></person-group><article-title>Distribution of p53 expression in tissue from 774 Danish ovarian tumour patients and its prognostic significance in ovarian carcinomas</article-title><source>APMIS</source><year>2008</year><volume>116</volume><fpage>400</fpage><lpage>409</lpage><pub-id pub-id-type=\"pmid\">18452430</pub-id></element-citation></ref><ref id=\"b30\"><label>30</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kupryjanczyk</surname><given-names>J</given-names></name><name><surname>Kraszewska</surname><given-names>E</given-names></name><name><surname>Ziolkowska-Seta</surname><given-names>I</given-names></name><name><surname>Madry</surname><given-names>R</given-names></name><name><surname>Timorek</surname><given-names>A</given-names></name><name><surname>Markowska</surname><given-names>J</given-names></name><etal/></person-group><article-title>TP53 status and taxane&#x02013;platinum versus platinum-based therapy in ovarian cancer patients: a non-randomized retrospective study</article-title><source>BMC Cancer</source><year>2008</year><volume>8</volume><fpage>27</fpage><pub-id pub-id-type=\"pmid\">18230133</pub-id></element-citation></ref><ref id=\"b31\"><label>31</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Donehower</surname><given-names>LA</given-names></name><name><surname>Harvey</surname><given-names>M</given-names></name><name><surname>Slagle</surname><given-names>BL</given-names></name><name><surname>McArthur</surname><given-names>MJ</given-names></name><name><surname>Montgomery</surname><given-names>CA</given-names><suffix>Jr</suffix></name><name><surname>Butel</surname><given-names>JS</given-names></name><etal/></person-group><article-title>Mice deficient for p53 are developmentally normal but susceptible to spontaneous tumours</article-title><source>Nature</source><year>1992</year><volume>356</volume><fpage>215</fpage><lpage>221</lpage><pub-id pub-id-type=\"pmid\">1552940</pub-id></element-citation></ref><ref id=\"b32\"><label>32</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Terzian</surname><given-names>T</given-names></name><name><surname>Suh</surname><given-names>YA</given-names></name><name><surname>Iwakuma</surname><given-names>T</given-names></name><name><surname>Post</surname><given-names>SM</given-names></name><name><surname>Neumann</surname><given-names>M</given-names></name><name><surname>Lang</surname><given-names>GA</given-names></name><etal/></person-group><article-title>The inherent instability of mutant p53 is alleviated by Mdm2 or p16INK4a loss</article-title><source>Genes Dev</source><year>2008</year><volume>22</volume><fpage>1337</fpage><lpage>1344</lpage><pub-id pub-id-type=\"pmid\">18483220</pub-id></element-citation></ref><ref id=\"b33\"><label>33</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Guessous</surname><given-names>F</given-names></name><name><surname>Kwon</surname><given-names>S</given-names></name><name><surname>Kumar</surname><given-names>M</given-names></name><name><surname>Ibidapo</surname><given-names>O</given-names></name><name><surname>Fuller</surname><given-names>L</given-names></name><etal/></person-group><article-title>PTEN has tumor-promoting properties in the setting of gain-of-function p53 mutations</article-title><source>Cancer Res</source><year>2008</year><volume>68</volume><fpage>1723</fpage><lpage>1731</lpage><pub-id pub-id-type=\"pmid\">18339852</pub-id></element-citation></ref><ref id=\"b34\"><label>34</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Phillips</surname><given-names>KA</given-names></name><name><surname>Nichol</surname><given-names>K</given-names></name><name><surname>Ozcelik</surname><given-names>H</given-names></name><name><surname>Knight</surname><given-names>J</given-names></name><name><surname>Done</surname><given-names>SJ</given-names></name><name><surname>Goodwin</surname><given-names>PJ</given-names></name><etal/></person-group><article-title>Frequency of p53 mutations in breast carcinomas from Ashkenazi Jewish carriers of BRCA1 mutations</article-title><source>J Natl Cancer Inst</source><year>1999</year><volume>91</volume><fpage>469</fpage><lpage>473</lpage><pub-id pub-id-type=\"pmid\">10070948</pub-id></element-citation></ref><ref id=\"b35\"><label>35</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Holstege</surname><given-names>H</given-names></name><name><surname>Joosse</surname><given-names>SA</given-names></name><name><surname>van Oostrom</surname><given-names>CT</given-names></name><name><surname>Nederlof</surname><given-names>PM</given-names></name><name><surname>de Vries</surname><given-names>A</given-names></name><name><surname>Jonkers</surname><given-names>J</given-names></name></person-group><article-title>High incidence of protein-truncating TP53 mutations in BRCA1-related breast cancer</article-title><source>Cancer Res</source><year>2009</year><volume>69</volume><fpage>3625</fpage><lpage>3633</lpage><pub-id pub-id-type=\"pmid\">19336573</pub-id></element-citation></ref><ref id=\"b36\"><label>36</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Turner</surname><given-names>N</given-names></name><name><surname>Tutt</surname><given-names>A</given-names></name><name><surname>Ashworth</surname><given-names>A</given-names></name></person-group><article-title>Hallmarks of &#x02018;BRCAness&#x02019; in sporadic cancers</article-title><source>Nature Rev Cancer</source><year>2004</year><volume>4</volume><fpage>814</fpage><lpage>819</lpage><pub-id pub-id-type=\"pmid\">15510162</pub-id></element-citation></ref><ref id=\"b37\"><label>37</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zorn</surname><given-names>KK</given-names></name><name><surname>Bonome</surname><given-names>T</given-names></name><name><surname>Gangi</surname><given-names>L</given-names></name><name><surname>Chandramouli</surname><given-names>GV</given-names></name><name><surname>Awtrey</surname><given-names>CS</given-names></name><name><surname>Gardner</surname><given-names>GJ</given-names></name><etal/></person-group><article-title>Gene expression profiles of serous, endometrioid, and clear cell subtypes of ovarian and endometrial cancer</article-title><source>Clin Cancer Res</source><year>2005</year><volume>11</volume><fpage>6422</fpage><lpage>6430</lpage><pub-id pub-id-type=\"pmid\">16166416</pub-id></element-citation></ref><ref id=\"b38\"><label>38</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kobel</surname><given-names>M</given-names></name><name><surname>Kalloger</surname><given-names>SE</given-names></name><name><surname>Boyd</surname><given-names>N</given-names></name><name><surname>McKinney</surname><given-names>S</given-names></name><name><surname>Mehl</surname><given-names>E</given-names></name><name><surname>Palmer</surname><given-names>C</given-names></name><etal/></person-group><article-title>Ovarian carcinoma subtypes are different diseases: implications for biomarker studies</article-title><source>PLoS Med</source><year>2008</year><volume>5</volume><fpage>e232</fpage><pub-id pub-id-type=\"pmid\">19053170</pub-id></element-citation></ref><ref id=\"b39\"><label>39</label><element-citation publication-type=\"journal\"><collab>ICON</collab><article-title>Paclitaxel plus carboplatin versus standard chemotherapy with either single-agent carboplatin or cyclophosphamide, doxorubicin, and cisplatin in women with ovarian cancer: the ICON3 randomised trial</article-title><source>Lancet</source><year>2002</year><volume>360</volume><fpage>505</fpage><lpage>515</lpage><pub-id pub-id-type=\"pmid\">12241653</pub-id></element-citation></ref><ref id=\"b40\"><label>40</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Etemadmoghadam</surname><given-names>D</given-names></name><name><surname>de Fazio</surname><given-names>A</given-names></name><name><surname>Beroukhim</surname><given-names>R</given-names></name><name><surname>Mermel</surname><given-names>C</given-names></name><name><surname>George</surname><given-names>J</given-names></name><name><surname>Getz</surname><given-names>G</given-names></name><etal/></person-group><article-title>Integrated genome-wide DNA copy number and expression analysis identifies distinct mechanisms of primary chemoresistance in ovarian carcinomas</article-title><source>Clin Cancer Res</source><year>2009</year><volume>15</volume><fpage>1417</fpage><lpage>1427</lpage><pub-id pub-id-type=\"pmid\">19193619</pub-id></element-citation></ref><ref id=\"b41\"><label>41</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rustin</surname><given-names>GJ</given-names></name><name><surname>Nelstrop</surname><given-names>AE</given-names></name><name><surname>McClean</surname><given-names>P</given-names></name><name><surname>Brady</surname><given-names>MF</given-names></name><name><surname>McGuire</surname><given-names>WP</given-names></name><name><surname>Hoskins</surname><given-names>WJ</given-names></name><etal/></person-group><article-title>Defining response of ovarian carcinoma to initial chemotherapy according to serum CA 125</article-title><source>J Clin Oncol</source><year>1996</year><volume>14</volume><fpage>1545</fpage><lpage>1551</lpage><pub-id pub-id-type=\"pmid\">8622070</pub-id></element-citation></ref><ref id=\"b42\"><label>42</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Vergote</surname><given-names>I</given-names></name><name><surname>Rustin</surname><given-names>GJ</given-names></name><name><surname>Eisenhauer</surname><given-names>EA</given-names></name><name><surname>Kristensen</surname><given-names>GB</given-names></name><name><surname>Pujade-Lauraine</surname><given-names>E</given-names></name><name><surname>Parmar</surname><given-names>MK</given-names></name><etal/></person-group><article-title>Re: new guidelines to evaluate the response to treatment in solid tumors [ovarian cancer]. Gynecologic Cancer Intergroup</article-title><source>J Natl Cancer Inst</source><year>2000</year><volume>92</volume><fpage>1534</fpage><lpage>1535</lpage><pub-id pub-id-type=\"pmid\">10995813</pub-id></element-citation></ref><ref id=\"b43\"><label>43</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kupryjanczyk</surname><given-names>J</given-names></name><name><surname>Thor</surname><given-names>AD</given-names></name><name><surname>Beauchamp</surname><given-names>R</given-names></name><name><surname>Merritt</surname><given-names>V</given-names></name><name><surname>Edgerton</surname><given-names>SM</given-names></name><name><surname>Bell</surname><given-names>DA</given-names></name><etal/></person-group><article-title>p53 gene mutations and protein accumulation in human ovarian cancer</article-title><source>Proc Natl Acad Sci U S A</source><year>1993</year><volume>90</volume><fpage>4961</fpage><lpage>4965</lpage><pub-id pub-id-type=\"pmid\">8506342</pub-id></element-citation></ref><ref id=\"b44\"><label>44</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shimizu</surname><given-names>Y</given-names></name><name><surname>Kamoi</surname><given-names>S</given-names></name><name><surname>Amada</surname><given-names>S</given-names></name><name><surname>Akiyama</surname><given-names>F</given-names></name><name><surname>Silverberg</surname><given-names>SG</given-names></name></person-group><article-title>Toward the development of a universal grading system for ovarian epithelial carcinoma: testing of a proposed system in a series of 461 patients with uniform treatment and follow-up</article-title><source>Cancer</source><year>1998</year><volume>82</volume><fpage>893</fpage><lpage>901</lpage><pub-id pub-id-type=\"pmid\">9486579</pub-id></element-citation></ref><ref id=\"b45\"><label>45</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Parsons</surname><given-names>DW</given-names></name><name><surname>Jones</surname><given-names>S</given-names></name><name><surname>Zhang</surname><given-names>X</given-names></name><name><surname>Lin</surname><given-names>JC</given-names></name><name><surname>Leary</surname><given-names>RJ</given-names></name><name><surname>Angenendt</surname><given-names>P</given-names></name><etal/></person-group><article-title>An integrated genomic analysis of human glioblastoma multiforme</article-title><source>Science</source><year>2008</year><volume>321</volume><fpage>1807</fpage><lpage>1812</lpage><pub-id pub-id-type=\"pmid\">18772396</pub-id></element-citation></ref><ref id=\"b46\"><label>46</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kuo</surname><given-names>KT</given-names></name><name><surname>Mao</surname><given-names>TL</given-names></name><name><surname>Jones</surname><given-names>S</given-names></name><name><surname>Veras</surname><given-names>E</given-names></name><name><surname>Ayhan</surname><given-names>A</given-names></name><name><surname>Wang</surname><given-names>TL</given-names></name><etal/></person-group><article-title>Frequent activating mutations of PIK3CA in ovarian clear cell carcinoma</article-title><source>Am J Pathol</source><year>2009</year><volume>174</volume><fpage>1597</fpage><lpage>1601</lpage><pub-id pub-id-type=\"pmid\">19349352</pub-id></element-citation></ref><ref id=\"b47\"><label>47</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bengtsson</surname><given-names>H</given-names></name><name><surname>Irizarry</surname><given-names>R</given-names></name><name><surname>Carvalho</surname><given-names>B</given-names></name><name><surname>Speed</surname><given-names>TP</given-names></name></person-group><article-title>Estimation and assessment of raw copy numbers at the single locus level</article-title><source>Bioinformatics</source><year>2008</year><volume>24</volume><fpage>759</fpage><lpage>767</lpage><pub-id pub-id-type=\"pmid\">18204055</pub-id></element-citation></ref><ref id=\"b48\"><label>48</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rozen</surname><given-names>S</given-names></name><name><surname>Skaletsky</surname><given-names>H</given-names></name></person-group><article-title>Primer3 on the WWW for general users and for biologist programmers</article-title><source>Methods Mol Biol</source><year>2000</year><volume>132</volume><fpage>365</fpage><lpage>386</lpage><pub-id pub-id-type=\"pmid\">10547847</pub-id></element-citation></ref><ref id=\"b49\"><label>49</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Riemenschneider</surname><given-names>MJ</given-names></name><name><surname>Buschges</surname><given-names>R</given-names></name><name><surname>Wolter</surname><given-names>M</given-names></name><name><surname>Reifenberger</surname><given-names>J</given-names></name><name><surname>Bostrom</surname><given-names>J</given-names></name><name><surname>Kraus</surname><given-names>JA</given-names></name><etal/></person-group><article-title>Amplification and overexpression of the MDM4 (MDMX) gene from 1q32 in a subset of malignant gliomas without TP53 mutation or MDM2 amplification</article-title><source>Cancer Res</source><year>1999</year><volume>59</volume><fpage>6091</fpage><lpage>6096</lpage><pub-id pub-id-type=\"pmid\">10626796</pub-id></element-citation></ref><ref id=\"b50\"><label>50</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>TL</given-names></name><name><surname>Maierhofer</surname><given-names>C</given-names></name><name><surname>Speicher</surname><given-names>MR</given-names></name><name><surname>Lengauer</surname><given-names>C</given-names></name><name><surname>Vogelstein</surname><given-names>B</given-names></name><name><surname>Kinzler</surname><given-names>KW</given-names></name><etal/></person-group><article-title>Digital karyotyping</article-title><source>Proc Natl Acad Sci U S A</source><year>2002</year><volume>99</volume><fpage>16156</fpage><lpage>16161</lpage><pub-id pub-id-type=\"pmid\">12461184</pub-id></element-citation></ref></ref-list><sec sec-type=\"supplementary-material\"><title>SUPPORTING INFORMATION ON THE INTERNET</title><p>The following supporting information may be found in the online version of this article.</p><p><bold>Supplementary experimental procedures.</bold></p><p><bold>Table S1.</bold> Samples listed by AOCS number with results of somatic <italic>TP53</italic> sequencing.</p><supplementary-material content-type=\"local-data\" id=\"SD1\"><media xlink:href=\"path0221-0049-SD1.pdf\" xlink:type=\"simple\" id=\"d32e3668\" position=\"anchor\" mimetype=\"application\" mime-subtype=\"pdf\"/></supplementary-material><p><bold>Table S2.</bold> Summary of clinical, pathology and molecular review of mutation-negative serous carcinomas.</p><supplementary-material content-type=\"local-data\" id=\"SD2\"><media xlink:href=\"path0221-0049-SD2.pdf\" xlink:type=\"simple\" id=\"d32e3674\" position=\"anchor\" mimetype=\"application\" mime-subtype=\"pdf\"/></supplementary-material><p><bold>Figure S1.</bold> Summary of all cases included in the pilot and validation sets.</p><supplementary-material content-type=\"local-data\" id=\"SD3\"><media xlink:href=\"path0221-0049-SD3.pdf\" xlink:type=\"simple\" id=\"d32e3680\" position=\"anchor\" mimetype=\"application\" mime-subtype=\"pdf\"/><media xlink:href=\"path0221-0049-SD4.doc\" xlink:type=\"simple\" id=\"d32e3681\" position=\"anchor\" mimetype=\"application\" mime-subtype=\"msword\"/></supplementary-material></sec></back></article>'}"
      ]
     },
     "execution_count": 365,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "x = articles.find_one({'_id': 'test'})\n",
    "x"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 366,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "None\n"
     ]
    }
   ],
   "source": [
    "write_article_meta_to_mongo(x)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 367,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'_id': 'test',\n",
       " 'annotations': {'disease-ontology-keywords': None},\n",
       " 'meta': {'article-ids': {'doi': '10.1002/path.2696',\n",
       "   'pmc': '3262968',\n",
       "   'pmid': '20229506'},\n",
       "  'article-type': 'research-article',\n",
       "  'keywords': ['p53',\n",
       "   'high-grade pelvic serous carcinoma',\n",
       "   'ovarian cancer',\n",
       "   'DNA sequence analysis',\n",
       "   'array-based genomic hybridization',\n",
       "   'histopathology',\n",
       "   'clinical outcome',\n",
       "   'BRCA']},\n",
       " 'nxml': '<!DOCTYPE article PUBLIC \"-//NLM//DTD JATS (Z39.96) Journal Archiving and Interchange DTD v1.0 20120330//EN\" \"JATS-archivearticle1.dtd\">\\n<article xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" article-type=\"research-article\"><?properties open_access?><front><journal-meta><journal-id journal-id-type=\"nlm-ta\">J Pathol</journal-id><journal-id journal-id-type=\"publisher-id\">path</journal-id><journal-title-group><journal-title>The Journal of Pathology</journal-title></journal-title-group><issn pub-type=\"ppub\">0022-3417</issn><issn pub-type=\"epub\">1096-9896</issn><publisher><publisher-name>John Wiley &#x00026; Sons, Ltd.</publisher-name></publisher></journal-meta><article-meta><article-id pub-id-type=\"pmid\">20229506</article-id><article-id pub-id-type=\"pmc\">3262968</article-id><article-id pub-id-type=\"doi\">10.1002/path.2696</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Original Paper</subject></subj-group></article-categories><title-group><article-title>Driver mutations in <italic>TP53</italic> are ubiquitous in high grade serous carcinoma of the ovary</article-title></title-group><contrib-group><contrib contrib-type=\"author\"><name><surname>Ahmed</surname><given-names>Ahmed Ashour</given-names></name><xref ref-type=\"aff\" rid=\"au1\">1</xref><xref ref-type=\"aff\" rid=\"au2\">2</xref></contrib><contrib contrib-type=\"author\"><name><surname>Etemadmoghadam</surname><given-names>Dariush</given-names></name><xref ref-type=\"aff\" rid=\"au3\">3</xref></contrib><contrib contrib-type=\"author\"><name><surname>Temple</surname><given-names>Jillian</given-names></name><xref ref-type=\"aff\" rid=\"au1\">1</xref><xref ref-type=\"aff\" rid=\"au2\">2</xref></contrib><contrib contrib-type=\"author\"><name><surname>Lynch</surname><given-names>Andy G</given-names></name><xref ref-type=\"aff\" rid=\"au2\">2</xref></contrib><contrib contrib-type=\"author\"><name><surname>Riad</surname><given-names>Mohamed</given-names></name><xref ref-type=\"aff\" rid=\"au4\">4</xref></contrib><contrib contrib-type=\"author\"><name><surname>Sharma</surname><given-names>Raghwa</given-names></name><xref ref-type=\"aff\" rid=\"au5\">5</xref></contrib><contrib contrib-type=\"author\"><name><surname>Stewart</surname><given-names>Colin</given-names></name><xref ref-type=\"aff\" rid=\"au6\">6</xref></contrib><contrib contrib-type=\"author\"><name><surname>Fereday</surname><given-names>Sian</given-names></name><xref ref-type=\"aff\" rid=\"au3\">3</xref></contrib><contrib contrib-type=\"author\"><name><surname>Caldas</surname><given-names>Carlos</given-names></name><xref ref-type=\"aff\" rid=\"au2\">2</xref><xref ref-type=\"aff\" rid=\"au7\">7</xref></contrib><contrib contrib-type=\"author\"><name><surname>deFazio</surname><given-names>Anna</given-names></name><xref ref-type=\"aff\" rid=\"au8\">8</xref></contrib><contrib contrib-type=\"author\"><name><surname>Bowtell</surname><given-names>David</given-names></name><xref ref-type=\"aff\" rid=\"au3\">3</xref><xref ref-type=\"aff\" rid=\"au9\">9</xref><xref ref-type=\"corresp\" rid=\"cor1\">*</xref></contrib><contrib contrib-type=\"author\"><name><surname>Brenton</surname><given-names>James D</given-names></name><xref ref-type=\"aff\" rid=\"au1\">1</xref><xref ref-type=\"aff\" rid=\"au2\">2</xref><xref ref-type=\"corresp\" rid=\"cor2\">*</xref></contrib></contrib-group><aff id=\"au1\"><label>1</label><institution>Functional Genomics of Ovarian Cancer Laboratory, Cancer Research UK Cambridge Research Institute, Li Ka Shing Centre</institution><addr-line>Robinson Way, Cambridge, CB2 0RE, UK</addr-line></aff><aff id=\"au2\"><label>2</label><institution>Department of Oncology, University of Cambridge, Hutchison/MRC Research Centre</institution><addr-line>Hills Road, Cambridge, CB2 0XZ, UK</addr-line></aff><aff id=\"au3\"><label>3</label><institution>Peter MacCallum Cancer Centre</institution><addr-line>Melbourne, Victoria, Australia</addr-line></aff><aff id=\"au4\"><label>4</label><institution>Department of Obstetrics and Gynaecology, El-Galaa Maternity Teaching Hospital</institution><addr-line>41 July 26th Street, Cairo 11511, Egypt</addr-line></aff><aff id=\"au5\"><label>5</label><institution>Department of Anatomical Pathology, University of Sydney and University of Western Sydney at Westmead Hospital</institution><addr-line>Sydney, Australia</addr-line></aff><aff id=\"au6\"><label>6</label><institution>Department of Pathology, King Edward Memorial Hospital</institution><addr-line>Western Australia, Australia</addr-line></aff><aff id=\"au7\"><label>7</label><institution>Functional Genomics of Breast Cancer Laboratory, Cancer Research UK Cambridge Research Institute, Li Ka Shing Centre</institution><addr-line>Robinson Way, Cambridge, CB2 0RE, UK</addr-line></aff><aff id=\"au8\"><label>8</label><institution>Westmead Institute for Cancer Research, University of Sydney at Westmead Millennium Institute, Westmead Hospital</institution><addr-line>New South Wales, Australia</addr-line></aff><aff id=\"au9\"><label>9</label><institution>Department of Biochemistry and Molecular Biology, University of Melbourne</institution><addr-line>Parkville, Victoria, Australia</addr-line></aff><author-notes><corresp id=\"cor1\">*Correspondence to: Professor David Bowtell, Research Division, Peter MacCallum Cancer Centre, Locked Bag 1, A\\'Beckett Street, Melbourne, 8006, Victoria, Australia. e-mail: <email>d.bowtell@petermac.org</email></corresp><corresp id=\"cor2\">*Correspondence to: Dr James D Brenton, Functional Genomics of Ovarian Cancer Laboratory, Cancer Research UK Cambridge Research Institute, Li Ka Shing Centre, Robinson Way, Cambridge, CB2 0RE, UK. e-mail: <email>james.brenton@cancer.org.uk</email></corresp><fn><p>No conflicts of interest were declared.</p></fn></author-notes><pub-date pub-type=\"ppub\"><month>5</month><year>2010</year></pub-date><volume>221</volume><issue>1</issue><fpage>49</fpage><lpage>56</lpage><history><date date-type=\"received\"><day>09</day><month>12</month><year>2009</year></date><date date-type=\"rev-recd\"><day>24</day><month>1</month><year>2010</year></date><date date-type=\"accepted\"><day>26</day><month>1</month><year>2010</year></date></history><permissions><copyright-statement>Copyright &#x000a9; 2010 Pathological Society of Great Britain and Ireland. Published by John Wiley &#x00026; Sons, Ltd.</copyright-statement><copyright-year>2010</copyright-year><license license-type=\"open-access\" xlink:href=\"http://creativecommons.org/licenses/by/2.5/\"><license-p>Re-use of this article is permitted in accordance with the Creative Commons Deed, Attribution 2.5, which does not permit commercial exploitation.</license-p></license></permissions><abstract><p>Numerous studies have tested the association between <italic>TP53</italic> mutations in ovarian cancer and prognosis but these have been consistently confounded by limitations in study design, methodology, and/or heterogeneity in the sample cohort. High-grade serous (HGS) carcinoma is the most clinically important histological subtype of ovarian cancer. As these tumours may arise from the ovary, Fallopian tube or peritoneum, they are collectively referred to as high-grade pelvic serous carcinoma (HGPSC). To identify the true prevalence of <italic>TP53</italic> mutations in HGPSC, we sequenced exons 2&#x02013;11 and intron&#x02013;exon boundaries in tumour DNA from 145 patients. HGPSC cases were defined as having histological grade 2 or 3 and FIGO stage III or IV. Surprisingly, pathogenic <italic>TP53</italic> mutations were identified in 96.7% (<italic>n</italic> = 119/123) of HGPSC cases. Molecular and pathological review of mutation-negative cases showed evidence of p53 dysfunction associated with copy number gain of <italic>MDM2</italic> or <italic>MDM4</italic>, or indicated the exclusion of samples as being low-grade serous tumours or carcinoma of uncertain primary site. Overall, p53 dysfunction rate approached 100% of confirmed HGPSCs. No association between <italic>TP53</italic> mutation and progression-free or overall survival was found. From this first comprehensive mapping of <italic>TP53</italic> mutation rate in a homogeneous group of HGPSC patients, we conclude that mutant <italic>TP53</italic> is a driver mutation in the pathogenesis of HGPSC cancers. Because <italic>TP53</italic> mutation is almost invariably present in HGPSC, it is not of substantial prognostic or predictive significance. Copyright &#x000a9; 2010 Pathological Society of Great Britain and Ireland. Published by John Wiley &#x00026; Sons, Ltd.</p></abstract><kwd-group><kwd>p53</kwd><kwd>high-grade pelvic serous carcinoma</kwd><kwd>ovarian cancer</kwd><kwd>DNA sequence analysis</kwd><kwd>array-based genomic hybridization</kwd><kwd>histopathology</kwd><kwd>clinical outcome</kwd><kwd>BRCA</kwd></kwd-group></article-meta></front><body><sec><title>Introduction</title><p>The gene <italic>TP53</italic> that encodes the tumour suppressor protein p53 is amongst the most commonly mutated genes in human cancer [<xref ref-type=\"bibr\" rid=\"b1\">1</xref>]. The frequent presence of <italic>TP53</italic> mutations in ovarian cancer has been suggested for almost two decades [<xref ref-type=\"bibr\" rid=\"b2\">2</xref>,<xref ref-type=\"bibr\" rid=\"b3\">3</xref>] and during this time, approximately 70 publications have described the relationship between <italic>TP53</italic> mutation status and clinical outcome [<xref ref-type=\"bibr\" rid=\"b4\">4</xref>]. For such an intensively studied question, there is a surprising degree of inconsistency in the published data for both the prevalence of mutation and association with prognosis or chemotherapy response.</p><p>A recent analysis of 64 publications reporting the relationship of <italic>TP53</italic> mutation to clinical outcome found that only six studies fulfilled minimum quality criteria for the method of detecting mutations or assignment of clinical response [<xref ref-type=\"bibr\" rid=\"b5\">5</xref>]. One of the most consistent limitations of previous p53 studies is a reliance on p53 immunostaining as a surrogate marker for <italic>TP53</italic> mutation [<xref ref-type=\"bibr\" rid=\"b4\">4</xref>], even though this leads to an unacceptable number of misclassifications [<xref ref-type=\"bibr\" rid=\"b6\">6</xref>]. Sequencing of tumour DNA is the gold standard to detect different types of mutation and relate these to clinical outcome. However, a recent review of 12 studies that sequenced <italic>TP53</italic> in ovarian cancer found variable mutation frequencies of 15&#x02013;80% [<xref ref-type=\"bibr\" rid=\"b5\">5</xref>]. Inconsistencies between these studies may relate to whether the whole gene was or only the most commonly mutated exons were sequenced. Additionally, most studies were confounded by the inclusion of cases with different histological types, stages, and grades of tumours [<xref ref-type=\"bibr\" rid=\"b4\">4</xref>].</p><p>Determining the true prevalence of <italic>TP53</italic> mutation is critical for understanding the pathogenesis of high-grade serous cancers arising as ovarian, Fallopian tube or primary peritoneal cancer. Although previously assumed to arise from the ovary, pathological examination of the Fallopian tube from carriers with <italic>BRCA1</italic> or <italic>BRCA2</italic> mutation has demonstrated that tubal intraepithelial carcinomas (TICs) arising from secretory epithelial cells in the fimbria are the early invasive lesion in high-grade serous (HGS) carcinoma [<xref ref-type=\"bibr\" rid=\"b7\">7</xref>,<xref ref-type=\"bibr\" rid=\"b8\">8</xref>]. In addition, TICs are found in patients without evidence of germline <italic>BRCA1</italic> or <italic>BRCA2</italic> lesions and show concomitant <italic>TP53</italic> mutations in the tube and in pelvic metastases, further supporting the model that TIC represents the origin of HGS carcinoma. HGS cancers may also arise from the peritoneum and although these may have different epidemiological risk factor profiles [<xref ref-type=\"bibr\" rid=\"b9\">9</xref>], they exist in a clinical continuum with ovarian and Fallopian tube cancers. Hence we use the term high-grade pelvic serous carcinoma (HGPSC) [<xref ref-type=\"bibr\" rid=\"b10\">10</xref>] to recognize the frequent lack of clarity regarding primary site in ovarian/Fallopian tube/peritoneal cancers and their overall biological and clinical similarity.</p><p>To address the limitations of previous studies and determine accurately the relationship between <italic>TP53</italic> mutation and HGPSC, we designed pilot and validation studies to measure the mutation frequency in a homogeneous group of advanced stage HGPSC cancers from a large population-based cohort.</p></sec><sec sec-type=\"materials|methods\"><title>Materials and methods</title><sec><title>Samples and study design</title><p>The results of this study are presented in accordance with reporting recommendations for tumour marker prognostic studies (REMARK) criteria [<xref ref-type=\"bibr\" rid=\"b11\">11</xref>]. Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD3\">Figure 1</xref> (Supporting information) summarizes the flow of patients through the pilot and validation study, including the number of patients included in each stage of the analysis and reasons for exclusion. Patients (<italic>n</italic> = 123) were selected from the Australian Ovarian Cancer Study (AOCS), a population-based, multicentre translational study that comprised prospective collection of bio-specimens and clinical and epidemiological data from patients with primary epithelial ovarian, primary peritoneal, and Fallopian tube cancer diagnosed between 2001 and 2005. Patients treated at Westmead Hospital, Sydney between 1992 and 2002 (<italic>n</italic> = 22) were also included in the pilot cohort, making an overall sample size of 145 HGPSCs. All women in the pilot cohort received first-line platinum-based chemotherapy (cisplatin or carboplatin) with (<italic>n</italic> = 39) or without (<italic>n</italic> = 6) the taxane paclitaxel. Similarly, the majority of patients in the validation set received combination platinum&#x02013;taxane treatment (<italic>n</italic> = 93) or platinum monotherapy (<italic>n</italic> = 4). Three patients (<italic>n</italic> = 3) did not receive any chemotherapy, including one patient with stage I cancer who declined treatment and two patients who were too ill to receive chemotherapy. All patients were prospectively consented using a protocol approved by the human research ethics committees at multiple participating clinical and research centres. Further details of the AOCS cohort can be found at <ext-link ext-link-type=\"uri\" xlink:href=\"http://www.aocstudy.org\">http://www.aocstudy.org</ext-link>. Statistical design and power calculations are described in the Supporting information, Supplementary methods.</p></sec><sec><title><italic>TP53</italic> sequencing</title><p>Coding <italic>TP53</italic> exons 2&#x02013;11 were amplified using primers that encompassed the entire exon and exon&#x02013;intron boundaries. Purified PCR products were se- quenced using an ABI 3100 genetic analyser (Applied Biosystems, Warrington, UK). Sequencing reactions were performed in forward and reverse directions. Mutational analysis was performed using SeqScape&#x000ae; Software v2.6 (Applied Biosystems). Sequence traces were aligned against the <italic>TP53</italic> reference sequence NC_000017. Independent, blinded sequencing of cases from the pilot study was performed by the CR-UK Mutation Detection Facility, Clinical Sciences Building, Level 6, St James\\'s University Hospital, Leeds, UK.</p></sec><sec><title>Characterization of mutation-negative HGPSC cancers</title><p>For single nucleotide polymorphism (SNP) mapping assay, SNP microarray data analysis, quantitative PCR (qPCR), and immunohistochemistry, the reader is referred to the Supporting information, Supplementary methods.</p></sec><sec><title>aCGH data</title><p>Raw data files from aCGH experiments are available from the Gene Expression Omnibus <ext-link ext-link-type=\"uri\" xlink:href=\"http://www.ncbi.nlm.nih.gov/geo/\">http://www.ncbi.nlm.nih.gov/geo/</ext-link> (Accession number GSE19416).</p></sec></sec><sec><title>Results</title><sec><title>Both treatment-responsive and primary resistant HGPSCs have a high frequency of <italic>TP53</italic> mutation</title><p>To estimate the number of samples required to detect a significant association between <italic>TP53</italic> mutations and survival, we performed a pilot study with 45 HGPSCs (ovarian, Fallopian tube, and primary peritoneal). The cohort included 20 chemotherapy-resistant patients who had tumour progression within 6 months of completion of treatment and 25 responsive patients who did not progress for at least 9 months. The two groups showed no statistically significant differences in age, tumour stage, residual disease or type of chemotherapy (<xref ref-type=\"table\" rid=\"tbl1\">Table 1</xref>). Review of pathology reports showed one patient with low-grade serous (LGS) carcinoma (case 450) and this case was excluded from further analysis. Unexpectedly, pathogenic <italic>TP53</italic> mutations were identified in 43 out of 44 patients included in the study (Supporting information, Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD1\">Table 1</xref> and Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD3\">Figure 1</xref>). Mutations were compared to Release 13 of the IARC <italic>TP53</italic> mutation database [<xref ref-type=\"bibr\" rid=\"b12\">12</xref>] and 20 cases had rare <italic>TP53</italic> mutations, defined as less than ten occurrences (Supporting information, Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD1\">Table 1</xref>). From these, 12 were randomly selected and independently confirmed by a clinical reference laboratory blinded to our results (data not shown). Further analysis of mutations is described below.</p><table-wrap id=\"tbl1\" position=\"float\"><label>Table 1</label><caption><p>A comparison between the clinical characteristics of the chemotherapy-resistant and -responsive patients in the pilot study</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th rowspan=\"1\" colspan=\"1\"/><th align=\"center\" rowspan=\"1\" colspan=\"1\">Resistant (<italic>n</italic> = 20)</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Responsive (<italic>n</italic> = 25)</th><th align=\"center\" rowspan=\"1\" colspan=\"1\"><italic>p</italic> (test)</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">PFS (months)</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Median</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">10.2</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">23.3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x0003c; 0.001 (log rank)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">OS (months)</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Median</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">21.4</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">57.3</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x0003c; 0.001 (log rank)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Age (years)</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Median</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">58.26</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">58.7</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">0.7 (Wilcoxon rank sum)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Range</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">22.9&#x02013;78</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">38.1&#x02013;77.7</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Stage (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;III</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">23</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">0.5 (Fisher)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;IV</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">&#x02014;</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">2</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Chemotherapy (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Adjuvant</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">25</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">0.4 (Fisher)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Neo-adjuvant</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Residual disease (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;No residual disease</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">0.2 (Fisher)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;&#x02264;1 cm</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">8</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;&#x0003e; 1 cm and &#x02264;2 cm</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">5</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;&#x0003e; 2 cm</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">6</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Unknown</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">TP53 (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Wild type</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">0.4 (Fisher)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Mutant</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">25</td><td rowspan=\"1\" colspan=\"1\"/></tr></tbody></table><table-wrap-foot><fn><p>OS = overall survival; PFS = progression-free survival, defined as the period from diagnosis to the date of documented progression. Six samples (2 resistant, 4 responsive) did not have precise values for size &#x0003c; 2 cm.</p></fn></table-wrap-foot></table-wrap></sec><sec><title>Confirmation of high frequency of <italic>TP53</italic> mutation in an unselected population-based cohort of serous cancers</title><p>To exclude the possibility that the high rate of <italic>TP53</italic> mutations was biased by the selection criteria used in the pilot study, we obtained additional serous cancer cases that were a representative subset of a large population-based series (<xref ref-type=\"table\" rid=\"tbl2\">Table 2</xref>). Mutations were detected in 76/82 advanced-stage HGPSCs from these cases, with no significant difference between the percentage of mutations in the matched validation study compared with the pilot study (<italic>p</italic> = 0.3; Fisher exact test).</p><table-wrap id=\"tbl2\" position=\"float\"><label>Table 2</label><caption><p>Clinical characteristics of the validation study patients</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th rowspan=\"1\" colspan=\"1\"/><th align=\"center\" rowspan=\"1\" colspan=\"1\">Validation set (<italic>n</italic> = 82</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">PFS (months)</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Median</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">13.3</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">OS (months)</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Median</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">34</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Age (years)</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Median</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">61.3</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Range</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">40.4&#x02013;79.3</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Stage (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;III</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">73</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;IV</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">9</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Grade (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;2</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">18</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">63</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Unknown</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Chemotherapy (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Adjuvant</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">71</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Neo-adjuvant</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">11</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Residual disease (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;No residual disease</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">16</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;&#x02264;1 cm</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">26</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;&#x0003e; 1 cm and &#x02264;2 cm</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">5</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;&#x0003e; 2 cm</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">25</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Unknown</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">10</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">TP53 (<italic>n</italic>)</td><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Wild type</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">6</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x02003;Mutant</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">76</td></tr></tbody></table><table-wrap-foot><fn><p>OS = overall survival; PFS = progression-free survival.</p></fn></table-wrap-foot></table-wrap></sec><sec><title><italic>TP53</italic> mutations arise at high frequency in exons 2&#x02013;4 and 9&#x02013;11 in HGPSC carcinoma</title><p>We analysed the 119 mutations found in 126 HGPSC cases from the pilot and validation set (Supporting information, Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD1\">Table 1</xref>). As expected, the majority of <italic>TP53</italic> mutations were missense mutations (56.3%, <italic>n</italic> = 67/119) occurring in exons 4&#x02013;8 (Supporting information, Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD1\">Table 1</xref>). The most frequently mutated codon was 273 (<italic>n</italic> = 9/67 cases, 13.4%) comprising R273C, R273H, and R273L mutants. Analysis of exon and splice sites for exons 2&#x02013;4 and 9&#x02013;11, outside the DNA binding domain, showed 12 (10.1%) and 10 (8.4%) mutations, respectively, which taken together comprised 18.5% (<italic>n</italic> = 22/119) of all mutations in our series. Of note, a total of eight mutations affected exon 10 (6.7%) in the tetramerization domain (codons 324&#x02013;356) and all were predicted to cause truncation of the protein. Four of these mutations occurred at g.16915 (codon 342). In 717 serous sub-type ovarian, Fallopian, and primary peritoneal cancer cases in Release 13 of the IARC <italic>TP53</italic> database, 30 (4.2%) had mutations in exons 2&#x02013;4, 18 had mutations affecting exon 10 alone (2.5%), and 20 (2.8%) had mutations in exons 9&#x02013;11. Twenty-two mutations from our series (<italic>n</italic> = 22/119, 18.5%) were not present in the IARC database. With the exception of one, all were insertions or deletions (Supporting information, Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD1\">Table 1</xref>). The remaining novel mutation in case 434 was at position g.11602_11603GC &#x0003e; AA, creating a unique tandem nonsense mutation.</p></sec><sec><title>Low-stage HGPSC cases have a high frequency of <italic>TP53</italic> mutations</title><p>The very high prevalence of mutation in high-stage cases suggested that <italic>TP53</italic> was an early event in the pathogenesis of HGPSC. From the validation set, 15 cases were stage 1 or 2 HGPSC and 13/15 (86.7%) had <italic>TP53</italic> mutation (Supporting information, Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD1\">Table 1</xref>). The negative cases 41307 and 60111 had mixed endometrioid/HGS or grade 1 pathology, respectively. As for high-stage cases, the majority of mutations (<italic>n</italic> = 10/13; 76.9%) were missense mutations in exons 5&#x02013;8. There was one nonsense mutation in exon 10 (7.7%) and two splice junction mutations (15.4%).</p></sec><sec><title>Mutation-negative cases have alternative mechanisms of p53 dysfunction</title><p>We considered the possibility that <italic>TP53</italic> mutations in the seven mutation-negative samples (pilot <italic>n</italic> = 1, validation <italic>n</italic> = 6) were missed because of contamination with normal DNA. Although all samples with significant stroma were needle-dissected prior to sequencing, pathological review of the mutation-negative cases indicated that they did not have increased stroma compared with mutation-positive cases (data not shown). Additionally, array-based comparative genomic hybridization (aCGH) showed that all mutation-negative cases had DNA copy number abnormalities (DCNAs), and the least aberrant, case 533, had distinct loss of chromosome arm 1p (<xref ref-type=\"fig\" rid=\"fig01\">Figure 1A</xref>). No samples showed evidence of homozygous deletion of the <italic>TP53</italic> locus (<xref ref-type=\"fig\" rid=\"fig01\">Figure 1</xref>B).</p><fig id=\"fig01\" position=\"float\"><label>Figure 1</label><caption><p>Characterization of <italic>TP53</italic> mutation-negative cancer samples. (A) Summary plots of whole genome DNA copy number data for seven mutation-negative samples. Red indicates chromosomal gain and blue shows regions of chromosomal loss. (B) High-resolution DNA copy number analysis of <italic>MDM2, MDM4</italic>, and <italic>TP53</italic> loci in each sample. The dotted line indicates the position of the gene within the region. Samples with chromosomal gain or loss are marked with a red or blue asterisk, respectively. (C) Confirmation of <italic>MDM2</italic> and <italic>MDM4</italic> copy number gain using quantitative PCR of tumour DNA. Samples with gain are indicated in red. (D) Immunohistochemical staining of p53 protein in the following selected mutation-negative cases: case 41358; case 60258 (<italic>MDM2</italic> gain); case 22029 (<italic>MDM4</italic> gain); and case 60049 (<italic>MDM4</italic> gain).</p></caption><graphic xlink:href=\"path0221-0049-f1\"/></fig><p>Pathology review of the mutation-negative cases resulted in the re-classification of case 60214 as LGS carcinoma, case 23221 as high-grade carcinoma of uncertain primary site, and case 533 as mixed LGS/HGS type (Supporting information, Supplementary <xref ref-type=\"supplementary-material\" rid=\"SD2\">Table 2</xref>). Furthermore, aCGH analysis of cases 533 and 60214 did not show high-frequency DCNAs that are typical of HGPSC [<xref ref-type=\"bibr\" rid=\"b11\">11</xref>]. The original diagnosis of HGPSC was confirmed in the remaining four cases (22029, 41358, 60049, and 60258). To further exclude the possibility of pathological misclassification of endometrioid, clear cell or LGS tumours in the mutation-negative cases, we also tested for mutations commonly associated with these subtypes in <italic>KRAS</italic> (exon 2), <italic>BRAF</italic> (exon 15), <italic>CTNNB1</italic> (exon 3), and <italic>PIK3CA</italic> (exons 10 and 21). Case 60214 had a missense mutation (G12C) in exon 2 of <italic>KRAS</italic>, consistent with LGS origin. No other mutations were detected (data not shown).</p><p>Other mutation-independent mechanisms of inactivation of the p53 pathway were investigated. p53 is targeted for degradation through the activity of MDM2 protein and is also regulated by the related MDM4 protein [<xref ref-type=\"bibr\" rid=\"b13\">13</xref>,<xref ref-type=\"bibr\" rid=\"b14\">14</xref>]. Amplification of <italic>MDM2</italic> is associated with loss of p53 activity in some solid tumours [<xref ref-type=\"bibr\" rid=\"b15\">15</xref>,<xref ref-type=\"bibr\" rid=\"b16\">16</xref>]. aCGH analysis of the remaining four cases showed copy number gain of <italic>MDM4</italic> in cases 22029 and 60049, and of <italic>MDM2</italic> in case 60258 (<xref ref-type=\"fig\" rid=\"fig01\">Figure 1B</xref>). These observations were confirmed by qPCR (<xref ref-type=\"fig\" rid=\"fig01\">Figure 1C</xref>). Interestingly, both cases with <italic>MDM4</italic> gain showed heterogeneous p53 immunostaining, including cytoplasmic staining (<xref ref-type=\"fig\" rid=\"fig01\">Figure 1D</xref>). Case 60258 with <italic>MDM2</italic> gain showed strong nuclear staining for p53 protein (<xref ref-type=\"fig\" rid=\"fig01\">Figure 1D</xref>). Interestingly, the remaining case with no MDM2 or MDM4 gain (41358) showed strong nuclear p53 immunostaining (<xref ref-type=\"fig\" rid=\"fig01\">Figure 1D</xref>).</p><p>In summary, the overall <italic>TP53</italic> mutation rate for selected HGPSC cases in the pilot and validation cohorts was 94.4% (<italic>n</italic> = 119/126). After removing the three mutation-negative cases with atypical or LGS histology, the adjusted overall rate was 96.7% (<italic>n</italic> = 119/123). In the remaining four mutation-negative HGPSC cases, three showed evidence of p53 dysfunction with increased p53 nuclear staining or cytoplasmic mislocalization that was correlated with gains at the <italic>MDM2/4</italic> loci.</p></sec><sec><title><italic>TP53</italic> mutation type is not prognostic in HGPSC</title><p>It has been previously reported that the type of <italic>TP53</italic> mutation may predict outcome in ovarian cancer [<xref ref-type=\"bibr\" rid=\"b17\">17</xref>]. Although our study was not adequately powered to detect small effects, we tested for trends in the data. For this analysis, we used all presumed HGPSC cases from the pilot study and the validation cohort (<italic>n</italic> = 127, <italic>n</italic> = 120 had <italic>TP53</italic> mutation; <xref ref-type=\"table\" rid=\"tbl3\">Tables 3</xref> and <xref ref-type=\"table\" rid=\"tbl4\">4</xref>). There was no indication of significant association between the frequency of <italic>TP53</italic> mutation or the type of mutation (missense versus not) and progression-free survival or overall survival. Similarly, whether or not the missense mutation affected the DNA binding motifs formed by the L2 and L3 loops (codons 164&#x02013;194 and 237&#x02013;250, respectively) or the LSH motifs (codons 119&#x02013;135 and 272&#x02013;287) [<xref ref-type=\"bibr\" rid=\"b18\">18</xref>] had no significant effect on progression-free survival or overall survival (<xref ref-type=\"table\" rid=\"tbl3\">Tables 3</xref> and <xref ref-type=\"table\" rid=\"tbl4\">4</xref>). There was also no effect of codon 72 polymorphism (Arg, Pro or heterozygous) on prognosis in any of the HGPSCs with mutations (data not shown) or in tumours with missense mutations (<xref ref-type=\"table\" rid=\"tbl3\">Tables 3</xref> and <xref ref-type=\"table\" rid=\"tbl4\">4</xref>). There was no significant interaction between the type of missense mutation (DNA binding or not) and codon 72 polymorphism (data not shown).</p><table-wrap id=\"tbl3\" position=\"float\"><label>Table 3</label><caption><p>The type of <italic>TP53</italic> mutation does not influence progression-free survival</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\">Index category (<italic>n</italic>)</th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Comparison (<italic>n</italic>)</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Hazard ratio</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">CI</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Log-rank <italic>p</italic> value</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Mutant (120)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Wild type (7)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1.4</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.6&#x02013;3.1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.5</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Not missense (52)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Missense (68)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.9</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.6&#x02013;1.4</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.8</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">DBM (44)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">NDBM (24)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.6&#x02013;1.7</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td></tr></tbody></table><table-wrap-foot><fn><p>DBM = DNA binding mutations; NDBM = non-DNA binding mutations.</p></fn></table-wrap-foot></table-wrap><table-wrap id=\"tbl4\" position=\"float\"><label>Table 4</label><caption><p>The type of <italic>TP53</italic> mutation does not influence overall survival</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\">Index category (<italic>n</italic>)</th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Comparison (<italic>n</italic>)</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Hazard ratio</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">CI</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Log-rank <italic>p</italic> value</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Mutant (120)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Wild type (7)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.6&#x02013;4.6</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.3</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Not missense (52)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Missense (68)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.7&#x02013;1.6</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">DBM (44)</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">NDBM (24)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.7&#x02013;2.4</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">0.4</td></tr></tbody></table><table-wrap-foot><fn><p>DBM = DNA binding mutations; NDBM = non-DNA binding mutations.</p></fn></table-wrap-foot></table-wrap></sec></sec><sec><title>Discussion</title><p>By addressing the limitations of numerous other studies of <italic>TP53</italic> mutation in ovarian cancer, we have demonstrated for the first time that HGPSCs have the highest frequency of p53 mutation of any solid cancer. Approximately two-thirds of mutations occurred in exons 5&#x02013;8, known to be mutation hotspots for <italic>TP53</italic>. By sequencing exons 2&#x02013;4 and 9&#x02013;11, we identified 22 mutations (<italic>n</italic> = 22/126, 17.5%) that would not have otherwise been identified in our series and are poorly represented in previous reports. The previously unreported mutations that we have discovered are mostly insertions/deletions predicted to result in truncated proteins, but it is notable that HGPSC cases have frequent involvement of exon 10 and specifically involvement of position g.16915 (codon 342) in the oligomerization domain of p53. This region also contains a putative nuclear export signal for p53 and a recent study of a novel missense mutation at codon 351 has shown that K351N results in significant loss of p53 tetramerization, reduced BAX activation, and reduced nuclear export of p53 [<xref ref-type=\"bibr\" rid=\"b19\">19</xref>]. Cytosolic export of p53 is required for cisplatin-induced apoptosis in A2780 cells, suggesting that combined loss of tetramerization and nuclear export is critical for platinum resistance.</p><p>Of the mutation-negative cases, approximately half were reclassified on pathological review as unrepresentative of HGPSC, including two tumours that were LGS and associated with borderline tumour. Serous cancers may follow two distinct routes to malignancy [<xref ref-type=\"bibr\" rid=\"b20\">20</xref>]. A minority of so-called type I tumours progress from borderline tumours, whereas the majority of HGS (type II) tumours arise with no evidence of LGS or borderline tumours. Borderline serous tumours and their type I invasive counterparts are generally low grade, have relatively limited genomic DNA copy number change, have frequent activating RAS pathway mutations, and have substantially lower rates of <italic>TP53</italic> mutation compared with type II tumours [<xref ref-type=\"bibr\" rid=\"b20\">20</xref>&#x02013;<xref ref-type=\"bibr\" rid=\"b23\">23</xref>]. Detailed analysis of six cases of HGPSC arising from LGS and borderline tumours showed no <italic>TP53</italic> mutation [<xref ref-type=\"bibr\" rid=\"b24\">24</xref>]. Consistent with this model, mutation-negative case 533 had both LGS and HGS components and a bland aCGH profile.</p><p>Exclusion of LGS cases and mixed high-grade serous cases increased the <italic>TP53</italic> mutation frequency to approximately 97% of HGPSCs. Of the remaining mutation-negative cases, three may have had inactivation of the p53 pathway through chromosomal gain at the <italic>MDM2</italic> or <italic>MDM4</italic> loci or potentially other genes that alter p53 function in <italic>trans</italic> such as <italic>RFWD2</italic> and <italic>RCHY1</italic> [<xref ref-type=\"bibr\" rid=\"b25\">25</xref>,<xref ref-type=\"bibr\" rid=\"b26\">26</xref>].</p><p>We are aware of over 70 publications that have sought to relate <italic>TP53</italic> mutation with clinical outcome in ovarian cancer, with conflicting conclusions about the importance of <italic>TP53</italic> mutations for tumour aggressiveness, response to treatment, and survival. There is no sign that a conclusion is at hand: since 2008, there have been a number of papers relating <italic>TP53</italic> mutation and clinical outcome [<xref ref-type=\"bibr\" rid=\"b27\">27</xref>&#x02013;<xref ref-type=\"bibr\" rid=\"b30\">30</xref>], with evidence presented for [<xref ref-type=\"bibr\" rid=\"b29\">29</xref>] and against [<xref ref-type=\"bibr\" rid=\"b28\">28</xref>] mutation being a significant prognostic factor. A recent meta-analysis of 62 studies concluded that <italic>TP53</italic> mutation had a modest impact on survival in ovarian cancer but the effect was insufficient to support clinical application [<xref ref-type=\"bibr\" rid=\"b4\">4</xref>]. Whilst meta-analyses such as that of de Graeff <italic>et al</italic> [<xref ref-type=\"bibr\" rid=\"b4\">4</xref>] have the potential to detect subtle trends that may be missed in all but the largest individual studies, they are confounded if there is a substantial false-negative rate, as our findings indicate. Given the near ubiquitous occurrence of <italic>TP53</italic> mutation, it difficult to conclude that it can be of significant prognostic or predictive significance in HGPSC. We have not, however, excluded the possibility that specific mutations can influence prognosis or response to therapy, an important consideration given the mounting evidence that some <italic>TP53</italic> mutations have specific neomorphic functions [<xref ref-type=\"bibr\" rid=\"b1\">1</xref>]. It is thought that inactivation of the p53 pathway is a common, perhaps mandatory, event in all solid cancers [<xref ref-type=\"bibr\" rid=\"b31\">31</xref>]. What is particularly unusual about HGPSC is an almost total reliance on mutation <italic>TP53</italic> itself for pathway inactivation, perhaps reflecting the requirement of certain tissues for neomorphic mutations for their transformation [<xref ref-type=\"bibr\" rid=\"b1\">1</xref>]. Future work will need to address interactions between specific mutations and other loci such as <italic>PTEN</italic> and <italic>CDKN2A</italic> [<xref ref-type=\"bibr\" rid=\"b32\">32</xref>,<xref ref-type=\"bibr\" rid=\"b33\">33</xref>].</p><p>Our findings have important implications for the new understanding of the pathogenesis of HGPSC. p53 immunostaining and mutation appear to be a feature of early-stage Fallopian tube lesions from <italic>BRCA1/2</italic> mutation carriers [<xref ref-type=\"bibr\" rid=\"b7\">7</xref>], suggesting that p53 dysfunction is essential for early tumourigenesis of HGPSC. The notion that <italic>TP53</italic> mutation is required to allow survival of <italic>BRCA</italic>-deficient ovarian precursor lesions is consistent with the extremely high rates of <italic>TP53</italic> mutation in breast cancers arising in <italic>BRCA1</italic> mutant women [<xref ref-type=\"bibr\" rid=\"b34\">34</xref>,<xref ref-type=\"bibr\" rid=\"b35\">35</xref>]. BRCA dysfunction may also occur in sporadic HGPSC via a variety of inactivating mechanisms but the very high rate of <italic>TP53</italic> mutation in our series of sporadic HGPSC cases suggests that most are &#x02018;BRCA-like&#x02019; with deficiencies for <italic>BRCA</italic> or closely related DNA repair pathways [<xref ref-type=\"bibr\" rid=\"b36\">36</xref>].</p><p>It has become increasingly clear that ovarian cancer is a series of distinctly different diseases with different aetiologies [<xref ref-type=\"bibr\" rid=\"b23\">23</xref>,<xref ref-type=\"bibr\" rid=\"b37\">37</xref>] and that the development of reliable biomarkers must be subtype-specific [<xref ref-type=\"bibr\" rid=\"b38\">38</xref>]. For example, several biomarkers that appeared to be prognostic in a cohort of all major subtypes of ovarian cancer were not informative within homogeneous subtypes. Our findings provide another example of the importance of refining the analysis of ovarian biomarkers by focusing on the most clinically significant group, HGPSC.</p><p>The key oncogenic and tumour suppressor genes for HGPSC have not been identified and as it has high rates of genomic instability, many of the alterations described may be passenger mutations. Our results for the prevalence of <italic>TP53</italic> mutation may be conservative as our prospectively determined selection criteria excluded low-stage HGPSC cases (<italic>n</italic> = 15). This was because true low-stage HGPSC is a rare entity and often suggests misdiagnosis of LGS histiotype. However, 13/15 of these cases had <italic>TP53</italic> mutation. Our data are therefore consistent with mutant <italic>TP53</italic> being an essential driver mutation early in the pathogenesis of HGPSC. Future studies will need to focus on studying the biological significance of the different types of <italic>TP53</italic> mutation in HGPSC and developing p53 synthetic-lethal therapies for patients with this disease.</p></sec></body><back><ack><p>We thank Sarah Hyland, Philip Chambers, Christian Bourne, and Nik Matthews for expert technical assistance. We acknowledge the support of the University of Cambridge, NIHR Cambridge Biomedical Research Centre, the Cambridge Experimental Cancer Medicine Centre, and Hutchison Whampoa Ltd. AAA holds a CR-UK Clinician Scientist Fellowship. D Bowtell, G Chenevix-Trench, A deFazio, A Green, D Gertig, and P Webb&#x02014;The AOCS Management Group&#x02014;gratefully acknowledge the contributions of the full AOCS Study Group, listed on <ext-link ext-link-type=\"uri\" xlink:href=\"http://www.aocstudy.org\">http://www.aocstudy.org</ext-link>, and express their gratitude to all women who participated in the study. The AOCS was supported by the US Army Medical Research and Material Command under DAMD17-01-1-0729, The Cancer Council Victoria, Queensland Cancer Fund, The Cancer Council New South Wales, The Cancer Council South Australia, The Cancer Foundation of Western Australia, The Cancer Council Tasmania, and the NH&#x00026;MRC. We also gratefully acknowledge Catherine Emmanuel, Catherine Kennedy, and Yoke-Eng Chiew for samples, and data from the Westmead Gynaecological Oncology Tissue Bank, a member of the Australasian Biospecimens Network-Oncology group funded by the NH&#x00026;MRC. This work was funded by Cancer Research UK (CR-UK), the Medical Research Council (MRC), and the National Health and Medical Research Council of Australia (NHMRC).</p></ack><ref-list><title>References</title><p>Note: References 39&#x02013;50 are cited in the Supporting information to this article.</p><ref id=\"b1\"><label>1</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Brosh</surname><given-names>R</given-names></name><name><surname>Rotter</surname><given-names>V</given-names></name></person-group><article-title>When mutants gain new powers: news from the mutant p53 field</article-title><source>Nature Rev Cancer</source><year>2009</year><volume>9</volume><fpage>701</fpage><lpage>713</lpage><pub-id pub-id-type=\"pmid\">19693097</pub-id></element-citation></ref><ref id=\"b2\"><label>2</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Marks</surname><given-names>JR</given-names></name><name><surname>Davidoff</surname><given-names>AM</given-names></name><name><surname>Kerns</surname><given-names>BJ</given-names></name><name><surname>Humphrey</surname><given-names>PA</given-names></name><name><surname>Pence</surname><given-names>JC</given-names></name><name><surname>Dodge</surname><given-names>RK</given-names></name><etal/></person-group><article-title>Overexpression and mutation of p53 in epithelial ovarian cancer</article-title><source>Cancer Res</source><year>1991</year><volume>51</volume><fpage>2979</fpage><lpage>2984</lpage><pub-id pub-id-type=\"pmid\">2032235</pub-id></element-citation></ref><ref id=\"b3\"><label>3</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Okamoto</surname><given-names>A</given-names></name><name><surname>Sameshima</surname><given-names>Y</given-names></name><name><surname>Yokoyama</surname><given-names>S</given-names></name><name><surname>Terashima</surname><given-names>Y</given-names></name><name><surname>Sugimura</surname><given-names>T</given-names></name><name><surname>Terada</surname><given-names>M</given-names></name><etal/></person-group><article-title>Frequent allelic losses and mutations of the p53 gene in human ovarian cancer</article-title><source>Cancer Res</source><year>1991</year><volume>51</volume><fpage>5171</fpage><lpage>5176</lpage><pub-id pub-id-type=\"pmid\">1680546</pub-id></element-citation></ref><ref id=\"b4\"><label>4</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>de Graeff</surname><given-names>P</given-names></name><name><surname>Crijns</surname><given-names>AP</given-names></name><name><surname>de Jong</surname><given-names>S</given-names></name><name><surname>Boezen</surname><given-names>M</given-names></name><name><surname>Post</surname><given-names>WJ</given-names></name><name><surname>de Vries</surname><given-names>EG</given-names></name><etal/></person-group><article-title>Modest effect of p53, EGFR and HER-2/neu on prognosis in epithelial ovarian cancer: a meta-analysis</article-title><source>Br J Cancer</source><year>2009</year><volume>101</volume><fpage>149</fpage><lpage>159</lpage><pub-id pub-id-type=\"pmid\">19513073</pub-id></element-citation></ref><ref id=\"b5\"><label>5</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hall</surname><given-names>J</given-names></name><name><surname>Paul</surname><given-names>J</given-names></name><name><surname>Brown</surname><given-names>R</given-names></name></person-group><article-title>Critical evaluation of p53 as a prognostic marker in ovarian cancer</article-title><source>Expert Rev Mol Med</source><year>2004</year><volume>6</volume><fpage>1</fpage><lpage>20</lpage><pub-id pub-id-type=\"pmid\">15147608</pub-id></element-citation></ref><ref id=\"b6\"><label>6</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Soussi</surname><given-names>T</given-names></name><name><surname>Ishioka</surname><given-names>C</given-names></name><name><surname>Claustres</surname><given-names>M</given-names></name><name><surname>Beroud</surname><given-names>C</given-names></name></person-group><article-title>Locus-specific mutation databases: pitfalls and good practice based on the p53 experience</article-title><source>Nature Rev Cancer</source><year>2006</year><volume>6</volume><fpage>83</fpage><lpage>90</lpage><pub-id pub-id-type=\"pmid\">16397528</pub-id></element-citation></ref><ref id=\"b7\"><label>7</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>Y</given-names></name><name><surname>Miron</surname><given-names>A</given-names></name><name><surname>Drapkin</surname><given-names>R</given-names></name><name><surname>Nucci</surname><given-names>MR</given-names></name><name><surname>Medeiros</surname><given-names>F</given-names></name><name><surname>Saleemuddin</surname><given-names>A</given-names></name><etal/></person-group><article-title>A candidate precursor to serous carcinoma that originates in the distal Fallopian tube</article-title><source>J Pathol</source><year>2007</year><volume>211</volume><fpage>26</fpage><lpage>35</lpage><pub-id pub-id-type=\"pmid\">17117391</pub-id></element-citation></ref><ref id=\"b8\"><label>8</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Crum</surname><given-names>CP</given-names></name><name><surname>Drapkin</surname><given-names>R</given-names></name><name><surname>Miron</surname><given-names>A</given-names></name><name><surname>Ince</surname><given-names>TA</given-names></name><name><surname>Muto</surname><given-names>M</given-names></name><name><surname>Kindelberger</surname><given-names>DW</given-names></name><etal/></person-group><article-title>The distal Fallopian tube: a new model for pelvic serous carcinogenesis</article-title><source>Curr Opin Obstet Gynecol</source><year>2007</year><volume>19</volume><fpage>3</fpage><lpage>9</lpage><pub-id pub-id-type=\"pmid\">17218844</pub-id></element-citation></ref><ref id=\"b9\"><label>9</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jordan</surname><given-names>SJ</given-names></name><name><surname>Green</surname><given-names>AC</given-names></name><name><surname>Whiteman</surname><given-names>DC</given-names></name><name><surname>Moore</surname><given-names>SP</given-names></name><name><surname>Bain</surname><given-names>CJ</given-names></name><name><surname>Gertig</surname><given-names>DM</given-names></name><etal/></person-group><article-title>Serous ovarian, Fallopian tube and primary peritoneal cancers: a comparative epidemiological analysis</article-title><source>Int J Cancer</source><year>2008</year><volume>122</volume><fpage>1598</fpage><lpage>1603</lpage><pub-id pub-id-type=\"pmid\">18058817</pub-id></element-citation></ref><ref id=\"b10\"><label>10</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Herrington</surname><given-names>CS</given-names></name><name><surname>McCluggage</surname><given-names>WG</given-names></name></person-group><article-title>The emerging role of the distal Fallopian tube and p53 in pelvic serous carcinogenesis</article-title><source>J Pathol</source><year>2007</year><volume>220</volume><fpage>5</fpage><lpage>6</lpage><pub-id pub-id-type=\"pmid\">19882674</pub-id></element-citation></ref><ref id=\"b11\"><label>11</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>McShane</surname><given-names>LM</given-names></name><name><surname>Altman</surname><given-names>DG</given-names></name><name><surname>Sauerbrei</surname><given-names>W</given-names></name><name><surname>Taube</surname><given-names>SE</given-names></name><name><surname>Gion</surname><given-names>M</given-names></name><name><surname>Clark</surname><given-names>GM</given-names></name></person-group><article-title>Reporting recommendations for tumor marker prognostic studies (REMARK)</article-title><source>J Natl Cancer Inst</source><year>2005</year><volume>97</volume><fpage>1180</fpage><lpage>1184</lpage><pub-id pub-id-type=\"pmid\">16106022</pub-id></element-citation></ref><ref id=\"b12\"><label>12</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Olivier</surname><given-names>M</given-names></name><name><surname>Petitjean</surname><given-names>A</given-names></name><name><surname>Marcel</surname><given-names>V</given-names></name><name><surname>Petre</surname><given-names>A</given-names></name><name><surname>Mounawar</surname><given-names>M</given-names></name><name><surname>Plymoth</surname><given-names>A</given-names></name><etal/></person-group><article-title>Recent advances in p53 research: an interdisciplinary perspective</article-title><source>Cancer Gene Ther</source><year>2009</year><volume>16</volume><fpage>1</fpage><lpage>12</lpage><pub-id pub-id-type=\"pmid\">18802452</pub-id></element-citation></ref><ref id=\"b13\"><label>13</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Montes de Oca Luna</surname><given-names>R</given-names></name><name><surname>Wagner</surname><given-names>DS</given-names></name><name><surname>Lozano</surname><given-names>G</given-names></name></person-group><article-title>Rescue of early embryonic lethality in mdm2-deficient mice by deletion of p53</article-title><source>Nature</source><year>1995</year><volume>378</volume><fpage>203</fpage><lpage>206</lpage><pub-id pub-id-type=\"pmid\">7477326</pub-id></element-citation></ref><ref id=\"b14\"><label>14</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Parant</surname><given-names>J</given-names></name><name><surname>Chavez-Reyes</surname><given-names>A</given-names></name><name><surname>Little</surname><given-names>NA</given-names></name><name><surname>Yan</surname><given-names>W</given-names></name><name><surname>Reinke</surname><given-names>V</given-names></name><name><surname>Jochemsen</surname><given-names>AG</given-names></name><etal/></person-group><article-title>Rescue of embryonic lethality in Mdm4-null mice by loss of Trp53 suggests a nonoverlapping pathway with MDM2 to regulate p53</article-title><source>Nature Genet</source><year>2001</year><volume>29</volume><fpage>92</fpage><lpage>95</lpage><pub-id pub-id-type=\"pmid\">11528400</pub-id></element-citation></ref><ref id=\"b15\"><label>15</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Oliner</surname><given-names>JD</given-names></name><name><surname>Kinzler</surname><given-names>KW</given-names></name><name><surname>Meltzer</surname><given-names>PS</given-names></name><name><surname>George</surname><given-names>DL</given-names></name><name><surname>Vogelstein</surname><given-names>B</given-names></name></person-group><article-title>Amplification of a gene encoding a p53-associated protein in human sarcomas</article-title><source>Nature</source><year>1992</year><volume>358</volume><fpage>80</fpage><lpage>83</lpage><pub-id pub-id-type=\"pmid\">1614537</pub-id></element-citation></ref><ref id=\"b16\"><label>16</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>McCann</surname><given-names>AH</given-names></name><name><surname>Kirley</surname><given-names>A</given-names></name><name><surname>Carney</surname><given-names>DN</given-names></name><name><surname>Corbally</surname><given-names>N</given-names></name><name><surname>Magee</surname><given-names>HM</given-names></name><name><surname>Keating</surname><given-names>G</given-names></name><etal/></person-group><article-title>Amplification of the MDM2 gene in human breast cancer and its association with MDM2 and p53 protein status</article-title><source>Br J Cancer</source><year>1995</year><volume>71</volume><fpage>981</fpage><lpage>985</lpage><pub-id pub-id-type=\"pmid\">7734324</pub-id></element-citation></ref><ref id=\"b17\"><label>17</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shahin</surname><given-names>MS</given-names></name><name><surname>Hughes</surname><given-names>JH</given-names></name><name><surname>Sood</surname><given-names>AK</given-names></name><name><surname>Buller</surname><given-names>RE</given-names></name></person-group><article-title>The prognostic significance of p53 tumor suppressor gene alterations in ovarian carcinoma</article-title><source>Cancer</source><year>2000</year><volume>89</volume><fpage>2006</fpage><lpage>2017</lpage><pub-id pub-id-type=\"pmid\">11064359</pub-id></element-citation></ref><ref id=\"b18\"><label>18</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Olivier</surname><given-names>M</given-names></name><name><surname>Langerod</surname><given-names>A</given-names></name><name><surname>Carrieri</surname><given-names>P</given-names></name><name><surname>Bergh</surname><given-names>J</given-names></name><name><surname>Klaar</surname><given-names>S</given-names></name><name><surname>Eyfjord</surname><given-names>J</given-names></name><etal/></person-group><article-title>The clinical value of somatic TP53 gene mutations in 1794 patients with breast cancer</article-title><source>Clin Cancer Res</source><year>2006</year><volume>12</volume><fpage>1157</fpage><lpage>1167</lpage><pub-id pub-id-type=\"pmid\">16489069</pub-id></element-citation></ref><ref id=\"b19\"><label>19</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Muscolini</surname><given-names>M</given-names></name><name><surname>Montagni</surname><given-names>E</given-names></name><name><surname>Caristi</surname><given-names>S</given-names></name><name><surname>Nomura</surname><given-names>T</given-names></name><name><surname>Kamada</surname><given-names>R</given-names></name><name><surname>Di Agostino</surname><given-names>S</given-names></name><etal/></person-group><article-title>Characterization of a new cancer-associated mutant of p53 with a missense mutation (K351N) in the tetramerization domain</article-title><source>Cell Cycle</source><year>2009</year><volume>8</volume><fpage>3396</fpage><lpage>3405</lpage><pub-id pub-id-type=\"pmid\">19806023</pub-id></element-citation></ref><ref id=\"b20\"><label>20</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Vang</surname><given-names>R</given-names></name><name><surname>Shih Ie</surname><given-names>M</given-names></name><name><surname>Kurman</surname><given-names>RJ</given-names></name></person-group><article-title>Ovarian low-grade and high-grade serous carcinoma: pathogenesis, clinicopathologic and molecular biologic features, and diagnostic problems</article-title><source>Adv Anat Pathol</source><year>2009</year><volume>16</volume><fpage>267</fpage><lpage>282</lpage><pub-id pub-id-type=\"pmid\">19700937</pub-id></element-citation></ref><ref id=\"b21\"><label>21</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shih Ie</surname><given-names>M</given-names></name><name><surname>Kurman</surname><given-names>RJ</given-names></name></person-group><article-title>Molecular pathogenesis of ovarian borderline tumors: new insights and old challenges</article-title><source>Clin Cancer Res</source><year>2005</year><volume>11</volume><fpage>7273</fpage><lpage>7279</lpage><pub-id pub-id-type=\"pmid\">16243797</pub-id></element-citation></ref><ref id=\"b22\"><label>22</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Meinhold-Heerlein</surname><given-names>I</given-names></name><name><surname>Bauerschlag</surname><given-names>D</given-names></name><name><surname>Hilpert</surname><given-names>F</given-names></name><name><surname>Dimitrov</surname><given-names>P</given-names></name><name><surname>Sapinoso</surname><given-names>LM</given-names></name><name><surname>Orlowska-Volk</surname><given-names>M</given-names></name><etal/></person-group><article-title>Molecular and prognostic distinction between serous ovarian carcinomas of varying grade and malignant potential</article-title><source>Oncogene</source><year>2005</year><volume>24</volume><fpage>1053</fpage><lpage>1065</lpage><pub-id pub-id-type=\"pmid\">15558012</pub-id></element-citation></ref><ref id=\"b23\"><label>23</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Salani</surname><given-names>R</given-names></name><name><surname>Kurman</surname><given-names>RJ</given-names></name><name><surname>Giuntoli</surname><given-names>R</given-names><suffix>2nd</suffix></name><name><surname>Gardner</surname><given-names>G</given-names></name><name><surname>Bristow</surname><given-names>R</given-names></name><name><surname>Wang</surname><given-names>TL</given-names></name><etal/></person-group><article-title>Assessment of TP53 mutation using purified tissue samples of ovarian serous carcinomas reveals a higher mutation rate than previously reported and does not correlate with drug resistance</article-title><source>Int J Gynecol Cancer</source><year>2008</year><volume>18</volume><fpage>487</fpage><lpage>491</lpage><pub-id pub-id-type=\"pmid\">17692090</pub-id></element-citation></ref><ref id=\"b24\"><label>24</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Dehari</surname><given-names>R</given-names></name><name><surname>Kurman</surname><given-names>RJ</given-names></name><name><surname>Logani</surname><given-names>S</given-names></name><name><surname>Shih</surname><given-names>IeM</given-names></name></person-group><article-title>The development of high-grade serous carcinoma from atypical proliferative (borderline) serous tumors and low-grade micropapillary serous carcinoma: a morphologic and molecular genetic analysis</article-title><source>Am J Surg Pathol</source><year>2007</year><volume>31</volume><fpage>1007</fpage><lpage>1012</lpage><pub-id pub-id-type=\"pmid\">17592266</pub-id></element-citation></ref><ref id=\"b25\"><label>25</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Leng</surname><given-names>RP</given-names></name><name><surname>Lin</surname><given-names>Y</given-names></name><name><surname>Ma</surname><given-names>W</given-names></name><name><surname>Wu</surname><given-names>H</given-names></name><name><surname>Lemmers</surname><given-names>B</given-names></name><name><surname>Chung</surname><given-names>S</given-names></name><etal/></person-group><article-title>Pirh2, a p53-induced ubiquitin-protein ligase, promotes p53 degradation</article-title><source>Cell</source><year>2003</year><volume>112</volume><fpage>779</fpage><lpage>791</lpage><pub-id pub-id-type=\"pmid\">12654245</pub-id></element-citation></ref><ref id=\"b26\"><label>26</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Dornan</surname><given-names>D</given-names></name><name><surname>Wertz</surname><given-names>I</given-names></name><name><surname>Shimizu</surname><given-names>H</given-names></name><name><surname>Arnott</surname><given-names>D</given-names></name><name><surname>Frantz</surname><given-names>GD</given-names></name><name><surname>Dowd</surname><given-names>P</given-names></name><etal/></person-group><article-title>The ubiquitin ligase COP1 is a critical negative regulator of p53</article-title><source>Nature</source><year>2004</year><volume>429</volume><fpage>86</fpage><lpage>92</lpage><pub-id pub-id-type=\"pmid\">15103385</pub-id></element-citation></ref><ref id=\"b27\"><label>27</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bartel</surname><given-names>F</given-names></name><name><surname>Jung</surname><given-names>J</given-names></name><name><surname>Bohnke</surname><given-names>A</given-names></name><name><surname>Gradhand</surname><given-names>E</given-names></name><name><surname>Zeng</surname><given-names>K</given-names></name><name><surname>Thomssen</surname><given-names>C</given-names></name><etal/></person-group><article-title>Both germ line and somatic genetics of the p53 pathway affect ovarian cancer incidence and survival</article-title><source>Clin Cancer Res</source><year>2008</year><volume>14</volume><fpage>89</fpage><lpage>96</lpage><pub-id pub-id-type=\"pmid\">18172257</pub-id></element-citation></ref><ref id=\"b28\"><label>28</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Darcy</surname><given-names>KM</given-names></name><name><surname>Brady</surname><given-names>WE</given-names></name><name><surname>McBroom</surname><given-names>JW</given-names></name><name><surname>Bell</surname><given-names>JG</given-names></name><name><surname>Young</surname><given-names>RC</given-names></name><name><surname>McGuire</surname><given-names>WP</given-names></name><etal/></person-group><article-title>Associations between p53 overexpression and multiple measures of clinical outcome in high-risk, early stage or suboptimally-resected, advanced stage epithelial ovarian cancers. A Gynecologic Oncology Group study</article-title><source>Gynecol Oncol</source><year>2008</year><volume>111</volume><fpage>487</fpage><lpage>495</lpage><pub-id pub-id-type=\"pmid\">18834621</pub-id></element-citation></ref><ref id=\"b29\"><label>29</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hogdall</surname><given-names>EV</given-names></name><name><surname>Christensen</surname><given-names>L</given-names></name><name><surname>Hogdall</surname><given-names>CK</given-names></name><name><surname>Frederiksen</surname><given-names>K</given-names></name><name><surname>Gayther</surname><given-names>S</given-names></name><name><surname>Blaakaer</surname><given-names>J</given-names></name><etal/></person-group><article-title>Distribution of p53 expression in tissue from 774 Danish ovarian tumour patients and its prognostic significance in ovarian carcinomas</article-title><source>APMIS</source><year>2008</year><volume>116</volume><fpage>400</fpage><lpage>409</lpage><pub-id pub-id-type=\"pmid\">18452430</pub-id></element-citation></ref><ref id=\"b30\"><label>30</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kupryjanczyk</surname><given-names>J</given-names></name><name><surname>Kraszewska</surname><given-names>E</given-names></name><name><surname>Ziolkowska-Seta</surname><given-names>I</given-names></name><name><surname>Madry</surname><given-names>R</given-names></name><name><surname>Timorek</surname><given-names>A</given-names></name><name><surname>Markowska</surname><given-names>J</given-names></name><etal/></person-group><article-title>TP53 status and taxane&#x02013;platinum versus platinum-based therapy in ovarian cancer patients: a non-randomized retrospective study</article-title><source>BMC Cancer</source><year>2008</year><volume>8</volume><fpage>27</fpage><pub-id pub-id-type=\"pmid\">18230133</pub-id></element-citation></ref><ref id=\"b31\"><label>31</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Donehower</surname><given-names>LA</given-names></name><name><surname>Harvey</surname><given-names>M</given-names></name><name><surname>Slagle</surname><given-names>BL</given-names></name><name><surname>McArthur</surname><given-names>MJ</given-names></name><name><surname>Montgomery</surname><given-names>CA</given-names><suffix>Jr</suffix></name><name><surname>Butel</surname><given-names>JS</given-names></name><etal/></person-group><article-title>Mice deficient for p53 are developmentally normal but susceptible to spontaneous tumours</article-title><source>Nature</source><year>1992</year><volume>356</volume><fpage>215</fpage><lpage>221</lpage><pub-id pub-id-type=\"pmid\">1552940</pub-id></element-citation></ref><ref id=\"b32\"><label>32</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Terzian</surname><given-names>T</given-names></name><name><surname>Suh</surname><given-names>YA</given-names></name><name><surname>Iwakuma</surname><given-names>T</given-names></name><name><surname>Post</surname><given-names>SM</given-names></name><name><surname>Neumann</surname><given-names>M</given-names></name><name><surname>Lang</surname><given-names>GA</given-names></name><etal/></person-group><article-title>The inherent instability of mutant p53 is alleviated by Mdm2 or p16INK4a loss</article-title><source>Genes Dev</source><year>2008</year><volume>22</volume><fpage>1337</fpage><lpage>1344</lpage><pub-id pub-id-type=\"pmid\">18483220</pub-id></element-citation></ref><ref id=\"b33\"><label>33</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Guessous</surname><given-names>F</given-names></name><name><surname>Kwon</surname><given-names>S</given-names></name><name><surname>Kumar</surname><given-names>M</given-names></name><name><surname>Ibidapo</surname><given-names>O</given-names></name><name><surname>Fuller</surname><given-names>L</given-names></name><etal/></person-group><article-title>PTEN has tumor-promoting properties in the setting of gain-of-function p53 mutations</article-title><source>Cancer Res</source><year>2008</year><volume>68</volume><fpage>1723</fpage><lpage>1731</lpage><pub-id pub-id-type=\"pmid\">18339852</pub-id></element-citation></ref><ref id=\"b34\"><label>34</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Phillips</surname><given-names>KA</given-names></name><name><surname>Nichol</surname><given-names>K</given-names></name><name><surname>Ozcelik</surname><given-names>H</given-names></name><name><surname>Knight</surname><given-names>J</given-names></name><name><surname>Done</surname><given-names>SJ</given-names></name><name><surname>Goodwin</surname><given-names>PJ</given-names></name><etal/></person-group><article-title>Frequency of p53 mutations in breast carcinomas from Ashkenazi Jewish carriers of BRCA1 mutations</article-title><source>J Natl Cancer Inst</source><year>1999</year><volume>91</volume><fpage>469</fpage><lpage>473</lpage><pub-id pub-id-type=\"pmid\">10070948</pub-id></element-citation></ref><ref id=\"b35\"><label>35</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Holstege</surname><given-names>H</given-names></name><name><surname>Joosse</surname><given-names>SA</given-names></name><name><surname>van Oostrom</surname><given-names>CT</given-names></name><name><surname>Nederlof</surname><given-names>PM</given-names></name><name><surname>de Vries</surname><given-names>A</given-names></name><name><surname>Jonkers</surname><given-names>J</given-names></name></person-group><article-title>High incidence of protein-truncating TP53 mutations in BRCA1-related breast cancer</article-title><source>Cancer Res</source><year>2009</year><volume>69</volume><fpage>3625</fpage><lpage>3633</lpage><pub-id pub-id-type=\"pmid\">19336573</pub-id></element-citation></ref><ref id=\"b36\"><label>36</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Turner</surname><given-names>N</given-names></name><name><surname>Tutt</surname><given-names>A</given-names></name><name><surname>Ashworth</surname><given-names>A</given-names></name></person-group><article-title>Hallmarks of &#x02018;BRCAness&#x02019; in sporadic cancers</article-title><source>Nature Rev Cancer</source><year>2004</year><volume>4</volume><fpage>814</fpage><lpage>819</lpage><pub-id pub-id-type=\"pmid\">15510162</pub-id></element-citation></ref><ref id=\"b37\"><label>37</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zorn</surname><given-names>KK</given-names></name><name><surname>Bonome</surname><given-names>T</given-names></name><name><surname>Gangi</surname><given-names>L</given-names></name><name><surname>Chandramouli</surname><given-names>GV</given-names></name><name><surname>Awtrey</surname><given-names>CS</given-names></name><name><surname>Gardner</surname><given-names>GJ</given-names></name><etal/></person-group><article-title>Gene expression profiles of serous, endometrioid, and clear cell subtypes of ovarian and endometrial cancer</article-title><source>Clin Cancer Res</source><year>2005</year><volume>11</volume><fpage>6422</fpage><lpage>6430</lpage><pub-id pub-id-type=\"pmid\">16166416</pub-id></element-citation></ref><ref id=\"b38\"><label>38</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kobel</surname><given-names>M</given-names></name><name><surname>Kalloger</surname><given-names>SE</given-names></name><name><surname>Boyd</surname><given-names>N</given-names></name><name><surname>McKinney</surname><given-names>S</given-names></name><name><surname>Mehl</surname><given-names>E</given-names></name><name><surname>Palmer</surname><given-names>C</given-names></name><etal/></person-group><article-title>Ovarian carcinoma subtypes are different diseases: implications for biomarker studies</article-title><source>PLoS Med</source><year>2008</year><volume>5</volume><fpage>e232</fpage><pub-id pub-id-type=\"pmid\">19053170</pub-id></element-citation></ref><ref id=\"b39\"><label>39</label><element-citation publication-type=\"journal\"><collab>ICON</collab><article-title>Paclitaxel plus carboplatin versus standard chemotherapy with either single-agent carboplatin or cyclophosphamide, doxorubicin, and cisplatin in women with ovarian cancer: the ICON3 randomised trial</article-title><source>Lancet</source><year>2002</year><volume>360</volume><fpage>505</fpage><lpage>515</lpage><pub-id pub-id-type=\"pmid\">12241653</pub-id></element-citation></ref><ref id=\"b40\"><label>40</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Etemadmoghadam</surname><given-names>D</given-names></name><name><surname>de Fazio</surname><given-names>A</given-names></name><name><surname>Beroukhim</surname><given-names>R</given-names></name><name><surname>Mermel</surname><given-names>C</given-names></name><name><surname>George</surname><given-names>J</given-names></name><name><surname>Getz</surname><given-names>G</given-names></name><etal/></person-group><article-title>Integrated genome-wide DNA copy number and expression analysis identifies distinct mechanisms of primary chemoresistance in ovarian carcinomas</article-title><source>Clin Cancer Res</source><year>2009</year><volume>15</volume><fpage>1417</fpage><lpage>1427</lpage><pub-id pub-id-type=\"pmid\">19193619</pub-id></element-citation></ref><ref id=\"b41\"><label>41</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rustin</surname><given-names>GJ</given-names></name><name><surname>Nelstrop</surname><given-names>AE</given-names></name><name><surname>McClean</surname><given-names>P</given-names></name><name><surname>Brady</surname><given-names>MF</given-names></name><name><surname>McGuire</surname><given-names>WP</given-names></name><name><surname>Hoskins</surname><given-names>WJ</given-names></name><etal/></person-group><article-title>Defining response of ovarian carcinoma to initial chemotherapy according to serum CA 125</article-title><source>J Clin Oncol</source><year>1996</year><volume>14</volume><fpage>1545</fpage><lpage>1551</lpage><pub-id pub-id-type=\"pmid\">8622070</pub-id></element-citation></ref><ref id=\"b42\"><label>42</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Vergote</surname><given-names>I</given-names></name><name><surname>Rustin</surname><given-names>GJ</given-names></name><name><surname>Eisenhauer</surname><given-names>EA</given-names></name><name><surname>Kristensen</surname><given-names>GB</given-names></name><name><surname>Pujade-Lauraine</surname><given-names>E</given-names></name><name><surname>Parmar</surname><given-names>MK</given-names></name><etal/></person-group><article-title>Re: new guidelines to evaluate the response to treatment in solid tumors [ovarian cancer]. Gynecologic Cancer Intergroup</article-title><source>J Natl Cancer Inst</source><year>2000</year><volume>92</volume><fpage>1534</fpage><lpage>1535</lpage><pub-id pub-id-type=\"pmid\">10995813</pub-id></element-citation></ref><ref id=\"b43\"><label>43</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kupryjanczyk</surname><given-names>J</given-names></name><name><surname>Thor</surname><given-names>AD</given-names></name><name><surname>Beauchamp</surname><given-names>R</given-names></name><name><surname>Merritt</surname><given-names>V</given-names></name><name><surname>Edgerton</surname><given-names>SM</given-names></name><name><surname>Bell</surname><given-names>DA</given-names></name><etal/></person-group><article-title>p53 gene mutations and protein accumulation in human ovarian cancer</article-title><source>Proc Natl Acad Sci U S A</source><year>1993</year><volume>90</volume><fpage>4961</fpage><lpage>4965</lpage><pub-id pub-id-type=\"pmid\">8506342</pub-id></element-citation></ref><ref id=\"b44\"><label>44</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shimizu</surname><given-names>Y</given-names></name><name><surname>Kamoi</surname><given-names>S</given-names></name><name><surname>Amada</surname><given-names>S</given-names></name><name><surname>Akiyama</surname><given-names>F</given-names></name><name><surname>Silverberg</surname><given-names>SG</given-names></name></person-group><article-title>Toward the development of a universal grading system for ovarian epithelial carcinoma: testing of a proposed system in a series of 461 patients with uniform treatment and follow-up</article-title><source>Cancer</source><year>1998</year><volume>82</volume><fpage>893</fpage><lpage>901</lpage><pub-id pub-id-type=\"pmid\">9486579</pub-id></element-citation></ref><ref id=\"b45\"><label>45</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Parsons</surname><given-names>DW</given-names></name><name><surname>Jones</surname><given-names>S</given-names></name><name><surname>Zhang</surname><given-names>X</given-names></name><name><surname>Lin</surname><given-names>JC</given-names></name><name><surname>Leary</surname><given-names>RJ</given-names></name><name><surname>Angenendt</surname><given-names>P</given-names></name><etal/></person-group><article-title>An integrated genomic analysis of human glioblastoma multiforme</article-title><source>Science</source><year>2008</year><volume>321</volume><fpage>1807</fpage><lpage>1812</lpage><pub-id pub-id-type=\"pmid\">18772396</pub-id></element-citation></ref><ref id=\"b46\"><label>46</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kuo</surname><given-names>KT</given-names></name><name><surname>Mao</surname><given-names>TL</given-names></name><name><surname>Jones</surname><given-names>S</given-names></name><name><surname>Veras</surname><given-names>E</given-names></name><name><surname>Ayhan</surname><given-names>A</given-names></name><name><surname>Wang</surname><given-names>TL</given-names></name><etal/></person-group><article-title>Frequent activating mutations of PIK3CA in ovarian clear cell carcinoma</article-title><source>Am J Pathol</source><year>2009</year><volume>174</volume><fpage>1597</fpage><lpage>1601</lpage><pub-id pub-id-type=\"pmid\">19349352</pub-id></element-citation></ref><ref id=\"b47\"><label>47</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bengtsson</surname><given-names>H</given-names></name><name><surname>Irizarry</surname><given-names>R</given-names></name><name><surname>Carvalho</surname><given-names>B</given-names></name><name><surname>Speed</surname><given-names>TP</given-names></name></person-group><article-title>Estimation and assessment of raw copy numbers at the single locus level</article-title><source>Bioinformatics</source><year>2008</year><volume>24</volume><fpage>759</fpage><lpage>767</lpage><pub-id pub-id-type=\"pmid\">18204055</pub-id></element-citation></ref><ref id=\"b48\"><label>48</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rozen</surname><given-names>S</given-names></name><name><surname>Skaletsky</surname><given-names>H</given-names></name></person-group><article-title>Primer3 on the WWW for general users and for biologist programmers</article-title><source>Methods Mol Biol</source><year>2000</year><volume>132</volume><fpage>365</fpage><lpage>386</lpage><pub-id pub-id-type=\"pmid\">10547847</pub-id></element-citation></ref><ref id=\"b49\"><label>49</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Riemenschneider</surname><given-names>MJ</given-names></name><name><surname>Buschges</surname><given-names>R</given-names></name><name><surname>Wolter</surname><given-names>M</given-names></name><name><surname>Reifenberger</surname><given-names>J</given-names></name><name><surname>Bostrom</surname><given-names>J</given-names></name><name><surname>Kraus</surname><given-names>JA</given-names></name><etal/></person-group><article-title>Amplification and overexpression of the MDM4 (MDMX) gene from 1q32 in a subset of malignant gliomas without TP53 mutation or MDM2 amplification</article-title><source>Cancer Res</source><year>1999</year><volume>59</volume><fpage>6091</fpage><lpage>6096</lpage><pub-id pub-id-type=\"pmid\">10626796</pub-id></element-citation></ref><ref id=\"b50\"><label>50</label><element-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>TL</given-names></name><name><surname>Maierhofer</surname><given-names>C</given-names></name><name><surname>Speicher</surname><given-names>MR</given-names></name><name><surname>Lengauer</surname><given-names>C</given-names></name><name><surname>Vogelstein</surname><given-names>B</given-names></name><name><surname>Kinzler</surname><given-names>KW</given-names></name><etal/></person-group><article-title>Digital karyotyping</article-title><source>Proc Natl Acad Sci U S A</source><year>2002</year><volume>99</volume><fpage>16156</fpage><lpage>16161</lpage><pub-id pub-id-type=\"pmid\">12461184</pub-id></element-citation></ref></ref-list><sec sec-type=\"supplementary-material\"><title>SUPPORTING INFORMATION ON THE INTERNET</title><p>The following supporting information may be found in the online version of this article.</p><p><bold>Supplementary experimental procedures.</bold></p><p><bold>Table S1.</bold> Samples listed by AOCS number with results of somatic <italic>TP53</italic> sequencing.</p><supplementary-material content-type=\"local-data\" id=\"SD1\"><media xlink:href=\"path0221-0049-SD1.pdf\" xlink:type=\"simple\" id=\"d32e3668\" position=\"anchor\" mimetype=\"application\" mime-subtype=\"pdf\"/></supplementary-material><p><bold>Table S2.</bold> Summary of clinical, pathology and molecular review of mutation-negative serous carcinomas.</p><supplementary-material content-type=\"local-data\" id=\"SD2\"><media xlink:href=\"path0221-0049-SD2.pdf\" xlink:type=\"simple\" id=\"d32e3674\" position=\"anchor\" mimetype=\"application\" mime-subtype=\"pdf\"/></supplementary-material><p><bold>Figure S1.</bold> Summary of all cases included in the pilot and validation sets.</p><supplementary-material content-type=\"local-data\" id=\"SD3\"><media xlink:href=\"path0221-0049-SD3.pdf\" xlink:type=\"simple\" id=\"d32e3680\" position=\"anchor\" mimetype=\"application\" mime-subtype=\"pdf\"/><media xlink:href=\"path0221-0049-SD4.doc\" xlink:type=\"simple\" id=\"d32e3681\" position=\"anchor\" mimetype=\"application\" mime-subtype=\"msword\"/></supplementary-material></sec></back></article>'}"
      ]
     },
     "execution_count": 367,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "x = articles.find_one({'_id': 'test', 'annotations.disease-ontology-keywords': {'$exists': True}})\n",
    "x"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 375,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<pymongo.results.DeleteResult at 0x11ae012d0>"
      ]
     },
     "execution_count": 375,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "articles.delete_one({'_id': \"test\"})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'iterate_infectious_disease_articles' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-16-5a5cc4bd510a>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0;32mfor\u001b[0m \u001b[0marticle\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0minfectious_diseases\u001b[0m \u001b[0;32min\u001b[0m \u001b[0miterate_infectious_disease_articles\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdb\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0marticlesubset\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0marticle\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'_id'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0minfectious_diseases\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mNameError\u001b[0m: name 'iterate_infectious_disease_articles' is not defined"
     ]
    }
   ],
   "source": [
    "for article, infectious_diseases in iterate_infectious_disease_articles(db.articlesubset):\n",
    "    print(article['_id'], infectious_diseases)\n",
    "    print(\"\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'total_article_count' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-10-c28412185261>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mtotal_article_count\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m: name 'total_article_count' is not defined"
     ]
    }
   ],
   "source": [
    "total_article_count"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'article_with_body_count' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-11-b7972d653aab>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0marticle_with_body_count\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m: name 'article_with_body_count' is not defined"
     ]
    }
   ],
   "source": [
    "article_with_body_count"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'infectious_disease_article_count' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-12-db9743c64222>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0minfectious_disease_article_count\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m: name 'infectious_disease_article_count' is not defined"
     ]
    }
   ],
   "source": [
    "infectious_disease_article_count"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
